%!TEX TS-program = xelatex
\documentclass[11pt]{article}

\usepackage[english]{babel}

\usepackage{amsmath,amssymb,amsfonts}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{stix2}
\usepackage[scaled]{helvet}
\usepackage[scaled]{inconsolata}

\usepackage{lastpage}

\usepackage{setspace}

\usepackage{ccicons}

\usepackage[hang,flushmargin]{footmisc}

\usepackage{geometry}

\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}

\usepackage{fancyhdr}
\renewcommand{\headrulewidth}{0pt}\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\makeatletter
\newcounter{tableno}
\newenvironment{tablenos:no-prefix-table-caption}{
  \caption@ifcompatibility{}{
    \let\oldthetable\thetable
    \let\oldtheHtable\theHtable
    \renewcommand{\thetable}{tableno:\thetableno}
    \renewcommand{\theHtable}{tableno:\thetableno}
    \stepcounter{tableno}
    \captionsetup{labelformat=empty}
  }
}{
  \caption@ifcompatibility{}{
    \captionsetup{labelformat=default}
    \let\thetable\oldthetable
    \let\theHtable\oldtheHtable
    \addtocounter{table}{-1}
  }
}
\makeatother

\usepackage{array}
\newcommand{\PreserveBackslash}[1]{\let\temp=\\#1\let\\=\temp}
\let\PBS=\PreserveBackslash

\usepackage[breaklinks=true]{hyperref}
\hypersetup{colorlinks,%
citecolor=blue,%
filecolor=blue,%
linkcolor=blue,%
urlcolor=blue}
\usepackage{url}

\usepackage{caption}
\setcounter{secnumdepth}{0}
\usepackage{cleveref}

\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth
\else\Gin@nat@width\fi}
\makeatother
\let\Oldincludegraphics\includegraphics
\renewcommand{\includegraphics}[1]{\Oldincludegraphics[width=\maxwidth]{#1}}

\usepackage{longtable}
\usepackage{booktabs}

\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}

\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for \widthof, \maxof
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\maxof{\widthof{#1}}{\csllabelwidth}}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}\geometry{verbose,letterpaper,tmargin=2.2cm,bmargin=2.2cm,lmargin=2.2cm,rmargin=2.2cm}

\usepackage{lineno}
\usepackage[nolists,noheads]{endfloat}

\pagestyle{plain}

\tolerance=1
\emergencystretch=\maxdimen
\hyphenpenalty=10000
\hbadness=10000

\doublespacing

\fancypagestyle{normal}
{
  \fancyhf{}
  \fancyfoot[R]{\footnotesize\sffamily\thepage\ of \pageref*{LastPage}}
}
\begin{document}
\raggedright
\thispagestyle{empty}
{\Large\bfseries\sffamily The biological interpretation of probabilistic
food webs}
\vskip 5em

%
\href{https://orcid.org/0000-0001-9051-0597}{Francis\,Banville}%
%
\,\textsuperscript{1,2,3,‡}\quad %
\href{https://orcid.org/0000-0001-6067-1349}{Tanya\,Strydom}%
%
\,\textsuperscript{1,3,‡}\quad %
\href{https://orcid.org/0000-0002-0735-5184}{Timothée\,Poisot}%
%
\,\textsuperscript{1,3}

\textsuperscript{1}\,Université de
Montréal\quad \textsuperscript{2}\,Université de
Sherbrooke\quad \textsuperscript{3}\,Quebec Centre for Biodiversity
Science

\textsuperscript{‡}\,These authors contributed equally to the work\\

\textbf{Correspondance to:}\\
Francis Banville --- \texttt{francis.banville@umontreal.ca}\\

\vfill
This work is released by its authors under a CC-BY 4.0 license\hfill\ccby\\
Last revision: \emph{\today}

\clearpage
\thispagestyle{empty}

\vfill
Community ecologists are increasingly thinking probabilistically when it
comes to food webs and other ecological networks. Assuredly, the
benefits of representing ecological interactions as probabilistic events
(e.g., how likely are species to interact?) instead of binary objects
(e.g., do species interact?) are numerous, from a better assessment of
the spatiotemporal variation of trophic interactions to an increase
capacity to reconstruct networks from sparse data. However,
probabilities need to be used with caution when working with species
interactions. Indeed, depending on the system at hand and the method
used to build probabilistic networks, probabilities can have different
interpretations that imply different ways to manipulate them. At the
core of these differences lie the distinction between assessing the
likelihood that two groups of individuals \emph{can} interact and the
likelihood that they \emph{will} interact. This impacts the spatial,
temporal, and taxonomic scaling of interaction probabilities, thus
enlightening the need to properly define them in their ecological
context. Published data on probabilistic species interactions are poorly
documented in the literature, which impedes our ability to use them
appropriately. With these challenges in mind, we propose a general
approach to thinking about probabilities in regards to ecological
interactions, with a strong focus on food webs, and call for better
definitions and conceptualizations of probabilistic ecological networks,
both at the local and regional scales.



\vfill

\clearpage
\linenumbers
\pagestyle{normal}

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

Cataloging species interactions across space is a gargantuan task. At
the core of this challenge lies the spatiotemporal variability of
ecological networks (\textbf{Poisot2012DisSpe?};
\textbf{Poisot2015SpeWhy?}), which makes documenting the location and
timing of interactions difficult. Indeed, it is not sufficient to know
that two species have the biological capacity to interact to infer the
realization of their interaction at a specific time and space (Dunne
2006). Taking food webs as an example, a predator species and its
potential prey must first co-occur on the same territory in order for a
trophic interaction to take place (\textbf{Blanchet2020CooNot?}). They
must then encounter, which is conditional on their relative abundances
in the ecosystem and the matching of their phenology
(\textbf{Poisot2015SpeWhy?}). Finally, the interaction occurs only if
the predators have a desire to consume their prey and are able to
capture and ingest them (\textbf{Pulliam1974TheOpt?}). Environmental
(e.g.~temperature and presence of shelters) and biological
(e.g.~physiological state of both species and availability of other prey
species) factors contribute to this variability by impacting species
co-occurrence (Araujo \emph{et al.} 2011) and the realization of their
interactions (\textbf{Poisot2015SpeWhy?}). In this context, it is
unsurprising that computational methods are being developed in ecology
to help alleviate the colossal sampling efforts required to document
species interactions across time and space
(\textbf{Strydom2021RoaPre?}). Having a better portrait of species
interactions and the emerging structure of their food webs is important
since it lays the groundwork for understanding the functioning,
dynamics, and resilience of ecosystems worldwide (e.g., Proulx \emph{et
al.} 2005; Pascual \emph{et al.} 2006; Delmas \emph{et al.} 2019).

The recognition of the intrinsic variability of species interactions and
the emergence of numerical methods have led ecologists to rethink their
representation of ecological networks, slowly moving from a binary to a
probabilistic view of species interactions (Poisot \emph{et al.} 2016).
This has several benefits. For example, probabilities represent the
limit of our knowledge about species interactions and can inform us
about the expected number of interactions and emerging network
properties despite this limited knowledge (Poisot \emph{et al.} 2016).
They are also very helpful in predicting the spatial distribution of
species within networks (\textbf{Cazelles2016TheSpe?}) and the temporal
variability of interactions (Poisot \emph{et al.} 2015), generating new
ecological data (e.g., Strydom \emph{et al.} 2022), and identifying
priority sampling locations of species interactions (see
\textbf{Andrade-Pacheco2020FinHot?} for an ecological example of a
sampling optimization problem). Moreover, the high rate of false
negatives in ecological network data, resulting from the difficulty of
witnessing interactions between rare species, makes it hard to interpret
non-observations of species interactions ecologically
(\textbf{Catchen2023MisLin?}). Using probabilities instead of yes-no
interactions accounts for these observation errors; in that case, only
forbidden interactions (\textbf{Jordano2003InvPro?};
\textbf{Olesen2010MisFor?}) would have a probability value of zero (but
see \textbf{Gonzalez-Varo2016LabLim?}). Many measures have been
developed to describe the structure (Poisot \emph{et al.} 2016) and
diversity (Ohlmann \emph{et al.} 2019; Godsoe \emph{et al.} n.d.) of
probabilistic interactions, which shows the potential of this framework
in the study of a variety of ecological phenomena.

However, representing species interactions probabilistically can also be
challenging. Beyond methodological difficulties in estimating these
numbers, there are important conceptual challenges in defining what we
mean by ``probability of interactions.'' To the best of our knowledge,
because the building blocks of this mathematical representation of food
webs are still being laid, there is no clear definition found in the
literature. This is worrisome, since working with probabilistic species
interactions without clear guidelines could be misleading as much for
field ecologists as for computational ecologists who use and generate
these data. In this contribution, we outline different ways to define
and interpret interactions probabilities in network ecology and propose
an approach to thinking about them. These definitions mostly depend on
the study system (e.g.~local network or metaweb) and on the method used
to generate them. We show that different definitions can have different
ecological implications, especially regarding spatial, temporal, and
taxonomic scaling. Although we will focus on food webs, our observations
and advice can be applied to all types of ecological networks, from
plant-pollinator to host-parasite networks. Specifically, we argue that
probabilities should be better documented, defined mathematically, and
used with caution when describing species interactions.

\hypertarget{definitions-and-interpretations}{%
\section{Definitions and
interpretations}\label{definitions-and-interpretations}}

The basic unit of food webs and other ecological networks are
individuals that interact with each others (e.g., by predation; Elton
2001). The aggregation of these individuals into more or less
homogeneous groups (e.g., populations, species, trophic species) allows
us to represent networks at different scales, which impacts the
properties and behaviour of these systems (Guimarães 2020). A network's
nodes can thus designate distinct levels of organization, whereas the
edges linking these nodes can describe a variety of interaction
measures. When using a Boolean (yes-no) representation of biotic
interactions, the observation that one individual from group (or node)
\(i\) interacts with another individual from group \(j\) is enough to
set the interaction \(A_{i,j}\) to 1. This simplified representation of
food webs is a highly valuable source of ecological information (Pascual
\emph{et al.} 2006) even though it overlooks important factors regarding
interaction strengths. These, in turn, can be represented using weighted
interactions, which better describe the energy flows, demographic
impacts or frequencies of interactions between nodes (Berlow \emph{et
al.} 2004; Borrett \& Scharler 2019), with \(A_{i,j} \in \mathbb{N}\) or
\(\mathbb{R}\) depending on the measure. For example, they can be used
to estimate the average number of prey individuals consumed by the
predators in a given time period. This extra amount of ecological
information typically comes at a cost of greater sampling effort or data
requirement in predictive models (Strydom \emph{et al.} 2021), which can
lead to high uncertainties when building these types of networks.

The uncertainty and spatiotemporal variability of both types of biotic
interactions (Boolean and weighted) can be represented
probabilistically. On one hand, Boolean interactions follow a Bernoulli
distribution \(A_{i,j} \sim {\rm Bernoulli}(p)\), with \(p\) being the
probability of interactions. The only two possible outcomes are the
presence (\(A_{i,j} = 1\)) or absence (\(A_{i,j} = 0\)) of an
interaction between the two nodes. Weighted interactions, on the other
hand, can follow various probability distributions depending on the
measure used. In this case, the event's outcome is the value of
interaction strength. For instance, weights can follow a Poisson
distribution \(A_{i,j} \sim {\rm Poisson}(\lambda)\) when predicting
frequencies of interactions between pairs of nodes, with \(\lambda\)
being the expected rate of interaction. The definition and
interpretation of parameters like \(p\) and \(\lambda\) are inextricably
linked to environmental and biological factors such as species relative
abundance, traits, area, and time, depending on the type of interaction.
Because Boolean species interactions are much more documented in the
literature, our primary focus in this contribution will be on addressing
the challenges in defining and interpretating \(p\) for pairwise species
interactions.

The first aspect to take into consideration when estimating or using
probabilities of interactions is knowing if they describe the likelihood
of potential or realized interactions. A potential interaction is
defined as the biological capacity of two species to interact (i.e., the
probability that they \emph{can} interact) whereas a realized
interaction refers to the materialization or observation of this
interaction in a delineated space and time period (i.e., the probability
that they \emph{will} interact). Here, we will use the terms
\emph{metaweb} to designate networks of potential interactions and
\emph{local networks} for those of realized interactions. Frequent
confusion arises among ecologists over the use of these two terms,
especially in a probabilistic context. Indeed, in many studies of
probabilistic ecological networks, it remains unclear when authors
describe potential or realized interactions, or when so-called
probabilities are actually \emph{interaction scores}. A better
understanding of these differences would alleviate interpretation errors
and help ecologists use these numbers more appropriately.

\hypertarget{probabilistic-metawebs}{%
\subsection{Probabilistic metawebs}\label{probabilistic-metawebs}}

What does a probability in the context of a metaweb mean?

\[P(i \rightarrow j)\]

\begin{itemize}
\tightlist
\item
  It means the probability that two taxa can interact, regardless of
  biological plasticity, environmental variability, or co-occurrence.
\item
  One observation is enough to set this probability to one.
\item
  Can we turn this into a local network realisation that is also
  probabilistic and intuitive?
\item
  Low probability with rare interactions?
\end{itemize}

\textbf{Papers:} (\textbf{Strydom2022FooWeb?})

\hypertarget{probabilistic-local-networks}{%
\subsection{Probabilistic local
networks}\label{probabilistic-local-networks}}

\[P(i \rightarrow j | C, A, N, E, t)\]

i = predator j = prey C = co-occurrence (and relative abundance?) A =
area N = relative abundance (or network?) E = environment (including
network?) t = time

What does a probability in the context of a local network mean? A
cautionary tale of how we define probabilities.

\begin{itemize}
\tightlist
\item
  It means the probability that two taxa will interact at a given
  location.
\item
  What do we mean by saying that two taxa will interact? We usually mean
  that at least one individual from one group will interact with (e.g.,
  predate) at least one other individual from the other group.
\item
  The probability is conditional on the environmental and local
  abundance contexts.
\item
  We should expect a certain number of interactions to be realized
  depending on the probability value. This number depends on the number
  of trials, which also depends on the ecological context (e.g.,
  environmental conditions, scale) in which probabilities were
  estimated. This is in contrast with probabilities in metawebs.
\end{itemize}

\hypertarget{scaling}{%
\section{Scaling}\label{scaling}}

\hypertarget{spatial-and-temporal-scales}{%
\subsection{Spatial and temporal
scales}\label{spatial-and-temporal-scales}}

How do interaction probabilities scale spatially and temporally?

\begin{itemize}
\tightlist
\item
  Why do probabilistic local food webs scale with area and time but not
  probabilistic metawebs?
\item
  In metawebs, interaction probabilities do not scale with space and
  time because they depend solely on the biological capacity of two
  species to interact.
\item
  In local food webs, interaction probabilities scale with space and
  time because there are more opportunities of interactions (e.g., more
  environmental conditions) in a larger area and longer time period.
\item
  What are some network area relationships in probabilistic local food
  webs?
\item
  We know that local networks can inform regional networks. However, can
  regional networks inform local networks?
\end{itemize}

\textbf{Figure:} Empirical example of the association between the number
of interactions in realized local food webs and the number of
interactions in the corresponding species subnetworks of regional
networks. We should expect the interaction to be linear below the 1:1
line.

\textbf{Papers:} there might be something in these McLeod \emph{et al.}
(2020); (\textbf{McLeod2021SamAsy?}); (\textbf{Wood2015EffSpa?})

\hypertarget{taxonomic-scale}{%
\subsection{Taxonomic scale}\label{taxonomic-scale}}

How do interaction probabilities scale taxonomically?

\begin{itemize}
\tightlist
\item
  There are different biological interpretations of probabilities in
  food webs at the individual level and at higher taxonomic levels.
\item
  How does the scaling up of the nodes from an individual to population
  to any higher taxonomic group change our interpretation of interaction
  probabilities? How does the aggregation change our interpretation?
\item
  How is it similar and different to spatial and temporal scaling?
  Basically, all kinds of scaling are just different ways to aggregate
  individuals or nodes.
\end{itemize}

\textbf{Figure:} Conceptual figure of how a scale up of the nodes from
an individual to a population to any higher taxonomic group change our
interpretation of the probability of interaction.

\hypertarget{concluding-remarks}{%
\section{Concluding remarks}\label{concluding-remarks}}

Here we present some advice moving forward.

\begin{itemize}
\tightlist
\item
  What can we learn from other systems/fields (e.g., social networks,
  probabilistic graph theory)?
\item
  What even are probabilities? What is the probability that we will ever
  know the answer to that?
\item
  Be careful of how we define probabilities. Be sure to be explicit
  about these things. Be sure to specify the type of interaction, the
  spatial, temporal, and taxonomic scale when presenting new data on
  interaction probabilities. We need better metadata documentation.
\item
  Be careful to use and manipulate interaction probabilities properly
  depending on how they were defined and obtained. Different
  interpretations imply different scaling, and thus different ways to
  manipulate these numbers.
\item
  Maybe mention thinking about a workflow to predict probabilistic local
  food webs from probabilistic metawebs.
\end{itemize}

\hypertarget{acknowledgement}{%
\section{Acknowledgement}\label{acknowledgement}}

We acknowledge that this study was conducted on land within the
traditional unceded territory of the Saint Lawrence Iroquoian,
Anishinabewaki, Mohawk, Huron-Wendat, and Omàmiwininiwak nations. This
work was supported by the Institute for Data Valorisation (IVADO) and
the Natural Sciences and Engineering Research Council of Canada (NSERC)
Collaborative Research and Training Experience (CREATE) program, through
the Computational Biodiversity Science and Services (BIOS²) program. A
special thank to all members of the Black Holes and Revelations working
group (organized by BIOS²) for their insightful discussions and valuable
feedbacks on this manuscript.

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\hypertarget{ref-Araujo2011UsiSpe}{}%
Araujo, M.B., Rozenfeld, A., Rahbek, C. \& Marquet, P.A. (2011). Using
species co-occurrence networks to assess the impacts of climate change.
\emph{Ecography}, 34, 897--908.

\leavevmode\hypertarget{ref-Berlow2004IntStr}{}%
Berlow, E.L., Neutel, A.-M., Cohen, J.E., De Ruiter, P.C., Ebenman, B.,
Emmerson, M., \emph{et al.} (2004). Interaction strengths in food webs:
Issues and opportunities. \emph{Journal of Animal Ecology}, 73,
585--598.

\leavevmode\hypertarget{ref-Borrett2019WalPar}{}%
Borrett, S.R. \& Scharler, U.M. (2019). Walk partitions of flow in
Ecological Network Analysis: Review and synthesis of methods and
indicators. \emph{Ecological Indicators}, 106, 105451.

\leavevmode\hypertarget{ref-Delmas2019AnaEco}{}%
Delmas, E., Besson, M., Brice, M.-H., Burkle, L.A., Dalla Riva, G.V.,
Fortin, M.-J., \emph{et al.} (2019). Analysing ecological networks of
species interactions. \emph{Biological Reviews}, 94, 16--36.

\leavevmode\hypertarget{ref-Dunne2006Network}{}%
Dunne, J.A. (2006). The Network Structure of Food Webs. In:
\emph{Ecological networks: Linking structure and dynamics} (eds. Dunne,
J.A. \& Pascual, M.). Oxford University Press, pp. 27--86.

\leavevmode\hypertarget{ref-Elton2001AniEco}{}%
Elton, C.S. (2001). \emph{Animal Ecology}. University of Chicago Press,
Chicago, IL.

\leavevmode\hypertarget{ref-GodsoeSpeInt}{}%
Godsoe, W., Murray, R. \& Iritani, R. (n.d.). Species interactions and
diversity: A unified framework using Hill numbers. \emph{Oikos}, n/a,
e09282.

\leavevmode\hypertarget{ref-Guimaraes2020StrEco}{}%
Guimarães, P.R. (2020). The Structure of Ecological Networks Across
Levels of Organization. \emph{Annual Review of Ecology, Evolution, and
Systematics}, 51, 433--460.

\leavevmode\hypertarget{ref-McLeod2020EffSpe}{}%
McLeod, A.M., Leroux, S.J. \& Chu, C. (2020). Effects of species traits,
motif profiles, and environment on spatial variation in multi-trophic
antagonistic networks. \emph{Ecosphere}, 11, e03018.

\leavevmode\hypertarget{ref-Ohlmann2019DivInd}{}%
Ohlmann, M., Miele, V., Dray, S., Chalmandrier, L., O'Connor, L. \&
Thuiller, W. (2019). Diversity indices for ecological networks: A
unifying framework using Hill numbers. \emph{Ecology Letters}, 22,
737--747.

\leavevmode\hypertarget{ref-Pascual2006EcoNet}{}%
Pascual, M., Dunne, J.A. \& Dunne, J.A. (2006). \emph{Ecological
Networks: Linking Structure to Dynamics in Food Webs}. Oxford University
Press, USA.

\leavevmode\hypertarget{ref-Poisot2016Structure}{}%
Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D., Fortin, M.-J. \&
Stouffer, D.B. (2016). The structure of probabilistic networks.
\emph{Methods in Ecology and Evolution}, 7, 303--312.

\leavevmode\hypertarget{ref-Poisot2015Speciesa}{}%
Poisot, T., Stouffer, D.B. \& Gravel, D. (2015). Beyond species: Why
ecological interaction networks vary through space and time.
\emph{Oikos}, 124, 243--251.

\leavevmode\hypertarget{ref-Proulx2005NetThi}{}%
Proulx, S.R., Promislow, D.E.L. \& Phillips, P.C. (2005). Network
thinking in ecology and evolution. \emph{Trends in Ecology \&
Evolution}, SPECIAL issue: BUMPER book REVIEW, 20, 345--353.

\leavevmode\hypertarget{ref-Strydom2022Food}{}%
Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron, D., Farrell,
M.J., \emph{et al.} (2022). Food web reconstruction through phylogenetic
transfer of low-rank network representation. \emph{Methods in Ecology
and Evolution}, 13.

\leavevmode\hypertarget{ref-Strydom2021Roadmapa}{}%
Strydom, T., Catchen, M.D., Banville, F., Caron, D., Dansereau, G.,
Desjardins-Proulx, P., \emph{et al.} (2021). A roadmap towards
predicting species interaction networks (across space and time).
\emph{Philosophical Transactions of the Royal Society B-Biological
Sciences}, 376, 20210063.

\end{CSLReferences}

\end{document}
