%!TEX TS-program = xelatex
\documentclass[11pt]{article}

\usepackage[english]{babel}

\usepackage{amsmath,amssymb,amsfonts}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{stix2}
\usepackage[scaled]{helvet}
\usepackage[scaled]{inconsolata}

\usepackage{lastpage}

\usepackage{setspace}

\usepackage{ccicons}

\usepackage[hang,flushmargin]{footmisc}

\usepackage{geometry}

\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}

\usepackage{fancyhdr}
\renewcommand{\headrulewidth}{0pt}\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\makeatletter
\newcounter{tableno}
\newenvironment{tablenos:no-prefix-table-caption}{
  \caption@ifcompatibility{}{
    \let\oldthetable\thetable
    \let\oldtheHtable\theHtable
    \renewcommand{\thetable}{tableno:\thetableno}
    \renewcommand{\theHtable}{tableno:\thetableno}
    \stepcounter{tableno}
    \captionsetup{labelformat=empty}
  }
}{
  \caption@ifcompatibility{}{
    \captionsetup{labelformat=default}
    \let\thetable\oldthetable
    \let\theHtable\oldtheHtable
    \addtocounter{table}{-1}
  }
}
\makeatother

\usepackage{array}
\newcommand{\PreserveBackslash}[1]{\let\temp=\\#1\let\\=\temp}
\let\PBS=\PreserveBackslash

\usepackage[breaklinks=true]{hyperref}
\hypersetup{colorlinks,%
citecolor=blue,%
filecolor=blue,%
linkcolor=blue,%
urlcolor=blue}
\usepackage{url}

\usepackage{caption}
\setcounter{secnumdepth}{0}
\usepackage{cleveref}

\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth
\else\Gin@nat@width\fi}
\makeatother
\let\Oldincludegraphics\includegraphics
\renewcommand{\includegraphics}[1]{\Oldincludegraphics[width=\maxwidth]{#1}}

\usepackage{longtable}
\usepackage{booktabs}

\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}

\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for \widthof, \maxof
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\maxof{\widthof{#1}}{\csllabelwidth}}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}\geometry{verbose,letterpaper,tmargin=2.2cm,bmargin=2.2cm,lmargin=2.2cm,rmargin=2.2cm}

\usepackage{lineno}
\usepackage[nolists,noheads]{endfloat}

\pagestyle{plain}

\tolerance=1
\emergencystretch=\maxdimen
\hyphenpenalty=10000
\hbadness=10000

\doublespacing

\fancypagestyle{normal}
{
  \fancyhf{}
  \fancyfoot[R]{\footnotesize\sffamily\thepage\ of \pageref*{LastPage}}
}
\begin{document}
\raggedright
\thispagestyle{empty}
{\Large\bfseries\sffamily The biological interpretation of probabilistic
food webs}
\vskip 5em

%
\href{https://orcid.org/0000-0001-9051-0597}{Francis\,Banville}%
%
\,\textsuperscript{1,2,3,‡}\quad %
\href{https://orcid.org/0000-0001-6067-1349}{Tanya\,Strydom}%
%
\,\textsuperscript{1,3,‡}\quad %
\href{https://orcid.org/0000-0002-0735-5184}{Timothée\,Poisot}%
%
\,\textsuperscript{1,3}

\textsuperscript{1}\,Université de
Montréal\quad \textsuperscript{2}\,Université de
Sherbrooke\quad \textsuperscript{3}\,Quebec Centre for Biodiversity
Science

\textsuperscript{‡}\,These authors contributed equally to the work\\

\textbf{Correspondance to:}\\
Francis Banville --- \texttt{francis.banville@umontreal.ca}\\

\vfill
This work is released by its authors under a CC-BY 4.0 license\hfill\ccby\\
Last revision: \emph{\today}

\clearpage
\thispagestyle{empty}

\vfill


        {\bfseries 1:}\,Community ecologists are increasingly thinking
probabilistically when it comes to food webs and other ecological
networks. Assuredly, the benefits of representing ecological
interactions as probabilistic events (e.g., how likely are species to
interact?) instead of binary objects (e.g., do species interact?) are
numerous, from a better assessment of the spatiotemporal variation of
trophic interactions to an increase capacity to reconstruct networks
from sparse data. However, probabilities need to be used with caution
when working with species interactions. Indeed, depending on the system
at hand and the method used to build probabilistic networks,
probabilities can have different interpretations that imply different
ways to manipulate them. At the core of these differences lie the
distinction between assessing the likelihood that two groups of
individuals \emph{can} interact and the likelihood that they \emph{will}
interact. This impacts the spatial, temporal, and taxonomic scaling of
interaction probabilities, thus enlightening the need to properly define
them in their ecological context. Published data on probabilistic
species interactions are poorly documented in the literature, which
impedes our ability to use them appropriately. With these challenges in
mind, we propose a general approach to thinking about probabilities in
regards to ecological interactions, with a strong focus on food webs,
and call for better definitions and conceptualizations of probabilistic
ecological networks, both at the local and regional scales.\\%
    
\vfill

\clearpage
\linenumbers
\pagestyle{normal}

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

Cataloging species interactions across space is a gargantuan task. At
the core of this challenge lies the spatiotemporal variability of
ecological networks (Poisot \emph{et al.} 2012, 2015), which makes
documenting the location and timing of interactions difficult. Indeed,
it is not sufficient to know if two species have the biological capacity
to interact to infer the realization of their interaction at a specific
time and space {[}TK{]}. Taking food webs as an example, a predator and
its potential prey must first co-occur on the same territory in order
for a trophic interaction to take place (Blanchet \emph{et al.} 2020).
They must then encounter, which is conditional on their relative
abundances in the ecosystem and the matching of their phenology (Poisot
\emph{et al.} 2015). Finally, the interaction occurs only if the
predators have a desire to consume their prey and are able to capture
and ingest them (Pulliam 1974). Environmental (e.g.~temperature and
presence of shelters) and biological (e.g.~physiological state of both
species and availability of other prey species) factors contribute to
this variability by impacting species co-occurrence {[}TK{]} and the
realization of their interactions {[}TK{]}. In this context, it is
unsurprising that computational methods are being developed in ecology
to help alleviate the colossal sampling efforts required to document
species interactions across time and space (Strydom \emph{et al.} 2021).
Having a better portrait of species interactions and the emerging
structure of food webs is important since it lays the groundwork for
understanding the functioning {[}TK{]}, dynamics {[}TK{]}, and
resilience {[}TK{]} of ecosystems worldwide.

The recognition of the variability of species interactions and the
emergence of numerical methods have led ecologists to rethink their
representation of ecological networks, slowly moving from a binary to a
probabilistic view of species interactions (Poisot \emph{et al.} 2016).
This has several benefits. For example, probabilities represent the
limit of our knowledge about species interactions and can indicate the
expected frequency of two species interacting with each other {[}TK{]}.
They are also very helpful in predictive models when modeling the
spatial distribution of species (Cazelles \emph{et al.} 2016) and the
temporal variability of ecological networks {[}TK{]}, generating new
ecological data (e.g., Strydom \emph{et al.} 2022), and identifying
priority sampling locations (see Andrade-Pacheco \emph{et al.} 2020 for
an ecological example of a sampling optimization problem). Moreover, the
high rate of false negatives in ecological network data, resulting from
the difficulty of witnessing interactions between rare species, makes it
hard to interpret non-observations of species interactions ecologically
(Catchen \emph{et al.} 2023). Using probabilities instead of yes-no
interactions accounts for these observation errors; in that case, only
forbidden interactions (Jordano \emph{et al.} 2003; Olesen \emph{et al.}
2010) would have a probability value of zero (but see Gonzalez-Varo \&
Traveset 2016).

However, representing species interactions probabilistically can also be
challenging. Beyond methodological difficulties in estimating these
numbers, there are important conceptual challenges in defining what we
mean by ``probability of interactions.'' To the best of our knowledge,
because the building blocks of this mathematical representation of food
webs are still being laid, there is no clear definition found in the
literature. This is worrisome, since working with probabilistic species
interactions without clear guidelines could be misleading as much for
field ecologists as for computational ecologists who use and generate
these data. In this contribution, we outline different ways to define
and interpret interactions probabilities in network ecology and propose
an approach to thinking about them. These definitions mostly depend on
the study system (e.g.~local network or metaweb) and on the method used
to generate them. We show that different definitions can have different
ecological implications, especially regarding spatial, temporal, and
taxonomic scaling. Although we will focus on food webs, our observations
and advice can be applied to all types of ecological networks, from
plant-pollinator to host-parasite networks. Specifically, we argue that
probabilities should be better documented, defined mathematically, and
used with caution when describing species interactions.

\hypertarget{definitions-and-interpretations}{%
\section{Definitions and
interpretations}\label{definitions-and-interpretations}}

\hypertarget{overview-of-interaction-probabilities}{%
\subsection{Overview of interaction
probabilities}\label{overview-of-interaction-probabilities}}

How are interaction probabilities defined in the literature? It might
not be as intuitive as one would think.

\begin{itemize}
\tightlist
\item
  There is a big difference in how we interpret the probability that two
  species \emph{can} interact (metaweb) and the probability that they
  \emph{will} interact (realized networks).
\item
  Interaction probabilities can be used to describe Boolean interactions
  (e.g., the probability that two species interact) and weighted
  interactions (e.g., the probability distribution of the amount of
  energy flow between two species).
\item
  In many studies, it is not obvious if authors use interaction scores
  or probabilities (in the latter case, it is rarely specified what
  these probabilities truly represent).
\end{itemize}

\hypertarget{probabilistic-metawebs}{%
\subsection{Probabilistic metawebs}\label{probabilistic-metawebs}}

What does a probability in the context of a metaweb mean?

\[P(i \rightarrow j)\]

\begin{itemize}
\tightlist
\item
  It means the probability that two taxa can interact, regardless of
  biological plasticity, environmental variability, or co-occurrence.
\item
  One observation is enough to set this probability to one.
\item
  Can we turn this into a local network realisation that is also
  probabilistic and intuitive?
\end{itemize}

\textbf{Papers:} (\textbf{Strydom2022FooWeb?})

\hypertarget{probabilistic-local-networks}{%
\subsection{Probabilistic local
networks}\label{probabilistic-local-networks}}

\[P(i \rightarrow j | C, A, N, E, t)\]

i = predator j = prey C = co-occurrence A = area N = relative abundance
E = environment (including network) t = time

What does a probability in the context of a local network mean? A
cautionary tale of how we define probabilities.

\begin{itemize}
\tightlist
\item
  It means the probability that two taxa will interact at a given
  location.
\item
  What do we mean by saying that two taxa will interact? We usually mean
  that at least one individual from one group will interact with (e.g.,
  predate) at least one other individual from the other group.
\item
  The probability is conditional on the environmental and local
  abundance contexts.
\item
  We should expect a certain number of interactions to be realized
  depending on the probability value. This number depends on the number
  of trials, which also depends on the ecological context (e.g.,
  environmental conditions, scale) in which probabilities were
  estimated. This is in contrast with probabilities in metawebs.
\end{itemize}

\hypertarget{scaling}{%
\section{Scaling}\label{scaling}}

\hypertarget{spatial-and-temporal-scales}{%
\subsection{Spatial and temporal
scales}\label{spatial-and-temporal-scales}}

How do interaction probabilities scale spatially and temporally?

\begin{itemize}
\tightlist
\item
  Why do probabilistic local food webs scale with area and time but not
  probabilistic metawebs?
\item
  In metawebs, interaction probabilities do not scale with space and
  time because they depend solely on the biological capacity of two
  species to interact.
\item
  In local food webs, interaction probabilities scale with space and
  time because there are more opportunities of interactions (e.g., more
  environmental conditions) in a larger area and longer time period.
\item
  What are some network area relationships in probabilistic local food
  webs?
\item
  We know that local networks can inform regional networks. However, can
  regional networks inform local networks?
\end{itemize}

\textbf{Figure:} Empirical example of the association between the number
of interactions in realized local food webs and the number of
interactions in the corresponding species subnetworks of regional
networks. We should expect the interaction to be linear below the 1:1
line.

\textbf{Papers:} there might be something in these McLeod \emph{et al.}
(2020); (\textbf{McLeod2021SamAsy?}); (\textbf{Wood2015EffSpa?})

\hypertarget{taxonomic-scale}{%
\subsection{Taxonomic scale}\label{taxonomic-scale}}

How do interaction probabilities scale taxonomically?

\begin{itemize}
\tightlist
\item
  There are different biological interpretations of probabilities in
  food webs at the individual level and at higher taxonomic levels.
\item
  How does the scaling up of the nodes from an individual to population
  to any higher taxonomic group change our interpretation of interaction
  probabilities? How does the aggregation change our interpretation?
\item
  How is it similar and different to spatial and temporal scaling?
  Basically, all kinds of scaling are just different ways to aggregate
  individuals or nodes.
\end{itemize}

\textbf{Figure:} Conceptual figure of how a scale up of the nodes from
an individual to a population to any higher taxonomic group change our
interpretation of the probability of interaction.

\hypertarget{concluding-remarks}{%
\section{Concluding remarks}\label{concluding-remarks}}

Here we present some advice moving forward.

\begin{itemize}
\tightlist
\item
  What can we learn from other systems/fields (e.g., social networks,
  probabilistic graph theory)?
\item
  What even are probabilities? What is the probability that we will ever
  know the answer to that?
\item
  Be careful of how we define probabilities. Be sure to be explicit
  about these things. Be sure to specify the type of interaction, the
  spatial, temporal, and taxonomic scale when presenting new data on
  interaction probabilities. We need better metadata documentation.
\item
  Be careful to use and manipulate interaction probabilities properly
  depending on how they were defined and obtained. Different
  interpretations imply different scaling, and thus different ways to
  manipulate these numbers.
\item
  Maybe mention thinking about a workflow to predict probabilistic local
  food webs from probabilistic metawebs.
\end{itemize}

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\hypertarget{ref-Andrade-Pacheco2020FinHot}{}%
Andrade-Pacheco, R., Rerolle, F., Lemoine, J., Hernandez, L., Meïté, A.,
Juziwelo, L., \emph{et al.} (2020). Finding hotspots: Development of an
adaptive spatial sampling approach. \emph{Scientific Reports}, 10,
10939.

\leavevmode\hypertarget{ref-Blanchet2020CooNot}{}%
Blanchet, F.G., Cazelles, K. \& Gravel, D. (2020). Co-occurrence is not
evidence of ecological interactions. \emph{Ecology Letters}, 23,
1050--1063.

\leavevmode\hypertarget{ref-Catchen2023MisLin}{}%
Catchen, M.D., Poisot, T., Pollock, L.J. \& Gonzalez, A. (2023). The
missing link: Discerning true from false negatives when sampling species
interaction networks.

\leavevmode\hypertarget{ref-Cazelles2016TheSpe}{}%
Cazelles, K., Araujo, M.B., Mouquet, N. \& Gravel, D. (2016). A theory
for species co-occurrence in interaction networks. \emph{Theoretical
Ecology}, 9, 39--48.

\leavevmode\hypertarget{ref-Gonzalez-Varo2016LabLim}{}%
Gonzalez-Varo, J.P. \& Traveset, A. (2016). The Labile Limits of
Forbidden Interactions. \emph{Trends in Ecology \& Evolution}, 31,
700--710.

\leavevmode\hypertarget{ref-Jordano2003InvPro}{}%
Jordano, P., Bascompte, J. \& Olesen, J.M. (2003). Invariant properties
in coevolutionary networks of plantanimal interactions. \emph{Ecology
Letters}, 6, 69--81.

\leavevmode\hypertarget{ref-McLeod2020EffSpe}{}%
McLeod, A.M., Leroux, S.J. \& Chu, C. (2020). Effects of species traits,
motif profiles, and environment on spatial variation in multi-trophic
antagonistic networks. \emph{Ecosphere}, 11, e03018.

\leavevmode\hypertarget{ref-Olesen2010MisFor}{}%
Olesen, J.M., Bascompte, J., Dupont, Y.L., Elberling, H., Rasmussen, C.
\& Jordano, P. (2010). Missing and forbidden links in mutualistic
networks. \emph{Proceedings of the Royal Society B: Biological
Sciences}, 278, 725--732.

\leavevmode\hypertarget{ref-Poisot2012DisSpe}{}%
Poisot, T., Canard, E., Mouillot, D., Mouquet, N. \& Gravel, D. (2012).
The dissimilarity of species interaction networks. \emph{Ecology
Letters}, 15, 1353--1361.

\leavevmode\hypertarget{ref-Poisot2016Structure}{}%
Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D., Fortin, M.-J. \&
Stouffer, D.B. (2016). The structure of probabilistic networks.
\emph{Methods in Ecology and Evolution}, 7, 303--312.

\leavevmode\hypertarget{ref-Poisot2015SpeWhy}{}%
Poisot, T., Stouffer, D.B. \& Gravel, D. (2015). Beyond species: Why
ecological interaction networks vary through space and time.
\emph{Oikos}, 124, 243--251.

\leavevmode\hypertarget{ref-Pulliam1974TheOpt}{}%
Pulliam, H.R. (1974). On the Theory of Optimal Diets. \emph{The American
Naturalist}, 108, 59--74.

\leavevmode\hypertarget{ref-Strydom2022Food}{}%
Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron, D., Farrell,
M.J., \emph{et al.} (2022). Food web reconstruction through phylogenetic
transfer of low-rank network representation. \emph{Methods in Ecology
and Evolution}, 13.

\leavevmode\hypertarget{ref-Strydom2021RoaPre}{}%
Strydom, T., Catchen, M.D., Banville, F., Caron, D., Dansereau, G.,
Desjardins-Proulx, P., \emph{et al.} (2021). A roadmap towards
predicting species interaction networks (across space and time).
\emph{Philosophical Transactions of the Royal Society B-Biological
Sciences}, 376, 20210063.

\end{CSLReferences}

\end{document}
