<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>The ecological interpretation of probabilistic
networks</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);

            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        html {
            background-color: white;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article>p {
            line-height: 135%;
        }

        article>h1 {
            padding-top: 3em;
        }

        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }

        a:hover {
            font-weight: 500;
        }

        .citation {
            color: var(--citation);
        }

        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }

        header>.container {
            margin-top: 4em;
        }

        code,
        pre {
            font-family: "Roboto Mono", monospace;
        }

        pre {
            white-space: pre-wrap;
        }

        h1,
        h2,
        h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }

        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }

        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }

        .header--authors>span {
            color: var(--light);
            font-weight: 400;
        }

        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }

        footer {
            background-color: var(--grey);
            font-weight: 300;
        }

        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }

        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }

        figure>img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }

        figure>figcaption {
            padding: var(--inner-space);
        }

        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }

        table>caption {
            padding: var(--inner-space);
        }

        figcaption,
        caption {
            color: var(--light);
            font-weight: 300;
        }

        thead {
            color: var(--structure);
        }

        td,
        th {
            padding: 3px;
        }

        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }

        tbody>tr:nth-child(even) {
            background: var(--grey)
        }

        ;

        tbody {
            border-bottom: 1px solid var(--structure);
        }

        #refs {
            font-size: 90%;
            column-count: 2;
        }

        #refs>.csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }

        .hanging-indent>.csl-entry {
            padding-left: 15px;
            text-indent: -15px;
        }

        .artifacts {
            text-align: center;
            border-radius: 4px;
            background-color: var(--grey);
            padding: 1em;
            margin-top: 4em;
        }

        .artifacts a{
            background-color: var(--structure);
            text-decoration: none;
            margin: 0px 2em;
            color: white;
            text-transform: capitalize;
            border-radius: 4px;
            text-align: center;
            padding: 4px 4px;
            width: 7em;
            display: inline-block;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.css"
        integrity="sha384-vKruj+a13U8yHIkAyGgK1J3ArTLzrFGBbBc0tDp4ad/EyewESeXE/Iv67Aj8gKZ0" crossorigin="anonymous">
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.js"
        integrity="sha384-PwRUT/YqbnEjkZO0zZxNqcxACrXe+j766U2amXcgMg5457rve2Y7I6ZJSm2A0mS4"
        crossorigin="anonymous"></script>
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/contrib/auto-render.min.js"
        integrity="sha384-+VBxd3r6XgURycqtZ117nYw44OOcIax56Z4dCRWbxyPt0Koah1uHoK0o4+/RRE05" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>

    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                    { left: "$$", right: "$$", display: true },
                    { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">The ecological interpretation of
probabilistic networks</div>
            <div class="header--authors">
                <span class="header--author">
                <a href="https://orcid.org/0000-0001-9051-0597" target="_blank">
                    Francis Banville
                </a>
                <sup class="affiliation">1,2,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0001-6067-1349" target="_blank">
                    Tanya Strydom
                </a>
                <sup class="affiliation">1,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-9368-6106" target="_blank">
                    Chris Brimacombe
                </a>
                <sup class="affiliation">4</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-6506-6487" target="_blank">
                    Michael Catchen
                </a>
                <sup class="affiliation">3,5</sup>
                </span><span class="header--author">
                Dominique Gravel
                <sup class="affiliation">2,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0003-2791-8383" target="_blank">
                    Gracielle Higino
                </a>
                <sup class="affiliation">2</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-2029-2325" target="_blank">
                    Kari Norman
                </a>
                <sup class="affiliation">1</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affiliation">1,3</sup>
                </span>
            </div>
        </div>
    </header>

    <main class="container">
        <section class="abstract">
            The stochastic nature of ecological interactions has led
            biologists to adopt a probabilistic view of ecological
            networks. Representing species interactions
            probabilistically (how likely are they occurring?) as
            opposed to deterministically (are they occurring?) allows a
            better assessment of their spatiotemporal variability and
            accounts for inherent uncertainties in both observations and
            predictions. However, despite this growing interest in
            probabilistic networks, guidelines for the estimation and
            documentation of probabilistic interaction data are still
            lacking. This is concerning given that our biological
            understanding of probabilistic interactions and their
            appropriate analyses depend on the often elusive methods and
            variables used for their estimation. We review how
            probabilistic interactions are defined at different spatial
            scales, from local interactions to regional networks
            (metawebs), with a strong emphasis on trophic interactions.
            These definitions are based on the distinction between the
            observation of an interaction at a specific time and space
            and its biological feasibility. We show that different
            network representations have different statistical
            properties when it comes to common ecological applications.
            Specifically, we argue that local probabilistic networks and
            metawebs differ in their spatial and temporal scaling of
            interactions, with potential interactions in metawebs
            remaining constant across spatial and temporal scales. This
            is in contrast with the taxonomic scaling of interactions,
            which does not qualitatively differ between both types of
            networks. We suggest two approaches to inferring
            deterministic networks from probabilistic webs that account
            for these differences and argue that systematic biases arise
            when directly inferring local networks from subsets of
            metawebs. To support our arguments, we build a
            spatiotemporally explicit model of probabilistic
            interactions and develop different case studies using
            empirical data on host-parasite interactions in Europe.
            Overall, our results underscore the importance of clear
            metadata for probabilistic ecological networks, both at the
            local and regional scales, to inform the appropriate reuse
            of interaction data.


            <div class="artifacts">
                <a href="Banville_et_al_2023_ecological_interpretation_draft.pdf" title="Draft" target="_blank">Draft</a>
                <a href="Banville_et_al_2023_ecological_interpretation_preprint.pdf" title="Preprint" target="_blank">Preprint</a>
                <a href="Banville_et_al_2023_ecological_interpretation.odt" title="OpenDocument" target="_blank">Libre Office</a>
            </div>
        </section>

        <article>
            <h1 id="introduction">Introduction</h1>
            <p>Cataloging ecological interactions is a substantial
            challenge. There are methodological and biological
            constraints that hinder our ability to observe all
            interactions, such as the spatial and temporal uncoupling of
            species (<span class="citation"
            data-cites="Jordano1987PatMut">Jordano (1987)</span>) and
            the large number of possible interactions in a community, of
            which the vast majority are rare (<span class="citation"
            data-cites="Jordano2016Samplingb">Jordano (2016)</span>).
            Documenting the location and timing of interactions becomes
            even more difficult when accounting for the spatiotemporal
            variability of ecological networks (<span class="citation"
            data-cites="Poisot2012Dissimilaritya">Poisot <em>et al.</em>
            (2012)</span>, <span class="citation"
            data-cites="Poisot2015Speciesa">Poisot <em>et al.</em>
            (2015)</span>). Indeed, knowing the biological capacity of
            two species to interact is necessary but not sufficient for
            inferring their interaction at a specific time and space.
            For example, <span class="citation"
            data-cites="Golubski2011ModMod">Golubski &amp; Abrams
            (2011)</span> presented many cases where trophic
            interactions in food webs depend on the presence or
            abundance of a third species (e.g., of a more profitable
            prey species). More generally, a handful of conditions must
            be satisfied for an interaction to be observed locally.
            First, both species must have overlapping geographic ranges,
            i.e. they must co-occur within the region of interest (<span
            class="citation"
            data-cites="Blanchet2020Cooccurrencea">Blanchet <em>et
            al.</em> (2020)</span>). Then, they must encounter locally.
            Probabilities of interspecific encounters are typically low,
            especially for rare species with low relative abundances
            (<span class="citation"
            data-cites="Canard2012Emergencea">Canard <em>et al.</em>
            (2012)</span>). Finally, their traits must be locally
            compatible (<span class="citation"
            data-cites="Poisot2015Speciesa">Poisot <em>et al.</em>
            (2015)</span>). This includes their phenology (<span
            class="citation" data-cites="Olesen2010Missing">Olesen
            <em>et al.</em> (2010)</span>, <span class="citation"
            data-cites="Singer2012GeoMos">Singer &amp; McBride
            (2012)</span>), behavioral choices (<span class="citation"
            data-cites="Pulliam1974Theory">Pulliam (1974)</span>, <span
            class="citation" data-cites="Choh2012PreRol">Choh <em>et
            al.</em> (2012)</span>) and phenotypes (<span
            class="citation" data-cites="Bolnick2011WhyInt">Bolnick
            <em>et al.</em> (2011)</span>, <span class="citation"
            data-cites="Stouffer2011RolBod">Stouffer <em>et al.</em>
            (2011)</span>, <span class="citation"
            data-cites="Gravel2013InfFooa">Gravel <em>et al.</em>
            (2013)</span>). Environmental factors, such as temperature
            (<span class="citation"
            data-cites="Angilletta2004TemGro">Angilletta <em>et al.</em>
            (2004)</span>), drought (<span class="citation"
            data-cites="Woodward2012CliCha">Woodward <em>et al.</em>
            (2012)</span>), climate change (<span class="citation"
            data-cites="Gilman2010FraCom">Gilman <em>et al.</em>
            (2010)</span>, <span class="citation"
            data-cites="Woodward2010ChaEco">Woodward <em>et al.</em>
            (2010)</span>, <span class="citation"
            data-cites="Araujo2011Usinga">Araujo <em>et al.</em>
            (2011)</span>), and habitat modifications (<span
            class="citation"
            data-cites="Tylianakis2007HabMod">Tylianakis <em>et al.</em>
            (2007)</span>), contribute to this spatiotemporal
            variability of interactions by impacting species abundance
            and traits. In this context, it is unsurprising that our
            knowledge of ecological interactions remains limited (<span
            class="citation" data-cites="Hortal2015SevSho">Hortal <em>et
            al.</em> (2015)</span>) despite extensive biodiversity data
            collection (<span class="citation"
            data-cites="Schmeller2015GloTer">Schmeller <em>et al.</em>
            (2015)</span>).</p>
            <p>The recognition of the intrinsic variability of species
            interactions has led ecologists to expand their
            representation of ecological networks to include a
            probabilistic view of interactions (<span class="citation"
            data-cites="Poisot2016Structure">Poisot <em>et al.</em>
            (2016)</span>). As opposed to binary deterministic networks,
            in which interactions are regarded as either occurring or
            not, probabilistic networks represent our degree of belief
            about the realization or feasibility of pairwise
            interactions at the local or regional scale, respectively.
            In other words, representing interactions probabilistically
            takes into account inherent uncertainties and variability
            associated with ecological data. In the broadest sense,
            binary networks are also a type of probabilistic network, in
            which the numerical value of an interaction is restrained to
            <span class="math inline">0</span> (non-occurring) or <span
            class="math inline">1</span> (occurring). In probabilistic
            networks, only forbidden interactions (i.e., interactions
            prohibited by biological traits, <span class="citation"
            data-cites="Jordano2003Invarianta">Jordano <em>et al.</em>
            (2003)</span>, <span class="citation"
            data-cites="Olesen2010Missing">Olesen <em>et al.</em>
            (2010)</span>) have a probability value of zero, provided
            that intraspecific trait variability is considered (<span
            class="citation"
            data-cites="Gonzalez-Varo2016Labilea">Gonzalez-Varo &amp;
            Traveset (2016)</span>). However, <em>neutrally</em>
            forbidden interactions (i.e., improbable interactions
            between rare species, <span class="citation"
            data-cites="Canard2012Emergencea">Canard <em>et al.</em>
            (2012)</span>) could have low probability values in local
            networks but high probability in a regional network
            (metaweb) describing the biological capacity of species to
            interact.</p>
            <p>By accounting for the uncertainty of interactions,
            probabilistic networks provide a more realistic portrait of
            species interactions and network structure
            (i.e. community-level properties), which is a major driver
            of the functioning, dynamics, and resilience of ecosystems
            worldwide (<span class="citation"
            data-cites="Proulx2005Networka">Proulx <em>et al.</em>
            (2005)</span>, <span class="citation"
            data-cites="McCann2007ProBio">McCann (2007)</span>, <span
            class="citation" data-cites="McCann2011FooWeb">McCann
            (2011)</span>, <span class="citation"
            data-cites="Rooney2012IntFoo">Rooney &amp; McCann
            (2012)</span>). Moreover, the application and development of
            computational methods in network ecology, often based on a
            probabilistic representation of interactions, can alleviate
            the sampling efforts required to document species
            interactions (<span class="citation"
            data-cites="Strydom2021Roadmapa">Strydom <em>et al.</em>
            (2021)</span>). For example, statistical models can be used
            to estimate the uncertainty of pairwise interactions (<span
            class="citation" data-cites="Cirtwill2019QuaFra">Cirtwill
            <em>et al.</em> (2019)</span>) and the probability of
            missing (false negatives) and spurious (false positives)
            interactions (<span class="citation"
            data-cites="Guimera2009MisSpu">Guimerà &amp; Sales-Pardo
            (2009)</span>). Considering the high rate of false negatives
            in species interaction data due to the difficulty of
            witnessing rare interactions (<span class="citation"
            data-cites="Catchen2023Missinga">Catchen <em>et al.</em>
            (2023)</span>), these models could inform the identification
            of priority sampling locations of ecological networks where
            data collection would yield the most valuable information,
            thereby reducing errors. Optimization models for sampling
            locations have mostly found applications in biological
            systems that are not networked, such as identifying priority
            sampling sites for disease hotspots (<span class="citation"
            data-cites="Andrade-Pacheco2020Finding">Andrade-Pacheco
            <em>et al.</em> (2020)</span>), but there is substantial
            promise in applying them to probabilistic ecological
            interactions. Statistical models can also be used to
            generate network predictions without prior knowledge of
            pairwise interactions, for instance using body size (<span
            class="citation" data-cites="Petchey2008SizFor">Petchey
            <em>et al.</em> (2008)</span>, <span class="citation"
            data-cites="Gravel2013InfFooa">Gravel <em>et al.</em>
            (2013)</span>), phylogeny (<span class="citation"
            data-cites="Elmasri2020HieBay">Elmasri <em>et al.</em>
            (2020)</span>, <span class="citation"
            data-cites="Strydom2022Food">Strydom <em>et al.</em>
            (2022)</span>), or a combination of niche and neutral
            processes (<span class="citation"
            data-cites="Bartomeus2016ComFra">Bartomeus <em>et al.</em>
            (2016)</span>, <span class="citation"
            data-cites="Pomeranz2019InfPre">Pomeranz <em>et al.</em>
            (2019)</span>) for inference. Topological null models (e.g.,
            <span class="citation"
            data-cites="Bascompte2003NesAss">Bascompte <em>et al.</em>
            (2003)</span>, <span class="citation"
            data-cites="Fortuna2006HabLos">Fortuna &amp; Bascompte
            (2006)</span>), which can be used to generate underlying
            distributions of network measures for null hypothesis
            significance testing, are other examples of common
            probabilistic network models. Many measures have been
            developed to describe the structure (<span class="citation"
            data-cites="Poisot2016Structure">Poisot <em>et al.</em>
            (2016)</span>) and diversity (<span class="citation"
            data-cites="Ohlmann2019Diversity">Ohlmann <em>et al.</em>
            (2019)</span>, <span class="citation"
            data-cites="Godsoe2022Species">Godsoe <em>et al.</em>
            (2022)</span>) of probabilistic networks. These models and
            measures support the use of this approach for the study of a
            wide range of ecological questions, from making better
            predictions of species distribution (<span class="citation"
            data-cites="Cazelles2016Theorya">Cazelles <em>et al.</em>
            (2016)</span>) to forecasting the impact of climate change
            on ecological networks (<span class="citation"
            data-cites="Gilman2010FraCom">Gilman <em>et al.</em>
            (2010)</span>).</p>
            <p>The lack of clear guidelines on the use of probabilistic
            interaction data is worrisome (<span class="citation"
            data-cites="Brimacombe2023ShoReu">Brimacombe <em>et al.</em>
            (2023)</span>), both for data producers and re-users who
            generate and manipulate these numbers. Besides
            methodological difficulties that may arise when assessing
            probabilistic interactions, a precise definition of
            probabilistic interactions appears to be lacking, making the
            estimation and use of these data more difficult. We aim to
            take a step back by outlining different ways in which
            probabilistic interactions are defined and used in network
            ecology. We distinguish two broad categories of
            probabilistic networks that necessitate distinct approaches
            when applied to key ecological questions: local networks of
            realized interactions, and regional networks (metawebs) of
            potential interactions. We highlight the distinctions in the
            ecological meaning of these two representations and show
            that they yield different statistical outcomes regarding
            e.g. the spatial and temporal scaling of interactions and
            the prediction of binary networks across space. Moreover,
            there is currently no metadata standard that could guide the
            documentation of all types of probabilistic interactions
            (<span class="citation" data-cites="Salim2022Data">Salim
            <em>et al.</em> (2022)</span> discuss data standards for
            deterministic mutualistic networks). Well-defined metadata
            for probabilistic networks would support more adequate
            manipulation and integration of interaction data from
            different sources and guard against the potential
            misinterpretations arising from ambiguous definitions of
            probabilistic networks. These metadata should outline the
            nature (i.e., potential or local) and type (e.g., predatory
            or pollination) of the interactions, provide information
            regarding the taxonomic level, identities, and
            characteristics (e.g., life stages) of the individuals
            involved in an interaction, present the mathematical
            formulation of probabilities, including clearly identified
            conditional variables, and describe the methods and contexts
            (e.g., location, time, environmental conditions) in which
            interactions were estimated. Inadequately documented
            probabilistic interaction data should be used with caution
            when analyzing ecological networks. Although our
            contribution focuses on food webs, our observations and
            advice can be applied to other types of ecological networks,
            from plant-pollinator to host-virus networks. Indeed,
            excluding networks of indirect interactions such as
            competition and facilitation networks (<span
            class="citation" data-cites="Kefi2015NetStr">Kéfi <em>et
            al.</em> (2015)</span>, <span class="citation"
            data-cites="Kefi2016HowStr">Kéfi <em>et al.</em>
            (2016)</span>), most ecological networks describe
            probabilities of direct interactions, which are conceptually
            and mathematically analogous regardless of their biological
            type (e.g., trophic and mutualistic interactions).</p>
            <h1
            id="probabilistic-representations-of-interactions">Probabilistic
            representations of interactions</h1>
            <p>One of the first aspects to take into consideration when
            estimating or interpreting probabilities of interactions is
            knowing if they describe potential or realized interactions.
            A potential interaction is defined as the biological
            capacity of two taxa to interact (i.e., the probability that
            they <em>can</em> theoretically interact) whereas a realized
            interaction is the materialization or observation of this
            interaction in a well-defined space and time (i.e., the
            probability that they interact locally). Here, we use the
            terms <em>metaweb</em> (<span class="citation"
            data-cites="Dunne2006Network">Dunne (2006)</span>) to
            designate networks of potential interactions and <em>local
            networks</em> (<span class="citation"
            data-cites="Poisot2012Dissimilaritya">Poisot <em>et al.</em>
            (2012)</span>) for those of realized interactions. Metawebs
            are the network analogues of the species pool, where local
            networks originate from a subset of both species (nodes) and
            interactions (edges) of the regional metaweb (<span
            class="citation" data-cites="Saravia2022Ecological">Saravia
            <em>et al.</em> (2022)</span>). Without clear documentation,
            it can be challenging to know if published probabilistic
            networks describe potential or realized interactions, or if
            so-called probabilities are in reality <em>interaction
            scores</em> (i.e., non-probabilistic quantitative
            interactions). When probabilistic potential interactions are
            used and interpreted as local interactions (and conversely),
            this may generate misleading findings during data analysis.
            We believe that a better understanding of the differences,
            similarities, and relationships between these two
            probabilistic representations of ecological networks would
            alleviate interpretation errors and facilitate a more
            adequate utilization of interaction data.</p>
            <h2
            id="pairwise-interactions-the-building-blocks-of-ecological-networks">Pairwise
            interactions: the building blocks of ecological
            networks</h2>
            <p>Local ecological networks and metawebs, like any type of
            network, are made of nodes and edges that can be represented
            at different levels of organization and precision. The basic
            unit of food webs and other ecological networks are
            individuals that interact with each other (e.g., by
            predation, <span class="citation"
            data-cites="Elton2001Animal">Elton (2001)</span>), forming
            individual-based networks (<span class="citation"
            data-cites="Melian2011EcoDyn">Melián <em>et al.</em>
            (2011)</span>). The aggregation of these individuals into
            more or less homogeneous groups (e.g., populations, species,
            trophic species, families) allows us to represent nodes at
            broader taxonomic scales, which affects our interpretation
            of the properties of these systems (<span class="citation"
            data-cites="Guimaraes2020Structurea">Guimarães
            (2020)</span>). Moreover, edges linking these nodes can
            describe a variety of interaction measures. Ecologists have
            traditionally represented interactions as binary objects
            that were considered realized after observing at least one
            individual from group <span
            class="math inline"><em>i</em></span> interact with at least
            another individual from group <span
            class="math inline"><em>j</em></span>. Boolean interactions
            can be viewed as the result of a Bernoulli process <span
            class="math inline">$A_{i,j} \sim {\rm Bernoulli}(P(i
            \rightarrow j))$</span>, with <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>
            being the probability of interaction between <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> that characterizes our
            limited knowledge of the system and its intrinsic
            spatiotemporal variability. Depending on the type of
            networks (local or metaweb), the mathematical formulation
            and interpretation of stochastic parameters like <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>
            can be linked to environmental and biological factors such
            as species relative abundance, traits, area, and time
            (tbl. 1), for example using logistic regression with a logit
            link function with continuous explanatory variables. In
            these probabilistic network representations in which <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>
            are edge values, the only two possible outcomes are the
            presence (<span
            class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> = 1</span>)
            or absence (<span
            class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> = 0</span>)
            of an interaction between each pair of nodes. When
            considering uncertainties around the estimation of <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>,
            a Beta distribution <span class="math inline">${\rm
            Beta}(\alpha, \beta)$</span> can also be used to encompass
            all possible probability values. Observing an interaction
            between two taxa at a given location and time provides
            important information that can be used to update previous
            estimates of <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>,
            informing us on the biological capacity of both taxa to
            interact and the environmental conditions that enabled them
            to interact locally.</p>
            <p>Even though binary networks constitute a highly valuable
            source of ecological information (<span class="citation"
            data-cites="Pascual2006Ecologicala">Pascual <em>et al.</em>
            (2006)</span>), they overlook important factors regarding
            interaction strengths. These are represented using
            quantitative interactions, which better describe the energy
            flows, demographic impacts or frequencies of interactions
            between nodes (<span class="citation"
            data-cites="Berlow2004Interaction">Berlow <em>et al.</em>
            (2004)</span>, <span class="citation"
            data-cites="Borrett2019Walk">Borrett &amp; Scharler
            (2019)</span>), with <span
            class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub></span>
            being a natural number <span class="math inline">ℕ</span> or
            a real number <span class="math inline">ℝ</span> depending
            on the measure. For example, they may represent local
            interaction rates between pairs of taxa (e.g., the flower
            visiting rates of pollinators in a mutualistic network,
            <span class="citation"
            data-cites="Herrera1989PolAbu">Herrera (1989)</span>).
            Because quantitative interactions can also describe
            predation pressure on prey taxa in food webs, they can be
            good estimators of the parameters describing species
            interactions in a Lotka-Volterra model (e.g., <span
            class="citation" data-cites="Emmerson2004Predatora">Emmerson
            &amp; Raffaelli (2004)</span>). The extra amount of
            ecological information in quantitative networks typically
            comes at a cost of greater sampling effort and data volume
            in predictive models (<span class="citation"
            data-cites="Strydom2021Roadmapa">Strydom <em>et al.</em>
            (2021)</span>), which can lead to relatively high levels of
            uncertainties when inferring quantitative networks with
            limited data. Just like binary networks, the uncertainty and
            spatiotemporal variability of quantitative interactions can
            be represented probabilistically. However, the need to
            estimate the probability distribution of all possible value
            of interaction strengths can make the inference of
            probabilities more challenging in quantitative networks
            compared to binary networks, which require only one
            probability estimate for each interaction. Quantitative
            interactions can follow various probability distributions
            depending on the measure used, the event’s outcome being the
            value of interaction strength. For instance, quantitative
            interactions can follow a Poisson distribution <span
            class="math inline">$A_{i,j} \sim {\rm Poisson}(\lambda_{i
            \rightarrow j})$</span> when predicting frequencies of
            interactions between pairs of nodes, with <span
            class="math inline"><em>λ</em><sub><em>i</em> → <em>j</em></sub></span>
            being the expected rate at which individuals of taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact (e.g., the
            average number of prey <span
            class="math inline"><em>j</em></span> consumed by all
            predators <span class="math inline"><em>i</em></span> in a
            given time period). The Poisson distribution can also be
            0-inflated after initially modeling non-interacting taxa
            (e.g., <span class="citation"
            data-cites="Boulangeat2012AccDis">Boulangeat <em>et al.</em>
            (2012)</span> employ a 0-inflated model to analyze species
            abundance following the modeling of species presence and
            absence), which constitute the majority of taxa pairs in
            most local networks due to their typically high sparseness
            (<span class="citation"
            data-cites="Jordano2016Samplingb">Jordano (2016)</span>).
            Because of the methodological difficulties typically
            encountered when building deterministic quantitative
            networks, binary networks, which are easier to sample (<span
            class="citation" data-cites="Jordano2016Samplingb">Jordano
            (2016)</span>) and predict (<span class="citation"
            data-cites="Strydom2021Roadmapa">Strydom <em>et al.</em>
            (2021)</span>), are much more documented and modeled in the
            literature. Moreover, most published probabilistic networks
            and methods describe Bernoulli interactions (whether
            interaction probabilities are regarded as constant or
            variable, e.g. represented by Beta distributions), which
            underlines the need for better guidelines regarding the
            interpretation and manipulation of these types of networks.
            For these reasons, our primary focus in this contribution
            will be on addressing the challenges in interpreting and
            using Bernoulli interactions, in both probabilistic local
            networks and metawebs.</p>
            <h2
            id="local-networks-communities-interacting-in-space-and-time">Local
            networks: communities interacting in space and time</h2>
            <p>Probabilistic local networks describe how likely taxa are
            to interact at a given location and time period (i.e.,
            interactions are contingent upon the environmental and
            biological conditions of the community). In local networks,
            edges commonly represent our degree of belief that two taxa
            interact in nature, but can also represent the probability
            of <em>observing</em> this interaction (<span
            class="citation" data-cites="Catchen2023Missinga">Catchen
            <em>et al.</em> (2023)</span>). For example, <span
            class="citation" data-cites="Kopelke2017FooStra">Kopelke
            <em>et al.</em> (2017)</span> assembled a dataset of binary
            local European food webs of willow-galling sawflies and
            their natural enemies, clearly referencing each food web in
            space and time. <span class="citation"
            data-cites="Gravel2019BriElt">Gravel <em>et al.</em>
            (2019)</span> used this dataset to infer the probabilities
            of locally observing interactions between co-occurring
            species, with the added aspect of situating local networks
            within the context of environmental variables. This was
            achieved by including temperature and precipitation as
            conditional variables in some of their models.</p>
            <p>We define space as the set of geographic coordinates
            <span
            class="math inline">(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            of the spatial boundaries delineating the system, whether
            sampled or targeted. Ecological interactions may exhibit
            variations spatially and along altitudinal gradients, as
            evidenced by changes in hummingbird-plant interactions
            (<span class="citation"
            data-cites="Weinstein2017PerBil">Weinstein &amp; Graham
            (2017b)</span> and <span class="citation"
            data-cites="Weinstein2017ComTra">Weinstein &amp; Graham
            (2017a)</span>) and mosquitoes biting rates (e.g., <span
            class="citation" data-cites="Kulkarni2006EntEva">Kulkarni
            <em>et al.</em> (2006)</span>) at different elevations. In
            contrast, time is defined as the specific time period within
            which interactions were either observed or predicted. With
            these definitions in mind, space and time can be
            conceptualized as distinct patches or time segments.
            Treating them as discrete dimensions aligns with the common
            sampling methods of ecological networks and provides actual
            probabilities of interactions across space and time. This
            differs from the approach of treating them as continuous
            variables, where edge values represent probability densities
            (i.e., relative likelihoods of interactions occurring at
            infinitesimal locations and instants in time) rather than
            discrete probability values (which can be obtained by
            integrating probability densities). By employing discrete
            locations and time periods, we can quantify both an area
            <span class="math inline"><em>A</em></span> and a duration
            <span class="math inline"><em>t</em></span>, which can be
            readily used in spatiotemporal analyses of ecological
            networks. For example, when studying network-area
            relationships (NAR, <span class="citation"
            data-cites="Galiana2018Spatiala">Galiana <em>et al.</em>
            (2018)</span>), we anticipate that local probabilities of
            interactions scale positively with area and duration because
            taxa have more opportunities to interact as these dimensions
            expand.</p>
            <p>The probability that two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact locally can
            be conditional on many environmental and biological factors.
            One of these is their co-occurrence <span
            class="math inline"><em>C</em><sub><em>i</em>, <em>j</em></sub></span>,
            which is usually a Boolean describing if the geographic
            distributions of both taxa overlap within the study area.
            Co-occurrence can be modeled probabilistically, in which
            case it may conform to a Bernoulli distribution <span
            class="math inline">$C_{i,j} \sim {\rm Bernoulli}(P_{i,j}(x,
            y, z))$</span>, where the probability of co-occurrence <span
            class="math inline"><em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            can be estimated through the application of joint species
            distribution models (e.g., <span class="citation"
            data-cites="Pollock2014UndCoo">Pollock <em>et al.</em>
            (2014)</span>). Considering that the probability that two
            non co-occurring taxa interact locally is zero (i.e., <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=0) = 0</span>),
            the probability of local interaction can be obtained by
            multiplying the probability of interaction given
            co-occurrence with the probability of co-occurrence:</p>
            <p><span id="eq:co-occur"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=1) × <em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>).   (1)</span></span></p>
            <p>Other important factors that can impact our estimation of
            interaction probabilities at the local scale are taxa
            relative abundance (<span class="citation"
            data-cites="Canard2012Emergencea">Canard <em>et al.</em>
            (2012)</span>) and traits (<span class="citation"
            data-cites="Poisot2015Speciesa">Poisot <em>et al.</em>
            (2015)</span>), as well as environmental factors such as
            temperature (<span class="citation"
            data-cites="Angilletta2004TemGro">Angilletta <em>et al.</em>
            (2004)</span>), precipitation (<span class="citation"
            data-cites="Woodward2012CliCha">Woodward <em>et al.</em>
            (2012)</span>), habitat structure (<span class="citation"
            data-cites="Klecka2014EffHab">Klecka &amp; Boukal
            (2014)</span>), and presence of other interacting taxa in
            the network (<span class="citation"
            data-cites="Pilosof2017MulNat">Pilosof <em>et al.</em>
            (2017)</span>, <span class="citation"
            data-cites="Kefi2012MorMea">Kéfi <em>et al.</em>
            (2012)</span>). Here, we use the variable <span
            class="math inline"><em>Ω</em></span> to describe the
            biological and ecological context in which interaction
            probabilities were estimated. For example, if a research
            team conducts a mesocosm experiment to estimate interaction
            probabilities between predators and prey with and without
            shelters, <span class="math inline"><em>Ω</em></span> would
            represent the presence or absence of these shelters. Like
            co-occurrence, <span class="math inline"><em>Ω</em></span>
            can also be modeled probabilistically when the stochasticity
            or uncertainty of environmental and biological factors is
            considered. In sum, <span
            class="math inline"><em>Ω</em></span> represents all
            ecological and biological variables that were taken into
            consideration when measuring interaction probabilities and
            is, therefore, a subset of all factors actually impacting
            ecological interactions.</p>
            <p>The probability that two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact in a local
            network <span class="math inline"><em>N</em></span> can thus
            be conditional on the area <span
            class="math inline"><em>A</em></span>, the time interval
            <span class="math inline"><em>t</em></span>, their
            co-occurrence <span class="math inline"><em>C</em></span>
            and chosen environmental and biological conditions <span
            class="math inline"><em>Ω</em></span>. This gives us the
            following equation when all of these conditions are included
            in the estimation of interaction probabilities:</p>
            <p><span id="eq:local"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>C</em>,<em>Ω</em>).   (2)</span></span></p>
            <p>The representation of the local context in which
            probabilities are estimated and the variables that should be
            taken into consideration depend on the study system, the
            objectives of the study, and the resources available to the
            researchers. In other words, these variables do not
            systematically need to be accounted for. For example, in
            <span class="citation" data-cites="Gravel2019BriElt">Gravel
            <em>et al.</em> (2019)</span>, for the purpose of model
            comparison, some models incorporated environmental variables
            as conditional factors to estimate interaction
            probabilities, while others did not. When accounted for,
            these variables should be clearly described in the
            documentation of the data (<span class="citation"
            data-cites="Brimacombe2023ShoReu">Brimacombe <em>et al.</em>
            (2023)</span>), preferentially in mathematical terms to
            avoid any confusion in their interpretation and to limit
            manipulation errors during their re-use. For instance,
            ecologists should be explicit about their consideration of
            co-occurrence in their estimation of local interaction
            probabilities. Indeed, it is important to specify if
            probability values are conditional <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=1)</span>
            or not <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>)</span>
            on co-occurrence since this can significantly impact the
            interpretation and analysis of the data. In tbl. 1, we
            present examples of studies that used these diverse
            formulations of probabilistic interactions and conditional
            variables.</p>
            <div id="tbl:prob">
            <table>
            <caption>Table 1: <strong>Notation of probabilistic
            interactions.</strong> The probability of interaction
            between two taxa <span class="math inline"><em>i</em></span>
            and <span class="math inline"><em>j</em></span> is
            interpreted differently in a metaweb <span
            class="math inline"><em>M</em></span> of potential
            interactions and a local network <span
            class="math inline"><em>N</em></span> of realized
            interactions. Each notation includes a different conditional
            variable, when applicable. An example of a study employing
            each of these notations and conditional variables is
            provided, with the specific variables used indicated in
            parentheses. The study marked with an asterisk has been
            carried out on binary networks. Note that interaction
            probabilities can be contingent upon multiple conditional
            variables, or none at all. Additionally, local interaction
            probabilities might represent the probability of observing
            an interaction, not necessarily its actual
            occurrence.</caption>
            <colgroup>
            <col style="width: 16%" />
            <col style="width: 12%" />
            <col style="width: 37%" />
            <col style="width: 33%" />
            </colgroup>
            <thead>
            <tr class="header">
            <th style="text-align: left;">Notation</th>
            <th style="text-align: left;">Type</th>
            <th style="text-align: left;">Description</th>
            <th style="text-align: right;">Reference</th>
            </tr>
            </thead>
            <tbody>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>)</span></td>
            <td style="text-align: left;">potential</td>
            <td style="text-align: left;">biological feasibility of the
            interaction</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Strydom2022Food">Strydom <em>et al.</em>
            (2022)</span></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the
            interaction</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Fortuna2006HabLos">Fortuna &amp; Bascompte
            (2006)</span> (null model)</td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            in a given area</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Galiana2018Spatiala">Galiana <em>et al.</em>
            (2018)</span> *</td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>t</em>)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            during a given time period</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Weinstein2017ComTra">Weinstein &amp; Graham
            (2017a)</span></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given that the taxa co-occur</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Gravel2019BriElt">Gravel <em>et al.</em>
            (2019)</span></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>Ω</em>)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given environmental conditions</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Gravel2019BriElt">Gravel <em>et al.</em>
            (2019)</span> (temperature and precipitation)</td>
            </tr>
            </tbody>
            </table>
            </div>
            <h2
            id="metawebs-regional-catalogs-of-interactions">Metawebs:
            regional catalogs of interactions</h2>
            <p>Metawebs are networks of potential interactions that have
            been designed for broad spatial, temporal, and taxonomic
            scales (e.g., species food webs at the continental scale).
            They represent the probability that taxa can biologically
            interact regardless of their co-occurrence and local
            environmental conditions. Potential interactions are by
            definition context-independent, i.e. they are not measured
            at a specific location and time. In contrast with
            probabilistic local networks, which represent the
            stochasticity of interactions occurring in nature,
            probabilistic metawebs measure our degree of belief in the
            capacity of two taxa to interact (i.e., the probability that
            their traits could support an interaction in the right
            conditions). In other words, potential interactions describe
            the probability that there exists at least one combination
            of phenotypes of taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> that can interact with
            each other if they were to encounter. To reduce any biases
            in our calculation of this probability, it is crucial to
            ensure that the set of traits sampled or considered
            accurately reflects the overall trait distribution in both
            taxa. This enhances our confidence in concluding that the
            taxa cannot interact when no positive observation has been
            made and increases our capacity to detect an interaction
            when it is biologically feasible. In addition to one-on-one
            interactions, the extent of sampling effort also contributes
            to shaping our assessment of metaweb properties, as sampling
            a larger number of local networks allows us to capture more
            potential interactions (<span class="citation"
            data-cites="McLeod2021SamAsy">McLeod <em>et al.</em>
            (2021a)</span>). A probability of potential interaction in a
            metaweb <span class="math inline"><em>M</em></span> can be
            expressed as</p>
            <p><span id="eq:metaweb"><span
            class="math display"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>),   (3)</span></span></p>
            <p>which, in contrast with eq. 2, is not conditional on any
            spatial, temporal, or environmental variables (tbl. 1).</p>
            <p>Starting from a selected set of taxa, which are usually
            distributed within a broad region of interest, probabilistic
            metawebs can be built using different data sources,
            including literature review, fieldwork, and predictive
            models (e.g., the metaweb of trophic interactions between
            Canadian mammals inferred by <span class="citation"
            data-cites="Strydom2022Food">Strydom <em>et al.</em>
            (2022)</span>). Every pair of taxa that have confidently
            been observed to interact at least once can be given a
            probability of <span class="math inline">1</span> (i.e.,
            <span
            class="math inline"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>) = 1</span>)
            since we know that they <em>can</em> interact. This is not
            the case in local probabilistic networks, in which
            interaction events usually remain stochastic (i.e., <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) &lt; 1</span>)
            after empirically observing interactions because of their
            intrinsic spatiotemporal variability. Similarly, although
            rare interactions typically have low probabilities in local
            networks, they can have high probabilities in metawebs when
            the traits of both taxa are congruent. On the other hand,
            interactions that were never observed can have low
            probability values in both metawebs and local networks,
            going as low as <span class="math inline">0</span> for
            forbidden links. However, because of observation errors due
            to taxonomic misidentifications and ecological
            misinterpretations (e.g., due to cryptic species and
            interactions, <span class="citation"
            data-cites="Pringle2020Resolving">Pringle &amp; Hutchinson
            (2020)</span>), many observations of interactions are in
            reality false positives. Likewise, forbidden interactions
            can be false negatives in metawebs, e.g. if they have been
            assessed for specific phenotypes, locations or time.
            Employing Bayesian models, whether they are mechanistic or
            phenomenological, has the potential to enhance the accuracy
            of our estimation of interaction probabilities within both
            local and potential networks (e.g., <span class="citation"
            data-cites="Bartomeus2016ComFra">Bartomeus <em>et al.</em>
            (2016)</span>, <span class="citation"
            data-cites="Cirtwill2019QuaFra">Cirtwill <em>et al.</em>
            (2019)</span>). This improvement is achieved in potential
            networks by leveraging prior information regarding the
            feasibility of interactions along with the empirical data on
            observed interactions.</p>
            <h1 id="properties-of-probabilistic-networks">Properties of
            probabilistic networks</h1>
            <p>Probabilistic local networks and metawebs differ in their
            type of interactions (i.e., local or potential) and in the
            conditional variables upon which interaction values depend.
            These differences are significant as they influence the
            characteristics of probabilistic networks. Neglecting to
            consider them may result in misleading results and
            interpretation errors when analyzing the properties of
            probabilistic networks, which could be particularly
            problematic when addressing crucial ecological questions
            about networks. Here we compare the characteristics of local
            networks and metawebs through the presentation of four
            common applications of probabilistic interactions.</p>
            <h2 id="host-parasite-network-data">Host-parasite network
            data</h2>
            <p>We use the collection of tripartite host-parasite
            networks sampled across Europe, created by <span
            class="citation" data-cites="Kopelke2017FooStra">Kopelke
            <em>et al.</em> (2017)</span>, in most of our case studies.
            This dataset contains well-resolved binary local
            interactions between willows (52 species), willow-galling
            sawflies (96 species), and their parasitoids (126 species).
            Given its replicated networks spanning large spatiotemporal
            scales, this dataset is well-suited for analyzing a variety
            of ecological hypotheses and processes. Out of a total of
            374 local networks, we retained those containing at least 5
            species, resulting in a set of 233 georeferenced local
            networks. We built a binary metaweb by aggregating all local
            interactions, which gave us a regional metaweb composed of
            274 species and 1080 interactions. In the first two panels
            of fig. 1, we show how the dissimilarity of interactions
            between common species (<span
            class="math inline"><em>β</em><sub><em>O</em><em>S</em></sub></span>)
            and the dissimilarity in species composition (<span
            class="math inline"><em>β</em><sub><em>S</em></sub></span>)
            between the metaweb and aggregated local networks (<span
            class="citation"
            data-cites="Poisot2012Dissimilaritya">Poisot <em>et al.</em>
            (2012)</span>) vary with the number of sampled local
            networks. This shows that networks of local interactions are
            highly dissimilar from the metaweb, both in terms of species
            and interactions, especially when only a limited number of
            sites has been sampled.</p>
            <figure>
            <img src="figures/network_accumulation.png"
            id="fig:accumulation"
            alt="Figure 1: Network accumulation curves. (a) Dissimilarity of interactions between common species and (b) dissimilarity in species composition between aggregated local networks and the binary metaweb of host-parasite interactions. Aggregated local networks were obtained by sequentially and randomly selecting a number of binary local networks and aggregating both their species and interactions. In both panels, the colored line represents the median dissimilarity across simulations and the grey areas cover the 50% and 95% percentile intervals. (c) Scaling of the number of links and (d) scaling of connectance with the number of sampled binary and probabilistic local networks. For a better comparison with binary networks, local probabilistic networks were derived from the probabilistic metaweb with a false positive and false negative rates of zero. A specific value of p (the local probability of interaction among potentially interacting species) was used for all local networks within a particular curve. Aggregated probabilistic local networks were obtained by sequentially and randomly selecting a number of probabilistic local networks and aggregating both their species and interactions (with the value of p adjusting according to eq. 5)." />
            <figcaption aria-hidden="true">Figure 1: <strong>Network
            accumulation curves.</strong> (a) Dissimilarity of
            interactions between common species and (b) dissimilarity in
            species composition between aggregated local networks and
            the binary metaweb of host-parasite interactions. Aggregated
            local networks were obtained by sequentially and randomly
            selecting a number of binary local networks and aggregating
            both their species and interactions. In both panels, the
            colored line represents the median dissimilarity across
            simulations and the grey areas cover the 50% and 95%
            percentile intervals. (c) Scaling of the number of links and
            (d) scaling of connectance with the number of sampled binary
            and probabilistic local networks. For a better comparison
            with binary networks, local probabilistic networks were
            derived from the probabilistic metaweb with a false positive
            and false negative rates of zero. A specific value of <span
            class="math inline"><em>p</em></span> (the local probability
            of interaction among potentially interacting species) was
            used for all local networks within a particular curve.
            Aggregated probabilistic local networks were obtained by
            sequentially and randomly selecting a number of
            probabilistic local networks and aggregating both their
            species and interactions (with the value of <span
            class="math inline"><em>p</em></span> adjusting according to
            eq. 5).</figcaption>
            </figure>
            <p>We converted these binary networks into probabilistic
            ones using models based on simple assumptions. Our models do
            not aim to precisely estimate the actual values of
            probabilistic interactions but rather to offer simplified
            networks as illustrative examples to underscore the
            differences between probabilistic local networks and
            metawebs. We created two probabilistic metawebs by employing
            constant false positive and false negative rates for all
            potential interactions. In the first metaweb, we set both
            false positive and false negative rates to zero to prevent
            artificially inflating the total number of links, enabling a
            more accurate comparison with binary networks. This gave us
            a probability of potential interaction of 1 when at least
            one interaction has been observed and of 0 in the absence of
            any observed interaction between a given pair of species. In
            the second metaweb, we introduced a 5% false positive rate
            to account for spurious interactions and a 10% false
            negative rate to capture the elevated occurrence of false
            negatives in ecological networks (<span class="citation"
            data-cites="Catchen2023Missinga">Catchen <em>et al.</em>
            (2023)</span>). Observed potential interactions were thus
            given a probability of 95%, whereas unobserved ones were
            assigned a probability of 10%.</p>
            <p>To build probabilistic local networks, we first recognize
            that local interactions must initially be biologically
            feasible before occurring at a specific time and space. A
            local probability of interaction <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>)</span>
            can be expressed as the product of the probability of local
            interaction given that the two taxa can potentially interact
            <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>M</em><sub><em>i</em> → <em>j</em></sub>=1)</span>,
            which we denote as <span
            class="math inline"><em>p</em></span> for the sake of
            simplicity, with their probability of potential interaction
            <span
            class="math inline"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>)</span>:</p>
            <p><span id="eq:local_meta"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>M</em><sub><em>i</em> → <em>j</em></sub>=1) × <em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>).   (4)</span></span></p>
            <p>We built the probabilistic local networks from the binary
            ones by using the probabilistic metawebs and a constant
            value of <span class="math inline"><em>p</em></span> across
            interactions. Lower values of <span
            class="math inline"><em>p</em></span> indicate that feasible
            interactions rarely occur locally, intermediate values
            around 50% suggest considerable spatiotemporal variability,
            while higher values indicate that potential interactions are
            nearly always realized. Following eq. 4, the local
            probabilities of interaction between a given pair of taxa
            consistently remained equal to or below their probability of
            potential interaction.</p>
            <p>In the last two panels of fig. 1, we show how the
            aggregated number of links and connectance (i.e., the
            proportion of all of the <span
            class="math inline"><em>S</em><sup>2</sup></span> possible
            links that are realized, where <span
            class="math inline"><em>S</em></span> represents species
            richness) scale with the number of sampled local
            probabilistic networks, according to different values of
            <span class="math inline"><em>p</em></span>. When
            aggregating local probabilistic networks, the constancy of
            the probability of potential interaction across the entire
            study area means that any rise in the probability of local
            interaction is solely attributable to an increase in <span
            class="math inline"><em>p</em></span>. The probability <span
            class="math inline"><em>p</em><sub>1, 2</sub></span> of
            local interaction among potentially interacting species in
            an aggregated network <span
            class="math inline"><em>N</em><sub>1, 2</sub></span> is
            obtained by:</p>
            <p><span id="eq:aggregate"><span
            class="math display"><em>p</em><sub>1, 2</sub> = 1 − (1−<em>p</em><sub>1</sub>) × (1−<em>p</em><sub>2</sub>),   (5)</span></span></p>
            <p>where <span
            class="math inline"><em>p</em><sub>1</sub></span> and <span
            class="math inline"><em>p</em><sub>2</sub></span> are the
            probabilities of local interaction among two potentially
            interacting species in the subnetworks <span
            class="math inline"><em>N</em><sub>1</sub></span> and <span
            class="math inline"><em>N</em><sub>2</sub></span>,
            respectively.</p>
            <p>By comparing the scaling relationships observed in binary
            and probabilistic networks, fig. 1 illustrates that high
            values of <span class="math inline"><em>p</em></span> lead
            to systematic overestimations in the number of links and
            connectance, especially when <span
            class="math inline"><em>p</em> = 1</span> (corresponding to
            the scenario where local probabilities of interactions are
            equivalent to the probabilities of potential interactions).
            However, these biases tend to diminish as the number of
            sampled networks increases.</p>
            <h2 id="taxonomic-scaling-of-interactions">Taxonomic scaling
            of interactions</h2>
            <p>Probabilistic networks offer a versatile approach to
            tackle a broad array of ecological questions, depending on
            their level of organization. For instance, the assemblage of
            interactions across ecological scales can be explored
            through species-based networks, while clade-based networks
            provide insights into macroevolutionary processes (e.g.,
            <span class="citation" data-cites="Gomez2010EcoInt">Gomez
            <em>et al.</em> (2010)</span>). Given that our
            interpretation of the properties and dynamics of ecological
            networks depends on their taxonomic scale (<span
            class="citation"
            data-cites="Guimaraes2020Structurea">Guimarães
            (2020)</span>), investigating the phylogenetic scaling of
            network structure emerges as a promising research avenue.
            Examining the same system at various taxonomic scales can
            yield meaningful and complementary ecological information,
            and, in our perspective, employing probabilistic networks is
            an effective approach for such analyses.</p>
            <p>There are no inherent differences in the taxonomic
            scaling between local networks and metawebs, as only the
            nodes are defined taxonomically. In other words, the
            probability values of edges in both local networks (eq. 2)
            and metawebs (eq. 3) are not conditional on any taxonomic
            scale. The phylogenetic scale is tied to the definition of
            the event itself (i.e., the interaction between two taxa),
            not to the conditional variables. In both types of networks,
            transitioning to a broader level of organization can be
            accomplished directly by using probabilities from finer
            scales. For example, in a network with <span
            class="math inline"><em>n</em><sub><em>A</em></sub></span>
            species from genus <span
            class="math inline"><em>A</em></span> and <span
            class="math inline"><em>n</em><sub><em>B</em></sub></span>
            species from genus <span
            class="math inline"><em>B</em></span>, one can compute the
            probability that at least one species from genus <span
            class="math inline"><em>A</em></span> interacts with at
            least one species from genus <span
            class="math inline"><em>B</em></span> as follows:</p>
            <p><span id="eq:taxo"><span class="math display">$$P(A
            \rightarrow B) = 1 - \prod_{i = 1}^{n_A}\prod_{j =
            1}^{n_B}(1 - P(A_i
            \rightarrow B_j)),\qquad{(6)}$$</span></span></p>
            <p>where <span
            class="math inline"><em>A</em><sub><em>i</em></sub></span>
            and <span
            class="math inline"><em>B</em><sub><em>j</em></sub></span>
            are the species of the corresponding genus. If it is known
            that at least two of these species interact (i.e., <span
            class="math inline"><em>P</em>(<em>A</em><sub><em>i</em></sub>→<em>B</em><sub><em>j</em></sub>) = 1</span>
            for at least one pair of <span
            class="math inline"><em>i</em>, <em>j</em></span>), it
            implies a probability of genus interaction equal to <span
            class="math inline">1</span>. <span class="citation"
            data-cites="Canard2012Emergencea">Canard <em>et al.</em>
            (2012)</span> built a species-based network following a
            similar approach, by using simulated interactions between
            individuals derived from a neutral model (i.e., a model that
            assumed ecological equivalence among individuals). In
            contrast, a more sophisticated approach is necessary when
            transitioning from a broader to a finer level of
            organization. This is because knowledge of an interaction
            between two genera does not guarantee that all possible
            pairwise combinations of their species will also interact.
            One possible method is to build a finer-scale network by
            generating probabilities of interactions through random
            sampling from a beta distribution, parameterized by the
            broader-scale network.</p>
            <p>Ideally, our biological interpretation of probabilistic
            interactions should remain consistent across a network even
            if it incorporates heterogenous levels of organization
            (e.g., a network whose nodes are composed of both species
            and trophic species). This is common in ecological networks
            where taxonomic resolution is typically low (<span
            class="citation"
            data-cites="Hemprich-Bennett2021AssImp">Hemprich-Bennett
            <em>et al.</em> (2021)</span>, <span class="citation"
            data-cites="VazquezSS2022Ecological">Vázquez <em>et al.</em>
            (2022)</span>). Interaction probabilities at broader
            taxonomic scales should be based on probabilities of
            interactions between individuals, for both local networks
            and metawebs. For instance, in local individual-based food
            webs, the probability that two individuals interact reflects
            our degree of belief that one individual will consume the
            other. Similarly, in local species-based food webs, the
            probability that two species interact represents our degree
            of belief that at least one individual from the predator
            species consumes at least another individual from the prey
            species. Furthermore, in local clade-based food webs, the
            probability that two clades interact represents our degree
            of belief that at least two species from these clades
            interact with each other or, equivalently, that at least two
            individuals from these clades interact with each other.
            Fundamentally, the taxonomic scaling of interactions
            involves aggregating interactions between individuals into
            larger groups, which may exhibit varying degrees of
            homogeneity based on the organisms and the study system. In
            that regard, taxonomic scaling is analogous to the spatial
            and temporal scaling of interactions, as they all represent
            different ways to aggregate individuals into broader groups
            (either spatially, temporally, or taxonomically).</p>
            <h2
            id="spatial-and-temporal-scaling-of-interactions">Spatial
            and temporal scaling of interactions</h2>
            <p>The investigation of network-area relationships (NARs)
            and interaction accumulation curves (IACs) is an important
            area of research in network ecology. First, NARs elucidate
            the scaling of network properties (such as modularity and
            trophic chain lengths) with spatial scale (e.g., <span
            class="citation" data-cites="Wood2015Effects">Wood <em>et
            al.</em> (2015)</span>, <span class="citation"
            data-cites="Galiana2018Spatiala">Galiana <em>et al.</em>
            (2018)</span>). The variations in network structure across
            spatial scales may stem from the scaling of species richness
            (species-area relationships, SARs) and the number of
            interactions (<span class="citation"
            data-cites="Brose2004UniSpa">Brose <em>et al.</em>
            (2004)</span>) with the sampled area. Additionally,
            ecological processes unfolding at distinct spatial scales,
            such as the spatial variability in local community
            composition resulting from different sequences of extinction
            and colonization events, can also contribute to this
            variation (<span class="citation"
            data-cites="Galiana2018Spatiala">Galiana <em>et al.</em>
            (2018)</span>). Next, IACs describe the scaling of the
            number of observed interactions with sampling effort (<span
            class="citation" data-cites="Jordano2016Samplingb">Jordano
            (2016)</span>), which can also impact various measures of
            network structure such as connectance (<span
            class="citation" data-cites="McLeod2021Sampling">McLeod
            <em>et al.</em> (2021b)</span>). Apart from sampling effort,
            the temporal scaling of interactions elucidates how network
            structure changes with the duration of the network (<span
            class="citation"
            data-cites="Poisot2012Dissimilaritya">Poisot <em>et al.</em>
            (2012)</span>), acknowledging that distinct interactions
            take place over time. Given that probabilistic local
            networks can quantify the spatiotemporal variability of
            interactions (eq. 2), they serve as valuable tools for
            developing and testing equations describing the scaling of
            network structure with space and time.</p>
            <p>Metawebs and local networks exhibit distinct
            relationships with spatial and temporal scales. On one hand,
            probabilistic metawebs, being independent of any local
            context, feature potential interactions that do not scale
            with space and time. This is because potential interactions
            depend solely on the biological capacity of two taxa to
            interact, regardless of their co-occurrence and specific
            environmental conditions. This implies that the probability
            of two taxa potentially interacting should theoretically be
            the same in all metawebs in which they are present, provided
            that the data and methods used for estimation are
            consistent. As a result, if a smaller metaweb <span
            class="math inline"><em>M</em><sub>1</sub></span> is derived
            from subsampling a regional metaweb <span
            class="math inline"><em>M</em><sub>0</sub></span> to
            represent potential interactions among a subset of taxa in a
            smaller region, their probabilities of interaction should be
            identical regardless of scale, i.e. <span
            class="math inline"><em>P</em><sub><em>M</em><sub>1</sub></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>M</em><sub>0</sub></sub>(<em>i</em>→<em>j</em>)</span>.
            However, because the number of taxa may be higher in the
            larger metaweb, it is expected that the overall number of
            interactions would be higher in the larger network, despite
            pairwise probabilities being identical.</p>
            <p>On the other hand, local interactions scale both
            spatially and temporally, given that they have more
            opportunities to be realized and observed in larger areas
            and longer durations. This is attributed to factors such as
            a higher number of individuals, greater trait variations,
            and increased opportunities for encounters, as highlighted
            by <span class="citation"
            data-cites="McLeod2020EffSpe">McLeod <em>et al.</em>
            (2020)</span>. For example, if a local probabilistic network
            <span class="math inline"><em>N</em><sub>1</sub></span> with
            an area <span
            class="math inline"><em>A</em><sub>1</sub></span> is derived
            from a larger network <span
            class="math inline"><em>N</em><sub>0</sub></span> with an
            area <span
            class="math inline"><em>A</em><sub>0</sub></span>, and <span
            class="math inline"><em>A</em><sub>1</sub></span> is
            entirely nested within <span
            class="math inline"><em>A</em><sub>0</sub></span>,
            interaction probabilities should be lower in the smaller
            network, i.e. <span
            class="math inline"><em>P</em><sub><em>N</em><sub>1</sub></sub>(<em>i</em>→<em>j</em>|<em>A</em><sub>1</sub>&lt;<em>A</em><sub>0</sub>) ≤ <em>P</em><sub><em>N</em><sub>0</sub></sub>(<em>i</em>→<em>j</em>|<em>A</em><sub>0</sub>)</span>.
            However, if <span
            class="math inline"><em>A</em><sub>1</sub></span> and <span
            class="math inline"><em>A</em><sub>0</sub></span> are
            disjoint, interaction probabilities could potentially be
            higher in the smaller area, contingent upon their
            environmental and biological conditions. Likewise,
            interaction probabilities are expected to be lower in
            networks with shorter durations when time intervals are
            nested. In fig. 2, we show how the expected number of local
            host-parasite interactions scales with area, represented as
            an expanding latitudinal window, in comparison with
            potential interactions. Even though we employed local
            probabilities of interactions equal to potential
            interactions for the purpose of comparison (i.e., using
            <span class="math inline"><em>p</em> = 1</span> here), we
            notice that the total number of potential interactions
            scales more rapidly than local interactions. This is due to
            the fact that numerous potential interactions involve
            species that never co-occur, and as a result, these
            interactions are not captured in local networks.</p>
            <figure>
            <img src="figures/spatial_scaling.png" id="fig:spatial"
            alt="Figure 2: Spatial scaling of interactions. Expected number of host-parasite interactions in a network aggregating all probabilistic (a) local and (b) potential interactions within a latitudinal window of a given length. Every dashed curve corresponds to a different expanding window characterized by a different central latitude, with the colored solid line representing the median number of interactions. Heatmaps of the expected number of (c) local and (d) potential interactions found in windows of specified length and central latitudes. Probabilities of potential interactions were obtained with a false positive rate of 5% and a false negative rate of 10%. Local probabilistic interactions were derived from the probabilistic metaweb by setting the value of p (the local probability of interaction among potentially interacting species) to 1, ensuring a conservative comparison between aggregated local networks and metawebs. Aggregated local networks were obtained by aggregating both the species and interactions found within a particular latitudinal window, with the values of p remaining at their maximum value of 1 following eq. 5." />
            <figcaption aria-hidden="true">Figure 2: <strong>Spatial
            scaling of interactions.</strong> Expected number of
            host-parasite interactions in a network aggregating all
            probabilistic (a) local and (b) potential interactions
            within a latitudinal window of a given length. Every dashed
            curve corresponds to a different expanding window
            characterized by a different central latitude, with the
            colored solid line representing the median number of
            interactions. Heatmaps of the expected number of (c) local
            and (d) potential interactions found in windows of specified
            length and central latitudes. Probabilities of potential
            interactions were obtained with a false positive rate of 5%
            and a false negative rate of 10%. Local probabilistic
            interactions were derived from the probabilistic metaweb by
            setting the value of <span
            class="math inline"><em>p</em></span> (the local probability
            of interaction among potentially interacting species) to
            <span class="math inline">1</span>, ensuring a conservative
            comparison between aggregated local networks and metawebs.
            Aggregated local networks were obtained by aggregating both
            the species and interactions found within a particular
            latitudinal window, with the values of <span
            class="math inline"><em>p</em></span> remaining at their
            maximum value of <span class="math inline">1</span>
            following eq. 5.</figcaption>
            </figure>
            <h2
            id="box-1-a-spatiotemporally-explicit-model-of-interactions">Box
            1: A spatiotemporally explicit model of interactions</h2>
            <p>Predicting local networks across time and space is a
            pivotal goal of network ecology (<span class="citation"
            data-cites="Strydom2021Roadmapa">Strydom <em>et al.</em>
            (2021)</span>). In a context of scarcity of interaction data
            (<span class="citation"
            data-cites="Jordano2016Samplingb">Jordano (2016)</span>),
            ecologists must resort to predictive models for
            reconstructing networks at fine spatial and temporal scales.
            For instance, real-time biomonitoring data coupled with
            appropriate numerical models (<span class="citation"
            data-cites="Bohan2017Nextgenerationa">Bohan <em>et al.</em>
            (2017)</span>) can be employed to reconstruct local
            ecological networks, opening avenues for in-depth studies on
            local ecosystem functioning and dynamics. Apart from their
            predictive applications, statistical models can also be
            crafted for descriptive purposes, elucidating key parameters
            of interest such as probabilities of interactions. In such
            instances, the parameter values themselves offer valuable
            ecological insights. Various types of models (e.g., Bayesian
            and machine learning models) have been used for both
            predictive and descriptive purposes in the realm of
            ecological interactions (<span class="citation"
            data-cites="Strydom2021Roadmapa">Strydom <em>et al.</em>
            (2021)</span>). The probabilistic representation of
            interactions acknowledges the inherent uncertainty in these
            models, typically expressed through probability
            distributions. We introduce and develop a simple generative
            mechanistic model for probabilistic local interactions that
            takes into consideration their spatiotemporal variability
            (i.e. a spatiotemporally explicit model of local
            interactions). It is essential to note that our model is not
            designed for potential interactions, which are
            scale-independent. Rather, it could prove valuable for
            predicting local interactions across time and space by
            generating new interaction data following parameter
            inference.</p>
            <p>As indicated by eq. 1, the probability that two taxa
            <span class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact at a specific
            location <span
            class="math inline">(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            is determined by the product of their probability of
            interaction given co-occurrence and their probability of
            co-occurrence. First, their co-occurrence probability can be
            calculated using their individual occurrence probabilities
            <span
            class="math inline"><em>P</em><sub><em>i</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            and <span
            class="math inline"><em>P</em><sub><em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)</span>,
            along with the strength of association <span
            class="math inline"><em>γ</em></span> between their
            occurrences and co-occurrence (<span class="citation"
            data-cites="Cazelles2016Theorya">Cazelles <em>et al.</em>
            (2016)</span>):</p>
            <p><span id="eq:modelcoprob"><span
            class="math display"><em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>) = <em>P</em><sub><em>i</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)<em>P</em><sub><em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)<em>γ</em>.   (7)</span></span></p>
            <p>When <span class="math inline"><em>γ</em> &gt; 1</span>,
            it signifies a positive association in the geographic
            distributions of both taxa, indicating that the presence of
            one taxon enhances the probability of occurrence of the
            other. In empirical networks, <span
            class="math inline"><em>γ</em> &gt; 1</span> holds true for
            the majority of species pairs (<span class="citation"
            data-cites="Catchen2023Missinga">Catchen <em>et al.</em>
            (2023)</span>). The co-occurrence of both taxa is modeled as
            the outcome of a Bernoulli trial</p>
            <p><span id="eq:modelco"><span
            class="math display"><em>C</em> ∼ <em>B</em><em>e</em><em>r</em><em>n</em><em>o</em><em>u</em><em>l</em><em>l</em><em>i</em>(<em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)).   (8)</span></span></p>
            <p>Next, the probability of interaction given co-occurrence
            can be made temporally explicit by modeling it as a Poisson
            process with rate parameter <span
            class="math inline"><em>λ</em></span>. This parameter
            represents the expected frequency of interaction between the
            taxa within a defined time interval and can be estimated
            using prior data on interaction strengths, when accessible.
            The probability that two co-occurring taxa engage in an
            interaction during a time period <span
            class="math inline"><em>t</em><sub>0</sub></span> is given
            by:</p>
            <p><span id="eq:modelrate"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=1) = 1 − <em>e</em><sup>−<em>λ</em><em>t</em><sub>0</sub></sup>,   (9)</span></span></p>
            <p>which tends toward <span class="math inline">1</span> as
            <span class="math inline"><em>t</em><sub>0</sub> → ∞</span>.
            It is important to note that the units of <span
            class="math inline"><em>λ</em></span> and <span
            class="math inline"><em>t</em><sub>0</sub></span> are
            complementary. For instance, if the duration <span
            class="math inline"><em>t</em><sub>0</sub></span> is
            measured in months, <span
            class="math inline"><em>λ</em></span> denote the expected
            number of interactions per month.</p>
            <p>The occurrence of an interaction between <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> can be modeled as a
            Bernoulli trial with a probability of <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>)</span>.
            Consequently, a Bayesian inference model can be built based
            on the preceding equations to estimate the value of the
            <span class="math inline"><em>λ</em></span> and <span
            class="math inline"><em>γ</em></span> parameters and
            generate novel interaction data:</p>
            <p><span id="eq:model"><span
            class="math display"><em>I</em> ∼ Bernoulli(<em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>))   (10)</span></span>
            <span id="eq:modeleq"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>i</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)<em>P</em><sub><em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)<em>γ</em>(1−<em>e</em><sup>−<em>λ</em><em>t</em><sub>0</sub></sup>)   (11)</span></span></p>
            <p><span id="eq:modelgamma"><span
            class="math display"><em>γ</em> ∼ Gamma(2,0.5)   (12)</span></span></p>
            <p><span id="eq:modellambda"><span
            class="math display"><em>λ</em> ∼ Exponential(2)   (13)</span></span></p>
            <p>In fig. 3, we show the variation in the probability of
            interaction under different parameter values. In the right
            panel, we notice that, irrespective of the interaction rate
            <span class="math inline"><em>λ</em></span>, the probability
            of interaction converges toward an asymptote determined by
            the co-occurrence <span
            class="math inline"><em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            (eq. 7). This model can be customized in different ways,
            such as by linking <span
            class="math inline"><em>λ</em></span> with specific
            environmental variables or explicitly incorporating
            observation errors (i.e., the probabilities of false
            negatives and false positives).</p>
            <figure>
            <img src="figures/spatiotemporal_model.png"
            id="fig:spatiotemporal"
            alt="Figure 3: Parameters of the spatiotemporally explicit model of interactions. (a) Probability of local interaction given by the process model (eq. 11) under different values of \lambda and \gamma, with t_0 = 1. The parameter values used in the right panel are denoted by the white stars. (b) Scaling of the probability of interaction with the duration parameter t_0 in eq. 11, for different values of \lambda and \gamma. In both panels, the individual probabilities of occurrence P_i(x, y, z) and P_j(x, y, z) are set to a constant value of 0.5." />
            <figcaption aria-hidden="true">Figure 3: <strong>Parameters
            of the spatiotemporally explicit model of
            interactions.</strong> (a) Probability of local interaction
            given by the process model (eq. 11) under different values
            of <span class="math inline"><em>λ</em></span> and <span
            class="math inline"><em>γ</em></span>, with <span
            class="math inline"><em>t</em><sub>0</sub> = 1</span>. The
            parameter values used in the right panel are denoted by the
            white stars. (b) Scaling of the probability of interaction
            with the duration parameter <span
            class="math inline"><em>t</em><sub>0</sub></span> in eq. 11,
            for different values of <span
            class="math inline"><em>λ</em></span> and <span
            class="math inline"><em>γ</em></span>. In both panels, the
            individual probabilities of occurrence <span
            class="math inline"><em>P</em><sub><em>i</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            and <span
            class="math inline"><em>P</em><sub><em>j</em></sub>(<em>x</em>,<em>y</em>,<em>z</em>)</span>
            are set to a constant value of <span
            class="math inline">0.5</span>.</figcaption>
            </figure>
            <h2 id="sampling-binary-networks">Sampling binary
            networks</h2>
            <p>Probabilistic networks can be used to predict binary
            interactions through random draws. This can be useful when
            analyzing the structure of a probabilistic network when
            analytical probabilistic measures are lacking. A binary
            network can be generated from independent Bernoulli trials
            for each interaction. The distribution of a network’s
            property can then be obtained after measuring the structure
            of all randomly generated networks (<span class="citation"
            data-cites="Poisot2016Structure">Poisot <em>et al.</em>
            (2016)</span>). Doing so allows us to represent the
            variability of network structure, although possibly with
            biases when connectance is low (<span class="citation"
            data-cites="Poisot2014WheEco">Poisot &amp; Gravel
            (2014)</span>, <span class="citation"
            data-cites="Chagnon2015ChaTop">Chagnon (2015)</span>). When
            binary networks are generated under a null model, this
            method can be used for null hypothesis significance testing,
            in which case the observed measure is compared to a null
            distribution (e.g., <span class="citation"
            data-cites="Bascompte2003NesAss">Bascompte <em>et al.</em>
            (2003)</span>). Furthermore, randomly generating binary
            networks across space and time can help us visualize the
            spatiotemporal variability of network structure and test
            ecological hypotheses on interactions across large spatial
            and temporal scales.</p>
            <p>There are at least two different approaches when sampling
            binary networks from probabilistic webs across space,
            e.g. if we want to predict a binary network for each of
            <span class="math inline"><em>n</em> × <em>n</em></span>
            grid cells. The first approach is to conduct a single
            Bernoulli trial for each pair of taxa found in the region of
            interest based on their regional probability of interaction.
            As a result, each pair of taxa that are predicted to
            interact in the regional network realization will interact
            in all of the <span
            class="math inline"><em>n</em><sup>2</sup></span> networks
            in which they co-occur. This sampling technique is best used
            with potential interactions that have no spatial variation.
            The second approach is to independently sample each of the
            <span class="math inline"><em>n</em><sup>2</sup></span>
            networks. In practice, this can be done by generating a
            different probabilistic network for each grid cell.
            Depending on how they were obtained, these networks can
            differ in their taxa composition (nodes) and/or interaction
            probabilities (edges). Then, binary networks can be
            independently sampled for each grid cell. Because this
            method generates spatial variation in binary interactions,
            it is best used with local interactions.</p>
            <p>The choice of sampling approach has an impact on the
            selection of grid cell size. In the first approach,
            interactions will be the same regardless of cell size
            because interactions are sampled only once from the regional
            network. However, in the second approach, local interaction
            probabilities depend on the network area. For example, let
            <span class="math inline"><em>N</em><sub>1</sub></span> and
            <span class="math inline"><em>N</em><sub>2</sub></span> be
            networks of area <span
            class="math inline">$\frac{1}{2}A_0$</span> nested within
            <span class="math inline"><em>A</em><sub>0</sub></span> and
            disjoint from each other, i.e. two contiguous cells that
            form <span
            class="math inline"><em>N</em><sub>0</sub></span>. If <span
            class="math inline"><em>N</em><sub>1</sub></span> and <span
            class="math inline"><em>N</em><sub>2</sub></span> are
            independent (which is rarely the case in reality because of
            spatial auto-correlation), the probability that two taxa
            <span class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact in <span
            class="math inline"><em>N</em><sub>0</sub></span> is given
            by:</p>
            <p><span id="eq:binary"><span
            class="math display"><em>P</em><sub><em>N</em><sub>0</sub></sub>(<em>i</em>→<em>j</em>) = 1 − (1−<em>P</em><sub><em>N</em><sub>1</sub></sub>(<em>i</em>→<em>j</em>)) × (1−<em>P</em><sub><em>N</em><sub>2</sub></sub>(<em>i</em>→<em>j</em>)).   (14)</span></span></p>
            <p>Because of its larger area, the probability that the two
            taxa interact in <span
            class="math inline"><em>N</em><sub>0</sub></span> is higher
            than in <span
            class="math inline"><em>N</em><sub>1</sub></span> and <span
            class="math inline"><em>N</em><sub>2</sub></span>. When
            sampling binary interactions from local networks, it is thus
            important to sample at the same spatial scale as the one for
            which probabilities were estimated. Otherwise, interaction
            probabilities must be adjusted to correspond to the targeted
            cell size and avoid systematic biases in prediction.</p>
            <p><img src="figures/network_sampling.png"
            id="fig:sampling" /></p>
            <h2
            id="prediction-of-local-networks-from-metawebs">Prediction
            of local networks from metawebs</h2>
            <p>Metawebs are an important source of ecological
            information that can be leveraged for predicting local
            networks. Indeed, binary local networks can be reconstructed
            by selecting a subset of both taxa and interactions from the
            metaweb (<span class="citation"
            data-cites="Dunne2006Network">Dunne (2006)</span>). This
            implies that there are always more interactions in a metaweb
            compared to its corresponding local networks. In practice,
            because a community’s composition is arguably easier to
            sample or predict than its interactions, the list of taxa to
            select can be known empirically or obtained using range maps
            or species distribution models. The challenge is rather to
            choose which interactions to select from the metaweb, which
            could require more sophisticated statistical models and
            ecological knowledge. Because the structure of local
            networks is constrained by the one of the metaweb (<span
            class="citation" data-cites="Saravia2022Ecological">Saravia
            <em>et al.</em> (2022)</span>), inferring their structure
            from the metaweb before predicting local pairwise
            interactions could prove promising (<span class="citation"
            data-cites="Strydom2021Roadmapa">Strydom <em>et al.</em>
            (2021)</span>).</p>
            <p>When building probabilistic local networks from a
            probabilistic metaweb, interaction probabilities decrease.
            This is because two taxa must first have the capacity to
            interact (i.e., <span
            class="math inline"><em>M</em><sub><em>i</em>, <em>j</em></sub><sup>*</sup> = 1</span>
            in the binary metaweb <span
            class="math inline"><em>M</em><sup>*</sup></span>) before
            interacting locally:</p>
            <p><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>) × <em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>M</em><sub><em>i</em>, <em>j</em></sub><sup>*</sup>=1).</span></p>
            <p>Therefore, inferring local networks from their metaweb
            keeping the same values of interaction probability would
            generate systematic biases in the prediction. In that case,
            these networks would instead represent smaller-scale
            metawebs of potential interactions, which could be
            misinterpreted as local interactions. As suggested by <span
            class="citation" data-cites="McLeod2021Sampling">McLeod
            <em>et al.</em> (2021b)</span>, even though metawebs do not
            capture the spatiotemporal variability of interactions, they
            set the upper limit of local interactions. In other words,
            the probability that two taxa interact at a given location
            and time is always lower or equal to the probability of
            their potential interaction:</p>
            <p><span id="eq:switch"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>C</em>,<em>Ω</em>) ≤ <em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>).   (15)</span></span></p>
            <p>Moreover, the probability that two taxa have the
            biological capacity to interact must be higher than the
            probability that they will ever interact because they might
            never co-occur or encounter locally. More precisely, the
            accumulated probability of realized interactions across all
            spatial, temporal, and environmental conditions must be
            lower than the probability of potential interaction,
            i.e. </p>
            <p><span id="eq:all"><span
            class="math display">∫<sub><em>Ω</em></sub>∫<sub><em>A</em></sub>∫<sub><em>t</em></sub><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>Ω</em>)<em>d</em><em>t</em><em>d</em><em>A</em><em>d</em><em>Ω</em> ≤ <em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>).   (16)</span></span></p>
            <h1 id="conclusion">Conclusion</h1>
            <p>In this contribution, we underline the importance of
            network metadata for adequately interpreting and
            manipulating probabilistic interaction data. The
            mathematical representation of probabilities and their
            statistical properties depend on the type of interactions
            (local or potential) and the conditions in which they were
            estimated. We showed that probabilistic local networks and
            metawebs differ in their relationship to spatial and
            temporal scales, with potential interactions remaining
            consistent across scales. In contrast with metawebs, local
            interactions are measured in a specific context (e.g., in a
            given area, time, and biological and environmental
            conditions) and depend on taxa co-occurrence. These
            important conceptual differences bring to light the need to
            use probabilistic data with caution, for instance when
            generating binary network realizations across space and
            predicting local networks from metawebs. Clear metadata
            describing the type of interaction and the variables used in
            their estimation are required to ensure adequate data
            manipulation. Better data practices and foundations for
            probabilistic thinking in network ecology could enable more
            reliable assessments of the spatiotemporal variability and
            uncertainty of biotic interactions.</p>
            <p>It is essential to enhance our comprehension of both
            potential and local interactions, especially considering the
            current scarcity of interaction data. However, while
            sampling biological communities does decrease the
            uncertainty of interactions by accumulating evidence for
            their feasibility and local realization, there is a limit to
            how much we can diminish their inherent randomness. In
            metawebs, probabilities reflect our limited knowledge of
            interactions (i.e., our degree of belief that interactions
            are feasible), which is expected to improve with a larger
            volume of data. We should anticipate that potential
            interactions will become more definitive (with probabilities
            approaching <span class="math inline">0</span> or <span
            class="math inline">1</span>) as we investigate various
            conditions, including different combinations of species
            traits. Conversely, in the case of local networks, which can
            be seen as random instances of metawebs, randomness cannot
            be as reduced. In local interactions, probabilities can be
            divided into a component representing uncertainty and
            another representing spatiotemporal variability. Owing to
            environmental heterogeneity, there will invariably be
            instances in which an interaction occurs and others in which
            it does not, across different times and locations,
            irrespective of the extent to which we can improve our
            knowledge of its biological feasibility and the local
            conditions that facilitate its occurrence. When local
            networks depict probabilities of observing interactions
            rather than just their actual occurrence, we must also
            consider the observation variability as an additional source
            of randomness. Every ecological process is stochastic but
            there is also a possibility that a phenomenon goes
            undetected. Quantifying and partitioning this stochasticity
            will enable us to make more accurate predictions about
            ecological interactions at various spatial and temporal
            scales.</p>
            <h1 id="acknowledgment">Acknowledgment</h1>
            <p>We acknowledge that this study was conducted on land
            within the traditional unceded territory of the Saint
            Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat,
            and Omàmiwininiwak nations. This work was supported by the
            Institute for Data Valorisation (IVADO) and the Natural
            Sciences and Engineering Research Council of Canada (NSERC)
            Collaborative Research and Training Experience (CREATE)
            program, through the Computational Biodiversity Science and
            Services (BIOS²) program. A special thanks to all members of
            the Black Holes and Revelations working group (organized by
            BIOS²) for their insightful discussions and valuable
            feedback on this manuscript.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs"
            class="references csl-bib-body hanging-indent"
            role="doc-bibliography">
            <div id="ref-Andrade-Pacheco2020Finding" class="csl-entry"
            role="doc-biblioentry">
            Andrade-Pacheco, R., Rerolle, F., Lemoine, J., Hernandez,
            L., Meïté, A., Juziwelo, L., <em>et al.</em> (2020). <a
            href="https://doi.org/10.1038/s41598-020-67666-3">Finding
            hotspots: Development of an adaptive spatial sampling
            approach</a>. <em>Scientific Reports</em>, 10, 10939.
            </div>
            <div id="ref-Angilletta2004TemGro" class="csl-entry"
            role="doc-biblioentry">
            Angilletta, M.J., Jr., Steury, T.D. &amp; Sears, M.W.
            (2004). <a
            href="https://doi.org/10.1093/icb/44.6.498">Temperature,
            Growth Rate, and Body Size in Ectotherms: Fitting Pieces of
            a Life-History Puzzle1</a>. <em>Integrative and Comparative
            Biology</em>, 44, 498–509.
            </div>
            <div id="ref-Araujo2011Usinga" class="csl-entry"
            role="doc-biblioentry">
            Araujo, M.B., Rozenfeld, A., Rahbek, C. &amp; Marquet, P.A.
            (2011). <a
            href="https://doi.org/10.1111/j.1600-0587.2011.06919.x">Using
            species co-occurrence networks to assess the impacts of
            climate change</a>. <em>Ecography</em>, 34, 897–908.
            </div>
            <div id="ref-Bartomeus2016ComFra" class="csl-entry"
            role="doc-biblioentry">
            Bartomeus, I., Gravel, D., Tylianakis, J.M., Aizen, M.A.,
            Dickie, I.A. &amp; Bernard-Verdier, M. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12666">A common
            framework for identifying linkage rules across different
            types of interactions</a>. <em>Functional Ecology</em>, 30,
            1894–1903.
            </div>
            <div id="ref-Bascompte2003NesAss" class="csl-entry"
            role="doc-biblioentry">
            Bascompte, J., Jordano, P., Melián, C.J. &amp; Olesen, J.M.
            (2003). <a
            href="https://doi.org/10.1073/pnas.1633576100">The nested
            assembly of plantanimal mutualistic networks</a>.
            <em>Proceedings of the National Academy of Sciences</em>,
            100, 9383–9387.
            </div>
            <div id="ref-Berlow2004Interaction" class="csl-entry"
            role="doc-biblioentry">
            Berlow, E.L., Neutel, A.-M., Cohen, J.E., De Ruiter, P.C.,
            Ebenman, B., Emmerson, M., <em>et al.</em> (2004). <a
            href="https://doi.org/10.1111/j.0021-8790.2004.00833.x">Interaction
            strengths in food webs: Issues and opportunities</a>.
            <em>Journal of Animal Ecology</em>, 73, 585–598.
            </div>
            <div id="ref-Blanchet2020Cooccurrencea" class="csl-entry"
            role="doc-biblioentry">
            Blanchet, F.G., Cazelles, K. &amp; Gravel, D. (2020). <a
            href="https://doi.org/10.1111/ele.13525">Co-occurrence is
            not evidence of ecological interactions</a>. <em>Ecology
            Letters</em>, 23, 1050–1063.
            </div>
            <div id="ref-Bohan2017Nextgenerationa" class="csl-entry"
            role="doc-biblioentry">
            Bohan, D.A., Vacher, C., Tamaddoni-Nezhad, A., Raybould, A.,
            Dumbrell, A.J. &amp; Woodward, G. (2017). <a
            href="https://doi.org/10.1016/j.tree.2017.03.001">Next-Generation
            Global Biomonitoring: Large-scale, Automated Reconstruction
            of Ecological Networks</a>. <em>Trends in Ecology &amp;
            Evolution</em>, 32, 477–487.
            </div>
            <div id="ref-Bolnick2011WhyInt" class="csl-entry"
            role="doc-biblioentry">
            Bolnick, D.I., Amarasekare, P., Araújo, M.S., Bürger, R.,
            Levine, J.M., Novak, M., <em>et al.</em> (2011). <a
            href="https://doi.org/10.1016/j.tree.2011.01.009">Why
            intraspecific trait variation matters in community
            ecology</a>. <em>Trends in Ecology &amp; Evolution</em>, 26,
            183–192.
            </div>
            <div id="ref-Borrett2019Walk" class="csl-entry"
            role="doc-biblioentry">
            Borrett, S.R. &amp; Scharler, U.M. (2019). <a
            href="https://doi.org/10.1016/j.ecolind.2019.105451">Walk
            partitions of flow in Ecological Network Analysis: Review
            and synthesis of methods and indicators</a>. <em>Ecological
            Indicators</em>, 106, 105451.
            </div>
            <div id="ref-Boulangeat2012AccDis" class="csl-entry"
            role="doc-biblioentry">
            Boulangeat, I., Gravel, D. &amp; Thuiller, W. (2012). <a
            href="https://doi.org/10.1111/j.1461-0248.2012.01772.x">Accounting
            for dispersal and biotic interactions to disentangle the
            drivers of species distributions and their abundances</a>.
            <em>Ecology Letters</em>, 15, 584–593.
            </div>
            <div id="ref-Brimacombe2023ShoReu" class="csl-entry"
            role="doc-biblioentry">
            Brimacombe, C., Bodner, K., Michalska-Smith, M., Poisot, T.
            &amp; Fortin, M.-J. (2023). <a
            href="https://doi.org/10.1371/journal.pbio.3002068">Shortcomings
            of reusing species interaction networks created by different
            sets of researchers</a>. <em>PLOS Biology</em>, 21,
            e3002068.
            </div>
            <div id="ref-Brose2004UniSpa" class="csl-entry"
            role="doc-biblioentry">
            Brose, U., Ostling, A., Harrison, K. &amp; Martinez, N.D.
            (2004). <a
            href="https://doi.org/10.1038/nature02297">Unified spatial
            scaling of species and their trophic interactions</a>.
            <em>Nature</em>, 428, 167–171.
            </div>
            <div id="ref-Canard2012Emergencea" class="csl-entry"
            role="doc-biblioentry">
            Canard, E., Mouquet, N., Marescot, L., Gaston, K.J., Gravel,
            D. &amp; Mouillot, D. (2012). <a
            href="https://doi.org/10.1371/journal.pone.0038295">Emergence
            of Structural Patterns in Neutral Trophic Networks</a>.
            <em>PLOS ONE</em>, 7, e38295.
            </div>
            <div id="ref-Catchen2023Missinga" class="csl-entry"
            role="doc-biblioentry">
            Catchen, M.D., Poisot, T., Pollock, L.J. &amp; Gonzalez, A.
            (2023). The missing link: Discerning true from false
            negatives when sampling species interaction networks.
            </div>
            <div id="ref-Cazelles2016Theorya" class="csl-entry"
            role="doc-biblioentry">
            Cazelles, K., Araujo, M.B., Mouquet, N. &amp; Gravel, D.
            (2016). <a
            href="https://doi.org/10.1007/s12080-015-0281-9">A theory
            for species co-occurrence in interaction networks</a>.
            <em>Theoretical Ecology</em>, 9, 39–48.
            </div>
            <div id="ref-Chagnon2015ChaTop" class="csl-entry"
            role="doc-biblioentry">
            Chagnon, P.-L. (2015). <a
            href="https://doi.org/10.1016/j.ecocom.2015.01.004">Characterizing
            topology of ecological networks along gradients: The limits
            of metrics’ standardization</a>. <em>Ecological
            Complexity</em>, 22, 36–39.
            </div>
            <div id="ref-Choh2012PreRol" class="csl-entry"
            role="doc-biblioentry">
            Choh, Y., Ignacio, M., Sabelis, M.W. &amp; Janssen, A.
            (2012). <a
            href="https://doi.org/10.1038/srep00728">Predator-prey role
            reversals, juvenile experience and adult antipredator
            behaviour</a>. <em>Scientific Reports</em>, 2, 728.
            </div>
            <div id="ref-Cirtwill2019QuaFra" class="csl-entry"
            role="doc-biblioentry">
            Cirtwill, A.R., Eklöf, A., Roslin, T., Wootton, K. &amp;
            Gravel, D. (2019). <a
            href="https://doi.org/10.1111/2041-210X.13180">A
            quantitative framework for investigating the reliability of
            empirical network construction</a>. <em>Methods in Ecology
            and Evolution</em>, 10, 902–911.
            </div>
            <div id="ref-Dunne2006Network" class="csl-entry"
            role="doc-biblioentry">
            Dunne, J.A. (2006). The Network Structure of Food Webs. In:
            <em>Ecological networks: Linking structure and dynamics</em>
            (eds. Dunne, J.A. &amp; Pascual, M.). Oxford University
            Press, pp. 27–86.
            </div>
            <div id="ref-Elmasri2020HieBay" class="csl-entry"
            role="doc-biblioentry">
            Elmasri, M., Farrell, M.J., Davies, T.J. &amp; Stephens,
            D.A. (2020). <a href="https://doi.org/10.1214/19-AOAS1296">A
            hierarchical Bayesian model for predicting ecological
            interactions using scaled evolutionary relationships</a>.
            <em>The Annals of Applied Statistics</em>, 14, 221–240.
            </div>
            <div id="ref-Elton2001Animal" class="csl-entry"
            role="doc-biblioentry">
            Elton, C.S. (2001). <em>Animal Ecology</em>. University of
            Chicago Press, Chicago, IL.
            </div>
            <div id="ref-Emmerson2004Predatora" class="csl-entry"
            role="doc-biblioentry">
            Emmerson, M.C. &amp; Raffaelli, D. (2004). <a
            href="https://doi.org/10.1111/j.0021-8790.2004.00818.x">Predatorprey
            body size, interaction strength and the stability of a real
            food web</a>. <em>Journal of Animal Ecology</em>, 73,
            399–409.
            </div>
            <div id="ref-Fortuna2006HabLos" class="csl-entry"
            role="doc-biblioentry">
            Fortuna, M.A. &amp; Bascompte, J. (2006). <a
            href="https://doi.org/10.1111/j.1461-0248.2005.00868.x">Habitat
            loss and the structure of plantanimal mutualistic
            networks</a>. <em>Ecology Letters</em>, 9, 281–286.
            </div>
            <div id="ref-Galiana2018Spatiala" class="csl-entry"
            role="doc-biblioentry">
            Galiana, N., Lurgi, M., Claramunt-López, B., Fortin, M.-J.,
            Leroux, S., Cazelles, K., <em>et al.</em> (2018). <a
            href="https://doi.org/10.1038/s41559-018-0517-3">The spatial
            scaling of species interaction networks</a>. <em>Nature
            Ecology &amp; Evolution</em>, 2, 782–790.
            </div>
            <div id="ref-Gilman2010FraCom" class="csl-entry"
            role="doc-biblioentry">
            Gilman, S.E., Urban, M.C., Tewksbury, J., Gilchrist, G.W.
            &amp; Holt, R.D. (2010). <a
            href="https://doi.org/10.1016/j.tree.2010.03.002">A
            framework for community interactions under climate
            change</a>. <em>Trends in Ecology &amp; Evolution</em>, 25,
            325–331.
            </div>
            <div id="ref-Godsoe2022Species" class="csl-entry"
            role="doc-biblioentry">
            Godsoe, W., Murray, R. &amp; Iritani, R. (2022). <a
            href="https://doi.org/10.1111/oik.09282">Species
            interactions and diversity: A unified framework using Hill
            numbers</a>. <em>Oikos</em>, n/a, e09282.
            </div>
            <div id="ref-Golubski2011ModMod" class="csl-entry"
            role="doc-biblioentry">
            Golubski, A.J. &amp; Abrams, P.A. (2011). <a
            href="https://doi.org/10.1111/j.1365-2656.2011.01852.x">Modifying
            modifiers: What happens when interspecific interactions
            interact?</a> <em>Journal of Animal Ecology</em>, 80,
            1097–1108.
            </div>
            <div id="ref-Gomez2010EcoInt" class="csl-entry"
            role="doc-biblioentry">
            Gomez, J.M., Verdu, M. &amp; Perfectti, F. (2010). <a
            href="https://doi.org/10.1038/nature09113">Ecological
            interactions are evolutionarily conserved across the entire
            tree of life</a>. <em>Nature</em>, 465, 918–U6.
            </div>
            <div id="ref-Gonzalez-Varo2016Labilea" class="csl-entry"
            role="doc-biblioentry">
            Gonzalez-Varo, J.P. &amp; Traveset, A. (2016). <a
            href="https://doi.org/10.1016/j.tree.2016.06.009">The Labile
            Limits of Forbidden Interactions</a>. <em>Trends in Ecology
            &amp; Evolution</em>, 31, 700–710.
            </div>
            <div id="ref-Gravel2019BriElt" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Baiser, B., Dunne, J.A., Kopelke, J.-P.,
            Martinez, N.D., Nyman, T., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/ecog.04006">Bringing Elton and
            Grinnell together: A quantitative framework to represent the
            biogeography of ecological interaction networks</a>.
            <em>Ecography</em>, 42, 401–415.
            </div>
            <div id="ref-Gravel2013InfFooa" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Poisot, T., Albouy, C., Velez, L. &amp;
            Mouillot, D. (2013). <a
            href="https://doi.org/10.1111/2041-210X.12103">Inferring
            food web structure from predatorprey body size
            relationships</a>. <em>Methods in Ecology and
            Evolution</em>, 4, 1083–1090.
            </div>
            <div id="ref-Guimaraes2020Structurea" class="csl-entry"
            role="doc-biblioentry">
            Guimarães, P.R. (2020). <a
            href="https://doi.org/10.1146/annurev-ecolsys-012220-120819">The
            Structure of Ecological Networks Across Levels of
            Organization</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 51, 433–460.
            </div>
            <div id="ref-Guimera2009MisSpu" class="csl-entry"
            role="doc-biblioentry">
            Guimerà, R. &amp; Sales-Pardo, M. (2009). <a
            href="https://doi.org/10.1073/pnas.0908366106">Missing and
            spurious interactions and the reconstruction of complex
            networks</a>. <em>Proceedings of the National Academy of
            Sciences</em>, 106, 22073–22078.
            </div>
            <div id="ref-Hemprich-Bennett2021AssImp" class="csl-entry"
            role="doc-biblioentry">
            Hemprich-Bennett, D.R., Oliveira, H.F.M., Le Comber, S.C.,
            Rossiter, S.J. &amp; Clare, E.L. (2021). <a
            href="https://doi.org/10.1002/ecy.3256">Assessing the impact
            of taxon resolution on network structure</a>.
            <em>Ecology</em>, 102, e03256.
            </div>
            <div id="ref-Herrera1989PolAbu" class="csl-entry"
            role="doc-biblioentry">
            Herrera, C.M. (1989). <a
            href="https://doi.org/10.1007/BF00380158">Pollinator
            abundance, morphology, and flower visitation rate: Analysis
            of the <span>“quantity”</span> component in a
            plant-pollinator system</a>. <em>Oecologia</em>, 80,
            241–248.
            </div>
            <div id="ref-Hortal2015SevSho" class="csl-entry"
            role="doc-biblioentry">
            Hortal, J., de Bello, F., Diniz-Filho, J.A.F., Lewinsohn,
            T.M., Lobo, J.M. &amp; Ladle, R.J. (2015). <a
            href="https://doi.org/10.1146/annurev-ecolsys-112414-054400">Seven
            Shortfalls that Beset Large-Scale Knowledge of
            Biodiversity</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 46, 523–549.
            </div>
            <div id="ref-Jordano1987PatMut" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P. (1987). <a
            href="https://doi.org/10.1086/284665">Patterns of
            Mutualistic Interactions in Pollination and Seed Dispersal:
            Connectance, Dependence Asymmetries, and Coevolution</a>.
            <em>The American Naturalist</em>, 129, 657–677.
            </div>
            <div id="ref-Jordano2016Samplingb" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12763">Sampling
            networks of ecological interactions</a>. <em>Functional
            Ecology</em>, 30, 1883–1893.
            </div>
            <div id="ref-Jordano2003Invarianta" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P., Bascompte, J. &amp; Olesen, J.M. (2003). <a
            href="https://doi.org/10.1046/j.1461-0248.2003.00403.x">Invariant
            properties in coevolutionary networks of plantanimal
            interactions</a>. <em>Ecology Letters</em>, 6, 69–81.
            </div>
            <div id="ref-Kefi2015NetStr" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Berlow, E.L., Wieters, E.A., Joppa, L.N., Wood,
            S.A., Brose, U., <em>et al.</em> (2015). <a
            href="https://doi.org/10.1890/13-1424.1">Network structure
            beyond food webs: Mapping non-trophic and trophic
            interactions on Chilean rocky shores</a>. <em>Ecology</em>,
            96, 291–303.
            </div>
            <div id="ref-Kefi2012MorMea" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Berlow, E.L., Wieters, E.A., Navarrete, S.A.,
            Petchey, O.L., Wood, S.A., <em>et al.</em> (2012). <a
            href="https://doi.org/10.1111/j.1461-0248.2011.01732.x">More
            than a meal… integrating non-feeding interactions into food
            webs</a>. <em>Ecology Letters</em>, 15, 291–300.
            </div>
            <div id="ref-Kefi2016HowStr" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Miele, V., Wieters, E.A., Navarrete, S.A. &amp;
            Berlow, E.L. (2016). <a
            href="https://doi.org/10.1371/journal.pbio.1002527">How
            Structured Is the Entangled Bank? The Surprisingly Simple
            Organization of Multiplex Ecological Networks Leads to
            Increased Persistence and Resilience</a>. <em>PLOS
            Biology</em>, 14, e1002527.
            </div>
            <div id="ref-Klecka2014EffHab" class="csl-entry"
            role="doc-biblioentry">
            Klecka, J. &amp; Boukal, D.S. (2014). <a
            href="https://doi.org/10.1007/s00442-014-3007-6">The effect
            of habitat structure on prey mortality depends on predator
            and prey microhabitat use</a>. <em>Oecologia</em>, 176,
            183–191.
            </div>
            <div id="ref-Kopelke2017FooStra" class="csl-entry"
            role="doc-biblioentry">
            Kopelke, J.-P., Nyman, T., Cazelles, K., Gravel, D.,
            Vissault, S. &amp; Roslin, T. (2017). <a
            href="https://doi.org/10.1002/ecy.1832">Food-web structure
            of willow-galling sawflies and their natural enemies across
            Europe</a>. <em>Ecology</em>, 98, 1730–1730.
            </div>
            <div id="ref-Kulkarni2006EntEva" class="csl-entry"
            role="doc-biblioentry">
            Kulkarni, M.A., Kweka, E., Nyale, E., Lyatuu, E., Mosha,
            F.W., Chandramohan, D., <em>et al.</em> (2006). <a
            href="https://doi.org/10.1093/jmedent/43.3.580">Entomological
            Evaluation of Malaria Vectors at Different Altitudes in Hai
            District, Northeastern Tanzania</a>. <em>Journal of Medical
            Entomology</em>, 43, 580–588.
            </div>
            <div id="ref-McCann2007ProBio" class="csl-entry"
            role="doc-biblioentry">
            McCann, K. (2007). Protecting biostructure. <em>Nature</em>,
            446, 29–29.
            </div>
            <div id="ref-McCann2011FooWeb" class="csl-entry"
            role="doc-biblioentry">
            McCann, K.S. (2011). Food webs (MPB-50). In: <em>Food Webs
            (MPB-50)</em>. Princeton University Press.
            </div>
            <div id="ref-McLeod2021SamAsy" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A., Leroux, S.J., Gravel, D., Chu, C., Cirtwill,
            A.R., Fortin, M.-J., <em>et al.</em> (2021a). <a
            href="https://doi.org/10.1111/oik.08650">Sampling and
            asymptotic network properties of spatial multi-trophic
            networks</a>. <em>Oikos</em>, 130, 2250–2259.
            </div>
            <div id="ref-McLeod2021Sampling" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A., Leroux, S.J., Gravel, D., Chu, C., Cirtwill,
            A.R., Fortin, M.-J., <em>et al.</em> (2021b). <a
            href="https://doi.org/10.1111/oik.08650">Sampling and
            asymptotic network properties of spatial multi-trophic
            networks</a>. <em>Oikos</em>, n/a.
            </div>
            <div id="ref-McLeod2020EffSpe" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A.M., Leroux, S.J. &amp; Chu, C. (2020). <a
            href="https://doi.org/10.1002/ecs2.3018">Effects of species
            traits, motif profiles, and environment on spatial variation
            in multi-trophic antagonistic networks</a>.
            <em>Ecosphere</em>, 11, e03018.
            </div>
            <div id="ref-Melian2011EcoDyn" class="csl-entry"
            role="doc-biblioentry">
            Melián, C.J., Vilas, C., Baldó, F., González-Ortegón, E.,
            Drake, P. &amp; Williams, R.J. (2011). <a
            href="https://doi.org/10.1016/B978-0-12-386475-8.00006-X">Eco-evolutionary
            Dynamics of Individual-Based Food Webs</a>. In: <em>Advances
            in Ecological Research</em>, The Role of Body Size in
            Multispecies Systems (ed. Belgrano, A.). Academic Press, pp.
            225–268.
            </div>
            <div id="ref-Ohlmann2019Diversity" class="csl-entry"
            role="doc-biblioentry">
            Ohlmann, M., Miele, V., Dray, S., Chalmandrier, L.,
            O’Connor, L. &amp; Thuiller, W. (2019). <a
            href="https://doi.org/10.1111/ele.13221">Diversity indices
            for ecological networks: A unifying framework using Hill
            numbers</a>. <em>Ecology Letters</em>, 22, 737–747.
            </div>
            <div id="ref-Olesen2010Missing" class="csl-entry"
            role="doc-biblioentry">
            Olesen, J.M., Bascompte, J., Dupont, Y.L., Elberling, H.,
            Rasmussen, C. &amp; Jordano, P. (2010). <a
            href="https://doi.org/10.1098/rspb.2010.1371">Missing and
            forbidden links in mutualistic networks</a>. <em>Proceedings
            of the Royal Society B: Biological Sciences</em>, 278,
            725–732.
            </div>
            <div id="ref-Pascual2006Ecologicala" class="csl-entry"
            role="doc-biblioentry">
            Pascual, M., Dunne, J.A. &amp; Dunne, J.A. (2006).
            <em>Ecological Networks: Linking Structure to Dynamics in
            Food Webs</em>. Oxford University Press, USA.
            </div>
            <div id="ref-Petchey2008SizFor" class="csl-entry"
            role="doc-biblioentry">
            Petchey, O.L., Beckerman, A.P., Riede, J.O. &amp; Warren,
            P.H. (2008). <a
            href="https://doi.org/10.1073/pnas.0710672105">Size,
            foraging, and food web structure</a>. <em>Proceedings of the
            National Academy of Sciences</em>, 105, 4191–4196.
            </div>
            <div id="ref-Pilosof2017MulNat" class="csl-entry"
            role="doc-biblioentry">
            Pilosof, S., Porter, M.A., Pascual, M. &amp; Kéfi, S.
            (2017). <a
            href="https://doi.org/10.1038/s41559-017-0101">The
            multilayer nature of ecological networks</a>. <em>Nature
            Ecology &amp; Evolution</em>, 1, 1–9.
            </div>
            <div id="ref-Poisot2012Dissimilaritya" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Canard, E., Mouillot, D., Mouquet, N. &amp;
            Gravel, D. (2012). <a
            href="https://doi.org/10.1111/ele.12002">The dissimilarity
            of species interaction networks</a>. <em>Ecology
            Letters</em>, 15, 1353–1361.
            </div>
            <div id="ref-Poisot2016Structure" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D.,
            Fortin, M.-J. &amp; Stouffer, D.B. (2016). <a
            href="https://doi.org/10.1111/2041-210X.12468">The structure
            of probabilistic networks</a>. <em>Methods in Ecology and
            Evolution</em>, 7, 303–312.
            </div>
            <div id="ref-Poisot2014WheEco" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T. &amp; Gravel, D. (2014). <a
            href="https://doi.org/10.7717/peerj.251">When is an
            ecological network complex? Connectance drives degree
            distribution and emerging network properties</a>.
            <em>PeerJ</em>, 2, e251.
            </div>
            <div id="ref-Poisot2015Speciesa" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Stouffer, D.B. &amp; Gravel, D. (2015). <a
            href="https://doi.org/10.1111/oik.01719">Beyond species: Why
            ecological interaction networks vary through space and
            time</a>. <em>Oikos</em>, 124, 243–251.
            </div>
            <div id="ref-Pollock2014UndCoo" class="csl-entry"
            role="doc-biblioentry">
            Pollock, L.J., Tingley, R., Morris, W.K., Golding, N.,
            O’Hara, R.B., Parris, K.M., <em>et al.</em> (2014). <a
            href="https://doi.org/10.1111/2041-210X.12180">Understanding
            co-occurrence bymodelling species simultaneously with a
            Joint Species DistributionModel (JSDM)</a>. <em>Methods in
            Ecology and Evolution</em>, 5, 397–406.
            </div>
            <div id="ref-Pomeranz2019InfPre" class="csl-entry"
            role="doc-biblioentry">
            Pomeranz, J.P.F., Thompson, R.M., Poisot, T. &amp; Harding,
            J.S. (2019). <a
            href="https://doi.org/10.1111/2041-210X.13125">Inferring
            predatorprey interactions in food webs</a>. <em>Methods in
            Ecology and Evolution</em>, 10, 356–367.
            </div>
            <div id="ref-Pringle2020Resolving" class="csl-entry"
            role="doc-biblioentry">
            Pringle, R.M. &amp; Hutchinson, M.C. (2020). <a
            href="https://doi.org/10.1146/annurev-ecolsys-110218-024908">Resolving
            Food-Web Structure</a>. <em>Annual Review of Ecology,
            Evolution, and Systematics</em>, 51, 55–80.
            </div>
            <div id="ref-Proulx2005Networka" class="csl-entry"
            role="doc-biblioentry">
            Proulx, S.R., Promislow, D.E.L. &amp; Phillips, P.C. (2005).
            <a href="https://doi.org/10.1016/j.tree.2005.04.004">Network
            thinking in ecology and evolution</a>. <em>Trends in Ecology
            &amp; Evolution</em>, SPECIAL ISSUE: BUMPER BOOK REVIEW, 20,
            345–353.
            </div>
            <div id="ref-Pulliam1974Theory" class="csl-entry"
            role="doc-biblioentry">
            Pulliam, H.R. (1974). <a
            href="https://doi.org/10.1086/282885">On the Theory of
            Optimal Diets</a>. <em>The American Naturalist</em>, 108,
            59–74.
            </div>
            <div id="ref-Rooney2012IntFoo" class="csl-entry"
            role="doc-biblioentry">
            Rooney, N. &amp; McCann, K.S. (2012). <a
            href="https://doi.org/10.1016/j.tree.2011.09.001">Integrating
            food web diversity, structure and stability</a>. <em>Trends
            in Ecology &amp; Evolution</em>, 27, 40–46.
            </div>
            <div id="ref-Salim2022Data" class="csl-entry"
            role="doc-biblioentry">
            Salim, J.A., Saraiva, A.M., Zermoglio, P.F., Agostini, K.,
            Wolowski, M., Drucker, D.P., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1093/gigascience/giac043">Data
            standardization of plantpollinator interactions</a>.
            <em>GigaScience</em>, 11, giac043.
            </div>
            <div id="ref-Saravia2022Ecological" class="csl-entry"
            role="doc-biblioentry">
            Saravia, L.A., Marina, T.I., Kristensen, N.P., De Troch, M.
            &amp; Momo, F.R. (2022). <a
            href="https://doi.org/10.1111/1365-2656.13652">Ecological
            network assembly: How the regional metaweb influences local
            food webs</a>. <em>Journal of Animal Ecology</em>, 91,
            630–642.
            </div>
            <div id="ref-Schmeller2015GloTer" class="csl-entry"
            role="doc-biblioentry">
            Schmeller, D.S., Julliard, R., Bellingham, P.J., Böhm, M.,
            Brummitt, N., Chiarucci, A., <em>et al.</em> (2015). <a
            href="https://doi.org/10.1016/j.jnc.2015.03.003">Towards a
            global terrestrial species monitoring program</a>.
            <em>Journal for Nature Conservation</em>, 25, 51–57.
            </div>
            <div id="ref-Singer2012GeoMos" class="csl-entry"
            role="doc-biblioentry">
            Singer, M.C. &amp; McBride, C.S. (2012). <a
            href="https://doi.org/10.1890/11-2078.1">Geographic mosaics
            of species’ association: A definition and an example driven
            by plantinsect phenological synchrony</a>. <em>Ecology</em>,
            93, 2658–2673.
            </div>
            <div id="ref-Stouffer2011RolBod" class="csl-entry"
            role="doc-biblioentry">
            Stouffer, D.B., Rezende, E.L. &amp; Amaral, L.A.N. (2011).
            <a
            href="https://doi.org/10.1111/j.1365-2656.2011.01812.x">The
            role of body mass in diet contiguity and food-web
            structure</a>. <em>Journal of Animal Ecology</em>, 80,
            632–639.
            </div>
            <div id="ref-Strydom2022Food" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron,
            D., Farrell, M.J., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1111/2041-210X.13835">Food web
            reconstruction through phylogenetic transfer of low-rank
            network representation</a>. <em>Methods in Ecology and
            Evolution</em>, 13.
            </div>
            <div id="ref-Strydom2021Roadmapa" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Catchen, M.D., Banville, F., Caron, D.,
            Dansereau, G., Desjardins-Proulx, P., <em>et al.</em>
            (2021). <a href="https://doi.org/10.1098/rstb.2021.0063">A
            roadmap towards predicting species interaction networks
            (across space and time)</a>. <em>Philosophical Transactions
            of the Royal Society B-Biological Sciences</em>, 376,
            20210063.
            </div>
            <div id="ref-Tylianakis2007HabMod" class="csl-entry"
            role="doc-biblioentry">
            Tylianakis, J.M., Tscharntke, T. &amp; Lewis, O.T. (2007).
            <a href="https://doi.org/10.1038/nature05429">Habitat
            modification alters the structure of tropical hostparasitoid
            food webs</a>. <em>Nature</em>, 445, 202–205.
            </div>
            <div id="ref-VazquezSS2022Ecological" class="csl-entry"
            role="doc-biblioentry">
            Vázquez, D.P., Peralta, G., Cagnolo, L., Santos, M. &amp;
            Igual, E. autores contribuyeron por. (2022). <a
            href="https://doi.org/10.25260/EA.22.32.2.1.1846">Ecological
            interaction networks. What we know, what we don’t, and why
            it matters</a>. <em>Ecología Austral</em>, 32, 670–697.
            </div>
            <div id="ref-Weinstein2017ComTra" class="csl-entry"
            role="doc-biblioentry">
            Weinstein, B.G. &amp; Graham, C.H. (2017a). <a
            href="https://doi.org/10.1016/j.fooweb.2017.05.002">On
            comparing traits and abundance for predicting species
            interactions with imperfect detection</a>. <em>Food
            Webs</em>, 11, 17–25.
            </div>
            <div id="ref-Weinstein2017PerBil" class="csl-entry"
            role="doc-biblioentry">
            Weinstein, B.G. &amp; Graham, C.H. (2017b). <a
            href="https://doi.org/10.1111/ele.12730">Persistent bill and
            corolla matching despite shifting temporal resources in
            tropical hummingbird-plant interactions</a>. <em>Ecology
            Letters</em>, 20, 326–335.
            </div>
            <div id="ref-Wood2015Effects" class="csl-entry"
            role="doc-biblioentry">
            Wood, S.A., Russell, R., Hanson, D., Williams, R.J. &amp;
            Dunne, J.A. (2015). <a
            href="https://doi.org/10.1002/ece3.1640">Effects of spatial
            scale of sampling on food web structure</a>. <em>Ecology and
            Evolution</em>, 5, 3769–3782.
            </div>
            <div id="ref-Woodward2010ChaEco" class="csl-entry"
            role="doc-biblioentry">
            Woodward, G., Benstead, J.P., Beveridge, O.S., Blanchard,
            J., Brey, T., Brown, L.E., <em>et al.</em> (2010). <a
            href="https://doi.org/10.1016/B978-0-12-381363-3.00002-2">Chapter
            2 - Ecological Networks in a Changing Climate</a>. In:
            <em>Advances in Ecological Research</em>, Ecological
            Networks (ed. Woodward, G.). Academic Press, pp. 71–138.
            </div>
            <div id="ref-Woodward2012CliCha" class="csl-entry"
            role="doc-biblioentry">
            Woodward, G., Brown, L.E., Edwards, F.K., Hudson, L.N.,
            Milner, A.M., Reuman, D.C., <em>et al.</em> (2012). <a
            href="https://doi.org/10.1098/rstb.2012.0245">Climate change
            impacts in multispecies systems: Drought alters food web
            size structure in a field experiment</a>. <em>Philosophical
            Transactions of the Royal Society B: Biological
            Sciences</em>, 367, 2990–2997.
            </div>
            </div>
        </article>
    </main>

    <footer>
                <div class="container">
            <h2>Supplementary Material</h2>
            <span class="appendix--list">
                <a href="appendix/Banville_et_al_2023_ecological_interpretation_s1.pdf" target="_blank">
                    <b>1:</b>&nbsp;Supplementary Material
                </a><br />
            </span>
        </div>
                <div class="container">
            <h2>Affiliations</h2>
            <sup>1</sup>&nbsp;Université de
            Montréal; <sup>2</sup>&nbsp;Université de
            Sherbrooke; <sup>3</sup>&nbsp;Quebec Centre for Biodiversity
            Science; <sup>4</sup>&nbsp;University of
            Toronto; <sup>5</sup>&nbsp;McGill University
            <br />
            
        </div>
    </footer>

</body>

</html>