<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Deciphering probabilistic species interaction
networks</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);

            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        html {
            background-color: white;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article>p {
            line-height: 135%;
        }

        article>h1 {
            padding-top: 3em;
        }

        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }

        a:hover {
            font-weight: 500;
        }

        .citation {
            color: var(--citation);
        }

        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }

        header>.container {
            margin-top: 4em;
        }

        code,
        pre {
            font-family: "Roboto Mono", monospace;
        }

        pre {
            white-space: pre-wrap;
        }

        h1,
        h2,
        h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }

        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }

        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }

        .header--authors>span {
            color: var(--light);
            font-weight: 400;
        }

        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }

        footer {
            background-color: var(--grey);
            font-weight: 300;
        }

        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }

        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }

        figure>img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }

        figure>figcaption {
            padding: var(--inner-space);
        }

        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }

        table>caption {
            padding: var(--inner-space);
        }

        figcaption,
        caption {
            color: var(--light);
            font-weight: 300;
        }

        thead {
            color: var(--structure);
        }

        td,
        th {
            padding: 3px;
        }

        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }

        tbody>tr:nth-child(even) {
            background: var(--grey)
        }

        ;

        tbody {
            border-bottom: 1px solid var(--structure);
        }

        #refs {
            font-size: 90%;
            column-count: 2;
        }

        #refs>.csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }

        .hanging-indent>.csl-entry {
            padding-left: 15px;
            text-indent: -15px;
        }

        .artifacts {
            text-align: center;
            border-radius: 4px;
            background-color: var(--grey);
            padding: 1em;
            margin-top: 4em;
        }

        .artifacts a{
            background-color: var(--structure);
            text-decoration: none;
            margin: 0px 2em;
            color: white;
            text-transform: capitalize;
            border-radius: 4px;
            text-align: center;
            padding: 4px 4px;
            width: 7em;
            display: inline-block;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.css"
        integrity="sha384-vKruj+a13U8yHIkAyGgK1J3ArTLzrFGBbBc0tDp4ad/EyewESeXE/Iv67Aj8gKZ0" crossorigin="anonymous">
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.js"
        integrity="sha384-PwRUT/YqbnEjkZO0zZxNqcxACrXe+j766U2amXcgMg5457rve2Y7I6ZJSm2A0mS4"
        crossorigin="anonymous"></script>
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/contrib/auto-render.min.js"
        integrity="sha384-+VBxd3r6XgURycqtZ117nYw44OOcIax56Z4dCRWbxyPt0Koah1uHoK0o4+/RRE05" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>

    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                    { left: "$$", right: "$$", display: true },
                    { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">Deciphering probabilistic species
interaction networks</div>
            <div class="header--authors">
                <span class="header--author">
                <a href="https://orcid.org/0000-0001-9051-0597" target="_blank">
                    Francis Banville
                </a>
                <sup class="affiliation">1,2,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0001-6067-1349" target="_blank">
                    Tanya Strydom
                </a>
                <sup class="affiliation">1,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0003-3071-9668" target="_blank">
                    Penelope S. A. Blyth
                </a>
                <sup class="affiliation">4</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-9368-6106" target="_blank">
                    Chris Brimacombe
                </a>
                <sup class="affiliation">5</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-6506-6487" target="_blank">
                    Michael Catchen
                </a>
                <sup class="affiliation">3,6</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-2212-3584" target="_blank">
                    Gabriel Dansereau
                </a>
                <sup class="affiliation">1,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-4498-7076" target="_blank">
                    Dominique Gravel
                </a>
                <sup class="affiliation">2,3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0003-2791-8383" target="_blank">
                    Gracielle Higino
                </a>
                <sup class="affiliation">2</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0001-5575-609X" target="_blank">
                    Thomas Malpas
                </a>
                <sup class="affiliation">4</sup>
                </span><span class="header--author">
                Hana Mayall
                <sup class="affiliation">4</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-2029-2325" target="_blank">
                    Kari Norman
                </a>
                <sup class="affiliation">1</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affiliation">1,3</sup>
                </span>
            </div>
        </div>
    </header>

    <main class="container">
        <section class="abstract">
            Representing species interactions probabilistically (how
            likely are they to occur?) as opposed to deterministically
            (are they occurring?) expresses uncertainties in our
            knowledge of interactions. The sources of uncertainty
            captured by interaction probabilities depend on the method
            used to evaluate them: uncertainty of predictive models,
            subjective assessment of experts, or empirical measurement
            of interaction spatiotemporal variability. However,
            guidelines for the estimation and documentation of
            probabilistic interaction data are still lacking. This is
            concerning because our understanding and analysis of
            interaction probabilities depend on their sometimes elusive
            definition and uncertainty sources. We review how
            probabilistic interactions are defined at different spatial
            scales, from local interactions to regional networks
            (metawebs), with a strong emphasis on host-parasite and
            trophic (predatory and herbivory) interactions. These
            definitions are based on the distinction between the
            realization of an interaction at a specific time and space
            (local) and its biological feasibility (regional). Through
            case studies using host-parasite interactions in Europe, we
            illustrate how these two network representations differ in
            their statistical properties, specifically: how local
            networks and metawebs differ in their spatial and temporal
            scaling of probabilistic interactions, but not in their
            taxonomic scaling. We present two approaches to inferring
            binary interactions from probabilistic ones that account for
            these differences and show that systematic biases arise when
            directly inferring local networks from metawebs. Our results
            underscore the importance of more rigorous descriptions of
            probabilistic species interaction networks that specify
            their type of interaction (local or regional) and
            uncertainty sources.


            <div class="artifacts">
                <a href="Banville_et_al_2024_deciphering_probabilistic_draft.pdf" title="Draft" target="_blank">Draft</a>
                <a href="Banville_et_al_2024_deciphering_probabilistic_preprint.pdf" title="Preprint" target="_blank">Preprint</a>
                <a href="Banville_et_al_2024_deciphering_probabilistic.odt" title="OpenDocument" target="_blank">Libre Office</a>
            </div>
        </section>

        <article>
            <h1 id="introduction">Introduction</h1>
            <p>As we try to navigate global biodiversity change, filling
            in knowledge gaps about biodiversity becomes instrumental to
            monitoring and mitigating those changes <span
            class="citation"
            data-cites="Hortal2015SevSho Gonzalez2022Monitor Abrego2021Accounting">(Abrego
            <em>et al.</em> 2021; Gonzalez &amp; Londoño 2022; Hortal
            <em>et al.</em> 2015)</span>. However, cataloging species,
            populations and, in particular, ecological interactions
            (e.g., predation, parasitism, and pollination) is a
            substantial challenge <span class="citation"
            data-cites="Polis1991Complex Pascual2006Ecologicala">(Pascual
            <em>et al.</em> 2006; Polis 1991)</span>. There are
            methodological and biological constraints that hinder our
            ability to document species interactions, inevitably leading
            to uncertainty in our knowledge of interactions. For
            example, the spatial and temporal uncoupling of species
            <span class="citation" data-cites="Jordano1987PatMut">(e.g.,
            nocturnal and diurnal species coexisting in the same space
            with different daily activity timings, Jordano 1987)</span>
            and the large number of rare and cryptic interactions in a
            community, contribute to these knowledge gaps <span
            class="citation" data-cites="Jordano2016Samplingb">(Jordano
            2016)</span>.</p>
            <p>More generally, several conditions must be satisfied for
            an interaction to be observed locally. First, both species
            must have overlapping geographic ranges, i.e. they must
            co-occur within the region of interest <span
            class="citation"
            data-cites="Morales-Castilla2015Inferringa Cazelles2016Theorya">(Cazelles
            <em>et al.</em> 2016; Morales-Castilla <em>et al.</em>
            2015)</span>. Second, they must have some probability of
            meeting <span class="citation"
            data-cites="Poisot2015Speciesa">(Poisot <em>et al.</em>
            2015)</span>. Probabilities of interspecific encounters are
            typically low, especially for rare species with low
            abundances. While species’ absolute abundances may impact
            interaction frequencies <span class="citation"
            data-cites="Vazquez2007Species">(Vázquez <em>et al.</em>
            2007)</span>, encounter probabilities are determined by
            their relative abundances <span class="citation"
            data-cites="Canard2012Emergencea">(Canard <em>et al.</em>
            2012)</span>. The probability that species meet also depends
            on their biology, such as their phenology <span
            class="citation"
            data-cites="Olesen2010Missing Singer2012GeoMos">(Olesen
            <em>et al.</em> 2010; Singer &amp; McBride 2012)</span> and
            discoverability <span class="citation"
            data-cites="Broom2005You">(Broom &amp; Ruxton 2005)</span>.
            Finally, when species do come into contact, an interaction
            occurs only if their traits are locally compatible <span
            class="citation" data-cites="Poisot2015Speciesa">(Poisot
            <em>et al.</em> 2015)</span>, including their phenotypes
            <span class="citation"
            data-cites="Bolnick2011WhyInt Stouffer2011RolBod Gravel2013InfFooa">(Bolnick
            <em>et al.</em> 2011; Gravel <em>et al.</em> 2013; Stouffer
            <em>et al.</em> 2011)</span> and behavior <span
            class="citation"
            data-cites="Pulliam1974Theory Choh2012PreRol">(Choh <em>et
            al.</em> 2012; Pulliam 1974)</span>. Interactions may also
            be influenced by a third species <span class="citation"
            data-cites="Golubski2011ModMod Sanders2012Indirect">(e.g., a
            more profitable prey species, Golubski &amp; Abrams 2011;
            Sanders &amp; van Veen 2012)</span>. Documenting the
            location and timing of interactions becomes even more
            difficult when accounting for the spatiotemporal variability
            of ecological interactions <span class="citation"
            data-cites="Poisot2012Dissimilaritya Poisot2015Speciesa">(Poisot
            <em>et al.</em> 2012, 2015)</span>. Environmental factors,
            such as temperature <span class="citation"
            data-cites="Angilletta2004TemGro">(Angilletta <em>et
            al.</em> 2004)</span>, drought <span class="citation"
            data-cites="Woodward2012CliCha">(Woodward <em>et al.</em>
            2012)</span>, climate change <span class="citation"
            data-cites="Gilman2010FraCom Woodward2010ChaEco Araujo2011Usinga">(Araujo
            <em>et al.</em> 2011; Gilman <em>et al.</em> 2010; Woodward
            <em>et al.</em> 2010)</span>, and habitat modifications
            <span class="citation"
            data-cites="Tylianakis2007HabMod">(Tylianakis <em>et
            al.</em> 2007)</span>, contribute to this spatiotemporal
            variability by impacting species abundance and traits. Even
            under favorable circumstances, there remains a possibility
            that the interaction does not occur locally, either due to
            the intricate nature of the system or simply by chance. If
            it does occur, it might go undetected, particularly if it
            happens infrequently. In this context, it is unsurprising
            that our knowledge of ecological interactions remains
            limited <span class="citation"
            data-cites="Hortal2015SevSho">(Hortal <em>et al.</em>
            2015)</span> despite extensive biodiversity data collection
            <span class="citation"
            data-cites="Schmeller2015GloTer">(Schmeller <em>et al.</em>
            2015)</span>.</p>
            <p>Knowing the biological capacity of two species to
            interact directly (via e.g., trophic interactions) is
            necessary but not sufficient for inferring their interaction
            at a specific time and space. The recognition of the
            intrinsic variability of species interactions has led
            ecologists to expand their representation of ecological
            networks to include a probabilistic view of interactions
            <span class="citation"
            data-cites="Poisot2016Structure Dallas2017Predictinga Fu2021Link">(Dallas
            <em>et al.</em> 2017; Fu <em>et al.</em> 2021; Poisot <em>et
            al.</em> 2016)</span>. This allows filling in the Eltonian
            shortfall <span class="citation"
            data-cites="Hortal2015SevSho">(Hortal <em>et al.</em>
            2015)</span> by modeling the probability of occurrence of
            interactions <span class="citation"
            data-cites="Gravel2019BriElt">(e.g., Gravel <em>et al.</em>
            2019)</span>, which can be an important tool for directing
            efforts and taking action <span class="citation"
            data-cites="Carlson2021Future">(Carlson <em>et al.</em>
            2021)</span>, especially in places where access and
            resources for research are scarce. The probabilistic
            representation of interactions has been applied to direct
            interactions, which are conceptually and mathematically
            analogous regardless of their biological type (e.g.,
            predation and pollination). This is in contrast with
            indirect interactions (e.g., interspecific competition),
            which arise from distinct ecological processes and are often
            not directly observable <span class="citation"
            data-cites="Kefi2015NetStr Kefi2016HowStr">(Kéfi <em>et
            al.</em> 2015, 2016)</span>.</p>
            <p>Representing direct interactions probabilistically can
            capture the spatiotemporal variability of the aforementioned
            ecological processes and the uncertainty in our knowledge of
            interactions. Networks of probabilistic interactions, within
            a Bayesian perspective, express our degree of belief (or
            confidence) regarding the occurrence or observation of
            interactions. In contrast, interactions are simply regarded
            as either occurring or not in networks of deterministic
            binary interactions. Based on the scale at which they are
            estimated, interaction probabilities may reflect our level
            of confidence in whether interactions will be observed,
            realized locally, or biologically feasible. As an
            illustration, we could outline a situation in which there is
            a <span class="math inline">50%</span> certainty that an
            interaction occurs <span class="math inline">50%</span> of
            the time. Our level of confidence should be more definitive
            (approaching either <span class="math inline">0</span> or
            <span class="math inline">1</span>) as we extend our
            sampling to a broader area and over a longer time period,
            thereby diminishing the uncertainty of our knowledge of
            interactions (but not necessarily the estimation of their
            variability). In the broadest sense, binary interactions are
            also a type of probabilistic interaction, in which the
            numerical value of an interaction is restrained to <span
            class="math inline">0</span> (non-occurring) or <span
            class="math inline">1</span> (occurring). In networks of
            probabilistic interactions, only forbidden interactions
            <span class="citation"
            data-cites="Jordano2003Invarianta Olesen2010Missing">(i.e.,
            interactions prohibited by biological traits or species
            absence, Jordano <em>et al.</em> 2003; Olesen <em>et
            al.</em> 2010)</span> have a probability value of zero,
            provided that intraspecific trait variability is considered
            <span class="citation"
            data-cites="Gonzalez-Varo2016Labilea">(Gonzalez-Varo &amp;
            Traveset 2016)</span>.</p>
            <p>By accounting for the uncertainty of interactions,
            networks of probabilistic interactions may provide a more
            realistic portrait of species interactions. However, how the
            uncertainty of pairwise interactions propagates to network
            structure <span class="citation"
            data-cites="McCann2007ProBio McCann2011FooWeb Proulx2005Networka Rooney2012IntFoo">(i.e.,
            community-level properties driving the functioning,
            dynamics, and resilience of ecosystems, McCann 2007; McCann
            2011; Proulx <em>et al.</em> 2005; Rooney &amp; McCann
            2012)</span> remains to be elucidated. The application and
            development of computational methods in network ecology,
            often based on a probabilistic representation of
            interactions, can alleviate (and guide) the sampling efforts
            required to document species interactions <span
            class="citation" data-cites="Strydom2021Roadmapa">(Strydom
            <em>et al.</em> 2021)</span>. For example, statistical
            models can be used to estimate the uncertainty of pairwise
            interactions <span class="citation"
            data-cites="Cirtwill2019QuaFra">(Cirtwill <em>et al.</em>
            2019)</span> and the probability of missing (false
            negatives) and spurious (false positives) interactions <span
            class="citation" data-cites="Guimera2009MisSpu">(Guimerà
            &amp; Sales-Pardo 2009)</span>. Statistical models can
            generate predictions of ecological networks without prior
            knowledge of pairwise interactions. They may do so using
            body size <span class="citation"
            data-cites="Petchey2008SizFor Gravel2013InfFooa">(Gravel
            <em>et al.</em> 2013; Petchey <em>et al.</em> 2008)</span>,
            phylogeny <span class="citation"
            data-cites="Elmasri2020HieBay Strydom2022Food">(Elmasri
            <em>et al.</em> 2020; Strydom <em>et al.</em> 2022)</span>,
            or a combination of niche and neutral processes <span
            class="citation"
            data-cites="Bartomeus2016ComFra Pomeranz2019InfPre">(Bartomeus
            <em>et al.</em> 2016; Pomeranz <em>et al.</em> 2019)</span>
            for inference. Topological null models, which generate
            networks of probabilistic interactions by preserving chosen
            characteristics of the adjacency matrix of binary
            interactions while intentionally omitting others <span
            class="citation"
            data-cites="Bascompte2003NesAss Fortuna2006HabLos">(Bascompte
            <em>et al.</em> 2003; Fortuna &amp; Bascompte 2006)</span>,
            are examples of common probabilistic interaction models.
            Null models can be used to produce underlying distributions
            of network measures for null hypothesis significance
            testing. Many measures have been developed to describe the
            structure <span class="citation"
            data-cites="Poisot2016Structure">(Poisot <em>et al.</em>
            2016)</span> and diversity <span class="citation"
            data-cites="Ohlmann2019Diversity Godsoe2022Species">(Godsoe
            <em>et al.</em> 2022; Ohlmann <em>et al.</em> 2019)</span>
            of probabilistic interaction networks. These models and
            measures support the use of this approach for the study of a
            wide range of ecological questions, from making better
            predictions of species distribution <span class="citation"
            data-cites="Cazelles2016Theorya">(Cazelles <em>et al.</em>
            2016)</span> to forecasting the impact of climate change on
            ecological networks <span class="citation"
            data-cites="Gilman2010FraCom">(Gilman <em>et al.</em>
            2010)</span>.</p>
            <p>Yet, a precise definition of probabilistic interactions
            appears to be lacking, making the estimation and use of
            these data more difficult. In this manuscript, we aim to
            take a step back by outlining different ways in which
            probabilistic interactions are defined and used in network
            ecology. We distinguish two broad categories of
            probabilistic interaction networks that necessitate distinct
            approaches: local networks describing probabilities of
            realized interactions, and regional networks (metawebs)
            describing probabilities of potential interactions. We
            highlight the distinctions in the ecological meaning of
            these two representations and examine some of their
            properties and relationships (particularly with space, time,
            and between each other) through empirical case studies.
            Moreover, the lack of clear guidelines on the use of
            probabilistic interaction data is worrisome, as it affects
            both data producers and re-users who generate and manipulate
            these numbers. This is concerning because sampling
            strategies and decisions regarding network construction can
            affect our understanding of network properties <span
            class="citation"
            data-cites="Brimacombe2023ShoReu">(Brimacombe <em>et
            al.</em> 2023)</span>. There is currently no reporting
            standard that could guide the documentation of all types of
            probabilistic interactions <span class="citation"
            data-cites="Salim2022Data">(although see e.g., Salim <em>et
            al.</em> 2022 who discuss data standards for deterministic
            mutualistic networks)</span>. Well-defined reporting
            standards for probabilistic interactions would support more
            adequate manipulation and integration of interaction data
            from different sources and guard against possible
            misinterpretations arising from ambiguous definitions of
            probabilistic interaction networks. This documentation
            should outline the nature (i.e., local or regional) and type
            (e.g., predatory or pollination) of the interactions,
            provide information regarding the taxonomic level,
            identities, and characteristics (e.g., life stages) of the
            individuals involved in an interaction, present the
            mathematical formulation of probabilities, including clearly
            identified conditional variables (e.g., spatial and temporal
            scales), and describe the methods and contexts (e.g.,
            location, time, environmental conditions) in which
            interactions were estimated. Inadequately documented
            probabilistic interaction data should be used with caution
            when analyzing ecological networks. The broad principles
            underlying our findings remain relevant and applicable
            across diverse ecological contexts.</p>
            <h1
            id="probabilistic-representations-of-interactions">Probabilistic
            representations of interactions</h1>
            <p>Consider a scenario where an avian predator has just
            established itself in a northern habitat home to a small
            rodent. Suppose these species have never co-occurred before,
            and as a result, their interaction has not been previously
            observed. What is the probability that the rodent is part of
            the diet of the avian predator, or put differently, what is
            the probability that they interact? Answering this question
            requires some clarification, as there are multiple ways to
            interpret and calculate interaction probabilities. We could
            calculate the probability that the traits of these species
            match, i.e. that the avian predator possesses the biological
            attributes to capture and consume the rodent. We could also
            calculate the probability that their traits support an
            interaction under the typical environmental conditions of
            the new habitat. For example, because avian predators hunt
            by sight, predation could be possible in the absence of snow
            but highly improbable when snow is present, as rodents may
            use it as a shelter to hide from predators. Finally, we
            could calculate the probability that the avian predator will
            consume the rodent at <em>that</em> particular location, for
            which the spatial and temporal boundaries need to be
            specified. The estimation of interaction probabilities,
            whether through predictive models or prior distributions,
            hinges on our understanding of these probabilities and the
            specific ecological processes we aim to capture.</p>
            <p>An important aspect to consider when using probabilities
            of interactions is knowing if they describe potential or
            realized interactions, as these two types of interactions
            have distinct conceptual underpinnings and sources of
            uncertainty. A potential (regional) interaction is defined
            as the biological capacity of two taxa to interact (i.e.,
            the probability that they interact if they were to encounter
            each other and given sufficient time) whereas a realized
            (local) interaction is the occurrence or observation of this
            interaction in a well-defined space and time (i.e., the
            probability that they interact locally). For two
            co-occurring taxa and over infinite time, the probability of
            local interaction is equivalent to the probability of
            regional (potential) interaction. We use the terms
            <em>metaweb</em> <span class="citation"
            data-cites="Dunne2006Network">(Dunne 2006)</span> to
            designate regional networks of potential interactions and
            <em>local networks</em> <span class="citation"
            data-cites="Poisot2012Dissimilaritya">(Poisot <em>et
            al.</em> 2012)</span> for those of realized interactions.
            Metawebs are the network analogs of the species pool, where
            local networks originate from a subset of both species
            (nodes) and interactions (edges) of the regional metaweb
            <span class="citation"
            data-cites="Saravia2022Ecological">(Saravia <em>et al.</em>
            2022)</span>. Without clear documentation, it can be
            challenging to know if published probabilistic interaction
            networks describe local or regional interactions (Tbl. 1
            provides examples of studies employing both types of
            probabilistic interaction networks), or if so-called
            probabilities are in reality a form of interaction score.
            When probabilistic regional interactions are used and
            interpreted incorrectly as local interactions (and
            conversely), this may generate misleading findings during
            data analysis. We believe that a better understanding of
            probabilistic local and regional interactions would prevent
            interpretation errors (e.g., when studying network-area
            relationships with metawebs or local networks) and
            facilitate a more adequate use of interaction data. In the
            following sections, we delve into these distinctions as we
            scale up from pairwise interactions to higher-level
            representations of ecological networks, i.e. local networks
            and metawebs.</p>
            <h1
            id="pairwise-interactions-the-building-blocks-of-ecological-networks">Pairwise
            interactions: the building blocks of ecological
            networks</h1>
            <p>Local networks and metawebs, like any type of network,
            are made of nodes and edges that may be represented at
            different levels of organization. The basic units of
            ecological networks are individuals that interact with each
            other <span class="citation"
            data-cites="Elton2001Animal">(e.g., by predation in food
            webs, Elton 2001)</span>, forming individual-based networks
            <span class="citation" data-cites="Melian2011EcoDyn">(Melián
            <em>et al.</em> 2011)</span>. The aggregation of these
            individuals into more or less homogeneous groups (e.g.,
            populations, species, families, feeding guilds) allows us to
            represent nodes at broader taxonomic scales, which affects
            our interpretation of the properties of these systems <span
            class="citation"
            data-cites="Guimaraes2020Structurea Hemprich-Bennett2021AssImp">(Guimarães
            2020; Hemprich-Bennett <em>et al.</em> 2021)</span>.</p>
            <p>Ecologists have traditionally represented interactions
            (edges) as binary objects that were considered realized
            after observing at least one individual from group <span
            class="math inline"><em>i</em></span> interact with at least
            another individual from group <span
            class="math inline"><em>j</em></span>. In an adjacency
            matrix <span class="math inline"><em>B</em></span> of binary
            interactions, the presence or absence of an interaction
            <span
            class="math inline"><em>B</em><sub><em>i</em> → <em>j</em></sub></span>
            between two taxa can be viewed as the result of a Bernoulli
            trial <span class="math inline">$B_{i \rightarrow j} \sim
            {\rm Bernoulli}(P(B_{i \rightarrow j}))$</span>, with <span
            class="math inline"><em>P</em>(<em>B</em><sub><em>i</em> → <em>j</em></sub>)</span>
            being the probability of interaction. This interaction
            probability characterizes our limited ecological knowledge
            and/or the intrinsic spatiotemporal variability of
            interactions. It may be estimated through predictive models
            (e.g., those based on biological traits and species
            abundances) or expert (prior) knowledge about the
            interaction. In networks of probabilistic interactions, the
            edge values <span
            class="math inline"><em>P</em>(<em>B</em><sub><em>i</em> → <em>j</em></sub>)</span>
            are probabilistic events whose only two possible outcomes
            are the presence (<span
            class="math inline"><em>B</em><sub><em>i</em> → <em>j</em></sub> = 1</span>)
            or absence (<span
            class="math inline"><em>B</em><sub><em>i</em> → <em>j</em></sub> = 0</span>)
            of an interaction between each pair of nodes. Depending on
            the type of probabilistic interaction network (local or
            metaweb), the mathematical formulation and interpretation of
            stochastic parameters like <span
            class="math inline"><em>P</em>(<em>B</em><sub><em>i</em> → <em>j</em></sub>)</span>
            can be linked to environmental and biological factors such
            as species abundance, traits, area, and time, for example
            using logistic regression with continuous explanatory
            variables.</p>
            <p>Predicting the number of local networks in which the
            interaction between two given taxa occurs can be achieved by
            using a Binomial distribution, assuming a constant
            interaction probability and independence between networks
            (trials). When considering uncertainties around the
            estimation of <span
            class="math inline"><em>P</em>(<em>B</em><sub><em>i</em> → <em>j</em></sub>)</span>
            or its spatiotemporal variability, a Beta distribution may
            be used to represent the relative likelihood of different
            probability values. For example, when calculating the
            probability of interaction between two taxa based on their
            local abundances, any spatiotemporal fluctuations in their
            abundances would introduce variability in the interaction
            probability at the local scale. If we take into account the
            stochasticity of the interaction probability, a
            Beta-Binomial distribution can be used to predict the number
            of networks in which the interaction occurs. Empirically
            observing an interaction between two taxa at a given
            location and time provides important information that can be
            used to update previous estimates of <span
            class="math inline"><em>P</em>(<em>B</em><sub><em>i</em> → <em>j</em></sub>)</span>,
            informing us on the biological capacity of both taxa to
            interact and the environmental conditions that enabled them
            to interact locally.</p>
            <p>Even though binary interaction networks constitute a
            highly valuable source of ecological information <span
            class="citation"
            data-cites="Pascual2006Ecologicala">(Pascual <em>et al.</em>
            2006)</span>, they overlook interaction strengths.
            Represented in a quantitative adjacency matrix <span
            class="math inline"><em>W</em></span>, interaction strengths
            better describe the energy flows, demographic impacts or
            frequencies of interactions between nodes <span
            class="citation"
            data-cites="Berlow2004Interaction Borrett2019Walk">(Berlow
            <em>et al.</em> 2004; Borrett &amp; Scharler 2019)</span>,
            with <span
            class="math inline"><em>W</em><sub><em>i</em> → <em>j</em></sub></span>
            being a natural <span class="math inline">ℕ</span> or real
            <span class="math inline">ℝ</span> number depending on the
            measure. For example, they may represent local interaction
            rates between pairs of taxa <span class="citation"
            data-cites="Herrera1989PolAbu">(e.g., the flower-visiting
            rates of pollinators in a mutualistic network, Herrera
            1989)</span>. When interaction strengths characterize
            predation pressure on prey, they can serve as parameters in
            a Lotka-Volterra model <span class="citation"
            data-cites="Emmerson2004Predatora">(e.g., Emmerson &amp;
            Raffaelli 2004)</span>. The extra amount of ecological
            information in quantitative networks typically comes at a
            cost of greater sampling effort and data volume in
            predictive models <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>, which can lead to relatively high levels of
            uncertainties when inferring quantitative networks with
            limited data.</p>
            <p>As for binary interaction networks, the uncertainty and
            variability of interaction strengths can be represented
            probabilistically. However, the need to estimate the
            probability distribution of interaction strengths makes the
            inference of probabilities more challenging in quantitative
            networks compared to binary interaction networks.
            Interaction strengths can follow many probability
            distributions depending on the method. For instance, they
            can follow a Poisson distribution <span
            class="math inline">$W_{i \rightarrow j} \sim {\rm
            Poisson}(\lambda_{i \rightarrow j})$</span> when predicting
            the number of interactions between pairs of nodes, with
            <span
            class="math inline"><em>λ</em><sub><em>i</em> → <em>j</em></sub></span>
            being the expected rate at which individuals of taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact (e.g., the
            average number of prey <span
            class="math inline"><em>j</em></span> consumed by all
            predators <span class="math inline"><em>i</em></span> in a
            given time period). The Poisson distribution can also be
            <span class="math inline">0</span>-inflated after initially
            modeling non-interacting taxa <span class="citation"
            data-cites="Boulangeat2012AccDis">(e.g., Boulangeat <em>et
            al.</em> 2012 employ a <span
            class="math inline">0</span>-inflated model to analyze
            species abundance following the modeling of species presence
            and absence)</span>, which constitute the majority of taxa
            pairs in most local networks due to their typically high
            sparseness <span class="citation"
            data-cites="Jordano2016Samplingb">(Jordano 2016)</span>.
            Because of the methodological difficulties typically
            encountered when building deterministic quantitative
            networks <span class="citation"
            data-cites="Plaganyi2004Criticala">(which are only partially
            mitigated by models such as Ecopath, Plagányi &amp;
            Butterworth 2004)</span>, binary interaction networks, which
            are easier to sample <span class="citation"
            data-cites="Jordano2016Samplingb">(Jordano 2016)</span> and
            predict <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>, have been more frequently studied and modeled.
            Moreover, most published probabilistic interaction networks
            and methods describe interactions whose outcome is binary
            (whether interaction probabilities are regarded as constant
            or variable, e.g. represented by a Beta distribution), which
            underlines the need for better guidelines regarding the
            interpretation and manipulation of these types of networks
            first. For these reasons, our primary focus is interpreting
            and using interaction probabilities in Bernoulli
            distributions, in both local networks and metawebs.</p>
            <h1
            id="local-networks-communities-interacting-in-space-and-time">Local
            networks: communities interacting in space and time</h1>
            <p>Local networks of probabilistic interactions describe how
            likely taxa are to interact at a given location and time
            period. Local interactions are contingent upon the
            environmental conditions of the community and the matching
            of taxa biological traits. In local networks, edges commonly
            represent our degree of belief that two taxa interact in
            nature, but can also represent the probability of
            empirically <em>observing</em> this interaction <span
            class="citation" data-cites="Catchen2023Missinga">(Catchen
            <em>et al.</em> 2023)</span>.</p>
            <p>We define space as the collection of geographic
            coordinates <span
            class="math inline">(<em>x</em>,<em>y</em>,<em>z</em>)</span>,
            with <span
            class="math inline">(<em>x</em>,<em>y</em>)</span>
            representing longitude and latitude coordinates, and <span
            class="math inline"><em>z</em></span> denoting either
            altitudes or depths. These point coordinates delineate the
            spatial boundaries of the system, which may be portrayed as
            a polyhedron. Ecological interactions may exhibit variations
            along latitudinal and altitudinal gradients, as evidenced by
            changes in hummingbird-plant interactions <span
            class="citation"
            data-cites="Weinstein2017ComTra Weinstein2017PerBil">(Weinstein
            &amp; Graham 2017a, b)</span> and mosquito biting rates
            <span class="citation"
            data-cites="Kulkarni2006EntEva">(e.g., Kulkarni <em>et
            al.</em> 2006)</span> at different elevations. In contrast,
            time is defined as the specific time period within which
            interactions were either observed or predicted. Even though
            space and time are continuous variables that should yield
            probability <em>densities</em> of interactions (i.e.,
            relative likelihoods of interactions occurring at
            infinitesimal locations and instants in time), these
            definitions enable them to be conceptualized as distinct
            patches and time segments. Treating space and time as
            discrete dimensions aligns with the common sampling methods
            of ecological networks and provides probabilities of
            interactions, which can be obtained by integrating
            probability densities over space and time. Furthermore, we
            can quantify both an area <span
            class="math inline"><em>A</em></span> and a duration <span
            class="math inline"><em>t</em></span>, which can be readily
            used in spatiotemporal analyses of ecological networks. When
            studying network-area relationships <span class="citation"
            data-cites="Galiana2018Spatiala">(NAR, Galiana <em>et
            al.</em> 2018)</span>, we anticipate that local
            probabilities of interactions scale positively with area and
            duration because taxa have more opportunities to interact as
            these dimensions expand.</p>
            <p>The probability that two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact in a local
            network <span
            class="math inline"><em>L</em><sub><em>x</em>, <em>y</em>, <em>z</em>, <em>t</em></sub></span>
            (spatial and temporal subscripts hereafter omitted or
            replaced by the shorter subscript <span
            class="math inline"><em>k</em></span> for clarity) can be
            conditioned on many environmental and biological factors. In
            addition to network area (or volume) and duration, they may
            be conditioned on taxa co-occurrence <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub></span>,
            which is usually a Boolean describing if the geographic
            distributions of both taxa overlap within the study area. As
            illustrated in Box 1, co-occurrence may be modeled
            probabilistically, in which case it may conform to a
            Bernoulli distribution <span class="math inline">$X_{i,j,k}
            \sim {\rm Bernoulli}(P(X_{i,k}, X_{j,k}))$</span>, where
            <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>X</em><sub><em>j</em>, <em>k</em></sub></span>
            are the local occurrences of both taxa. The probability of
            co-occurrence <span
            class="math inline"><em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>,<em>X</em><sub><em>j</em>, <em>k</em></sub>)</span>
            can be estimated through the application of joint species
            distribution models <span class="citation"
            data-cites="Pollock2014UndCoo">(e.g., Pollock <em>et
            al.</em> 2014)</span>, potentially taking into account
            biotic interactions <span class="citation"
            data-cites="Staniczenko2017Linking">(Staniczenko <em>et
            al.</em> 2017)</span>. Given that the probability that two
            non-co-occurring taxa interact locally is zero (i.e., <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub>=0) = 0</span>),
            the probability of local interaction can be obtained by
            multiplying the probability of interaction given
            co-occurrence with the probability of co-occurrence:</p>
            <p><span id="eq:co-occur"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>) = <em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub>=1) × <em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>=1,<em>X</em><sub><em>j</em>, <em>k</em></sub>=1).   (1)</span></span></p>
            <p>Other important factors that can impact interaction
            probabilities at the local scale are taxa local abundances
            <span
            class="math inline"><em>N</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>N</em><sub><em>j</em>, <em>k</em></sub></span>,
            which affect encounter probabilities <span class="citation"
            data-cites="Canard2012Emergencea">(Canard <em>et al.</em>
            2012)</span>, and local traits distributions <span
            class="math inline"><em>T</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>T</em><sub><em>j</em>, <em>k</em></sub></span>
            <span class="citation"
            data-cites="Caron2024Traitmatching">(e.g., body mass,
            longevity, and habitat breadth, Caron <em>et al.</em>
            2024)</span>, which determine the ability of individuals to
            interact after encountering each other <span
            class="citation" data-cites="Poisot2015Speciesa">(Poisot
            <em>et al.</em> 2015)</span>. Moreover, local interactions
            may be conditional on local environmental factors such as
            temperature <span class="citation"
            data-cites="Angilletta2004TemGro">(Angilletta <em>et
            al.</em> 2004)</span>, precipitation <span class="citation"
            data-cites="Woodward2012CliCha">(Woodward <em>et al.</em>
            2012)</span>, habitat structure <span class="citation"
            data-cites="Klecka2014EffHab">(Klecka &amp; Boukal
            2014)</span>, and presence of other taxa in the network
            <span class="citation"
            data-cites="Pilosof2017MulNat Kefi2012MorMea">(Kéfi <em>et
            al.</em> 2012; Pilosof <em>et al.</em> 2017)</span>, as
            described above. Here, we use the variable <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>
            to describe the local environmental context in which
            interaction probabilities were estimated. For example, in a
            mesocosm experiment estimating interaction probabilities
            between predators and prey with and without shelters, <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>
            would represent the presence or absence of these shelters.
            Like co-occurrence, <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>
            can also be modeled probabilistically when the stochasticity
            or uncertainty of environmental factors is considered. <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>
            represents all environmental variables that were taken into
            consideration when measuring interaction probabilities; it
            is a subset of all environmental factors acting on
            ecological interactions. Finally, local interaction
            probabilities may be conditioned on higher-level properties
            of the network, which we denote by <span
            class="math inline"><em>f</em>(<em>L</em>)</span>. Many
            topological null models (i.e., statistical models that
            randomize interactions by retaining certain properties of
            the network while excluding others) provide interaction
            probabilities from selected measures of network structure,
            such as connectance <span class="citation"
            data-cites="Fortuna2006HabLos">(Fortuna &amp; Bascompte
            2006)</span> and the degree distribution <span
            class="citation" data-cites="Bascompte2003NesAss">(Bascompte
            <em>et al.</em> 2003)</span>.</p>
            <p>Local interactions must be biologically feasible before
            occurring at a specific time and space. A local probability
            of interaction <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>)</span>
            can be expressed as the product of the probability of local
            interaction given that the two taxa can potentially interact
            <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>=1)</span>,
            which we sometimes denote as <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            for the sake of simplicity, with their probability of
            regional interaction <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>:</p>
            <p><span id="eq:local_meta"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>) = <em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>=1) × <em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>=1).   (2)</span></span></p>
            <p>Low values of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            indicate that feasible interactions rarely occur locally,
            intermediate values around <span
            class="math inline">50%</span> suggest considerable
            spatiotemporal variability, while high values indicate that
            regional interactions are nearly always realized locally.
            The local probability of interaction between a given pair of
            taxa is always equal to or below their probability of
            regional interaction.</p>
            <p>The probability that two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact in a local
            network <span class="math inline"><em>L</em></span> can thus
            be conditioned on their co-occurrence <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub></span>
            (or more explicitly on their occurrences <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>X</em><sub><em>j</em>, <em>k</em></sub></span>),
            local abundances <span
            class="math inline"><em>N</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>N</em><sub><em>j</em>, <em>k</em></sub></span>,
            local traits distributions <span
            class="math inline"><em>T</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>T</em><sub><em>j</em>, <em>k</em></sub></span>,
            local environmental conditions <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>,
            network area (or volume) <span
            class="math inline"><em>A</em></span>, time interval <span
            class="math inline"><em>t</em></span>, network properties
            <span class="math inline"><em>f</em>(<em>L</em>)</span> and
            biological (or regional) feasibility <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>.</p>
            <p>Although these variables correspond to distinct
            ecological inquiries or mechanisms related to ecological
            interactions, they may covary with each other, such as the
            possible dependence of <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>
            on spatial and temporal scales. When estimating interaction
            probabilities using e.g. a generalized linear model with
            multiple explanatory variables that might not be
            independent, it may become important to address
            collinearity. In such a case, to mitigate this issue, it may
            be necessary to use variable selection techniques before
            fitting the model to data. The probability that a local
            interaction is realized is described by the following
            expression when all these conditional variables are
            included:</p>
            <p><span id="eq:local"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>X</em><sub><em>i</em>, <em>k</em></sub>,<em>X</em><sub><em>j</em>, <em>k</em></sub>,<em>N</em><sub><em>i</em>, <em>k</em></sub>,<em>N</em><sub><em>j</em>, <em>k</em></sub>,<em>T</em><sub><em>i</em>, <em>k</em></sub>,<em>T</em><sub><em>j</em>, <em>k</em></sub>,<em>E</em><sub><em>k</em></sub>,<em>A</em>,<em>t</em>,<em>f</em>(<em>L</em>),<em>M</em><sub><em>i</em> → <em>j</em></sub>)   (3)</span></span></p>
            <p>The representation of the local context in which
            probabilities are estimated and the variables that should be
            taken into consideration depend on the study system, the
            objectives of the study, and the resources available to the
            researchers. In other words, these variables do not
            systematically need to be accounted for. For example, <span
            class="citation" data-cites="Gravel2019BriElt">Gravel <em>et
            al.</em> (2019)</span> analyzed local European trophic
            networks of willow-galling sawflies and their natural
            enemies, depicting binary interactions, all referenced in
            space and time, to infer probabilities of local interactions
            between co-occurring species. This was achieved by including
            temperature and precipitation as conditional variables in
            their models. In Box 2, we show the extent of variation
            among these local host-parasite networks. We do so by
            measuring their dissimilarity with the regional network
            (metaweb aggregating all local interactions), both in terms
            of species composition and interactions. We build local
            probabilistic networks following eq. 2, showing that
            insufficient local variation (high probability of local
            interaction among potentially interacting species) results
            in an overestimation in both the number of interactions and
            connectance (i.e., the proportion of all of the
            non-forbidden links that are realized).</p>
            <p>When accounted for, conditional variables should be
            clearly described in the documentation of the data <span
            class="citation"
            data-cites="Brimacombe2023ShoReu">(Brimacombe <em>et
            al.</em> 2023)</span>, preferentially in mathematical terms
            to avoid any confusion in their interpretation and to limit
            manipulation errors during their re-use. For instance,
            ecologists should be explicit about their consideration of
            co-occurrence in their estimation of local interaction
            probabilities. Indeed, it is important to specify if
            probability values are conditional (<span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub>=1)</span>)
            or not (<span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>)</span>)
            on co-occurrence, as this can change the interpretation of
            the data. In Tbl. 1, we present examples of studies that
            used these different formulations of probabilistic
            interactions and conditional variables. We have included the
            probability of empirically observing an interaction that is
            realized locally <span
            class="math inline"><em>P</em>(<em>O</em><sub><em>i</em> → <em>j</em></sub>|<em>L</em><sub><em>i</em> → <em>j</em></sub>)</span>
            to underscore the distinction between local observations and
            actual realizations of interactions, even though the focus
            of this manuscript is not on the observation of
            interactions.</p>
            <div id="tbl:prob">
            <table>
            <caption>Table 1: <strong>Mathematical expressions of
            probabilistic interactions.</strong> The probability of
            interaction between two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> is interpreted
            differently in a local network <span
            class="math inline"><em>L</em></span> of realized
            interactions and a metaweb <span
            class="math inline"><em>M</em></span> of potential
            interactions (representing the <em>biological</em>
            feasibility of interactions). Each expression emphasizes a
            different conditional variable, the ellipsis serving as a
            placeholder for other variables not explicitly stated in the
            expression. The outcome of each of these probabilistic
            events, along with common uncertainty sources (often
            stemming from the model used for estimation), is presented
            alongside examples of studies that employed them (with
            specific variables indicated in parentheses, when
            applicable). The study marked with an asterisk has been
            conducted on binary interaction networks. Note that we also
            included the expression for a local network <span
            class="math inline"><em>O</em></span> of observed
            interactions and a metaweb <span
            class="math inline"><em>M</em><sup>*</sup></span> of
            potential interactions (representing the <em>ecological</em>
            feasibility of interactions) as they were mentioned in the
            main text, even though they are not the focus of this
            manuscript.</caption>
            <colgroup>
            <col style="width: 19%" />
            <col style="width: 8%" />
            <col style="width: 30%" />
            <col style="width: 19%" />
            <col style="width: 22%" />
            </colgroup>
            <thead>
            <tr class="header">
            <th style="text-align: left;">Expression</th>
            <th style="text-align: left;">Type</th>
            <th style="text-align: left;">Outcome</th>
            <th style="text-align: left;">Variability sources</th>
            <th style="text-align: right;">Reference</th>
            </tr>
            </thead>
            <tbody>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>X</em><sub><em>i</em>, <em>k</em></sub>,<em>X</em><sub><em>j</em>, <em>k</em></sub>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given taxa co-occurrence</td>
            <td style="text-align: left;">spatiotemporal
            variability</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Gravel2019BriElt">Gravel <em>et al.</em>
            (2019)</span></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>N</em><sub><em>i</em>, <em>k</em></sub>,<em>N</em><sub><em>j</em>, <em>k</em></sub>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given taxa abundances</td>
            <td style="text-align: left;">neutral models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Canard2014Empiricala">Canard <em>et al.</em>
            (2014)</span></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em>, <em>k</em></sub>,<em>T</em><sub><em>j</em>, <em>k</em></sub>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given local traits</td>
            <td style="text-align: left;">trait matching models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Caron2024Traitmatching">Caron <em>et al.</em>
            (2024)</span></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>E</em><sub><em>k</em></sub>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given local environmental conditions</td>
            <td style="text-align: left;">environmental-based
            models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Gravel2019BriElt">Gravel <em>et al.</em>
            (2019)</span> (temperature and precipitation)</td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>A</em>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            in a given area or volume</td>
            <td style="text-align: left;">spatial models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Galiana2018Spatiala">Galiana <em>et al.</em>
            (2018)</span> *</td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>t</em>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            during a given time period</td>
            <td style="text-align: left;">temporal models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Weinstein2017ComTra">Weinstein &amp; Graham
            (2017a)</span></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>f</em>(<em>L</em>),...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given network structure</td>
            <td style="text-align: left;">topological null models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Fortuna2006HabLos">Fortuna &amp; Bascompte
            (2006)</span> (connectance)</td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">realization of the interaction
            given that the taxa can biologically interact</td>
            <td style="text-align: left;">spatiotemporal
            variability</td>
            <td style="text-align: right;">this study</td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>O</em><sub><em>i</em> → <em>j</em></sub>|<em>L</em><sub><em>i</em> → <em>j</em></sub>,...)</span></td>
            <td style="text-align: left;">local</td>
            <td style="text-align: left;">observation of the interaction
            given that it is realized locally</td>
            <td style="text-align: left;">sampling model</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Catchen2023Missinga">Catchen <em>et al.</em>
            (2023)</span></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>)</span></td>
            <td style="text-align: left;">regional</td>
            <td style="text-align: left;">biological feasibility of the
            interaction given regional traits (non-forbiddenness)</td>
            <td style="text-align: left;">trait matching models</td>
            <td style="text-align: right;"><span class="citation"
            data-cites="Strydom2022Food">Strydom <em>et al.</em>
            (2022)</span></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub><sup>*</sup>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>,<em>E</em>)</span></td>
            <td style="text-align: left;">regional</td>
            <td style="text-align: left;">ecological feasibility of the
            interaction given regional traits and environmental
            conditions</td>
            <td style="text-align: left;">trait matching and
            environmental-based models</td>
            <td style="text-align: right;">this study</td>
            </tr>
            </tbody>
            </table>
            </div>
            <p>When using multiple models to estimate local interaction
            probabilities, rather than selecting a single model that
            best fits the data, model averaging may enhance our
            estimations. In this approach, the weighting is based on the
            likelihood of each model. Model weights represent the
            probability that each model is the most suitable for
            explaining the data, and may be measured using Akaike
            weights <span class="citation"
            data-cites="Burnham2004Multimodel Wagenmakers2004Aic">(Burnham
            &amp; Anderson 2004; Wagenmakers &amp; Farrell 2004)</span>.
            For instance, considering two models <span
            class="math inline"><em>m</em><em>o</em><em>d</em><sub>1</sub></span>
            and <span
            class="math inline"><em>m</em><em>o</em><em>d</em><sub>2</sub></span>
            with respective probabilities (or weights) <span
            class="math inline"><em>P</em>(<em>m</em><em>o</em><em>d</em><sub>1</sub>)</span>
            and <span
            class="math inline"><em>P</em>(<em>m</em><em>o</em><em>d</em><sub>2</sub>)</span>,
            the average probability of interaction <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|...)</span>
            can be calculated as follows:</p>
            <p><span id="eq:model_ens"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|...) = <em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>m</em><em>o</em><em>d</em><sub>1</sub>,...) × <em>P</em>(<em>m</em><em>o</em><em>d</em><sub>1</sub>) + <em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>m</em><em>o</em><em>d</em><sub>2</sub>,...) × <em>P</em>(<em>m</em><em>o</em><em>d</em><sub>2</sub>),   (4)</span></span></p>
            <p>where the ellipsis serves as a placeholder for the
            conditional variables incorporated in these models.</p>
            <blockquote>
            <h2
            id="box-1-a-spatiotemporally-explicit-model-of-interactions"
            class="unnumbered">Box 1: A spatiotemporally explicit model
            of interactions</h2>
            <p>Ecologists may resort to predictive models (e.g.,
            generative Bayesian models) to reconstruct local networks
            across time and space <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>. Uncertainty in model parameters, model
            structure, and input data leads to probabilistic
            interactions. We introduce and develop a simple generative
            mechanistic model for probabilistic local interactions that
            takes into consideration their spatiotemporal variability
            (i.e., a spatiotemporally explicit model of local
            interactions). This model is not designed for regional
            interactions, which do not vary spatially and temporally.
            Rather, it may prove valuable for predicting local
            interactions across time and space by generating new
            interaction data following parameter inference.</p>
            <p>As indicated by Eq. 1, the probability that two taxa
            <span class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact at a specific
            location and time <span
            class="math inline"><em>k</em></span> may be determined by
            the product of their probability of interaction given
            co-occurrence and their probability of co-occurrence. First,
            their co-occurrence probability can be calculated using
            their individual (marginal) occurrence probabilities <span
            class="math inline"><em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>)</span>
            and <span
            class="math inline"><em>P</em>(<em>X</em><sub><em>j</em>, <em>k</em></sub>)</span>.
            Given that taxa occurrences are not independent of each
            other, the joint probability of both taxa occurring together
            can be calculated by multiplying the probability of one
            taxon being present by the conditional probability of the
            other occurring when the first one is present.
            Alternatively, the joint occurrence probability can be
            obtained by multiplying the marginal probabilities by the
            strength of association <span
            class="math inline"><em>γ</em></span> between the
            occurrences of both taxa <span class="citation"
            data-cites="Cazelles2016Theorya">(Cazelles <em>et al.</em>
            2016)</span>:</p>
            <p><span id="eq:modelcoprob"><span
            class="math display"><em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>,<em>X</em><sub><em>j</em>, <em>k</em></sub>) = <em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>|<em>X</em><sub><em>j</em>, <em>k</em></sub>)<em>P</em>(<em>X</em><sub><em>j</em>, <em>k</em></sub>) = <em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>)<em>P</em>(<em>X</em><sub><em>j</em>, <em>k</em></sub>)<em>γ</em>.   (5)</span></span></p>
            <p>When <span class="math inline"><em>γ</em> &gt; 1</span>,
            there is a positive association in the geographic
            distributions of both taxa, indicating that the presence of
            one taxon enhances the probability of occurrence of the
            other. Attractions may be the result of positive
            interactions (e.g., mutualism) or positive/negative
            interactions <span class="citation"
            data-cites="Cazelles2016Theorya">(e.g., antagonism between
            predators and prey, Cazelles <em>et al.</em> 2016)</span>.
            In empirical networks, <span
            class="math inline"><em>γ</em> &gt; 1</span> holds for the
            majority of species pairs <span class="citation"
            data-cites="Catchen2023Missinga">(Catchen <em>et al.</em>
            2023)</span>. In contrast, repulsions (<span
            class="math inline"><em>γ</em> &lt; 1</span>) may be caused
            e.g. by strong interspecific competition <span
            class="citation" data-cites="Cazelles2016Theorya">(Cazelles
            <em>et al.</em> 2016)</span>. We model the co-occurrence
            <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub></span>
            of both taxa as the outcome of a Bernoulli trial</p>
            <p><span id="eq:modelco"><span
            class="math display"><em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub> ∼ <em>B</em><em>e</em><em>r</em><em>n</em><em>o</em><em>u</em><em>l</em><em>l</em><em>i</em>(<em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>,<em>X</em><sub><em>j</em>, <em>k</em></sub>)).   (6)</span></span></p>
            <p>Next, the probability of interaction given co-occurrence
            can be made temporally explicit by modeling it as a Poisson
            process with rate parameter <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>.
            This parameter represents the local expected frequency of
            interaction between the taxa within a defined time interval
            and can be estimated using prior data on interaction
            strengths, when accessible. The probability that two
            co-occurring taxa engage in an interaction during a time
            period <span
            class="math inline"><em>t</em><sub>0</sub></span> is given
            by:</p>
            <p><span id="eq:modelrate"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub>=1) = 1 − <em>e</em><sup>−<em>λ</em><sub><em>k</em></sub><em>t</em><sub>0</sub></sup>,   (7)</span></span></p>
            <p>which tends toward <span class="math inline">1</span> as
            <span class="math inline"><em>t</em><sub>0</sub> → ∞</span>
            if <span class="math inline"><em>λ</em> &gt; 0</span>. In
            other words, two co-occurring taxa with a nonzero rate of
            interaction will inevitably interact in a sufficiently long
            time period. It is important to note that the units of <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            and <span class="math inline"><em>t</em><sub>0</sub></span>
            are complementary. For instance, if the duration <span
            class="math inline"><em>t</em><sub>0</sub></span> is
            measured in months, <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            would denote the expected number of interactions per
            month.</p>
            <p>The occurrence of an interaction between <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> can be modeled as a
            Bernoulli trial with a probability of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>)</span>.
            Consequently, a Bayesian inference model can be built based
            on the preceding equations to estimate the value of the
            <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            and <span class="math inline"><em>γ</em></span> parameters
            and generate novel interaction data:</p>
            <p><span id="eq:model"><span
            class="math display"><em>L</em><sub><em>i</em> → <em>j</em></sub> ∼ Bernoulli(<em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>))   (8)</span></span></p>
            <p><span id="eq:modeleq"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>) = <em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>)<em>P</em>(<em>X</em><sub><em>j</em>, <em>k</em></sub>)<em>γ</em>(1−<em>e</em><sup>−<em>λ</em><sub><em>k</em></sub><em>t</em><sub>0</sub></sup>)   (9)</span></span></p>
            <p><span id="eq:modelgamma"><span
            class="math display"><em>γ</em> ∼ Gamma(2,0.5)   (10)</span></span></p>
            <p><span id="eq:modellambda"><span
            class="math display"><em>λ</em><sub><em>k</em></sub> ∼ Exponential(2)   (11)</span></span></p>
            <p>In Fig. 1, we show the variation in the probability of
            interaction under different parameter values. In the right
            panel, we notice that, irrespective of the interaction rate
            <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>,
            the probability of interaction converges toward an asymptote
            determined by the probability of co-occurrence <span
            class="math inline"><em>P</em>(<em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub>)</span>
            (Eq. 5). This model can be customized in different ways,
            such as by linking <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            with specific environmental variables or explicitly
            incorporating observation errors (i.e., the probabilities of
            false negatives and false positives).</p>
            <figure>
            <img src="figures/spatiotemporal_model.png"
            id="fig:spatiotemporal"
            alt="Figure 1: Parameters of the spatiotemporally explicit model of interactions. (a) Probability of local interaction given by the process model (Eq. 9) under different values of \lambda_k (interaction rate) and \gamma (strength of co-occurrence), with t_0 = 1 (duration). Parameters t_0 and \lambda_k have complementary units (e.g., t_0 in months and \lambda_k in number of interactions per month). The parameter values used in the right panel are denoted by the white stars. (b) Scaling of the probability of interaction with the duration parameter t_0 in Eq. 9, for different values of \lambda_k and \gamma. In both panels, the marginal probabilities of occurrence P(X_{i,k}) and P(X_{j,k}) are set to a constant value of 0.5." />
            <figcaption aria-hidden="true">Figure 1: <strong>Parameters
            of the spatiotemporally explicit model of
            interactions.</strong> (a) Probability of local interaction
            given by the process model (Eq. 9) under different values of
            <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            (interaction rate) and <span
            class="math inline"><em>γ</em></span> (strength of
            co-occurrence), with <span
            class="math inline"><em>t</em><sub>0</sub> = 1</span>
            (duration). Parameters <span
            class="math inline"><em>t</em><sub>0</sub></span> and <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            have complementary units (e.g., <span
            class="math inline"><em>t</em><sub>0</sub></span> in months
            and <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            in number of interactions per month). The parameter values
            used in the right panel are denoted by the white stars. (b)
            Scaling of the probability of interaction with the duration
            parameter <span
            class="math inline"><em>t</em><sub>0</sub></span> in Eq. 9,
            for different values of <span
            class="math inline"><em>λ</em><sub><em>k</em></sub></span>
            and <span class="math inline"><em>γ</em></span>. In both
            panels, the marginal probabilities of occurrence <span
            class="math inline"><em>P</em>(<em>X</em><sub><em>i</em>, <em>k</em></sub>)</span>
            and <span
            class="math inline"><em>P</em>(<em>X</em><sub><em>j</em>, <em>k</em></sub>)</span>
            are set to a constant value of <span
            class="math inline">0.5</span>.</figcaption>
            </figure>
            </blockquote>
            <blockquote>
            <h2 id="box-2-dissimilarity-of-local-host-parasite-networks"
            class="unnumbered">Box 2: Dissimilarity of local
            host-parasite networks</h2>
            <p>We use the collection of tripartite host-parasite
            networks of <span class="citation"
            data-cites="Kopelke2017FooStra">Kopelke <em>et al.</em>
            (2017)</span>, consisting of willows, willow-galling
            sawflies, and their natural enemies, sampled across Europe
            (<span class="math inline">233</span> local networks used).
            Given its replicated networks spanning large spatiotemporal
            scales, this dataset is well-suited for analyzing network
            variability, both in terms of species composition and
            interactions. Further details regarding data manipulation
            and network construction can be found in the supplementary
            material. All code and data to reproduce these analyses are
            available at the Open Science Framework (TBD).</p>
            <p>In Fig. 2a-b, we show how the dissimilarity between the
            metaweb of binary interactions and aggregated local networks
            varies with the number of sampled local networks. To do so,
            we randomly selected one local network of binary
            interactions and sequentially sampled additional networks
            while aggregating both their species and interactions. We
            compared the metaweb and the aggregated local networks using
            the dissimilarity in species composition (<span
            class="math inline"><em>β</em><sub><em>S</em></sub></span>,
            Fig. 2a) and the dissimilarity of interactions between
            common species (<span
            class="math inline"><em>β</em><sub><em>O</em><em>S</em></sub></span>,
            Fig. 2b) indices <span class="citation"
            data-cites="Poisot2012Dissimilaritya">(Poisot <em>et
            al.</em> 2012)</span>. We repeated this sampling process one
            hundred times and highlighted the median dissimilarity
            values across simulations, as well as the <span
            class="math inline">50%</span> and <span
            class="math inline">95%</span> percentile intervals. Both
            dissimilarity indices were calculated based on the number of
            items shared by the two networks (<span
            class="math inline"><em>c</em><sub><em>L</em><em>M</em></sub></span>)
            and the number of items unique to the metaweb (<span
            class="math inline"><em>u</em><sub><em>M</em></sub></span>)
            and the aggregated local network (<span
            class="math inline"><em>u</em><sub><em>L</em></sub></span>).
            The <span
            class="math inline"><em>β</em><sub><em>S</em></sub></span>
            dissimilarity index uses species (nodes) as items being
            compared, while the <span
            class="math inline"><em>β</em><sub><em>O</em><em>S</em></sub></span>
            index assesses dissimilarity based on interactions between
            shared species. Both indices were calculated following the
            <span
            class="math inline"><em>β</em><sub><em>W</em></sub></span>
            index of <span class="citation"
            data-cites="Whittaker1960Vegetation">Whittaker
            (1960)</span>:</p>
            <p><span id="eq:diss"><span class="math display">$$\beta_W =
            \frac{c_{LM} + u_L + u_M}{(2 c_{LM} + u_L + u_M) / 2} -
            1.\qquad{(12)}$$</span></span></p>
            <figure>
            <img src="figures/network_accumulation.png"
            id="fig:accumulation"
            alt="Figure 2: Network accumulation curves. (a) Dissimilarity in species composition and (b) dissimilarity of interactions between common species between aggregated local networks and the metaweb of binary host-parasite interactions. Aggregated local networks were obtained by sequentially and randomly selecting a number of local networks of binary interactions and aggregating both their species and interactions. In both panels, the colored line represents the median dissimilarity across simulations and the grey areas cover the 50\% and 95\% percentile intervals. (c) Scaling of the number of interactions and (d) scaling of connectance with the number of sampled (aggregated) binary and probabilistic local interaction networks. For a better comparison with binary interactions, local networks of probabilistic interactions were derived from a metaweb of probabilistic interactions with a false positive and false negative rate of zero. A specific value of P(L_{i \rightarrow j}|M_{i \rightarrow j}) (the local probability of interaction among potentially interacting species, indices omitted in the figure for simplicity) was used for all local networks within a particular curve. Aggregated local networks of probabilistic interactions were obtained by sequentially and randomly selecting a number of local networks and aggregating both their species and interactions (with the value of P(L_{i \rightarrow j}|M_{i \rightarrow j}) adjusting according to Eq. 13)." />
            <figcaption aria-hidden="true">Figure 2: <strong>Network
            accumulation curves.</strong> (a) Dissimilarity in species
            composition and (b) dissimilarity of interactions between
            common species between aggregated local networks and the
            metaweb of binary host-parasite interactions. Aggregated
            local networks were obtained by sequentially and randomly
            selecting a number of local networks of binary interactions
            and aggregating both their species and interactions. In both
            panels, the colored line represents the median dissimilarity
            across simulations and the grey areas cover the <span
            class="math inline">50%</span> and <span
            class="math inline">95%</span> percentile intervals. (c)
            Scaling of the number of interactions and (d) scaling of
            connectance with the number of sampled (aggregated) binary
            and probabilistic local interaction networks. For a better
            comparison with binary interactions, local networks of
            probabilistic interactions were derived from a metaweb of
            probabilistic interactions with a false positive and false
            negative rate of zero. A specific value of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            (the local probability of interaction among potentially
            interacting species, indices omitted in the figure for
            simplicity) was used for all local networks within a
            particular curve. Aggregated local networks of probabilistic
            interactions were obtained by sequentially and randomly
            selecting a number of local networks and aggregating both
            their species and interactions (with the value of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            adjusting according to Eq. 13).</figcaption>
            </figure>
            <p>Expectingly, local networks are highly dissimilar from
            the metaweb in terms of species composition, especially when
            only a limited number of sites has been sampled. This is
            because species in the metaweb form the regional species
            pool, and few species occur locally. Moreover, we observe a
            peak in the dissimilarity of interactions between common
            species at intermediate sampling levels. This suggests that
            species are collected faster than their interactions. With a
            limited number of sampled local networks, few regional
            interactions are observed locally. Adding more sites brings
            new species, but not always their interactions. Interactions
            are more comprehensively captured when the number of sampled
            sites is high. Quadratic relationships of network properties
            with sampling effort were also observed by <span
            class="citation" data-cites="McLeod2021Sampling">McLeod
            <em>et al.</em> (2021)</span>.</p>
            <p>Next, we investigate how the number of local interactions
            and connectance scale with the number of sampled
            (aggregated) local networks of probabilistic or binary
            interactions (Fig. 2c-d). We built probabilistic local
            networks following Eq. 2, and set <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            to <span class="math inline">1</span> when the interaction
            was observed at least once, and to <span
            class="math inline">0</span> otherwise (i.e., no false
            positive or false negative in the metaweb). We used a
            constant value of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            for all interactions.</p>
            <p>When aggregating local networks of probabilistic
            interactions, the constancy of the probability of regional
            interaction across the entire study area means that any rise
            in the probability of local interaction is solely
            attributable to an increase in <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>.
            For example, let <span
            class="math inline"><em>L</em><sub>1</sub></span> and <span
            class="math inline"><em>L</em><sub>2</sub></span> be two
            local networks and <span
            class="math inline"><em>L</em><sub>1, 2</sub></span> the
            aggregated network. If <span
            class="math inline"><em>P</em>(<em>L</em><sub>1, <em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            and <span
            class="math inline"><em>P</em>(<em>L</em><sub>2, <em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            are the probabilities that two potentially interacting taxa
            interact respectively in <span
            class="math inline"><em>L</em><sub>1</sub></span> and <span
            class="math inline"><em>L</em><sub>2</sub></span>, the
            probability <span
            class="math inline"><em>P</em>(<em>L</em><sub>1, 2, <em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            that these taxa interact in the aggregated network <span
            class="math inline"><em>L</em><sub>1, 2</sub></span> is
            obtained by:</p>
            <p><span id="eq:aggregate"><span
            class="math display"><em>P</em>(<em>L</em><sub>1, 2, <em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 1 − [1−<em>P</em>(<em>L</em><sub>1, <em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)] × [1−<em>P</em>(<em>L</em><sub>2, <em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)],   (13)</span></span></p>
            <p>assuming independence between the interaction of the two
            taxa in different networks. This equation represents the
            probability that the interaction is realized in either (1)
            exclusively the local network <span
            class="math inline"><em>L</em><sub>1</sub></span>, (2)
            exclusively the local network <span
            class="math inline"><em>L</em><sub>2</sub></span> or (3)
            both, given that the two taxa have the biological capacity
            to interact.</p>
            <p>By comparing the scaling relationships observed in local
            networks of binary (empirical) and probabilistic (modeled)
            interactions, Fig. 2c-d shows that high values of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            lead to systematic overestimations in the number of
            interactions and connectance, especially when <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 1</span>
            (corresponding to the scenario where local probabilities of
            interactions are equivalent to the probabilities of regional
            interactions). This suggests that high values of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            do not adequately capture the variability of local
            interactions. However, these biases tend to diminish as the
            number of sampled networks increases, indicating that most
            interactions are eventually captured when <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            is high. In contrast, low values of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            lead to missing interactions, resulting in an
            underestimation of the number of interactions and
            connectance. These results underscore the importance of
            using the appropriate level of variability when estimating
            local interaction probabilities.</p>
            </blockquote>
            <h1
            id="metawebs-regional-catalogs-of-interactions">Metawebs:
            regional catalogs of interactions</h1>
            <p>Metawebs <span class="citation"
            data-cites="Dunne2006Network">(Dunne 2006)</span> are
            networks of potential interactions over broad spatial,
            temporal, and taxonomic scales (e.g., food webs at the
            continental scale). Potential interactions describe the
            biological capacity of taxa to interact, which is typically
            assessed at the regional scale. Metawebs of probabilistic
            interactions are particularly useful in situations where
            there is uncertainty in the ability of taxa to interact.
            This uncertainty frequently arises due to insufficient
            interaction data, especially for taxa that have not yet been
            observed to co-occur, and uncertainties in trait-matching
            models. As data accumulates, interactions in metawebs should
            tend towards binarity, approaching probability values of
            <span class="math inline">0</span> (repeatedly failing to
            observe an interaction between two co-occurring taxa) and
            <span class="math inline">1</span> (observing an interaction
            at least once). The extent of sampling effort thus
            influences our evaluation of probabilities of potential
            interactions, as sampling over a larger area or for a longer
            duration enables us to capture a greater number of regional
            interactions <span class="citation"
            data-cites="McLeod2021Sampling">(McLeod <em>et al.</em>
            2021)</span>. However, in contrast with local networks of
            probabilistic interactions, which describe local interaction
            stochasticity, regional interactions are not evaluated for
            any particular local context. In Box 3, we discuss the
            differences in spatial and temporal scaling of regional
            interactions compared to local interactions. We do so by
            using the host-parasite networks of <span class="citation"
            data-cites="Kopelke2017FooStra">Kopelke <em>et al.</em>
            (2017)</span> as an illustration of spatial scaling.
            Although <em>neutrally</em> forbidden interactions <span
            class="citation" data-cites="Canard2012Emergencea">(i.e.,
            between rare species, Canard <em>et al.</em> 2012)</span>
            tend to have low probability values in local networks, they
            may have higher probabilities in the metaweb (i.e., when
            species’ biological traits can support an interaction if
            they were to encounter each other). Likewise,
            non-co-occurring taxa may have a non-zero probability of
            interaction in the metaweb.</p>
            <p>Potential interactions describe what we refer to as the
            <em>biological</em> feasibility of interactions, which is
            based solely on the regional traits distributions <span
            class="math inline"><em>T</em><sub><em>i</em></sub></span>
            and <span
            class="math inline"><em>T</em><sub><em>j</em></sub></span>
            of taxa <span class="math inline"><em>i</em></span> and
            <span class="math inline"><em>j</em></span>, respectively. A
            probability of potential interaction in a metaweb <span
            class="math inline"><em>M</em></span> describing the
            biological feasibility of interactions may be expressed
            as:</p>
            <p><span id="eq:metaweb"><span
            class="math display"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>),   (14)</span></span></p>
            <p>which, in contrast with local networks, is not
            conditioned on any spatial, temporal, co-occurrence or
            environmental variables (Tbl. 1). Regional traits may differ
            from local traits <span
            class="math inline"><em>T</em><sub><em>i</em>, <em>k</em></sub></span>
            and <span
            class="math inline"><em>T</em><sub><em>j</em>, <em>k</em></sub></span>,
            which may vary spatially and temporally due to phenotypic
            plasticity <span class="citation"
            data-cites="Berg2010Traita">(Berg &amp; Ellers 2010)</span>.
            The taxonomic level at which interactions are evaluated
            influences the distribution of regional and local traits.
            However, as explained in Box 4, there is no fundamental
            difference in the taxonomic scaling of regional and local
            interactions (i.e., how interaction probabilities change
            with taxonomic level), unlike their spatial and temporal
            scaling.</p>
            <p>The biological feasibility of interactions expresses our
            degree of belief that there exists at least one combination
            of phenotypes that could support an interaction if they were
            to encounter each other, assuming they had infinite time to
            interact. Evaluating this probability is conducted without
            incorporating the environmental conditions under which they
            encounter each other into the model, as the complement of
            the probability <span
            class="math inline"><em>P</em>(<em>F</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>)</span>
            of forbidden interactions based uniquely on biological
            traits (i.e., the probability that their traits do not
            support an interaction):</p>
            <p><span id="eq:forbidden"><span
            class="math display"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>) = 1 − <em>P</em>(<em>F</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>).   (15)</span></span></p>
            <p>For example, let <span
            class="math inline"><em>i</em></span> be a western
            diamondback rattlesnake (<em>Crotalus atrox</em>) and <span
            class="math inline"><em>j</em></span>, a wood lemming
            (<em>Myopus schisticolor</em>). These two taxa never
            co-occur, the rattlesnake being adapted to warm regions of
            North America <span class="citation"
            data-cites="Castoe2007Phylogeographic">(Castoe <em>et
            al.</em> 2007)</span> and the lemming, to northern habitats
            of Eurasia <span class="citation"
            data-cites="Fedorov2008Comparative">(Fedorov <em>et al.</em>
            2008)</span>. As we lack direct observations of an
            interaction between these two species, we have to rely on
            expert knowledge or trait-matching models to estimate their
            probability of potential interaction. To accurately estimate
            this probability using trait-matching models, it is crucial
            to ensure that the set of traits considered reflects the
            overall traits distributions of both taxa. We could for
            instance consider their average body mass and the average
            phylogenetic proximity of lemmings to rattlesnakes’ prey.
            Doing so, we might find a high probability of potential
            interaction based on these traits.</p>
            <p>The biological feasibility of interactions should not be
            confused with what we refer to as the <em>ecological</em>
            feasibility of interactions. A probability of potential
            interaction in a metaweb <span
            class="math inline"><em>M</em><sup>*</sup></span> describing
            ecological feasibility of interactions may be expressed
            as:</p>
            <p><span id="eq:metaweb2"><span
            class="math display"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub><sup>*</sup>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>,<em>E</em>),   (16)</span></span></p>
            <p>where <span class="math inline"><em>E</em></span> is the
            environmental conditions under which potential interactions
            are evaluated (Tbl. 1). Unlike <span
            class="math inline"><em>E</em><sub><em>k</em></sub></span>,
            these environmental conditions do not represent conditions
            occurring at specific locations. Ecological feasibility
            represents the probability that two taxa interact if they
            were to encounter each other under given environmental
            conditions, assuming they had infinite time to interact.
            Incorporating environmental conditions into a trait-matching
            model may be important when there is high covariance between
            the environment and traits. For instance, in our example
            involving rattlesnakes and lemmings, the probability of
            potential interaction between these two species may be low
            in most environmental conditions. Western diamondback
            rattlesnakes may be unactive under low temperatures <span
            class="citation" data-cites="Kissner1997Rattling">(Kissner
            <em>et al.</em> 1997)</span>, whereas wood lemmings may have
            low tolerance to high temperatures <span class="citation"
            data-cites="Kausrud2008Linking">(Kausrud <em>et al.</em>
            2008)</span>. The probability that an interaction is
            ecologically feasible is always lower than the probability
            that it is biologically feasible, even across all
            environmental conditions:</p>
            <p><span id="eq:feasibility"><span
            class="math display">∫<sub><em>E</em></sub><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub><sup>*</sup>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>,<em>E</em>)<em>d</em><em>E</em> ≤ <em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>).   (17)</span></span></p>
            <p>This is because the biological feasibility of an
            interaction is a prerequisite for its ecological
            feasibility. Biological feasibility is necessary but not
            sufficient for an interaction to be ecologically feasible.
            Our discussion of metawebs will focus on the biological
            feasibility of interactions since most methods developed for
            inferring probabilities of regional interactions do not
            explicitly take into account environmental conditions <span
            class="citation" data-cites="Strydom2022Food">(e.g., Strydom
            <em>et al.</em> 2022)</span>.</p>
            <p>Starting from a selected set of taxa, which are usually
            distributed within a broad region of interest, metawebs can
            be built using different data sources, including literature
            review <span class="citation"
            data-cites="Maiorano2020Tetraeua">(e.g., Maiorano <em>et
            al.</em> 2020)</span>, aggregated interaction data <span
            class="citation"
            data-cites="Gravel2019BriElt Saravia2022Ecological">(e.g.,
            Gravel <em>et al.</em> 2019; Saravia <em>et al.</em>
            2022)</span>, trait-matching models <span class="citation"
            data-cites="Shaw2024Framework Strydom2022Food">(e.g., Shaw
            <em>et al.</em> 2024; Strydom <em>et al.</em> 2022)</span>,
            and expert knowledge. Every pair of taxa that have
            confidently been observed to interact at least once can be
            given a probability of <span class="math inline">1</span>
            (i.e., <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 1</span>)
            since we know that they <em>can</em> interact. This differs
            from local networks of probabilistic interactions, where
            interaction events may remain stochastic (i.e., <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>) &lt; 1</span>)
            even after empirically observing interactions due to their
            spatiotemporal variability. Interactions that were never
            observed typically have low probability values in local
            networks and vary from low to high values in metawebs,
            contingent upon taxa traits distributions (reaching <span
            class="math inline">0</span> for forbidden links).</p>
            <p>When incorporating local network data to estimate
            probabilities of regional interactions, repeatedly failing
            to observe an interaction between two co-occurring taxa
            (i.e., <span
            class="math inline"><em>P</em>(<em>O</em><sub><em>i</em> → <em>j</em></sub>=0)</span>)
            should decrease the probability that the interaction is
            biologically feasible. Using Bayes’ theorem, the probability
            that the interaction is biologically feasible given that it
            was never observed locally, <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>=1|<em>O</em><sub><em>i</em> → <em>j</em></sub>=0,...)</span>,
            may be calculated as follows:</p>
            <p><span id="eq:emp_sampling"><span
            class="math display">$$P(M_{i \rightarrow j} = 1 | O_{i
            \rightarrow j} = 0,...) = \frac{P(O_{i
            \rightarrow j} = 0 |M_{i \rightarrow j} = 1, ...) \times
            P(M_{i \rightarrow j} =
            1 | ...)}{P(O_{i \rightarrow j} = 0
            |...)}.\qquad{(18)}$$</span></span></p>
            <p>The reduction in the probability of regional interaction
            after considering that it was never observed locally (i.e.,
            <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>=1|<em>O</em><sub><em>i</em> → <em>j</em></sub>=0,...) &lt; <em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>=1|...)</span>)
            occurs because <span
            class="math inline"><em>P</em>(<em>O</em><sub><em>i</em> → <em>j</em></sub>=0|<em>M</em><sub><em>i</em> → <em>j</em></sub>=1,...)</span>
            must be lower than <span
            class="math inline"><em>P</em>(<em>O</em><sub><em>i</em> → <em>j</em></sub>=0,...)</span>,
            i.e. there is a higher chance of observing an interaction
            when we know it is biologically feasible.</p>
            <p>Many observations of interactions are false positives
            because of observation errors due to taxonomic
            misidentifications and ecological misinterpretations <span
            class="citation" data-cites="Pringle2020Resolving">(e.g.,
            due to cryptic species and interactions, Pringle &amp;
            Hutchinson 2020)</span>. Likewise, forbidden interactions
            may be false negatives, e.g. if they have been evaluated
            based on unrepresentative or incomplete traits
            distributions. Employing Bayesian models proves valuable
            when estimating interaction probabilities in metawebs <span
            class="citation"
            data-cites="Bartomeus2016ComFra Cirtwill2019QuaFra">(e.g.,
            Bartomeus <em>et al.</em> 2016; Cirtwill <em>et al.</em>
            2019)</span>. This improvement is achieved by updating prior
            information regarding the feasibility of interactions (e.g.,
            experts’ prior assessments of interaction probabilities)
            with empirical data on interactions and traits.</p>
            <blockquote>
            <h2 id="box-3-spatial-and-temporal-scaling-of-interactions"
            class="unnumbered">Box 3: Spatial and temporal scaling of
            interactions</h2>
            <p>Network-Area Relationships document the scaling of
            network properties (such as modularity and trophic chain
            lengths) with spatial scale <span class="citation"
            data-cites="Galiana2018Spatiala Wood2015Effects">(e.g.,
            Galiana <em>et al.</em> 2018; Wood <em>et al.</em>
            2015)</span>. The variations in network structure across
            spatial scales may stem from the scaling of species richness
            (species-area relationships, SARs), the number of
            interactions <span class="citation"
            data-cites="Brose2004UniSpa">(Brose <em>et al.</em>
            2004)</span>, and many other higher-level properties of the
            system <span class="citation"
            data-cites="Thompson2005Foodweb">(e.g., environmental
            heterogeneity, Thompson &amp; Townsend 2005)</span> with the
            sampled area. Additionally, ecological processes occurring
            at distinct spatial scales, such as the spatial variability
            in local community composition resulting from different
            sequences of extinction and colonization events, can also
            contribute to this variation <span class="citation"
            data-cites="Galiana2018Spatiala">(Galiana <em>et al.</em>
            2018)</span>. Next, interaction accumulation curves describe
            the scaling of the number of observed interactions with
            sampling effort <span class="citation"
            data-cites="Jordano2016Samplingb">(Jordano 2016)</span>.
            Sampling effort, which may correspond to the duration of the
            sampling period used to construct the network, can impact
            connectance <span class="citation"
            data-cites="Bersier1999Scale">(Bersier <em>et al.</em>
            1999)</span> and various measures of network structure <span
            class="citation"
            data-cites="Banasek-Richter2004Sampling McLeod2021Sampling">(Banašek-Richter
            <em>et al.</em> 2004; McLeod <em>et al.</em> 2021)</span>.
            Apart from sampling effort, the temporal scaling of
            interactions also elucidates how network structure changes
            with the temporal resolution of the network, acknowledging
            that distinct interactions take place over time, ranging
            from short-term fluctuations of interactions to long-term
            trends. As local networks of probabilistic interactions may
            explicitly account for the spatiotemporal variability of
            interactions, they offer a distinct approach to
            investigating the scaling of network structure with space
            and time, in contrast to networks of binary and quantitative
            interactions, by making the stochasticity of interactions
            the focal point of the modeling process.</p>
            <p>Local networks and metawebs exhibit distinct
            relationships with spatial and temporal scales. On one hand,
            metawebs of probabilistic interactions, representing
            biological feasibility, feature regional interactions that
            do not scale with space and time. This is because regional
            interactions depend solely on the biological capacity of two
            taxa to interact, regardless of their co-occurrence and
            local environmental conditions. However, probabilities of
            potential interactions may change (tending to become more
            definitive) upon updating previous estimates with increased
            sampling effort, even though they do not vary in a specific
            direction with the spatial and temporal extent (boundaries)
            of the network. The probability of two taxa potentially
            interacting should theoretically be the same in all metawebs
            in which they are present, provided that the data and
            methods used for estimation are consistent. For example, if
            a smaller metaweb <span
            class="math inline"><em>M</em><sub>1</sub></span> is derived
            from a larger metaweb <span
            class="math inline"><em>M</em><sub>0</sub></span> by
            selecting the subset of taxa present in the region described
            by <span class="math inline"><em>M</em><sub>1</sub></span>
            and retaining all their interactions, their probabilities of
            interaction should be identical regardless of scale,
            i.e. <span
            class="math inline"><em>P</em>(<em>M</em><sub>1, <em>i</em> → <em>j</em></sub>) = <em>P</em>(<em>M</em><sub>0, <em>i</em> → <em>j</em></sub>)</span>.
            With a larger or equal number of taxa in <span
            class="math inline"><em>M</em><sub>0</sub></span> as
            compared to <span
            class="math inline"><em>M</em><sub>1</sub></span>, the total
            number of interactions is expected to be higher or at least
            equal in the larger network, even though pairwise
            probabilities remain identical.</p>
            <p>On the other hand, local interactions scale both
            spatially and temporally, given that they have more
            opportunities to be realized and observed in larger areas
            and longer durations. This is attributed to factors such as
            a higher number of individuals, greater trait variations,
            and increased opportunities for encounters, as highlighted
            by <span class="citation"
            data-cites="McLeod2020EffSpe">McLeod <em>et al.</em>
            (2020)</span>. For example, if a local network of
            probabilistic interactions <span
            class="math inline"><em>L</em><sub>1</sub></span> with an
            area <span class="math inline"><em>A</em><sub>1</sub></span>
            is derived from a larger network <span
            class="math inline"><em>L</em><sub>0</sub></span> with an
            area <span
            class="math inline"><em>A</em><sub>0</sub></span>, and <span
            class="math inline"><em>A</em><sub>1</sub></span> is
            entirely nested within <span
            class="math inline"><em>A</em><sub>0</sub></span>,
            interaction probabilities should be lower in the smaller
            network, i.e. <span
            class="math inline"><em>P</em>(<em>L</em><sub>1, <em>i</em> → <em>j</em></sub>|<em>A</em><sub>1</sub>&lt;<em>A</em><sub>0</sub>) ≤ <em>P</em>(<em>L</em><sub>0, <em>i</em> → <em>j</em></sub>|<em>A</em><sub>0</sub>)</span>.
            However, if <span
            class="math inline"><em>A</em><sub>1</sub></span> and <span
            class="math inline"><em>A</em><sub>0</sub></span> are
            disjoint, interaction probabilities could be higher in the
            smaller area, contingent upon their environmental and
            biological conditions. Likewise, interaction probabilities
            are expected to be lower in networks with shorter durations
            when time intervals are nested. In Fig. 3, we show how the
            expected number of local host-parasite interactions scales
            with area, represented as an expanding latitudinal window,
            in comparison with regional interactions. Even though we
            employed local probabilities of interactions equal to
            regional interactions for the purpose of comparison (i.e.,
            using <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 1</span>
            here), we notice that the total number of regional
            interactions scales more rapidly than local interactions.
            This is because numerous regional interactions involve
            species that never co-occur, and as a result, these
            interactions are not captured in local networks.</p>
            <figure>
            <img src="figures/spatial_scaling.png" id="fig:spatial"
            alt="Figure 3: Spatial scaling of interactions. Expected number of host-parasite interactions in a network aggregating all (a) local and (b) regional probabilistic interactions within a latitudinal window of a given length. Every dashed curve corresponds to a different expanding window characterized by a different central latitude, with the colored solid line representing the median number of interactions. Heatmaps of the expected number of (c) local and (d) regional interactions found in windows of specified length and central latitudes. Probabilities of regional interactions were obtained with a false positive rate of 5\% and a false negative rate of 10\%. Local probabilistic interactions were derived from probabilistic regional interactions by setting the value of P(L_{i \rightarrow j}|M_{i \rightarrow j}) (the local probability of interaction among potentially interacting species) to 1, ensuring a conservative comparison between aggregated local networks and metawebs. Aggregated local networks were obtained by aggregating both the species and interactions found within a particular latitudinal window, with the values of P(L_{i \rightarrow j}|M_{i \rightarrow j}) remaining at their maximum value of 1 following Eq. 13." />
            <figcaption aria-hidden="true">Figure 3: <strong>Spatial
            scaling of interactions.</strong> Expected number of
            host-parasite interactions in a network aggregating all (a)
            local and (b) regional probabilistic interactions within a
            latitudinal window of a given length. Every dashed curve
            corresponds to a different expanding window characterized by
            a different central latitude, with the colored solid line
            representing the median number of interactions. Heatmaps of
            the expected number of (c) local and (d) regional
            interactions found in windows of specified length and
            central latitudes. Probabilities of regional interactions
            were obtained with a false positive rate of <span
            class="math inline">5%</span> and a false negative rate of
            <span class="math inline">10%</span>. Local probabilistic
            interactions were derived from probabilistic regional
            interactions by setting the value of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            (the local probability of interaction among potentially
            interacting species) to <span class="math inline">1</span>,
            ensuring a conservative comparison between aggregated local
            networks and metawebs. Aggregated local networks were
            obtained by aggregating both the species and interactions
            found within a particular latitudinal window, with the
            values of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            remaining at their maximum value of <span
            class="math inline">1</span> following Eq. 13.</figcaption>
            </figure>
            </blockquote>
            <blockquote>
            <h2 id="box-4-taxonomic-scaling-of-interactions"
            class="unnumbered">Box 4: Taxonomic scaling of
            interactions</h2>
            <p>Probabilistic interaction networks offer a versatile
            approach to tackle a broad array of ecological questions,
            depending on their level of organization. For instance, the
            assemblage of interactions across ecological scales can be
            explored through species-based networks, while clade-based
            networks provide insights into macroevolutionary processes
            <span class="citation" data-cites="Gomez2010EcoInt">(e.g.,
            Gomez <em>et al.</em> 2010)</span>. Given that our
            interpretation of the properties and dynamics of ecological
            networks depends on their taxonomic level <span
            class="citation"
            data-cites="Guimaraes2020Structurea">(Guimarães
            2020)</span>, investigating the taxonomic scaling of
            interactions (i.e., how interaction probabilities change
            with taxonomic level) emerges as a promising research
            avenue. Examining the same system at various taxonomic
            scales can yield meaningful and complementary ecological
            information, and, in our perspective, employing networks of
            probabilistic interactions is an effective approach for such
            analyses.</p>
            <p>There are no inherent differences in the taxonomic
            scaling between local and metawebs, as only the nodes are
            defined taxonomically. In other words, the probability
            values of edges in both local (Eq. 3) and metawebs (Eq. 14)
            are not conditional on any taxonomic level. The taxonomic
            scale is tied to the definition of the event itself (i.e.,
            the interaction between two taxa, defined at the desired
            taxonomical scale), not to the variables on which
            interaction probabilities are conditional. In both types of
            networks, transitioning to a broader level of organization
            (e.g., from a species-level network <span
            class="math inline"><em>S</em></span> to a genus-level
            network <span class="math inline"><em>G</em></span>) can be
            accomplished directly by using probabilities from finer
            scales. For example, in a network with <span
            class="math inline"><em>n</em><sub>1</sub></span> species
            from genus <span
            class="math inline"><em>g</em><sub>1</sub></span> and <span
            class="math inline"><em>n</em><sub>2</sub></span> species
            from genus <span
            class="math inline"><em>g</em><sub>2</sub></span>, one can
            compute the probability that at least one species from genus
            <span class="math inline"><em>g</em><sub>1</sub></span>
            interacts with at least one species from genus <span
            class="math inline"><em>g</em><sub>2</sub></span> (i.e.,
            that there is a non-zero number of species-level
            interactions or, equivalently, that the genus-level
            interaction occurs) as follows:</p>
            <p><span id="eq:taxo"><span class="math display">$$P(G_{g_1
            \rightarrow g_2} = 1) = 1 - \prod_{i = 1}^{n_1}\prod_{j =
            1}^{n_2}(1
            - P(S_{g_{1i} \rightarrow g_{2j}} =
            1)),\qquad{(19)}$$</span></span></p>
            <p>where <span
            class="math inline"><em>g</em><sub>1, <em>i</em></sub></span>
            and <span
            class="math inline"><em>g</em><sub>2, <em>j</em></sub></span>
            are the species of the corresponding genus and assuming
            independence between species-level interactions. If it is
            known that at least two of these species interact (i.e.,
            <span
            class="math inline"><em>P</em>(<em>S</em><sub><em>g</em><sub>1, <em>i</em></sub> → <em>g</em><sub>2, <em>j</em></sub></sub>) = 1</span>
            for at least one pair of <span
            class="math inline">(<em>g</em><sub>1, <em>i</em></sub>,<em>g</em><sub>2, <em>j</em></sub>)</span>),
            it implies a probability of genus interaction equal to <span
            class="math inline">1</span>. <span class="citation"
            data-cites="Canard2012Emergencea">Canard <em>et al.</em>
            (2012)</span> built a species-based network following a
            similar approach, by using simulated interactions between
            individuals derived from a neutral model (i.e., a model that
            assumed ecological equivalence among individuals). In
            contrast, a more sophisticated approach is necessary when
            transitioning from a broader to a finer level of
            organization. This is because knowledge of an interaction
            between two genera does not guarantee that all possible
            pairwise combinations of their species will also interact.
            One possible method is to build a finer-scale network by
            generating probabilities of interactions through random
            sampling from a beta distribution, parameterized by the
            broader-scale network.</p>
            <p>Ideally, our biological interpretation of probabilistic
            interactions should remain consistent across a network even
            if it incorporates heterogeneous levels of organization,
            such as a network whose nodes represent both species and
            trophic species (groups of species sharing similar predators
            and prey). This is common in ecological networks where
            taxonomic resolution is typically low <span class="citation"
            data-cites="Hemprich-Bennett2021AssImp VazquezSS2022Ecological">(Hemprich-Bennett
            <em>et al.</em> 2021; Vázquez <em>et al.</em> 2022)</span>.
            Interaction probabilities at broader taxonomic scales should
            be based on probabilities of interactions between
            individuals, for both local networks and metawebs. For
            instance, in local individual-based food webs, the
            probability that two individuals interact reflects our
            degree of belief that one individual will consume the other.
            Similarly, in local species-based food webs, the probability
            that two species interact represents our degree of belief
            that at least one individual from the predator species
            consumes at least another individual from the prey species.
            Furthermore, in local clade-based food webs, the probability
            that two clades interact represents our degree of belief
            that at least two species from these clades interact with
            each other or, equivalently, that at least two individuals
            from these clades interact with each other. Fundamentally,
            the taxonomic scaling of interactions involves aggregating
            interactions between individuals into larger groups, which
            may exhibit varying degrees of homogeneity based on the
            organisms and the study system. In that regard, taxonomic
            scaling is analogous to the spatial and temporal scaling of
            interactions, as they all represent different ways to
            aggregate individuals into broader groups (either spatially,
            temporally, or taxonomically).</p>
            </blockquote>
            <blockquote>
            <h2 id="box-5-sampling-for-binary-interaction-networks"
            class="unnumbered">Box 5: Sampling for binary interaction
            networks</h2>
            <p>The prediction of binary interactions through Bernoulli
            trials is an important application of probabilistic
            interaction networks. This approach proves beneficial for
            analyzing the structural characteristics of probabilistic
            interaction networks, particularly in the absence of
            specific analytical formulas for measures describing the
            structure of the complete network <span class="citation"
            data-cites="Poisot2016Structure">(Poisot <em>et al.</em>
            2016)</span>. By performing independent Bernoulli trials for
            each interaction, a network of binary interactions may be
            generated. A probability distribution of network properties
            can be obtained by measuring network structure across
            multiple randomly generated networks <span class="citation"
            data-cites="Poisot2016Structure">(Poisot <em>et al.</em>
            2016)</span>. This method enables the representation of the
            variability or uncertainty of network structure, albeit with
            possible biases when connectance is low <span
            class="citation"
            data-cites="Poisot2014WheEco Chagnon2015ChaTop">(Chagnon
            2015; Poisot &amp; Gravel 2014)</span>. Employing this
            strategy to generate binary interaction networks under a
            null model facilitates null hypothesis significance testing,
            wherein the observed measure is compared against the
            simulated distribution <span class="citation"
            data-cites="Bascompte2003NesAss">(e.g., Bascompte <em>et
            al.</em> 2003)</span>. Yet, inferring network structure
            through sampling for binary interactions assumes
            independence among interactions, which might not accurately
            represent reality. Covariation among interactions could
            exist even if we do not explicitly condition interactions on
            others. For example, an interaction between two taxa could
            be more probable when another interaction occurs. The
            consequences of this assumption of independence on the
            prediction of network structure have yet to be empirically
            examined. In Box 5, we argue that predicting binary
            interactions across space using regional interaction
            probabilities differs from doing so with local interaction
            probabilities. We illustrate both sampling processes using
            the host-parasite networks of <span class="citation"
            data-cites="Kopelke2017FooStra">Kopelke <em>et al.</em>
            (2017)</span>. The random generation of binary interaction
            networks, from a local network that accounts for the
            spatiotemporal variability of interactions, may effectively
            capture network structure across space and time,
            facilitating the investigation of ecological hypotheses
            about interactions at broad spatial and temporal scales.</p>
            <p>There are at least two distinct approaches to sample
            binary interaction networks from probabilistic interaction
            networks across space, for example, when attempting to
            predict a binary interaction network for each location <span
            class="math inline"><em>k</em></span> within a given region.
            Both approaches assume independence between interactions.
            The first approach involves performing a singular Bernoulli
            trial for each pair of taxa based on their regional
            probability of interaction:</p>
            <p><span class="math display">$$M_{i \rightarrow j} \sim
            {\rm Bernoulli}(P(M_{i \rightarrow j})).$$</span></p>
            <p>In employing this approach, we predict a single metaweb
            of binary interactions for each simulation. Every pair of
            taxa predicted to interact in this metaweb will be treated
            as interacting in all local networks where they co-occur,
            i.e. <span
            class="math inline"><em>L</em><sub><em>l</em>, <em>i</em> → <em>j</em></sub> = <em>M</em><sub><em>i</em> → <em>j</em></sub></span>
            when <span
            class="math inline"><em>X</em><sub><em>i</em>, <em>j</em>, <em>k</em></sub> = 1</span>.
            This will result in local pairwise interactions without
            spatial variation.</p>
            <p>The second approach is to independently sample each local
            network of probabilistic interactions:</p>
            <p><span class="math display">$$L_{l, i \rightarrow j} \sim
            {\rm Bernoulli}(P(L_{l, i \rightarrow j})).$$</span></p>
            <p>This can be achieved by first generating distinct
            probabilistic interaction networks for each location. These
            local networks of probabilistic interactions may vary in
            taxa composition and interaction probabilities. Because
            binary interaction networks are sampled independently for
            each location, this second approach introduces spatial
            variation in binary interactions.</p>
            <p>In Fig. 4, we compare the average connectance of binary
            interaction networks resulting from these two sampling
            techniques. We drew regional and local interactions from our
            host-parasite networks of probabilistic interactions,
            generating a number of binary interaction network
            realizations for each site in the dataset. These two
            sampling approaches yield different outcomes, particularly
            for lower values of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>,
            which represent instances where regional interactions do not
            consistently manifest locally. Small discrepancies between
            these techniques are also apparent when we equate the
            probability of local interaction to the probability of
            regional interaction (i.e., when using <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 1.0</span>
            in Eq. 2, Fig. 4a), especially when the number of
            simulations of binary interaction networks for each location
            is low (<span class="math inline"><em>n</em> = 1</span>). As
            anticipated, we observe that sampling binary interactions
            from the metaweb tends to overestimate connectance on
            average compared to sampling them from local networks
            (Fig. 4a-c). Furthermore, we observe an increase in the
            variability of connectance when employing a single
            simulation (Fig. 4, gray markers), which in our opinion is a
            more tangible representation of the process leading to the
            realization of local interactions in nature.</p>
            <figure>
            <img src="figures/network_sampling.png" id="fig:sampling"
            alt="Figure 4: Connectance of sampled binary interaction networks. Comparison between the average connectance of binary interaction network samples obtained from the local networks and metawebs of probabilistic interactions. Each dot corresponds to a different site. The local probability of interaction between potentially interacting species was set to three different values: (a) P(L_{i \rightarrow j}|M_{i \rightarrow j}) = 1.0, (b) P(L_{i \rightarrow j}|M_{i \rightarrow j}) = 0.75, and (c) P(L_{i \rightarrow j}|M_{i \rightarrow j}) = 0.50. Grey dots represent the outcome of a single simulation, while colored dots represent the average connectance of each network across 100 simulations. (d) Reduction in the mean squared logarithmic error between the average connectance of binary interaction networks obtained from these two sampling methods as the number of simulations increases, for the same values of P(L_{i \rightarrow j}|M_{i \rightarrow j}) used in panels a-c. Probabilities of regional interactions were obtained with a false positive rate of 5\% and a false negative rate of 10\%. Metaweb samples were obtained by randomly sampling binary interactions from the probabilistic interaction metaweb, and then propagating this result to all local networks that include the species potentially engaged in the interactions. Local binary interaction networks were generated by independently sampling binary interactions for each local network of probabilistic interactions." />
            <figcaption aria-hidden="true">Figure 4: <strong>Connectance
            of sampled binary interaction networks.</strong> Comparison
            between the average connectance of binary interaction
            network samples obtained from the local networks and
            metawebs of probabilistic interactions. Each dot corresponds
            to a different site. The local probability of interaction
            between potentially interacting species was set to three
            different values: (a) <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 1.0</span>,
            (b) <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 0.75</span>,
            and (c) <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>) = 0.50</span>.
            Grey dots represent the outcome of a single simulation,
            while colored dots represent the average connectance of each
            network across <span class="math inline">100</span>
            simulations. (d) Reduction in the mean squared logarithmic
            error between the average connectance of binary interaction
            networks obtained from these two sampling methods as the
            number of simulations increases, for the same values of
            <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            used in panels a-c. Probabilities of regional interactions
            were obtained with a false positive rate of <span
            class="math inline">5%</span> and a false negative rate of
            <span class="math inline">10%</span>. Metaweb samples were
            obtained by randomly sampling binary interactions from the
            probabilistic interaction metaweb, and then propagating this
            result to all local networks that include the species
            potentially engaged in the interactions. Local binary
            interaction networks were generated by independently
            sampling binary interactions for each local network of
            probabilistic interactions.</figcaption>
            </figure>
            <p>The choice of a sampling approach can influence the
            selection of grid cell size when delineating local
            communities within a broader region of interest. In the
            first approach, pairwise interactions remain constant
            irrespective of cell size because they are sampled only once
            from the metaweb. However, in the second approach, local
            interaction probabilities are contingent on network area.
            For instance, consider the local networks <span
            class="math inline"><em>L</em><sub>1</sub></span> and <span
            class="math inline"><em>L</em><sub>2</sub></span> with an
            area of <span class="math inline">$\frac{1}{2}A_0$</span>,
            both nested within <span
            class="math inline"><em>A</em><sub>0</sub></span> but
            disjoint from each other, forming <span
            class="math inline"><em>L</em><sub>0</sub></span>. If we
            treat <span
            class="math inline"><em>L</em><sub>1</sub></span> and <span
            class="math inline"><em>L</em><sub>2</sub></span> as
            independent, the probability of interaction between taxa
            <span class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> in <span
            class="math inline"><em>L</em><sub>0</sub></span> is given
            by:</p>
            <p><span id="eq:binary"><span
            class="math display"><em>P</em>(<em>L</em><sub>0, <em>i</em> → <em>j</em></sub>) = 1 − [1−<em>P</em>(<em>L</em><sub>1, <em>i</em> → <em>j</em></sub>)] × [1−<em>P</em>(<em>L</em><sub>2, <em>i</em> → <em>j</em></sub>)].   (20)</span></span></p>
            <p>Due to its larger area, the probability that the two taxa
            interact in <span
            class="math inline"><em>L</em><sub>0</sub></span> is equal
            or greater than in <span
            class="math inline"><em>L</em><sub>1</sub></span> and <span
            class="math inline"><em>L</em><sub>2</sub></span>. When
            sampling binary interactions from local networks, it is
            crucial to sample at the same spatial scale for which
            probabilities were estimated. Otherwise, interaction
            probabilities must be adjusted to align with the intended
            cell size, preventing systematic biases in predictions.</p>
            </blockquote>
            <h1 id="conclusion">Conclusion</h1>
            <p>In this contribution, we underline the importance of
            network documentation for adequately interpreting and
            manipulating probabilistic interaction data. The
            mathematical representation of probabilities and their
            statistical properties depend on the type of interactions
            (local or regional) and the conditions under which these
            interactions were evaluated. We showed that local networks
            and metawebs of probabilistic interactions differ in their
            relationship to spatial and temporal scales (Box 3), with
            regional interactions remaining consistent across scales. In
            contrast with metawebs, local interactions are measured in a
            specific context (e.g., in a given area, time, and
            biological and environmental conditions) and depend on taxa
            co-occurrence. These differences bring to light the need to
            use probabilistic data with caution, for instance when
            generating network realizations of binary interactions
            across space (Box 5). Clear documentation describing the
            type of interaction and the variables used in their
            estimation are required to ensure adequate data
            manipulation. Sound data practices and foundations for
            probabilistic thinking in network ecology facilitate
            reliable assessments of the spatiotemporal variability and
            uncertainty of biotic interactions. Here we identify key
            research priorities for improving our understanding of local
            and regional interactions and their stochasticity.</p>
            <h2 id="predicting-local-networks-from-metawebs">Predicting
            local networks from metawebs</h2>
            <p>Metawebs are a valuable source of ecological information
            for predicting local networks across time and space. Local
            networks of binary interactions can be reconstructed by
            selecting a subset of taxa and interactions from the metaweb
            <span class="citation" data-cites="Dunne2006Network">(Dunne
            2006)</span>. Metawebs thus contain more interactions than
            local networks, even though their connectance is usually
            much smaller than the one of local networks <span
            class="citation" data-cites="Gravel2011Trophic">(Gravel
            <em>et al.</em> 2011)</span>. Determining the list of taxa
            to select can be achieved empirically (e.g., observed
            occurrence data for a site) or numerically (e.g., species
            distribution models). As species composition is arguably
            easier to sample or predict than pairwise interactions, the
            primary challenge lies in deciding which interactions to
            select from the metaweb. Inferring the structure of local
            networks from the metaweb before predicting local pairwise
            interactions could hold promise <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>, considering that the structure of local
            networks is constrained by the metaweb <span
            class="citation" data-cites="Saravia2022Ecological">(Saravia
            <em>et al.</em> 2022)</span>.</p>
            <p>Inferring local networks of probabilistic interactions
            from a metaweb comes with its own set of challenges. For
            example, <span class="citation"
            data-cites="Dansereau2023Spatially">Dansereau <em>et
            al.</em> (2023)</span> inferred spatially-explicit food webs
            from a metaweb of probabilistic trophic interactions between
            Canadian mammals. Their predicted localized food webs are
            downscaled versions of the metaweb (i.e., localized metawebs
            with the same interaction probabilities as those in the
            regional metaweb). To infer local networks as defined in
            this manuscript (i.e., describing local realizations of
            interactions), local interaction probabilities must be
            smaller than regional interaction probabilities. This
            decrease is due to the prerequisite that two taxa must
            initially possess the capacity to interact before engaging
            in local interactions (Eq. 2). Inferring local networks from
            a metaweb by maintaining identical interaction probability
            values introduces systematic biases into the predictions, as
            discussed in Box 1 (unless networks are seen as downscaled
            metawebs).</p>
            <p>As suggested by <span class="citation"
            data-cites="McLeod2021Sampling">McLeod <em>et al.</em>
            (2021)</span>, although metawebs do not capture the
            spatiotemporal variability of interactions, they establish
            an upper limit for local interactions <span class="citation"
            data-cites="Strydom2023Graph">(similarly for metawebs of
            probabilistic interactions, Strydom <em>et al.</em>
            2023)</span>. In other words, the probability that two taxa
            interact at a specific location and time is consistently
            lower or equal to the probability of their regional
            interaction, regardless of the conditional variables
            considered:</p>
            <p><span id="eq:switch"><span
            class="math display"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|...) ≤ <em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>).   (21)</span></span></p>
            <p>Moreover, the probability that two taxa possess the
            biological capacity to interact must be higher than the
            probability of them interacting at any location and time
            because they may never co-occur or encounter locally.
            Specifically, the cumulative probability of local
            interaction across all spatial, temporal, and environmental
            conditions must be less than the probability of regional
            interaction, i.e.</p>
            <p><span id="eq:all"><span
            class="math display">∫<sub><em>E</em><sub><em>k</em></sub></sub>∫<sub><em>A</em></sub>∫<sub><em>t</em></sub><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>E</em><sub><em>k</em></sub>,<em>A</em>,<em>t</em>) d<em>t</em> d<em>A</em> d<em>E</em><sub><em>k</em></sub> ≤ <em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|<em>T</em><sub><em>i</em></sub>,<em>T</em><sub><em>j</em></sub>).   (22)</span></span></p>
            <p>Estimating more precisely the probability <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>
            that two taxa interact locally if they can potentially
            interact allows for improved predictions of local networks
            from the metaweb of probabilistic interactions. This task is
            challenging due to the variability of this probability
            across space and time, as well as its variability across
            pairwise interactions within a network. Using simple models
            of <span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|<em>M</em><sub><em>i</em> → <em>j</em></sub>)</span>,
            as shown in Box 1, represents an initial step toward the
            overarching objective of reconstructing local networks from
            metawebs.</p>
            <h2
            id="quantifying-and-reducing-interaction-uncertainty">Quantifying
            and reducing interaction uncertainty</h2>
            <p>While sampling biological communities decreases the
            uncertainty of interactions by accumulating evidence for
            their feasibility and local realization, there is a limit to
            how much we can reduce uncertainty. In metawebs,
            probabilities reflect our limited knowledge of interactions
            (i.e., our degree of belief that interactions are feasible),
            which is expected to improve with a larger volume of data.
            Regional interactions should become more definitive (with
            probabilities approaching <span class="math inline">0</span>
            or <span class="math inline">1</span>) as we investigate
            various conditions, including different combinations of
            species traits.</p>
            <p>In comparison, although local networks can be seen as
            random instances of metawebs, their stochasticity cannot be
            reduced to the same extent. Local interaction probabilities
            may represent both their uncertainty and spatiotemporal
            variability. Owing to environmental heterogeneity, there
            will invariably be instances in which an interaction occurs
            and others in which it does not, across different times and
            locations, irrespective of the extent to which we can
            improve our knowledge of its biological feasibility and the
            local conditions that facilitate its occurrence. When local
            networks describe probabilities of observing interactions
            rather than their actual occurrence, we must also consider
            observation variability (sampling error) as an additional
            source of stochasticity. Every ecological process is
            stochastic but there is also a possibility that a phenomenon
            goes undetected. Quantifying and partitioning this
            stochasticity will enable us to make more accurate
            predictions about ecological interactions at various spatial
            and temporal scales. This will prove to be of vital
            importance as our time to understand nature runs out,
            especially at locations where the impacts of climate change
            and habitat loss hit harder.</p>
            <h2 id="relaxing-the-independence-assumption">Relaxing the
            independence assumption</h2>
            <p>Estimating local interaction probabilities independently
            for each taxa pair and assembling them into a network of
            probabilistic interactions comes with limitations.
            Predicting local networks of binary interactions based on
            these interaction probabilities assumes independence among
            interactions, a condition seldom respected in practice <span
            class="citation" data-cites="Golubski2011ModMod">(Golubski
            &amp; Abrams 2011)</span>. Relaxing this assumption is the
            next logical step in the stochastic representation of
            interactions.</p>
            <p>A more accurate representation of the stochasticity of
            ecological networks involves creating <em>probabilistic
            networks</em> (<span
            class="math inline"><em>P</em>(<em>L</em>|...)</span> and
            <span
            class="math inline"><em>P</em>(<em>M</em>|...)</span>),
            rather than networks of <em>probabilistic interactions</em>
            (<span
            class="math inline"><em>P</em>(<em>L</em><sub><em>i</em> → <em>j</em></sub>|...)</span>
            and <span
            class="math inline"><em>P</em>(<em>M</em><sub><em>i</em> → <em>j</em></sub>|...)</span>).
            Probabilistic networks describe the probability that a
            particular network of binary (or quantitative) interactions
            (its whole adjacency matrix) is realized. For example, <span
            class="citation" data-cites="Young2021Reconstructiona">Young
            <em>et al.</em> (2021)</span> used a Bayesian approach to
            estimate the probability of different plant-pollinator
            network structures derived from imperfect observational
            data. A probability distribution of ecological networks may
            also be derived using the principle of maximum entropy given
            structural constrained <span class="citation"
            data-cites="Cimini2019Statistical Park2004Statisticala">(e.g.,
            Cimini <em>et al.</em> 2019; Park &amp; Newman
            2004)</span>.</p>
            <p>Regardless of the method employed, generating
            probabilistic local networks could lead to more accurate
            predictions of local networks of binary interactions by
            bypassing the independence assumption. Probabilistic
            networks could serve as an alternative to null hypothesis
            significance testing when comparing the structure of a local
            network to some random expectations or, as done in <span
            class="citation"
            data-cites="Pellissier2018Comparing">Pellissier <em>et
            al.</em> (2018)</span>, to the metaweb. These random
            expectations are typically derived by performing a series of
            Bernoulli trials on probabilistic interactions, assuming
            independence, to generate a distribution of networks of
            binary interactions to calculate their structure <span
            class="citation" data-cites="Poisot2016Structure">(Poisot
            <em>et al.</em> 2016)</span>. One could for instance compare
            the likelihood of an observed network to the one of the most
            likely network structure (according to the probabilistic
            network distribution), thereby directly obtaining a measure
            of discrepancy of the empirical network. Generating
            probabilistic ecological networks represents a tangible
            challenge, one that, in the coming years, promises to unlock
            doors to more advanced and adequate analyses of ecological
            networks.</p>
            <h1 id="acknowledgment">Acknowledgment</h1>
            <p>We acknowledge that this study was conducted on land
            within the traditional unceded territory of the Saint
            Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat,
            and Omàmiwininiwak nations. This work was supported by the
            Institute for Data Valorisation (IVADO) and the Natural
            Sciences and Engineering Research Council of Canada (NSERC)
            Collaborative Research and Training Experience (CREATE)
            program, through the Computational Biodiversity Science and
            Services (BIOS²) program. A special thanks to all members of
            the Black Holes and Revelations working group (organized by
            BIOS²) for their insightful discussions and valuable
            feedback on this manuscript.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs"
            class="references csl-bib-body hanging-indent"
            role="doc-bibliography">
            <div id="ref-Abrego2021Accounting" class="csl-entry"
            role="doc-biblioentry">
            Abrego, N., Roslin, T., Huotari, T., Ji, Y., Schmidt, N.M.,
            Wang, J., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/ecog.05547">Accounting for
            species interactions is necessary for predicting how arctic
            arthropod communities respond to climate change</a>.
            <em>Ecography</em>, 44, 885–896.
            </div>
            <div id="ref-Angilletta2004TemGro" class="csl-entry"
            role="doc-biblioentry">
            Angilletta, M.J., Jr., Steury, T.D. &amp; Sears, M.W.
            (2004). <a
            href="https://doi.org/10.1093/icb/44.6.498">Temperature,
            Growth Rate, and Body Size in Ectotherms: Fitting Pieces of
            a Life-History Puzzle1</a>. <em>Integrative and Comparative
            Biology</em>, 44, 498–509.
            </div>
            <div id="ref-Araujo2011Usinga" class="csl-entry"
            role="doc-biblioentry">
            Araujo, M.B., Rozenfeld, A., Rahbek, C. &amp; Marquet, P.A.
            (2011). <a
            href="https://doi.org/10.1111/j.1600-0587.2011.06919.x">Using
            species co-occurrence networks to assess the impacts of
            climate change</a>. <em>Ecography</em>, 34, 897–908.
            </div>
            <div id="ref-Banasek-Richter2004Sampling" class="csl-entry"
            role="doc-biblioentry">
            Banašek-Richter, C., Cattin, M.-F. &amp; Bersier, L.-F.
            (2004). <a
            href="https://doi.org/10.1016/S0022-5193(03)00305-9">Sampling
            effects and the robustness of quantitative and qualitative
            food-web descriptors</a>. <em>Journal of Theoretical
            Biology</em>, 226, 23–32.
            </div>
            <div id="ref-Bartomeus2016ComFra" class="csl-entry"
            role="doc-biblioentry">
            Bartomeus, I., Gravel, D., Tylianakis, J.M., Aizen, M.A.,
            Dickie, I.A. &amp; Bernard-Verdier, M. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12666">A common
            framework for identifying linkage rules across different
            types of interactions</a>. <em>Functional Ecology</em>, 30,
            1894–1903.
            </div>
            <div id="ref-Bascompte2003NesAss" class="csl-entry"
            role="doc-biblioentry">
            Bascompte, J., Jordano, P., Melián, C.J. &amp; Olesen, J.M.
            (2003). <a
            href="https://doi.org/10.1073/pnas.1633576100">The nested
            assembly of plant–animal mutualistic networks</a>.
            <em>Proceedings of the National Academy of Sciences</em>,
            100, 9383–9387.
            </div>
            <div id="ref-Berg2010Traita" class="csl-entry"
            role="doc-biblioentry">
            Berg, M.P. &amp; Ellers, J. (2010). <a
            href="https://doi.org/10.1007/s10682-009-9347-8">Trait
            plasticity in species interactions: A driving force of
            community dynamics</a>. <em>Evolutionary Ecology</em>, 24,
            617–629.
            </div>
            <div id="ref-Berlow2004Interaction" class="csl-entry"
            role="doc-biblioentry">
            Berlow, E.L., Neutel, A.-M., Cohen, J.E., De Ruiter, P.C.,
            Ebenman, B., Emmerson, M., <em>et al.</em> (2004). <a
            href="https://doi.org/10.1111/j.0021-8790.2004.00833.x">Interaction
            strengths in food webs: Issues and opportunities</a>.
            <em>Journal of Animal Ecology</em>, 73, 585–598.
            </div>
            <div id="ref-Bersier1999Scale" class="csl-entry"
            role="doc-biblioentry">
            Bersier, L.-F., Dixon, P. &amp; Sugihara, G. (1999). <a
            href="https://doi.org/10.1086/303200">Scale-Invariant or
            Scale-Dependent Behavior of the Link Density Property in
            Food Webs: A Matter of Sampling Effort?</a> <em>The American
            Naturalist</em>, 153, 676–682.
            </div>
            <div id="ref-Bolnick2011WhyInt" class="csl-entry"
            role="doc-biblioentry">
            Bolnick, D.I., Amarasekare, P., Araújo, M.S., Bürger, R.,
            Levine, J.M., Novak, M., <em>et al.</em> (2011). <a
            href="https://doi.org/10.1016/j.tree.2011.01.009">Why
            intraspecific trait variation matters in community
            ecology</a>. <em>Trends in Ecology &amp; Evolution</em>, 26,
            183–192.
            </div>
            <div id="ref-Borrett2019Walk" class="csl-entry"
            role="doc-biblioentry">
            Borrett, S.R. &amp; Scharler, U.M. (2019). <a
            href="https://doi.org/10.1016/j.ecolind.2019.105451">Walk
            partitions of flow in Ecological Network Analysis: Review
            and synthesis of methods and indicators</a>. <em>Ecological
            Indicators</em>, 106, 105451.
            </div>
            <div id="ref-Boulangeat2012AccDis" class="csl-entry"
            role="doc-biblioentry">
            Boulangeat, I., Gravel, D. &amp; Thuiller, W. (2012). <a
            href="https://doi.org/10.1111/j.1461-0248.2012.01772.x">Accounting
            for dispersal and biotic interactions to disentangle the
            drivers of species distributions and their abundances</a>.
            <em>Ecology Letters</em>, 15, 584–593.
            </div>
            <div id="ref-Brimacombe2023ShoReu" class="csl-entry"
            role="doc-biblioentry">
            Brimacombe, C., Bodner, K., Michalska-Smith, M., Poisot, T.
            &amp; Fortin, M.-J. (2023). <a
            href="https://doi.org/10.1371/journal.pbio.3002068">Shortcomings
            of reusing species interaction networks created by different
            sets of researchers</a>. <em>PLOS Biology</em>, 21,
            e3002068.
            </div>
            <div id="ref-Broom2005You" class="csl-entry"
            role="doc-biblioentry">
            Broom, M. &amp; Ruxton, G.D. (2005). <a
            href="https://doi.org/10.1093/beheco/ari024">You can run—or
            you can hide: Optimal strategies for cryptic prey against
            pursuit predators</a>. <em>Behavioral Ecology</em>, 16,
            534–540.
            </div>
            <div id="ref-Brose2004UniSpa" class="csl-entry"
            role="doc-biblioentry">
            Brose, U., Ostling, A., Harrison, K. &amp; Martinez, N.D.
            (2004). <a
            href="https://doi.org/10.1038/nature02297">Unified spatial
            scaling of species and their trophic interactions</a>.
            <em>Nature</em>, 428, 167–171.
            </div>
            <div id="ref-Burnham2004Multimodel" class="csl-entry"
            role="doc-biblioentry">
            Burnham, K.P. &amp; Anderson, D.R. (2004). <a
            href="https://doi.org/10.1177/0049124104268644">Multimodel
            Inference: Understanding AIC and BIC in Model Selection</a>.
            <em>Sociological Methods &amp; Research</em>, 33, 261–304.
            </div>
            <div id="ref-Canard2014Empiricala" class="csl-entry"
            role="doc-biblioentry">
            Canard, E.F., Mouquet, N., Mouillot, D., Stanko, M.,
            Miklisova, D. &amp; Gravel, D. (2014). <a
            href="https://doi.org/10.1086/675363">Empirical Evaluation
            of Neutral Interactions in Host-Parasite Networks.</a>
            <em>The American Naturalist</em>, 183, 468–479.
            </div>
            <div id="ref-Canard2012Emergencea" class="csl-entry"
            role="doc-biblioentry">
            Canard, E., Mouquet, N., Marescot, L., Gaston, K.J., Gravel,
            D. &amp; Mouillot, D. (2012). <a
            href="https://doi.org/10.1371/journal.pone.0038295">Emergence
            of Structural Patterns in Neutral Trophic Networks</a>.
            <em>PLOS ONE</em>, 7, e38295.
            </div>
            <div id="ref-Carlson2021Future" class="csl-entry"
            role="doc-biblioentry">
            Carlson, C.J., Farrell, M.J., Grange, Z., Han, B.A.,
            Mollentze, N., Phelan, A.L., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1098/rstb.2020.0358">The future of
            zoonotic risk prediction</a>. <em>Philosophical Transactions
            of the Royal Society B: Biological Sciences</em>, 376,
            20200358.
            </div>
            <div id="ref-Caron2024Traitmatching" class="csl-entry"
            role="doc-biblioentry">
            Caron, D., Brose, U., Lurgi, M., Blanchet, F.G., Gravel, D.
            &amp; Pollock, L.J. (2024). <a
            href="https://doi.org/10.1111/geb.13807">Trait-matching
            models predict pairwise interactions across regions, not
            food web properties</a>. <em>Global Ecology and
            Biogeography</em>, 33, e13807.
            </div>
            <div id="ref-Castoe2007Phylogeographic" class="csl-entry"
            role="doc-biblioentry">
            Castoe, T.A., Spencer, C.L. &amp; Parkinson, C.L. (2007). <a
            href="https://doi.org/10.1016/j.ympev.2006.07.002">Phylogeographic
            structure and historical demography of the western
            diamondback rattlesnake (<em>Crotalus</em> <em>Atrox</em>):
            A perspective on North American desert biogeography</a>.
            <em>Molecular Phylogenetics and Evolution</em>, 42, 193–212.
            </div>
            <div id="ref-Catchen2023Missinga" class="csl-entry"
            role="doc-biblioentry">
            Catchen, M.D., Poisot, T., Pollock, L.J. &amp; Gonzalez, A.
            (2023). The missing link: Discerning true from false
            negatives when sampling species interaction networks.
            </div>
            <div id="ref-Cazelles2016Theorya" class="csl-entry"
            role="doc-biblioentry">
            Cazelles, K., Araujo, M.B., Mouquet, N. &amp; Gravel, D.
            (2016). <a
            href="https://doi.org/10.1007/s12080-015-0281-9">A theory
            for species co-occurrence in interaction networks</a>.
            <em>Theoretical Ecology</em>, 9, 39–48.
            </div>
            <div id="ref-Chagnon2015ChaTop" class="csl-entry"
            role="doc-biblioentry">
            Chagnon, P.-L. (2015). <a
            href="https://doi.org/10.1016/j.ecocom.2015.01.004">Characterizing
            topology of ecological networks along gradients: The limits
            of metrics’ standardization</a>. <em>Ecological
            Complexity</em>, 22, 36–39.
            </div>
            <div id="ref-Choh2012PreRol" class="csl-entry"
            role="doc-biblioentry">
            Choh, Y., Ignacio, M., Sabelis, M.W. &amp; Janssen, A.
            (2012). <a
            href="https://doi.org/10.1038/srep00728">Predator-prey role
            reversals, juvenile experience and adult antipredator
            behaviour</a>. <em>Scientific Reports</em>, 2, 728.
            </div>
            <div id="ref-Cimini2019Statistical" class="csl-entry"
            role="doc-biblioentry">
            Cimini, G., Squartini, T., Saracco, F., Garlaschelli, D.,
            Gabrielli, A. &amp; Caldarelli, G. (2019). <a
            href="https://doi.org/10.1038/s42254-018-0002-6">The
            statistical physics of real-world networks</a>. <em>Nature
            Reviews Physics</em>, 1, 58–71.
            </div>
            <div id="ref-Cirtwill2019QuaFra" class="csl-entry"
            role="doc-biblioentry">
            Cirtwill, A.R., Eklöf, A., Roslin, T., Wootton, K. &amp;
            Gravel, D. (2019). <a
            href="https://doi.org/10.1111/2041-210X.13180">A
            quantitative framework for investigating the reliability of
            empirical network construction</a>. <em>Methods in Ecology
            and Evolution</em>, 10, 902–911.
            </div>
            <div id="ref-Dallas2017Predictinga" class="csl-entry"
            role="doc-biblioentry">
            Dallas, T., Park, A.W. &amp; Drake, J.M. (2017). <a
            href="https://doi.org/10.1371/journal.pcbi.1005557">Predicting
            cryptic links in host-parasite networks</a>. <em>PLOS
            Computational Biology</em>, 13, e1005557.
            </div>
            <div id="ref-Dansereau2023Spatially" class="csl-entry"
            role="doc-biblioentry">
            Dansereau, G., Barros, C. &amp; Poisot, T. (2023). Spatially
            explicit predictions of food web structure from regional
            level data.
            </div>
            <div id="ref-Dunne2006Network" class="csl-entry"
            role="doc-biblioentry">
            Dunne, J.A. (2006). The Network Structure of Food Webs. In:
            <em>Ecological networks: Linking structure and dynamics</em>
            (eds. Dunne, J.A. &amp; Pascual, M.). Oxford University
            Press, pp. 27–86.
            </div>
            <div id="ref-Elmasri2020HieBay" class="csl-entry"
            role="doc-biblioentry">
            Elmasri, M., Farrell, M.J., Davies, T.J. &amp; Stephens,
            D.A. (2020). <a href="https://doi.org/10.1214/19-AOAS1296">A
            hierarchical Bayesian model for predicting ecological
            interactions using scaled evolutionary relationships</a>.
            <em>The Annals of Applied Statistics</em>, 14, 221–240.
            </div>
            <div id="ref-Elton2001Animal" class="csl-entry"
            role="doc-biblioentry">
            Elton, C.S. (2001). <em>Animal Ecology</em>. University of
            Chicago Press, Chicago, IL.
            </div>
            <div id="ref-Emmerson2004Predatora" class="csl-entry"
            role="doc-biblioentry">
            Emmerson, M.C. &amp; Raffaelli, D. (2004). <a
            href="https://doi.org/10.1111/j.0021-8790.2004.00818.x">Predator–prey
            body size, interaction strength and the stability of a real
            food web</a>. <em>Journal of Animal Ecology</em>, 73,
            399–409.
            </div>
            <div id="ref-Fedorov2008Comparative" class="csl-entry"
            role="doc-biblioentry">
            Fedorov, V.B., Goropashnaya, A.V., Boeskorov, G.G. &amp;
            Cook, J.A. (2008). <a
            href="https://doi.org/10.1111/j.1365-294X.2007.03595.x">Comparative
            phylogeography and demographic history of the wood lemming
            (Myopus schisticolor): Implications for late Quaternary
            history of the taiga species in Eurasia</a>. <em>Molecular
            Ecology</em>, 17, 598–610.
            </div>
            <div id="ref-Fortuna2006HabLos" class="csl-entry"
            role="doc-biblioentry">
            Fortuna, M.A. &amp; Bascompte, J. (2006). <a
            href="https://doi.org/10.1111/j.1461-0248.2005.00868.x">Habitat
            loss and the structure of plant–animal mutualistic
            networks</a>. <em>Ecology Letters</em>, 9, 281–286.
            </div>
            <div id="ref-Fu2021Link" class="csl-entry"
            role="doc-biblioentry">
            Fu, X., Seo, E., Clarke, J. &amp; Hutchinson, R.A. (2021).
            <a href="https://doi.org/10.1109/TKDE.2019.2962031">Link
            Prediction Under Imperfect Detection: Collaborative
            Filtering for Ecological Networks</a>. <em>IEEE Transactions
            on Knowledge and Data Engineering</em>, 33, 3117–3128.
            </div>
            <div id="ref-Galiana2018Spatiala" class="csl-entry"
            role="doc-biblioentry">
            Galiana, N., Lurgi, M., Claramunt-López, B., Fortin, M.-J.,
            Leroux, S., Cazelles, K., <em>et al.</em> (2018). <a
            href="https://doi.org/10.1038/s41559-018-0517-3">The spatial
            scaling of species interaction networks</a>. <em>Nature
            Ecology &amp; Evolution</em>, 2, 782–790.
            </div>
            <div id="ref-Gilman2010FraCom" class="csl-entry"
            role="doc-biblioentry">
            Gilman, S.E., Urban, M.C., Tewksbury, J., Gilchrist, G.W.
            &amp; Holt, R.D. (2010). <a
            href="https://doi.org/10.1016/j.tree.2010.03.002">A
            framework for community interactions under climate
            change</a>. <em>Trends in Ecology &amp; Evolution</em>, 25,
            325–331.
            </div>
            <div id="ref-Godsoe2022Species" class="csl-entry"
            role="doc-biblioentry">
            Godsoe, W., Murray, R. &amp; Iritani, R. (2022). <a
            href="https://doi.org/10.1111/oik.09282">Species
            interactions and diversity: A unified framework using Hill
            numbers</a>. <em>Oikos</em>, n/a, e09282.
            </div>
            <div id="ref-Golubski2011ModMod" class="csl-entry"
            role="doc-biblioentry">
            Golubski, A.J. &amp; Abrams, P.A. (2011). <a
            href="https://doi.org/10.1111/j.1365-2656.2011.01852.x">Modifying
            modifiers: What happens when interspecific interactions
            interact?</a> <em>Journal of Animal Ecology</em>, 80,
            1097–1108.
            </div>
            <div id="ref-Gomez2010EcoInt" class="csl-entry"
            role="doc-biblioentry">
            Gomez, J.M., Verdu, M. &amp; Perfectti, F. (2010). <a
            href="https://doi.org/10.1038/nature09113">Ecological
            interactions are evolutionarily conserved across the entire
            tree of life</a>. <em>Nature</em>, 465, 918–U6.
            </div>
            <div id="ref-Gonzalez2022Monitor" class="csl-entry"
            role="doc-biblioentry">
            Gonzalez, A. &amp; Londoño, M.C. (2022). <a
            href="https://doi.org/10.1126/science.adg1506">Monitor
            biodiversity for action</a>. <em>Science</em>, 378,
            1147–1147.
            </div>
            <div id="ref-Gonzalez-Varo2016Labilea" class="csl-entry"
            role="doc-biblioentry">
            Gonzalez-Varo, J.P. &amp; Traveset, A. (2016). <a
            href="https://doi.org/10.1016/j.tree.2016.06.009">The Labile
            Limits of Forbidden Interactions</a>. <em>Trends in Ecology
            &amp; Evolution</em>, 31, 700–710.
            </div>
            <div id="ref-Gravel2019BriElt" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Baiser, B., Dunne, J.A., Kopelke, J.-P.,
            Martinez, N.D., Nyman, T., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/ecog.04006">Bringing Elton and
            Grinnell together: A quantitative framework to represent the
            biogeography of ecological interaction networks</a>.
            <em>Ecography</em>, 42, 401–415.
            </div>
            <div id="ref-Gravel2011Trophic" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Massol, F., Canard, E., Mouillot, D. &amp;
            Mouquet, N. (2011). <a
            href="https://doi.org/10.1111/j.1461-0248.2011.01667.x">Trophic
            theory of island biogeography</a>. <em>Ecology Letters</em>,
            14, 1010–1016.
            </div>
            <div id="ref-Gravel2013InfFooa" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Poisot, T., Albouy, C., Velez, L. &amp;
            Mouillot, D. (2013). <a
            href="https://doi.org/10.1111/2041-210X.12103">Inferring
            food web structure from predator–prey body size
            relationships</a>. <em>Methods in Ecology and
            Evolution</em>, 4, 1083–1090.
            </div>
            <div id="ref-Guimaraes2020Structurea" class="csl-entry"
            role="doc-biblioentry">
            Guimarães, P.R. (2020). <a
            href="https://doi.org/10.1146/annurev-ecolsys-012220-120819">The
            Structure of Ecological Networks Across Levels of
            Organization</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 51, 433–460.
            </div>
            <div id="ref-Guimera2009MisSpu" class="csl-entry"
            role="doc-biblioentry">
            Guimerà, R. &amp; Sales-Pardo, M. (2009). <a
            href="https://doi.org/10.1073/pnas.0908366106">Missing and
            spurious interactions and the reconstruction of complex
            networks</a>. <em>Proceedings of the National Academy of
            Sciences</em>, 106, 22073–22078.
            </div>
            <div id="ref-Hemprich-Bennett2021AssImp" class="csl-entry"
            role="doc-biblioentry">
            Hemprich-Bennett, D.R., Oliveira, H.F.M., Le Comber, S.C.,
            Rossiter, S.J. &amp; Clare, E.L. (2021). <a
            href="https://doi.org/10.1002/ecy.3256">Assessing the impact
            of taxon resolution on network structure</a>.
            <em>Ecology</em>, 102, e03256.
            </div>
            <div id="ref-Herrera1989PolAbu" class="csl-entry"
            role="doc-biblioentry">
            Herrera, C.M. (1989). <a
            href="https://doi.org/10.1007/BF00380158">Pollinator
            abundance, morphology, and flower visitation rate: Analysis
            of the <span>“quantity”</span> component in a
            plant-pollinator system</a>. <em>Oecologia</em>, 80,
            241–248.
            </div>
            <div id="ref-Hortal2015SevSho" class="csl-entry"
            role="doc-biblioentry">
            Hortal, J., de Bello, F., Diniz-Filho, J.A.F., Lewinsohn,
            T.M., Lobo, J.M. &amp; Ladle, R.J. (2015). <a
            href="https://doi.org/10.1146/annurev-ecolsys-112414-054400">Seven
            Shortfalls that Beset Large-Scale Knowledge of
            Biodiversity</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 46, 523–549.
            </div>
            <div id="ref-Jordano1987PatMut" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P. (1987). <a
            href="https://doi.org/10.1086/284665">Patterns of
            Mutualistic Interactions in Pollination and Seed Dispersal:
            Connectance, Dependence Asymmetries, and Coevolution</a>.
            <em>The American Naturalist</em>, 129, 657–677.
            </div>
            <div id="ref-Jordano2016Samplingb" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12763">Sampling
            networks of ecological interactions</a>. <em>Functional
            Ecology</em>, 30, 1883–1893.
            </div>
            <div id="ref-Jordano2003Invarianta" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P., Bascompte, J. &amp; Olesen, J.M. (2003). <a
            href="https://doi.org/10.1046/j.1461-0248.2003.00403.x">Invariant
            properties in coevolutionary networks of plant–animal
            interactions</a>. <em>Ecology Letters</em>, 6, 69–81.
            </div>
            <div id="ref-Kausrud2008Linking" class="csl-entry"
            role="doc-biblioentry">
            Kausrud, K.L., Mysterud, A., Steen, H., Vik, J.O., Østbye,
            E., Cazelles, B., <em>et al.</em> (2008). <a
            href="https://doi.org/10.1038/nature07442">Linking climate
            change to lemming cycles</a>. <em>Nature</em>, 456, 93–97.
            </div>
            <div id="ref-Kefi2015NetStr" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Berlow, E.L., Wieters, E.A., Joppa, L.N., Wood,
            S.A., Brose, U., <em>et al.</em> (2015). <a
            href="https://doi.org/10.1890/13-1424.1">Network structure
            beyond food webs: Mapping non-trophic and trophic
            interactions on Chilean rocky shores</a>. <em>Ecology</em>,
            96, 291–303.
            </div>
            <div id="ref-Kefi2012MorMea" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Berlow, E.L., Wieters, E.A., Navarrete, S.A.,
            Petchey, O.L., Wood, S.A., <em>et al.</em> (2012). <a
            href="https://doi.org/10.1111/j.1461-0248.2011.01732.x">More
            than a meal… integrating non-feeding interactions into food
            webs</a>. <em>Ecology Letters</em>, 15, 291–300.
            </div>
            <div id="ref-Kefi2016HowStr" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Miele, V., Wieters, E.A., Navarrete, S.A. &amp;
            Berlow, E.L. (2016). <a
            href="https://doi.org/10.1371/journal.pbio.1002527">How
            Structured Is the Entangled Bank? The Surprisingly Simple
            Organization of Multiplex Ecological Networks Leads to
            Increased Persistence and Resilience</a>. <em>PLOS
            Biology</em>, 14, e1002527.
            </div>
            <div id="ref-Kissner1997Rattling" class="csl-entry"
            role="doc-biblioentry">
            Kissner, K.J., Forbes, M.R. &amp; Secoy, D.M. (1997). <a
            href="https://doi.org/10.1111/j.1439-0310.1997.tb00146.x">Rattling
            Behavior of Prairie Rattlesnakes (Crotalus viridis viridis,
            Viperidae) in Relation to Sex, Reproductive Status, Body
            Size, and Body Temperature</a>. <em>Ethology</em>, 103,
            1042–1050.
            </div>
            <div id="ref-Klecka2014EffHab" class="csl-entry"
            role="doc-biblioentry">
            Klecka, J. &amp; Boukal, D.S. (2014). <a
            href="https://doi.org/10.1007/s00442-014-3007-6">The effect
            of habitat structure on prey mortality depends on predator
            and prey microhabitat use</a>. <em>Oecologia</em>, 176,
            183–191.
            </div>
            <div id="ref-Kopelke2017FooStra" class="csl-entry"
            role="doc-biblioentry">
            Kopelke, J.-P., Nyman, T., Cazelles, K., Gravel, D.,
            Vissault, S. &amp; Roslin, T. (2017). <a
            href="https://doi.org/10.1002/ecy.1832">Food-web structure
            of willow-galling sawflies and their natural enemies across
            Europe</a>. <em>Ecology</em>, 98, 1730–1730.
            </div>
            <div id="ref-Kulkarni2006EntEva" class="csl-entry"
            role="doc-biblioentry">
            Kulkarni, M.A., Kweka, E., Nyale, E., Lyatuu, E., Mosha,
            F.W., Chandramohan, D., <em>et al.</em> (2006). <a
            href="https://doi.org/10.1093/jmedent/43.3.580">Entomological
            Evaluation of Malaria Vectors at Different Altitudes in Hai
            District, Northeastern Tanzania</a>. <em>Journal of Medical
            Entomology</em>, 43, 580–588.
            </div>
            <div id="ref-Maiorano2020Tetraeua" class="csl-entry"
            role="doc-biblioentry">
            Maiorano, L., Montemaggiori, A., Ficetola, G.F., O’Connor,
            L. &amp; Thuiller, W. (2020). <a
            href="https://doi.org/10.1111/geb.13138">TETRA-EU 1.0: A
            species-level trophic metaweb of European tetrapods</a>.
            <em>Global Ecology and Biogeography</em>, 29, 1452–1457.
            </div>
            <div id="ref-McCann2007ProBio" class="csl-entry"
            role="doc-biblioentry">
            McCann, K. (2007). Protecting biostructure. <em>Nature</em>,
            446, 29–29.
            </div>
            <div id="ref-McCann2011FooWeb" class="csl-entry"
            role="doc-biblioentry">
            McCann, K.S. (2011). Food webs (MPB-50). In: <em>Food Webs
            (MPB-50)</em>. Princeton University Press.
            </div>
            <div id="ref-McLeod2021Sampling" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A., Leroux, S.J., Gravel, D., Chu, C., Cirtwill,
            A.R., Fortin, M.-J., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/oik.08650">Sampling and
            asymptotic network properties of spatial multi-trophic
            networks</a>. <em>Oikos</em>, n/a.
            </div>
            <div id="ref-McLeod2020EffSpe" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A.M., Leroux, S.J. &amp; Chu, C. (2020). <a
            href="https://doi.org/10.1002/ecs2.3018">Effects of species
            traits, motif profiles, and environment on spatial variation
            in multi-trophic antagonistic networks</a>.
            <em>Ecosphere</em>, 11, e03018.
            </div>
            <div id="ref-Melian2011EcoDyn" class="csl-entry"
            role="doc-biblioentry">
            Melián, C.J., Vilas, C., Baldó, F., González-Ortegón, E.,
            Drake, P. &amp; Williams, R.J. (2011). <a
            href="https://doi.org/10.1016/B978-0-12-386475-8.00006-X">Eco-evolutionary
            Dynamics of Individual-Based Food Webs</a>. In: <em>Advances
            in Ecological Research</em>, The Role of Body Size in
            Multispecies Systems (ed. Belgrano, A.). Academic Press, pp.
            225–268.
            </div>
            <div id="ref-Morales-Castilla2015Inferringa"
            class="csl-entry" role="doc-biblioentry">
            Morales-Castilla, I., Matias, M.G., Gravel, D. &amp; Araújo,
            M.B. (2015). <a
            href="https://doi.org/10.1016/j.tree.2015.03.014">Inferring
            biotic interactions from proxies</a>. <em>Trends in Ecology
            &amp; Evolution</em>, 30, 347–356.
            </div>
            <div id="ref-Ohlmann2019Diversity" class="csl-entry"
            role="doc-biblioentry">
            Ohlmann, M., Miele, V., Dray, S., Chalmandrier, L.,
            O’Connor, L. &amp; Thuiller, W. (2019). <a
            href="https://doi.org/10.1111/ele.13221">Diversity indices
            for ecological networks: A unifying framework using Hill
            numbers</a>. <em>Ecology Letters</em>, 22, 737–747.
            </div>
            <div id="ref-Olesen2010Missing" class="csl-entry"
            role="doc-biblioentry">
            Olesen, J.M., Bascompte, J., Dupont, Y.L., Elberling, H.,
            Rasmussen, C. &amp; Jordano, P. (2010). <a
            href="https://doi.org/10.1098/rspb.2010.1371">Missing and
            forbidden links in mutualistic networks</a>. <em>Proceedings
            of the Royal Society B: Biological Sciences</em>, 278,
            725–732.
            </div>
            <div id="ref-Park2004Statisticala" class="csl-entry"
            role="doc-biblioentry">
            Park, J. &amp; Newman, M.E.J. (2004). <a
            href="https://doi.org/10.1103/PhysRevE.70.066117">Statistical
            mechanics of networks</a>. <em>Physical Review E</em>, 70,
            066117.
            </div>
            <div id="ref-Pascual2006Ecologicala" class="csl-entry"
            role="doc-biblioentry">
            Pascual, M., Dunne, J.A. &amp; Dunne, J.A. (2006).
            <em>Ecological Networks: Linking Structure to Dynamics in
            Food Webs</em>. Oxford University Press, USA.
            </div>
            <div id="ref-Pellissier2018Comparing" class="csl-entry"
            role="doc-biblioentry">
            Pellissier, L., Albouy, C., Bascompte, J., Farwig, N.,
            Graham, C., Loreau, M., <em>et al.</em> (2018). <a
            href="https://doi.org/10.1111/brv.12366">Comparing species
            interaction networks along environmental gradients</a>.
            <em>Biological Reviews</em>, 93, 785–800.
            </div>
            <div id="ref-Petchey2008SizFor" class="csl-entry"
            role="doc-biblioentry">
            Petchey, O.L., Beckerman, A.P., Riede, J.O. &amp; Warren,
            P.H. (2008). <a
            href="https://doi.org/10.1073/pnas.0710672105">Size,
            foraging, and food web structure</a>. <em>Proceedings of the
            National Academy of Sciences</em>, 105, 4191–4196.
            </div>
            <div id="ref-Pilosof2017MulNat" class="csl-entry"
            role="doc-biblioentry">
            Pilosof, S., Porter, M.A., Pascual, M. &amp; Kéfi, S.
            (2017). <a
            href="https://doi.org/10.1038/s41559-017-0101">The
            multilayer nature of ecological networks</a>. <em>Nature
            Ecology &amp; Evolution</em>, 1, 1–9.
            </div>
            <div id="ref-Plaganyi2004Criticala" class="csl-entry"
            role="doc-biblioentry">
            Plagányi, É.E. &amp; Butterworth, D.S. (2004). <a
            href="https://doi.org/10.2989/18142320409504061">A critical
            look at the potential of Ecopath with ecosim to assist in
            practical fisheries management</a>. <em>African Journal of
            Marine Science</em>, 26, 261–287.
            </div>
            <div id="ref-Poisot2012Dissimilaritya" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Canard, E., Mouillot, D., Mouquet, N. &amp;
            Gravel, D. (2012). <a
            href="https://doi.org/10.1111/ele.12002">The dissimilarity
            of species interaction networks</a>. <em>Ecology
            Letters</em>, 15, 1353–1361.
            </div>
            <div id="ref-Poisot2016Structure" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D.,
            Fortin, M.-J. &amp; Stouffer, D.B. (2016). <a
            href="https://doi.org/10.1111/2041-210X.12468">The structure
            of probabilistic networks</a>. <em>Methods in Ecology and
            Evolution</em>, 7, 303–312.
            </div>
            <div id="ref-Poisot2014WheEco" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T. &amp; Gravel, D. (2014). <a
            href="https://doi.org/10.7717/peerj.251">When is an
            ecological network complex? Connectance drives degree
            distribution and emerging network properties</a>.
            <em>PeerJ</em>, 2, e251.
            </div>
            <div id="ref-Poisot2015Speciesa" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Stouffer, D.B. &amp; Gravel, D. (2015). <a
            href="https://doi.org/10.1111/oik.01719">Beyond species: Why
            ecological interaction networks vary through space and
            time</a>. <em>Oikos</em>, 124, 243–251.
            </div>
            <div id="ref-Polis1991Complex" class="csl-entry"
            role="doc-biblioentry">
            Polis, G.A. (1991). <a
            href="https://www.jstor.org/stable/2462536">Complex Trophic
            Interactions in Deserts: An Empirical Critique of Food-Web
            Theory</a>. <em>The American Naturalist</em>, 138, 123–155.
            </div>
            <div id="ref-Pollock2014UndCoo" class="csl-entry"
            role="doc-biblioentry">
            Pollock, L.J., Tingley, R., Morris, W.K., Golding, N.,
            O’Hara, R.B., Parris, K.M., <em>et al.</em> (2014). <a
            href="https://doi.org/10.1111/2041-210X.12180">Understanding
            co-occurrence bymodelling species simultaneously with a
            Joint Species DistributionModel (JSDM)</a>. <em>Methods in
            Ecology and Evolution</em>, 5, 397–406.
            </div>
            <div id="ref-Pomeranz2019InfPre" class="csl-entry"
            role="doc-biblioentry">
            Pomeranz, J.P.F., Thompson, R.M., Poisot, T. &amp; Harding,
            J.S. (2019). <a
            href="https://doi.org/10.1111/2041-210X.13125">Inferring
            predator–prey interactions in food webs</a>. <em>Methods in
            Ecology and Evolution</em>, 10, 356–367.
            </div>
            <div id="ref-Pringle2020Resolving" class="csl-entry"
            role="doc-biblioentry">
            Pringle, R.M. &amp; Hutchinson, M.C. (2020). <a
            href="https://doi.org/10.1146/annurev-ecolsys-110218-024908">Resolving
            Food-Web Structure</a>. <em>Annual Review of Ecology,
            Evolution, and Systematics</em>, 51, 55–80.
            </div>
            <div id="ref-Proulx2005Networka" class="csl-entry"
            role="doc-biblioentry">
            Proulx, S.R., Promislow, D.E.L. &amp; Phillips, P.C. (2005).
            <a href="https://doi.org/10.1016/j.tree.2005.04.004">Network
            thinking in ecology and evolution</a>. <em>Trends in Ecology
            &amp; Evolution</em>, SPECIAL ISSUE: BUMPER BOOK REVIEW, 20,
            345–353.
            </div>
            <div id="ref-Pulliam1974Theory" class="csl-entry"
            role="doc-biblioentry">
            Pulliam, H.R. (1974). <a
            href="https://doi.org/10.1086/282885">On the Theory of
            Optimal Diets</a>. <em>The American Naturalist</em>, 108,
            59–74.
            </div>
            <div id="ref-Rooney2012IntFoo" class="csl-entry"
            role="doc-biblioentry">
            Rooney, N. &amp; McCann, K.S. (2012). <a
            href="https://doi.org/10.1016/j.tree.2011.09.001">Integrating
            food web diversity, structure and stability</a>. <em>Trends
            in Ecology &amp; Evolution</em>, 27, 40–46.
            </div>
            <div id="ref-Salim2022Data" class="csl-entry"
            role="doc-biblioentry">
            Salim, J.A., Saraiva, A.M., Zermoglio, P.F., Agostini, K.,
            Wolowski, M., Drucker, D.P., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1093/gigascience/giac043">Data
            standardization of plant–pollinator interactions</a>.
            <em>GigaScience</em>, 11, giac043.
            </div>
            <div id="ref-Sanders2012Indirect" class="csl-entry"
            role="doc-biblioentry">
            Sanders, D. &amp; van Veen, F.J.F. (2012). <a
            href="https://doi.org/10.1098/rsbl.2012.0572">Indirect
            commensalism promotes persistence of secondary consumer
            species</a>. <em>Biology Letters</em>, 8, 960–963.
            </div>
            <div id="ref-Saravia2022Ecological" class="csl-entry"
            role="doc-biblioentry">
            Saravia, L.A., Marina, T.I., Kristensen, N.P., De Troch, M.
            &amp; Momo, F.R. (2022). <a
            href="https://doi.org/10.1111/1365-2656.13652">Ecological
            network assembly: How the regional metaweb influences local
            food webs</a>. <em>Journal of Animal Ecology</em>, 91,
            630–642.
            </div>
            <div id="ref-Schmeller2015GloTer" class="csl-entry"
            role="doc-biblioentry">
            Schmeller, D.S., Julliard, R., Bellingham, P.J., Böhm, M.,
            Brummitt, N., Chiarucci, A., <em>et al.</em> (2015). <a
            href="https://doi.org/10.1016/j.jnc.2015.03.003">Towards a
            global terrestrial species monitoring program</a>.
            <em>Journal for Nature Conservation</em>, 25, 51–57.
            </div>
            <div id="ref-Shaw2024Framework" class="csl-entry"
            role="doc-biblioentry">
            Shaw, J.O., Dunhill, A.M., Beckerman, A.P., Dunne, J.A.
            &amp; Hull, P.M. (2024). <a
            href="https://doi.org/10.1101/2024.01.30.578036">A framework
            for reconstructing ancient food webs using functional trait
            data</a>.
            </div>
            <div id="ref-Singer2012GeoMos" class="csl-entry"
            role="doc-biblioentry">
            Singer, M.C. &amp; McBride, C.S. (2012). <a
            href="https://doi.org/10.1890/11-2078.1">Geographic mosaics
            of species’ association: A definition and an example driven
            by plant–insect phenological synchrony</a>.
            <em>Ecology</em>, 93, 2658–2673.
            </div>
            <div id="ref-Staniczenko2017Linking" class="csl-entry"
            role="doc-biblioentry">
            Staniczenko, P.P.A., Sivasubramaniam, P., Suttle, K.B. &amp;
            Pearson, R.G. (2017). <a
            href="https://doi.org/10.1111/ele.12770">Linking
            macroecology and community ecology: Refining predictions of
            species distributions using biotic interaction networks</a>.
            <em>Ecology Letters</em>, 20, 693–707.
            </div>
            <div id="ref-Stouffer2011RolBod" class="csl-entry"
            role="doc-biblioentry">
            Stouffer, D.B., Rezende, E.L. &amp; Amaral, L.A.N. (2011).
            <a
            href="https://doi.org/10.1111/j.1365-2656.2011.01812.x">The
            role of body mass in diet contiguity and food-web
            structure</a>. <em>Journal of Animal Ecology</em>, 80,
            632–639.
            </div>
            <div id="ref-Strydom2022Food" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron,
            D., Farrell, M.J., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1111/2041-210X.13835">Food web
            reconstruction through phylogenetic transfer of low-rank
            network representation</a>. <em>Methods in Ecology and
            Evolution</em>, 13.
            </div>
            <div id="ref-Strydom2023Graph" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron,
            D., Farrell, M.J., <em>et al.</em> (2023). <a
            href="https://doi.org/10.1111/2041-210X.14228">Graph
            embedding and transfer learning can help predict potential
            species interaction networks despite data limitations</a>.
            <em>Methods in Ecology and Evolution</em>, 14, 2917–2930.
            </div>
            <div id="ref-Strydom2021Roadmapa" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Catchen, M.D., Banville, F., Caron, D.,
            Dansereau, G., Desjardins-Proulx, P., <em>et al.</em>
            (2021). <a href="https://doi.org/10.1098/rstb.2021.0063">A
            roadmap towards predicting species interaction networks
            (across space and time)</a>. <em>Philosophical Transactions
            of the Royal Society B-Biological Sciences</em>, 376,
            20210063.
            </div>
            <div id="ref-Thompson2005Foodweb" class="csl-entry"
            role="doc-biblioentry">
            Thompson, R.M. &amp; Townsend, C.R. (2005). <a
            href="https://doi.org/10.1890/04-1352">Food-Web Topology
            Varies with Spatial Scale in a Patchy Environment</a>.
            <em>Ecology</em>, 86, 1916–1925.
            </div>
            <div id="ref-Tylianakis2007HabMod" class="csl-entry"
            role="doc-biblioentry">
            Tylianakis, J.M., Tscharntke, T. &amp; Lewis, O.T. (2007).
            <a href="https://doi.org/10.1038/nature05429">Habitat
            modification alters the structure of tropical
            host–parasitoid food webs</a>. <em>Nature</em>, 445,
            202–205.
            </div>
            <div id="ref-Vazquez2007Species" class="csl-entry"
            role="doc-biblioentry">
            Vázquez, D.P., Melián, C.J., Williams, N.M., Blüthgen, N.,
            Krasnov, B.R. &amp; Poulin, R. (2007). <a
            href="https://doi.org/10.1111/j.0030-1299.2007.15828.x">Species
            abundance and asymmetric interaction strength in ecological
            networks</a>. <em>Oikos</em>, 116, 1120–1127.
            </div>
            <div id="ref-VazquezSS2022Ecological" class="csl-entry"
            role="doc-biblioentry">
            Vázquez, D.P., Peralta, G., Cagnolo, L. &amp; Santos, M.
            (2022). <a
            href="https://doi.org/10.25260/EA.22.32.2.1.1846">Ecological
            interaction networks. What we know, what we don’t, and why
            it matters</a>. <em>Ecología Austral</em>, 32, 670–697.
            </div>
            <div id="ref-Wagenmakers2004Aic" class="csl-entry"
            role="doc-biblioentry">
            Wagenmakers, E.-J. &amp; Farrell, S. (2004). <a
            href="https://doi.org/10.3758/BF03206482">AIC model
            selection using Akaike weights</a>. <em>Psychonomic Bulletin
            &amp; Review</em>, 11, 192–196.
            </div>
            <div id="ref-Weinstein2017ComTra" class="csl-entry"
            role="doc-biblioentry">
            Weinstein, B.G. &amp; Graham, C.H. (2017a). <a
            href="https://doi.org/10.1016/j.fooweb.2017.05.002">On
            comparing traits and abundance for predicting species
            interactions with imperfect detection</a>. <em>Food
            Webs</em>, 11, 17–25.
            </div>
            <div id="ref-Weinstein2017PerBil" class="csl-entry"
            role="doc-biblioentry">
            Weinstein, B.G. &amp; Graham, C.H. (2017b). <a
            href="https://doi.org/10.1111/ele.12730">Persistent bill and
            corolla matching despite shifting temporal resources in
            tropical hummingbird-plant interactions</a>. <em>Ecology
            Letters</em>, 20, 326–335.
            </div>
            <div id="ref-Whittaker1960Vegetation" class="csl-entry"
            role="doc-biblioentry">
            Whittaker, R.H. (1960). <a
            href="https://doi.org/10.2307/1943563">Vegetation of the
            Siskiyou Mountains, Oregon and California</a>.
            <em>Ecological Monographs</em>, 30, 279–338.
            </div>
            <div id="ref-Wood2015Effects" class="csl-entry"
            role="doc-biblioentry">
            Wood, S.A., Russell, R., Hanson, D., Williams, R.J. &amp;
            Dunne, J.A. (2015). <a
            href="https://doi.org/10.1002/ece3.1640">Effects of spatial
            scale of sampling on food web structure</a>. <em>Ecology and
            Evolution</em>, 5, 3769–3782.
            </div>
            <div id="ref-Woodward2010ChaEco" class="csl-entry"
            role="doc-biblioentry">
            Woodward, G., Benstead, J.P., Beveridge, O.S., Blanchard,
            J., Brey, T., Brown, L.E., <em>et al.</em> (2010). <a
            href="https://doi.org/10.1016/B978-0-12-381363-3.00002-2">Chapter
            2 - Ecological Networks in a Changing Climate</a>. In:
            <em>Advances in Ecological Research</em>, Ecological
            Networks (ed. Woodward, G.). Academic Press, pp. 71–138.
            </div>
            <div id="ref-Woodward2012CliCha" class="csl-entry"
            role="doc-biblioentry">
            Woodward, G., Brown, L.E., Edwards, F.K., Hudson, L.N.,
            Milner, A.M., Reuman, D.C., <em>et al.</em> (2012). <a
            href="https://doi.org/10.1098/rstb.2012.0245">Climate change
            impacts in multispecies systems: Drought alters food web
            size structure in a field experiment</a>. <em>Philosophical
            Transactions of the Royal Society B: Biological
            Sciences</em>, 367, 2990–2997.
            </div>
            <div id="ref-Young2021Reconstructiona" class="csl-entry"
            role="doc-biblioentry">
            Young, J.-G., Valdovinos, F.S. &amp; Newman, M.E.J. (2021).
            <a
            href="https://doi.org/10.1038/s41467-021-24149-x">Reconstruction
            of plant–pollinator networks from observational data</a>.
            <em>Nature Communications</em>, 12, 3911.
            </div>
            </div>
        </article>
    </main>

    <footer>
                <div class="container">
            <h2>Supplementary Material</h2>
            <span class="appendix--list">
                <a href="appendix/Banville_et_al_2024_deciphering_probabilistic_s1.pdf" target="_blank">
                    <b>1:</b>&nbsp;Supplementary Material
                </a><br />
            </span>
        </div>
                <div class="container">
            <h2>Affiliations</h2>
            <sup>1</sup>&nbsp;Université de
            Montréal; <sup>2</sup>&nbsp;Université de
            Sherbrooke; <sup>3</sup>&nbsp;Quebec Centre for Biodiversity
            Science; <sup>4</sup>&nbsp;University of
            Sheffield; <sup>5</sup>&nbsp;University of
            Toronto; <sup>6</sup>&nbsp;McGill University
            <br />
            
        </div>
    </footer>

</body>

</html>