<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>The ecological interpretation of probabilistic
networks</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);

            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        html {
            background-color: white;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article>p {
            line-height: 135%;
        }

        article>h1 {
            padding-top: 3em;
        }

        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }

        a:hover {
            font-weight: 500;
        }

        .citation {
            color: var(--citation);
        }

        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }

        header>.container {
            margin-top: 4em;
        }

        code,
        pre {
            font-family: "Roboto Mono", monospace;
        }

        pre {
            white-space: pre-wrap;
        }

        h1,
        h2,
        h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }

        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }

        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }

        .header--authors>span {
            color: var(--light);
            font-weight: 400;
        }

        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }

        footer {
            background-color: var(--grey);
            font-weight: 300;
        }

        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }

        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }

        figure>img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }

        figure>figcaption {
            padding: var(--inner-space);
        }

        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }

        table>caption {
            padding: var(--inner-space);
        }

        figcaption,
        caption {
            color: var(--light);
            font-weight: 300;
        }

        thead {
            color: var(--structure);
        }

        td,
        th {
            padding: 3px;
        }

        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }

        tbody>tr:nth-child(even) {
            background: var(--grey)
        }

        ;

        tbody {
            border-bottom: 1px solid var(--structure);
        }

        #refs {
            font-size: 90%;
            column-count: 2;
        }

        #refs>.csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }

        .hanging-indent>.csl-entry {
            padding-left: 15px;
            text-indent: -15px;
        }

        .artifacts {
            text-align: center;
            border-radius: 4px;
            background-color: var(--grey);
            padding: 1em;
            margin-top: 4em;
        }

        .artifacts a{
            background-color: var(--structure);
            text-decoration: none;
            margin: 0px 2em;
            color: white;
            text-transform: capitalize;
            border-radius: 4px;
            text-align: center;
            padding: 4px 4px;
            width: 7em;
            display: inline-block;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.css"
        integrity="sha384-vKruj+a13U8yHIkAyGgK1J3ArTLzrFGBbBc0tDp4ad/EyewESeXE/Iv67Aj8gKZ0" crossorigin="anonymous">
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.js"
        integrity="sha384-PwRUT/YqbnEjkZO0zZxNqcxACrXe+j766U2amXcgMg5457rve2Y7I6ZJSm2A0mS4"
        crossorigin="anonymous"></script>
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/contrib/auto-render.min.js"
        integrity="sha384-+VBxd3r6XgURycqtZ117nYw44OOcIax56Z4dCRWbxyPt0Koah1uHoK0o4+/RRE05" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>

    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                    { left: "$$", right: "$$", display: true },
                    { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">The ecological interpretation of
probabilistic networks</div>
            <div class="header--authors">
                <span class="header--author">
                <a href="https://orcid.org/0000-0001-9051-0597" target="_blank">
                    Francis Banville
                </a>
                <sup class="affiliation">1,2,3,‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0001-6067-1349" target="_blank">
                    Tanya Strydom
                </a>
                <sup class="affiliation">1,3,‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affiliation">1,3</sup>
                </span>
            </div>
        </div>
    </header>

    <main class="container">
        <section class="abstract">
            The stochastic nature of ecological interactions has led
            many biologists to adopt a probabilistic view of ecological
            networks. Representing species interactions
            probabilistically (how likely are they to interact?) as
            opposed to deterministically (do they interact?) allows a
            better assessment of their spatiotemporal variability and
            accounts for inherent uncertainties in observations and
            predictions. However, despite this growing interest in
            probabilistic networks, general guidelines regarding the
            estimation and documentation of probabilistic interaction
            data are still lacking. This is concerning given that their
            biological interpretation and statistical manipulation are
            contingent upon the methods and variables used to estimate
            them. In this contribution, we review how probabilistic
            interactions are defined in the literature at different
            spatial scales, from local interactions to regional networks
            (metawebs), with a strong emphasis on food webs. These
            definitions are based on the distinction between the
            realization of an interaction at a specific time and space
            and its biological feasibility. We show that different
            network representations have different statistical
            behaviours when it comes to common ecological applications.
            Specifically, we argue that local probabilistic networks and
            metawebs differ in their spatial and temporal scaling of
            interactions, with potential interactions being
            scale-independent. This is in contrast with the taxonomic
            scaling of interactions, which does not qualitatively differ
            between both types of networks. Moreover, we suggest two
            approaches to sampling deterministic networks from
            probabilistic webs that account for these differences and
            argue that systematic biases arise when directly inferring
            local networks from subsets of metawebs. To support our
            arguments, we built a spatiotemporally explicit model of
            probabilistic interactions and developed different case
            studies using empirical and simulated data. Overall, our
            results emphasize the need for better documentation of
            probabilistic ecological networks, both at the local and
            regional scales, to inform the appropriate reuse of these
            data.


            <div class="artifacts">
                <a href="Banville_et_al_2023_ecological_interpretation_draft.pdf" title="Draft" target="_blank">Draft</a>
                <a href="Banville_et_al_2023_ecological_interpretation_preprint.pdf" title="Preprint" target="_blank">Preprint</a>
                <a href="Banville_et_al_2023_ecological_interpretation.odt" title="OpenDocument" target="_blank">Libre Office</a>
            </div>
        </section>

        <article>
            <h1 id="introduction">Introduction</h1>
            <p>Cataloging ecological interactions is a gargantuan task.
            Regardless of sampling effort, there are practical and
            biological constraints that hinder our ability to observe
            all interactions in nature, such as the spatial and temporal
            uncoupling of species and the large number of potential
            interactions in a community, of which the vast majority are
            rare <span class="citation"
            data-cites="Jordano2016Samplingb">(Jordano 2016)</span>.
            Documenting the location and timing of interactions becomes
            even more challenging when accounting for the spatiotemporal
            variability of ecological networks <span class="citation"
            data-cites="Poisot2012Dissimilaritya Poisot2015Speciesa">(Poisot
            <em>et al.</em> 2012, 2015)</span>. Indeed, it is now
            recognized that knowing the biological capacity of two
            species to interact is necessary but not sufficient for
            inferring their interaction at a specific time and space.
            For example, <span class="citation"
            data-cites="Golubski2011ModMod">Golubski &amp; Abrams
            (2011)</span> presented many cases where trophic
            interactions in food webs depend on the presence or
            abundance of a third species (e.g., of a more profitable
            prey species). More generally, a handful of conditions must
            be satisfied for an interaction to be observed locally.
            First, both species must have overlapping geographic ranges,
            i.e. they must co-occur within the region of interest <span
            class="citation"
            data-cites="Blanchet2020Cooccurrencea">(Blanchet <em>et
            al.</em> 2020)</span>. Then, they must encounter locally.
            Probabilities of interspecific encounters are typically low,
            especially for rare species with low relative abundances
            <span class="citation"
            data-cites="Canard2012Emergencea">(Canard <em>et al.</em>
            2012)</span>. Finally, their traits must be locally
            compatible <span class="citation"
            data-cites="Poisot2015Speciesa">(Poisot <em>et al.</em>
            2015)</span>. This includes their phenology <span
            class="citation"
            data-cites="Olesen2010Missing Singer2012GeoMos">(Olesen
            <em>et al.</em> 2010; Singer &amp; McBride 2012)</span>,
            behavioral choices <span class="citation"
            data-cites="Pulliam1974Theory Choh2012PreRol">(Pulliam 1974;
            Choh <em>et al.</em> 2012)</span> and phenotypes
            (Bolnick2011WhyInt; <span class="citation"
            data-cites="Stouffer2011RolBod">Stouffer <em>et al.</em>
            (2011)</span>; Gravel2013InfFooa). Environmental factors,
            such as temperature <span class="citation"
            data-cites="Angilletta2004TemGro">(Angilletta <em>et
            al.</em> 2004)</span>, drought <span class="citation"
            data-cites="Woodward2012CliCha">(Woodward <em>et al.</em>
            2012)</span>, climate change <span class="citation"
            data-cites="Gilman2010FraCom Woodward2010ChaEco Araujo2011Usinga">(Gilman
            <em>et al.</em> 2010; Woodward <em>et al.</em> 2010; Araujo
            <em>et al.</em> 2011)</span>, and habitat modifications
            <span class="citation"
            data-cites="Tylianakis2007HabMod">(Tylianakis <em>et
            al.</em> 2007)</span>, contribute to this spatiotemporal
            variability of interactions by impacting species abundance
            and traits. In this context, it is unsurprising that our
            knowledge of ecological interactions remains limited <span
            class="citation" data-cites="Hortal2015SevSho">(Hortal
            <em>et al.</em> 2015)</span> despite extensive biodiversity
            data collection <span class="citation"
            data-cites="Schmeller2015GloTer">(Schmeller <em>et al.</em>
            2015)</span>.</p>
            <p>The recognition of the intrinsic variability of species
            interactions has led ecologists to expand their
            representation of ecological networks to include a
            probabilistic view of interactions <span class="citation"
            data-cites="Poisot2016Structure">(Poisot <em>et al.</em>
            2016)</span>. As opposed to binary deterministic networks,
            in which interactions are either observed or not,
            probabilistic networks represent our degree of belief about
            the realization or feasibility of pairwise interactions at
            the local or regional scale, respectively. In other words,
            representing interactions probabilistically considers
            inherent uncertainties and observation errors associated
            with ecological data. In the broadest sense, binary networks
            are also a type of probabilistic network, in which the value
            of interactions is restrained to <span
            class="math inline">0</span> (non-observed) or <span
            class="math inline">1</span> (observed). In probabilistic
            networks, only forbidden interactions <span class="citation"
            data-cites="Jordano2003Invarianta Olesen2010Missing">(Jordano
            <em>et al.</em> 2003; Olesen <em>et al.</em> 2010)</span>
            have a probability value of zero <span class="citation"
            data-cites="Gonzalez-Varo2016Labilea">(but see Gonzalez-Varo
            &amp; Traveset 2016)</span>. However, <em>neutral</em>
            forbidden interactions <span class="citation"
            data-cites="Canard2012Emergencea">(i.e., improbable
            interactions between rare species, Canard <em>et al.</em>
            2012)</span> could have low probability values in a local
            network but high probability in a regional network (metaweb)
            describing the biological capacity of species to
            interact.</p>
            <p>By accounting for the uncertainty of interactions,
            probabilistic networks provide a more realistic portrait of
            species interactions and their emerging structure. This is
            important given that network structure is one of the major
            drivers of the functioning, dynamics, and resilience of
            ecosystems worldwide <span class="citation"
            data-cites="Proulx2005Networka McCann2007ProBio McCann2011FooWeb Rooney2012IntFoo">(Proulx
            <em>et al.</em> 2005; McCann 2007; McCann 2011; Rooney &amp;
            McCann 2012)</span>. Moreover, the application and
            development of computational methods in network ecology,
            which are often based on a probabilistic representation of
            interactions, can help alleviate the colossal sampling
            efforts required to document species interactions <span
            class="citation" data-cites="Strydom2021Roadmapa">(Strydom
            <em>et al.</em> 2021)</span>. For example, statistical
            models can be used to estimate the uncertainty of pairwise
            interactions <span class="citation"
            data-cites="Cirtwill2019QuaFra">(Cirtwill <em>et al.</em>
            2019)</span> and the probability of missing (false
            negatives) and spurious (false positives) interactions <span
            class="citation" data-cites="Guimera2009MisSpu">(Guimerà
            &amp; Sales-Pardo 2009)</span>. Considering the high rate of
            false negatives in species interaction data due to the
            difficulty of witnessing rare interactions <span
            class="citation" data-cites="Catchen2023Missinga">(Catchen
            <em>et al.</em> 2023)</span>, these models can inform the
            identification of priority sampling locations of ecological
            networks <span class="citation"
            data-cites="Andrade-Pacheco2020Finding">(e.g.,
            Andrade-Pacheco <em>et al.</em> 2020 present an approach to
            identify priority sampling locations of disease
            hotspots)</span>. Statistical models can also be used to
            generate network predictions without prior knowledge of
            pairwise interactions, for instance using body size <span
            class="citation"
            data-cites="Petchey2008SizFor Gravel2013InfFooa">(Petchey
            <em>et al.</em> 2008; Gravel <em>et al.</em> 2013)</span>,
            phylogeny <span class="citation"
            data-cites="Elmasri2020HieBay Strydom2022Food">(Elmasri
            <em>et al.</em> 2020; Strydom <em>et al.</em> 2022)</span>,
            or a combination of niche and neutral processes <span
            class="citation"
            data-cites="Bartomeus2016ComFra Pomeranz2019InfPre">(Bartomeus
            <em>et al.</em> 2016; Pomeranz <em>et al.</em> 2019)</span>
            for inference. Topological null models <span
            class="citation"
            data-cites="Bascompte2003NesAss Fortuna2006HabLos">(e.g.,
            Bascompte <em>et al.</em> 2003; Fortuna &amp; Bascompte
            2006)</span>, which can be used to generate underlying
            distributions of network measures for null hypothesis
            significance testing, are other examples of common
            probabilistic network models. Many measures have been
            developed to describe the structure <span class="citation"
            data-cites="Poisot2016Structure">(Poisot <em>et al.</em>
            2016)</span> and diversity <span class="citation"
            data-cites="Ohlmann2019Diversity Godsoe2022Species">(Ohlmann
            <em>et al.</em> 2019; Godsoe <em>et al.</em> 2022)</span> of
            probabilistic networks. These models and measures support
            the use of this approach for the study of a wide range of
            ecological questions, from making better predictions of
            species distribution <span class="citation"
            data-cites="Cazelles2016Theorya">(Cazelles <em>et al.</em>
            2016)</span> to forecasting the impact of climate change on
            ecological networks <span class="citation"
            data-cites="Gilman2010FraCom">(Gilman <em>et al.</em>
            2010)</span>.</p>
            <p>Despite these advances and opportunities, the lack of
            clear guidelines on the use of probabilistic interaction
            data is worrisome, especially for field and computational
            ecologists who manipulate and generate these numbers.
            Indeed, beyond methodological challenges encountered when
            evaluating them, there are important and perhaps more
            fundamental conceptual challenges when it comes to defining
            them. To the best of our knowledge, there is currently no
            data standard that could guide the estimation and
            documentation of interaction probabilities <span
            class="citation" data-cites="Salim2022Data">(Salim <em>et
            al.</em> 2022 discuss data standards for deterministic
            mutualistic networks)</span>. General guidelines could
            support more adequate manipulation and integration of
            interaction data from different sources and prevent
            ecologists from being misled by ambiguous and often
            diverging interpretations of probabilistic networks. In this
            contribution, we aim to take a step back by outlining
            different ways in which they were defined and used in
            network ecology and propose an approach to thinking about
            them. We distinguish two broad categories of probabilistic
            networks that have different statistical behaviors when
            applied to key ecological questions: local networks of
            realized interactions and regional networks (metawebs) of
            potential interactions. We show that these representations
            have different ecological and statistical implications,
            especially regarding the spatial and temporal scaling of
            interactions and the prediction of binary networks across
            space. Although we focus on food webs, our observations and
            advice can be applied to most types of ecological networks,
            from plant-pollinator to host-parasite networks. Indeed,
            excepting networks of indirect interactions such as
            competition and facilitation networks <span class="citation"
            data-cites="Kefi2015NetStr Kefi2016HowStr">(Kéfi <em>et
            al.</em> 2015, 2016)</span>, most ecological networks
            describe probabilities of direct interactions, which are
            conceptually and mathematically analogous to each other
            regardless of their biological type (e.g., trophic and
            parasitic interactions). Overall, we argue that
            probabilistic networks should be better documented, clearly
            defined in mathematical terms, and used with caution when
            analyzing ecological interactions.</p>
            <h1
            id="probabilistic-representations-of-interactions">Probabilistic
            representations of interactions</h1>
            <p>One of the first aspects to take into consideration when
            estimating or interpreting probabilities of interactions is
            knowing if they describe potential or realized interactions.
            A potential interaction is defined as the biological
            capacity of two taxa to interact (i.e., the probability that
            they <em>can</em> theoretically interact) whereas a realized
            interaction refers to the materialization or observation of
            this interaction in a delineated space and time (i.e., the
            probability that they interact locally). Here, we use the
            terms <em>metaweb</em> to designate networks of potential
            interactions and <em>local networks</em> for those of
            realized interactions. Metawebs are the network analog of
            the species pool, where local networks originate from a
            subset of both species (nodes) and interactions (edges) of
            the regional metaweb <span class="citation"
            data-cites="Saravia2022Ecological">(Saravia <em>et al.</em>
            2022)</span>. Frequent confusion arises among ecologists
            over the use of these two terms, especially in a
            probabilistic context. Indeed, it can be difficult to know
            when published probabilistic networks describe potential or
            realized interactions, or when so-called probabilities are
            in reality <em>interaction scores</em> (i.e., a type of
            non-probabilistic quantitative interactions). Likewise,
            probabilistic potential interactions are often used and
            interpreted as realized interactions (and conversely), which
            may generate misleading findings when analyzing these data.
            We believe that a better understanding of the differences,
            similarities, and relationships between these two
            probabilistic representations of ecological networks would
            alleviate interpretation errors and help ecologists use
            these numbers more appropriately.</p>
            <h2
            id="pairwise-interactions-the-building-blocks-of-ecological-networks">Pairwise
            interactions: the building blocks of ecological
            networks</h2>
            <p>Local ecological networks and metawebs, like any type of
            networks, are made of nodes and edges that can be
            represented at different levels of organization and
            precision. The basic unit of food webs and other ecological
            networks are individuals that interact with each other
            (e.g., by predation, <span class="citation"
            data-cites="Elton2001Animal">Elton (2001)</span>), forming
            individual-based networks. The aggregation of these
            individuals into more or less homogeneous groups (e.g.,
            populations, species, trophic species, families) allows us
            to represent nodes at broader taxonomic scales, which
            impacts our interpretation of the properties and behavior of
            these systems <span class="citation"
            data-cites="Guimaraes2020Structurea">(Guimarães
            2020)</span>. Moreover, edges linking these nodes can
            describe a variety of interaction measures. Ecologists have
            traditionally represented interactions as binary objects
            that were considered realized after observing at least one
            individual from group <span
            class="math inline"><em>i</em></span> interact with at least
            another individual from group <span
            class="math inline"><em>j</em></span>. Boolean interactions
            are actually the result of a Bernoulli process <span
            class="math inline">$A_{i,j} \sim {\rm Bernoulli}(P(i
            \rightarrow j))$</span>, with <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>
            being the probability of interaction between <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> that characterizes our
            limited knowledge of the system and its intrinsic
            spatiotemporal variability. Depending on the type of
            networks (local or metaweb), the mathematical formulation
            and interpretation of stochastic parameters like <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>
            can be linked to environmental and biological factors such
            as species relative abundance, traits, area, and time
            (tbl. 1). In these probabilistic network representations in
            which <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>
            are edge values, the only two possible outcomes are the
            presence (<span
            class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> = 1</span>)
            or absence (<span
            class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> = 0</span>)
            of an interaction between each pair of nodes. Observing an
            interaction between two taxa at a given location and time
            provides important information that can be used to update
            previous estimates of <span
            class="math inline"><em>P</em>(<em>i</em>→<em>j</em>)</span>,
            informing us on the biological capacity of both taxa to
            interact and the environmental conditions that enabled them
            to interact locally.</p>
            <p>Even though binary networks constitute a highly valuable
            source of ecological information <span class="citation"
            data-cites="Pascual2006Ecologicala">(Pascual <em>et al.</em>
            2006)</span>, they overlook important factors regarding
            interaction strengths. These are represented using
            quantitative interactions, which better describe the energy
            flows, demographic impacts or frequencies of interactions
            between nodes <span class="citation"
            data-cites="Berlow2004Interaction Borrett2019Walk">(Berlow
            <em>et al.</em> 2004; Borrett &amp; Scharler 2019)</span>,
            with <span
            class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub></span>
            being a natural number <span class="math inline">ℕ</span> or
            a real number <span class="math inline">ℝ</span> depending
            on the measure. For example, they can represent the average
            number of prey individuals consumed by a predator in a given
            time period (e.g., the average number of fish in the stomach
            of a piscivorous species). Because quantitative interactions
            can describe predation pressure on prey taxa, they can be
            good estimators of the parameters describing species
            interactions in a Lotka-Volterra model <span
            class="citation" data-cites="Emmerson2004Predatora">(e.g.,
            Emmerson &amp; Raffaelli 2004)</span>. However, this extra
            amount of ecological information typically comes at a cost
            of greater sampling effort or data requirement in predictive
            models <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>, which can lead to relatively high levels of
            uncertainties when inferring quantitative networks with
            limited data. Just like binary networks, the uncertainty and
            spatiotemporal variability of quantitative interactions can
            be represented probabilistically, with the difference that
            quantitative interactions can follow various probability
            distributions depending on the measure used, the event’s
            outcome being the value of interaction strength. For
            instance, quantitative interactions can follow a Poisson
            distribution <span class="math inline">$A_{i,j} \sim {\rm
            Poisson}(\lambda_{i \rightarrow j})$</span> when predicting
            frequencies of interactions between pairs of nodes, with
            <span
            class="math inline"><em>λ</em><sub><em>i</em> → <em>j</em></sub></span>
            being the expected rate at which individuals of taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact (e.g., the
            average number of prey <span
            class="math inline"><em>j</em></span> consumed by all
            predators <span class="math inline"><em>i</em></span>). The
            Poisson distribution can also be 0-inflated when considering
            non-interacting taxa, which constitute the majority of taxa
            pairs in most local networks due to their typically high
            sparseness <span class="citation"
            data-cites="Jordano2016Samplingb">(Jordano 2016)</span>.
            Because of the methodological difficulties typically
            encountered when building deterministic quantitative
            networks, binary networks, which are easier to sample <span
            class="citation" data-cites="Jordano2016Samplingb">(Jordano
            2016)</span> and predict <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>, are much more documented and modeled in the
            literature. Moreover, most published probabilistic networks
            and methods describe Bernoulli interactions, which
            underlines the need for better guidelines regarding the
            interpretation and manipulation of these types of networks.
            For these reasons, our primary focus in this contribution
            will be on addressing the challenges in estimating and using
            Bernoulli interactions, in both probabilistic local networks
            and metawebs.</p>
            <h2
            id="local-networks-communities-interacting-in-space-and-time">Local
            networks: communities interacting in space and time</h2>
            <p>As opposed to metawebs, probabilistic local networks
            describe how likely taxa are to interact at a given location
            and time period (i.e., they are context-dependent). In local
            networks, edges commonly represent our degree of belief that
            two taxa interact in nature, but can also document the
            probability of <em>observing</em> this interaction <span
            class="citation" data-cites="Catchen2023Missinga">(Catchen
            <em>et al.</em> 2023)</span>. For example, <span
            class="citation" data-cites="Kopelke2017FooStra">Kopelke
            <em>et al.</em> (2017)</span> assembled a dataset of
            deterministic local European food webs of willow-galling
            sawflies and their natural enemies, clearly referencing each
            food web in space and time. Because of its large number of
            replicated samples, this dataset can be used to infer the
            probability of locally observing an interaction between a
            pair of taxa <span class="citation"
            data-cites="Gravel2019BriElt">(Gravel <em>et al.</em>
            2019)</span>. More generally, we define space as the
            geographic coordinates <span
            class="math inline">(<em>x</em>,<em>y</em>)</span> of the
            spatial boundaries delineating the system (sampled or
            targeted) and time as the time interval during which
            interactions were sampled or for which they were predicted.
            Given that space and time are in reality continuous
            variables, the probability that an interaction occurs within
            a particular spatial and temporal setting is given by the
            integral of the probability density function describing the
            relative likelihood that this interaction is realized at any
            specific and infinitely small location and time. Therefore,
            the edge value could represent a probability density or a
            probability mass depending on how space and time are
            measured. For simplicity reasons, we will consider space and
            time as discrete dimensions that provide actual
            probabilities of interactions, which is conform to how
            ecological interactions are usually sampled. Using space and
            time intervals allows us to measure an area <span
            class="math inline"><em>A</em></span> and duration <span
            class="math inline"><em>t</em></span>, which can be directly
            used in spatiotemporal analyses of ecological networks. For
            example, when studying network-area relationships <span
            class="citation" data-cites="Galiana2018Spatiala">(NAR,
            Galiana <em>et al.</em> 2018)</span>, we should expect local
            probabilities of interactions to scale with area and
            duration because taxa have more opportunities to
            interact.</p>
            <p>The probability that two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact locally can
            also be conditional on many environmental and biological
            factors. One of these is their co-occurrence <span
            class="math inline"><em>C</em><sub><em>i</em>, <em>j</em></sub></span>,
            which is usually a Boolean describing if the geographic
            distribution of both taxa overlaps within the study area. In
            local networks, the probability that the interaction is
            realized must be <span class="math inline">0</span> when
            taxa do not co-occur, i.e. <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=0) = 0</span>.
            Co-occurrence can also be modeled probabilistically. In that
            case, it follows a Bernoulli distribution <span
            class="math inline">$C_{i,j} \sim {\rm Bernoulli}(P_{i,j}(x,
            y))$</span>, where the probability of co-occurrence <span
            class="math inline"><em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>)</span>
            can be estimated using species distribution models <span
            class="citation" data-cites="Pollock2014UndCoo">(e.g.,
            Pollock <em>et al.</em> 2014)</span>. More generally, the
            probability that two taxa interact locally can be obtained
            by the product of their probability of interaction given
            co-occurrence with their probability of co-occurrence:</p>
            <p><span id="eq:co-occur"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=1) × <em>P</em><sub><em>i</em>, <em>j</em></sub>(<em>x</em>,<em>y</em>).   (1)</span></span></p>
            <p>Other important factors that can impact our estimation of
            interaction probabilities at the local scale are taxa
            relative abundance <span class="citation"
            data-cites="Canard2012Emergencea">(Canard <em>et al.</em>
            2012)</span> and traits <span class="citation"
            data-cites="Poisot2015Speciesa">(Poisot <em>et al.</em>
            2015)</span>, as well as environmental factors such as
            temperature <span class="citation"
            data-cites="Angilletta2004TemGro">(Angilletta <em>et
            al.</em> 2004)</span>, precipitation <span class="citation"
            data-cites="Woodward2012CliCha">(Woodward <em>et al.</em>
            2012)</span>, habitat structure <span class="citation"
            data-cites="Klecka2014EffHab">(Klecka &amp; Boukal
            2014)</span>, and presence of other interacting taxa in the
            network <span class="citation"
            data-cites="Pilosof2017MulNat Kefi2012MorMea">(Kéfi <em>et
            al.</em> 2012; Pilosof <em>et al.</em> 2017)</span>. Here,
            we will use the variable <span
            class="math inline"><em>Ω</em></span> to describe the
            biological and ecological context in which interaction
            probabilities were estimated. For example, if a research
            team conducts a mesocosm experiment to estimate interaction
            probabilities between predators and prey with and without
            shelters, <span class="math inline"><em>Ω</em></span> would
            represent the presence or absence of these shelters. Like
            co-occurrence, <span class="math inline"><em>Ω</em></span>
            can also be modeled probabilistically when the stochasticity
            or uncertainty of environmental and biological factors is
            considered. In sum, <span
            class="math inline"><em>Ω</em></span> represents all
            ecological and biological variables that were taken into
            consideration when measuring interaction probabilities and
            is, therefore, a subset of all factors actually impacting
            ecological interactions.</p>
            <p>The probability that two taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interact in a local
            network <span class="math inline"><em>N</em></span> can thus
            be conditional on the area <span
            class="math inline"><em>A</em></span>, the time interval
            <span class="math inline"><em>t</em></span>, their
            co-occurrence <span class="math inline"><em>C</em></span>
            and chosen environmental and biological conditions <span
            class="math inline"><em>Ω</em></span>. This gives us the
            following equation when all these conditions are included in
            the estimation of interaction probabilities:</p>
            <p><span id="eq:local"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>C</em>,<em>Ω</em>).   (2)</span></span></p>
            <p>The local context in which probabilities are estimated
            and the variables that should be taken into consideration
            depend on the study system, the objective of the study, and
            the resources available to the researchers. In other words,
            these variables do not systematically need to be accounted
            for. However, when they are, they should be specified in the
            documentation of the data, preferentially in mathematical
            terms to avoid any confusion in their interpretation and to
            limit manipulation errors during their re-use. For example,
            ecologists should be explicit about their consideration of
            co-occurrence in their estimation of local interaction
            probabilities. Indeed, it is important to specify if
            probability values are conditional <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>=1)</span>
            or not <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>)</span>
            on co-occurrence since this can significantly impact the
            interpretation and analysis of the data. In tbl. 1, we
            present a handful of studies of probabilistic ecological
            networks and their formulation of probabilistic
            interactions. This table illustrates the variety of
            definitions of probabilistic interactions found in the
            literature and emphasizes the need to carefully describe
            interaction data before integrating and analyzing them.</p>
            <div id="tbl:prob">
            <table>
            <caption>Table 1: Interaction probabilities are interpreted
            differently in metawebs and local networks. Each formula
            includes different conditional variables and is described in
            plain text. A non-exhaustive list of studies using these
            formulas is included, with the variables used specified in
            parentheses.</caption>
            <colgroup>
            <col style="width: 24%" />
            <col style="width: 54%" />
            <col style="width: 21%" />
            </colgroup>
            <thead>
            <tr class="header">
            <th style="text-align: left;">Formula</th>
            <th style="text-align: left;">Description</th>
            <th style="text-align: right;">Studies</th>
            </tr>
            </thead>
            <tbody>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is biologically feasible</td>
            <td style="text-align: right;"></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is realized locally</td>
            <td style="text-align: right;"></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is realized locally given network area</td>
            <td style="text-align: right;"></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>t</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is realized locally given duration</td>
            <td style="text-align: right;"></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is realized locally given co-occurrence</td>
            <td style="text-align: right;"></td>
            </tr>
            <tr class="even">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>Ω</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is realized locally given chosen environmental
            and biological factors</td>
            <td style="text-align: right;"></td>
            </tr>
            <tr class="odd">
            <td style="text-align: left;"><span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>C</em>,<em>Ω</em>)</span></td>
            <td style="text-align: left;">probability that the
            interaction is realized locally given many conditional
            factors</td>
            <td style="text-align: right;"></td>
            </tr>
            </tbody>
            </table>
            </div>
            <h2
            id="metawebs-regional-catalogs-of-interactions">Metawebs:
            regional catalogs of interactions</h2>
            <p>Metawebs are networks of potential interactions that have
            been designed for broad spatial, temporal, and taxonomic
            scales (e.g, species food webs at the continental scale).
            They represent the probability that two taxa can
            biologically interact regardless of their co-occurrence and
            local environmental conditions. Indeed, potential
            interactions are by definition context-independent,
            i.e. they are not measured at a specific location and time.
            In contrast with probabilistic local networks, which
            represent the stochasticity of interactions occurring in
            nature, probabilistic metawebs measure our degree of belief
            in the capacity of two taxa to interact (i.e., the
            probability that their traits could support an interaction
            in the right conditions). In other words, potential
            interactions describe the probability that there exists at
            least one combination of phenotypes of taxa <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> that can interact with
            each other if they were to encounter. This probability of
            interaction, in a metaweb <span
            class="math inline"><em>M</em></span>, can be expressed
            as</p>
            <p><span id="eq:metaweb"><span
            class="math display"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>),   (3)</span></span></p>
            <p>which, in contrast with eq. 2, is not conditional on any
            spatial, temporal, or environmental variables (tbl. 1).</p>
            <p>Starting from a selected set of taxa, which are usually
            distributed within a broad region of interest, metawebs can
            be built using different data sources, including literature
            review, fieldwork, and predictive models <span
            class="citation" data-cites="Strydom2022Food">(e.g., the
            metaweb of Canadian mammals inferred by Strydom <em>et
            al.</em> 2022)</span>. Every pair of taxa that have
            confidently been observed to interact at least once can be
            given a probability of <span class="math inline">1</span>
            (i.e., <span
            class="math inline"><em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>) = 1</span>)
            since we know that they <em>can</em> interact. This is
            usually not the case in local probabilistic networks, in
            which probabilities usually remain stochastic (i.e., <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>) &lt; 1</span>)
            after empirically observing interactions because of their
            intrinsic spatiotemporal variability. Similarly, although
            rare interactions typically have low probabilities in local
            networks, they can have high probabilities in metawebs if
            the traits of both taxa match. On the other hand,
            interactions that were never observed can have low
            probability values in both metawebs and local networks,
            going as low as <span class="math inline">0</span> for
            forbidden links. However, because of observation errors due
            to taxonomic misidentifications and ecological
            misinterpretations (e.g., due to cryptic species and
            interactions, <span class="citation"
            data-cites="Pringle2020Resolving">Pringle &amp; Hutchinson
            (2020)</span>), many observations of interactions are
            actually false positives. Similarly, forbidden interactions
            can be false negatives in metawebs, e.g. if they have been
            assessed for specific phenotypes, locations or time.
            Implementing a Bayesian framework, which updates prior
            probabilities with empirical data (e.g., <span
            class="citation" data-cites="Bartomeus2016ComFra">Bartomeus
            <em>et al.</em> (2016)</span>, <span class="citation"
            data-cites="Cirtwill2019QuaFra">Cirtwill <em>et al.</em>
            (2019)</span>), could improve our estimation of interaction
            probabilities in both systems.</p>
            <h1
            id="statistical-behaviors-of-probabilistic-networks">Statistical
            behaviors of probabilistic networks</h1>
            <p>The differences in the mathematical formulations of
            probabilistic local and potential interactions can affect
            their statistical behaviors when applied to key ecological
            questions. These disparities must therefore be taken into
            account when analyzing probabilistic interaction data to
            prevent misleading results and minimize interpretation
            errors. Here we show four common applications of
            probabilistic interactions and compare the characteristics
            of local networks and metawebs using simulated and empirical
            data.</p>
            <h2
            id="taxonomic-agglomeration-and-division-of-nodes">Taxonomic
            agglomeration and division of nodes</h2>
            <p>Probabilistic networks can be used to address a wide
            range of ecological questions based on their level of
            organization. For example, the assemblage of interactions
            across ecological scales can be studied using species-based
            networks, whereas clade-based networks can be used to study
            macroevolutionary processes <span class="citation"
            data-cites="Gomez2010EcoInt">(e.g., Gomez <em>et al.</em>
            2010)</span>. Because our interpretation of the properties
            and dynamics of ecological networks depends on their
            taxonomic scale <span class="citation"
            data-cites="Guimaraes2020Structurea">(Guimarães
            2020)</span>, examining the phylogenetic scaling of network
            structure is also a promising research avenue. Analyzing the
            same system at different taxonomic scales can thus provide
            meaningful and complementary ecological information and is,
            in our perspective, best conducted using probabilistic
            networks.</p>
            <p>Local networks are analogous to metawebs in their
            taxonomic scaling of interactions because only the nodes are
            defined taxonomically. In other words, the edge values of
            local networks (eq. 2) and metawebs (eq. 3) are not
            conditional on any taxonomic scale. It is the definition of
            the event itself (i.e., the interaction of the two taxa)
            that has a given phylogenetic scale, not their conditional
            variables. In both types of networks, shifting to a broader
            level of organization can be directly done using
            probabilities at finer scales. For example, if we have a
            network of <span
            class="math inline"><em>n</em><sub><em>A</em></sub></span>
            species from genus <span
            class="math inline"><em>A</em></span> and <span
            class="math inline"><em>n</em><sub><em>B</em></sub></span>
            species from genus <span
            class="math inline"><em>B</em></span>, we can calculate the
            probability that at least one species from genus <span
            class="math inline"><em>A</em></span> interact with at least
            one species from genus <span
            class="math inline"><em>B</em></span> as follows:</p>
            <p><span id="eq:taxo"><span class="math display">$$P(A
            \rightarrow B) = 1 - \prod_{i = 1}^{n_A}\prod_{j =
            1}^{n_B}(1 - P(A_i
            \rightarrow B_j)),\qquad{(4)}$$</span></span></p>
            <p>where <span
            class="math inline"><em>A</em><sub><em>i</em></sub></span>
            and <span
            class="math inline"><em>B</em><sub><em>j</em></sub></span>
            are the species of the corresponding genus. Knowing that two
            species interact (i.e., <span
            class="math inline"><em>P</em>(<em>A</em><sub><em>i</em></sub>→<em>B</em><sub><em>j</em></sub>) = 1</span>)
            gives a probability of genus interaction of <span
            class="math inline">1</span>. <span class="citation"
            data-cites="Canard2012Emergencea">Canard <em>et al.</em>
            (2012)</span> built a species-based network from neutrally
            simulated interactions between individuals using a similar
            approach. In contrast, more sophisticated models need to be
            built when shifting to a finer level of organization.
            Indeed, knowing that two genera interact does not imply that
            all of their pairwise species combinations can also
            interact. One could, for example, build a finer-scale
            network by generating probabilities of species interactions
            by randomly sampling them from a beta distribution
            parametrized by the broader-scale network.</p>
            <p>The biological interpretation of probabilistic
            interactions should not differ across a network even if it
            has heterogenous levels of organization, i.e. if different
            nodes are represented at different taxonomic scales (e.g., a
            network composed of species and trophic species). This is
            frequent in ecological networks where taxonomic resolution
            is typically low <span class="citation"
            data-cites="Hemprich-Bennett2021AssImp VazquezSS2022Ecological">(Hemprich-Bennett
            <em>et al.</em> 2021; Vázquez <em>et al.</em> 2022)</span>.
            Indeed, these definitions should be based on probabilities
            of interactions between individuals, either locally or
            potentially. In local individual-based food webs, the
            probability that two individuals interact represents the
            degree of belief that one individual will actually consume
            the other. Similarly, in local species-based food webs, the
            probability that two species interact represents the degree
            of belief that at least one individual from the predator
            species consumes at least another individual from the prey
            species. Moreover, in local clade-based food webs, the
            probability that two clades interact represents the degree
            of belief that at least two species from these clades
            interact with each other or, equivalently, that at least two
            individuals from these clades interact with each other.
            Fundamentally, the taxonomic scaling of interactions is an
            aggregation of interactions between individuals into larger
            groups, which could be more or less homogeneous depending on
            the organisms and the study system. This type of scaling is
            analogous to the spatial and temporal scaling of
            interactions to the extent that these are different ways to
            aggregate individuals into broader nodes, either spatially,
            temporally, or taxonomically.</p>
            <h2
            id="spatial-and-temporal-scaling-of-probabilistic-interactions">Spatial
            and temporal scaling of probabilistic interactions</h2>
            <p>Metawebs and local networks intrinsically differ in their
            relation to scale. On one hand, as mentioned above,
            probabilistic metawebs are context-independent, i.e.,
            probabilistic pairwise interactions do not scale with space
            and time because they depend solely on the biological
            capacity of the two taxa to interact. This implies that the
            estimated likelihood that two species can potentially
            interact should be the same among all metawebs in which they
            are present. In practice, this is rarely the case because
            ecologists use different methods and data to estimate these
            probabilities of interactions (e.g., different sampling area
            and time period). However, in the case where local metawebs
            <span
            class="math inline"><em>M</em><sub><em>x</em>, <em>y</em></sub></span>
            are subsampled from their regional counterpart <span
            class="math inline"><em>M</em><sub>0</sub></span>, we should
            expect edge values to be identical among all networks,
            regardless of their spatial scale, i.e. <span
            class="math inline"><em>P</em><sub><em>M</em><sub><em>x</em>, <em>y</em></sub></sub>(<em>i</em>→<em>j</em>) = <em>P</em><sub><em>M</em><sub>0</sub></sub>(<em>i</em>→<em>j</em>)</span>.
            On the other hand, local probabilistic networks are
            indissociable from their spatial and temporal contexts
            because there are more opportunities of interactions (e.g.,
            more individuals, more trait variations, more chance of
            encounter) in a larger area and longer time period <span
            class="citation" data-cites="McLeod2020EffSpe">(McLeod
            <em>et al.</em> 2020)</span>. Let <span
            class="math inline"><em>N</em><sub>0</sub></span> be a local
            probabilistic food web delineated in an area <span
            class="math inline"><em>A</em><sub>0</sub></span> and <span
            class="math inline"><em>N</em><sub>1</sub></span> a network
            of area <span
            class="math inline"><em>A</em><sub>1</sub> &lt; <em>A</em><sub>0</sub></span>
            within <span
            class="math inline"><em>A</em><sub>0</sub></span>. We should
            expect the probability that <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interacts to scale
            spatially, i.e. <span
            class="math inline"><em>P</em><sub><em>N</em><sub>1</sub></sub>(<em>i</em>→<em>j</em>|<em>A</em>&lt;<em>A</em><sub>0</sub>) ≤ <em>P</em><sub><em>N</em><sub>0</sub></sub>(<em>i</em>→<em>j</em>|<em>A</em>=<em>A</em><sub>0</sub>)</span>.
            Similarly, the temporal scaling of probabilistic local food
            webs could be manifested through the effect of sampling
            effort on the observation of interactions <span
            class="citation"
            data-cites="Jordano2016Samplingb McLeod2021Sampling">(Jordano
            2016; McLeod <em>et al.</em> 2021)</span> or of time itself
            on their realization <span class="citation"
            data-cites="Poisot2012Dissimilaritya">(Poisot <em>et
            al.</em> 2012)</span>. There are many network-area
            relationships <span class="citation"
            data-cites="Wood2015Effects Galiana2018Spatiala">(e.g., Wood
            <em>et al.</em> 2015; Galiana <em>et al.</em> 2018)</span>
            and interaction accumulation curves <span class="citation"
            data-cites="Jordano2016Samplingb">(e.g, Jordano 2016)</span>
            explored in the literature. These could inspire the
            development and testing of different equations describing
            the spatiotemporal scaling of probabilistic pairwise
            interactions in local networks, which are over the scope of
            this manuscript.</p>
            <h3
            id="a-spatiotemporally-explicit-model-of-probabilistic-interactions">A
            spatiotemporally explicit model of probabilistic
            interactions</h3>
            <p>The variability of species interactions spurred the
            development of methods aiming at predicting ecological
            networks at fine spatial and temporal scales. For example,
            <span class="citation"
            data-cites="Bohan2017Nextgenerationa">Bohan <em>et al.</em>
            (2017)</span> proposed a framework to reconstruct networks
            in real time using continuous biomonitoring. Here, we will
            build on these studies by proposing a simple model to make
            probabilistic local networks spatiotemporally explicit.
            These types of models could prove useful when inferring food
            webs across time and space from sparse data. However, they
            are not suitable for metawebs, which are static objects.</p>
            <p>One way that probabilistic food webs can be made
            spatiotemporally explicit is by modelling interactions
            between co-occurring species as a Poisson process with rate
            <span class="math inline"><em>λ</em></span>. Specifically,
            if the total observation time for a location is <span
            class="math inline"><em>t</em><sub>0</sub></span>, the
            probability that two co-occurring species <span
            class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> will interact during
            this time period is <span
            class="math inline"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>C</em><sub><em>i</em>, <em>j</em></sub>=1,<em>t</em>=<em>t</em><sub>0</sub>) = 1 − <em>e</em><sup>−<em>λ</em><em>t</em><sub>0</sub></sup></span>,
            which approaches <span class="math inline">1</span> when
            <span class="math inline"><em>t</em> → ∞</span>. The value
            of the parameter <span class="math inline"><em>λ</em></span>
            could be estimated using prior data on interaction
            strengths, if available. Additionally, we can estimate the
            probability of co-occurrence at location <span
            class="math inline">(<em>x</em>,<em>y</em>)</span> with
            <span
            class="math inline"><em>P</em><sub><em>x</em>, <em>y</em></sub>(<em>C</em><sub><em>i</em>, <em>j</em></sub>=1) = <em>P</em><sub><em>x</em>, <em>y</em></sub>(<em>i</em>)<em>P</em><sub><em>x</em>, <em>y</em></sub>(<em>j</em>)<em>γ</em></span>,
            where <span
            class="math inline"><em>P</em><sub><em>x</em>, <em>y</em></sub>(<em>i</em>)</span>
            and <span
            class="math inline"><em>P</em><sub><em>x</em>, <em>y</em></sub>(<em>j</em>)</span>
            are respectively the probabilities of occurrence of species
            <span class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> and <span
            class="math inline"><em>γ</em></span> is the strength of
            association between occurrence and co-occurrence, as defined
            in <span class="citation"
            data-cites="Cazelles2016Theorya">Cazelles <em>et al.</em>
            (2016)</span>. Note that in empirical networks, <span
            class="math inline"><em>γ</em></span> is typically <span
            class="math inline"> &gt; 1</span> <span class="citation"
            data-cites="Catchen2023Missinga">(Catchen <em>et al.</em>
            2023)</span>. The observation of this interaction would thus
            follow a Bernoulli distribution with parameter <span
            class="math inline"><em>p</em> = <em>p</em><sub><em>A</em></sub>(<em>x</em>,<em>y</em>)<em>p</em><sub><em>B</em></sub>(<em>x</em>,<em>y</em>)<em>γ</em>(1−<em>e</em><sup>−<em>λ</em><em>t</em><sub>0</sub></sup>)</span>.
            This simple model could be customized in many ways, e.g. by
            linking <span class="math inline"><em>λ</em></span> with
            given environmental variables or by adding in observation
            errors (i.e., probability of false negatives and false
            positives; <span class="citation"
            data-cites="Catchen2023Missinga">Catchen <em>et al.</em>
            (2023)</span>).</p>
            <h2
            id="binary-conversion-of-probabilistic-networks-through-random-draws">Binary
            conversion of probabilistic networks through random
            draws</h2>
            <p>Another conceptual challenge encountered when using
            probabilistic food webs is the prediction of Boolean
            networks across space. Lets take <span
            class="math inline"><em>n</em> × <em>n</em></span> grid
            cells each representing a probabilistic food web. If they
            contain potential interactions, a single random trial must
            be conducted for each pairwise interaction across the region
            (i.e., we should have only one random realization of the
            regional metaweb). On the contrary, if they represent
            probabilities of realized interactions, each food web must
            be independently sampled (i.e., <span
            class="math inline"><em>n</em><sup>2</sup></span>
            independent random draws). This has direct implications on
            the spatial scaling of interactions. For example, let <span
            class="math inline"><em>N</em><sub>1</sub></span> and <span
            class="math inline"><em>N</em><sub>2</sub></span> be
            networks of area <span
            class="math inline"> &lt; <em>A</em><sub>0</sub></span>
            within a bigger area <span
            class="math inline"><em>A</em><sub>0</sub></span> and
            disjoint from each other, such as <span
            class="math inline"><em>N</em><sub>1</sub></span> and <span
            class="math inline"><em>N</em><sub>2</sub></span> form <span
            class="math inline"><em>N</em><sub>0</sub></span> (think of
            two contiguous cells that together delineate <span
            class="math inline"><em>N</em><sub>0</sub></span>). All
            other things being equal, we should expect the probability
            that <span class="math inline"><em>i</em></span> and <span
            class="math inline"><em>j</em></span> interacts in <span
            class="math inline"><em>A</em><sub>0</sub></span> to be
            <span
            class="math inline"><em>P</em><sub><em>N</em><sub>0</sub></sub>(<em>i</em>→<em>j</em>) = 1 − (1−<em>P</em><sub><em>N</em><sub>1</sub></sub>(<em>i</em>→<em>j</em>)) × (1−<em>P</em><sub><em>N</em><sub>2</sub></sub>(<em>i</em>→<em>j</em>))</span>
            if <span class="math inline"><em>N</em><sub>1</sub></span>
            and <span class="math inline"><em>N</em><sub>2</sub></span>
            are independently sampled. This also implies that we should
            expect interactions to be realized in a certain number of
            local networks depending on the probability value, which is
            not the case with metawebs. Note that spatial
            auto-correlation and the concept of meta-network (i.e.,
            networks of networks) could invalidate the statistical
            assumption of independence. Nevertheless, the fundamental
            difference in sampling metawebs and local networks stands
            even when considering these factors. This difference in
            sampling further sheds light on the importance of clearly
            defining interaction probabilities. What we consider as a
            <em>Bernoulli trial</em>, when randomly drawing
            deterministic networks from probabilistic food webs, depends
            on our biological interpretation of these probabilities.</p>
            <p>Quantitative interactions can be converted to
            probabilistic interactions by normalizing.</p>
            <h2
            id="prediction-of-local-networks-from-probabilistic-metawebs">Prediction
            of local networks from probabilistic metawebs</h2>
            <p>Even though the spatiotemporal variability of
            interactions is not considered in metawebs, they can still
            be useful to reconstruct local networks of realized
            interactions. Indeed, local networks are formed from subsets
            of their metaweb (called subnetworks), which are obtained by
            selecting a subset of both species and interactions <span
            class="citation" data-cites="Dunne2006Network">(Dunne
            2006)</span>. Because a community’s composition is arguably
            easier to sample (or predict) than its interactions, the
            biggest challenge is to sample links from the metaweb. This
            becomes a conceptual issue when we consider how potential
            and realized interactions differ. Despite these concerns,
            metawebs remain an important source of ecological
            information that can be leveraged for realistically
            predicting spatially explicit networks. First, metawebs set
            the upper limit of species interactions <span
            class="citation" data-cites="McLeod2021Sampling">(McLeod
            <em>et al.</em> 2021)</span>, i.e. the probability that two
            species interact at a specific location is always lower or
            equal to the probability of their potential interaction:</p>
            <p><span id="eq:switch"><span
            class="math display"><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>C</em>,<em>Ω</em>) ≤ <em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>).   (5)</span></span></p>
            <p>Probabilistic metawebs give limited information on local
            networks. Additionally, our degree of belief that two taxa
            have the capacity to interact must be higher than the
            probability that they will actually interact (or that they
            will ever interact). This implies that the accumulated
            probability of realized interactions across all spatial,
            temporal, and environmental conditions must be lower than
            the probability of potential interaction, i.e. </p>
            <p><span id="eq:all"><span
            class="math display">∫<sub><em>Ω</em></sub>∫<sub><em>A</em></sub>∫<sub><em>t</em></sub><em>P</em><sub><em>N</em></sub>(<em>i</em>→<em>j</em>|<em>A</em>,<em>t</em>,<em>Ω</em>)<em>d</em><em>t</em><em>d</em><em>A</em><em>d</em><em>Ω</em> ≤ <em>P</em><sub><em>M</em></sub>(<em>i</em>→<em>j</em>),   (6)</span></span></p>
            <p>since both taxa might never co-occur or encounter
            locally.</p>
            <p>Therefore, inferring local networks from their metaweb
            keeping the same values of interaction probability would
            generate systematic biases in the prediction. In that case,
            these networks would instead be called <em>spatially
            explicit</em> or <em>local</em> metawebs (i.e.,
            smaller-scale networks of potential interactions). Second,
            the structure of local networks is constrained by the one of
            their metaweb <span class="citation"
            data-cites="Saravia2022Ecological">(Saravia <em>et al.</em>
            2022)</span>. This suggests that a metaweb not only
            constrains the pairwise interactions of its corresponding
            local networks, but also their emerging properties.
            Inferring the structure of local networks from the metaweb
            could thus help estimate more realistically the likelihood
            that potential interactions are realized and observed
            locally <span class="citation"
            data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em>
            2021)</span>.</p>
            <h1 id="conclusion">Conclusion</h1>
            <p>In this contribution, we underlined the importance of
            network metadata for adequately interpreting and
            manipulating probabilistic interaction data. Indeed, the
            mathematical representation of probabilities and their
            statistical behaviors depend on the type of interactions
            (local or potential) and the conditions in which they were
            estimated. We showed that probabilistic local networks and
            metawebs differ in their relationship to spatial and
            temporal scales, with potential interactions being
            scale-independent. In contrast, local interactions are
            measured in a specific context (e.g., in a given area, time,
            and biological and environmental conditions) and are
            conditional on taxa co-occurrence. These important
            conceptual differences bring to light the need to use
            probabilistic data with caution, for instance when
            generating binary network realizations across space and
            predicting local networks from metawebs. Clear metadata
            describing the type of interaction and the variables used in
            their estimation are required to ensure adequate data
            manipulation. Better data practices and rigorous foundations
            for probabilistic thinking in network ecology could enable
            more reliable assessments of the spatiotemporal variability
            and measurement uncertainty of biotic interactions.</p>
            <h1 id="acknowledgment">Acknowledgment</h1>
            <p>We acknowledge that this study was conducted on land
            within the traditional unceded territory of the Saint
            Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat,
            and Omàmiwininiwak nations. This work was supported by the
            Institute for Data Valorisation (IVADO) and the Natural
            Sciences and Engineering Research Council of Canada (NSERC)
            Collaborative Research and Training Experience (CREATE)
            program, through the Computational Biodiversity Science and
            Services (BIOS²) program. A special thank to all members of
            the Black Holes and Revelations working group (organized by
            BIOS²) for their insightful discussions and valuable
            feedback on this manuscript.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs"
            class="references csl-bib-body hanging-indent"
            role="doc-bibliography">
            <div id="ref-Andrade-Pacheco2020Finding" class="csl-entry"
            role="doc-biblioentry">
            Andrade-Pacheco, R., Rerolle, F., Lemoine, J., Hernandez,
            L., Meïté, A., Juziwelo, L., <em>et al.</em> (2020). <a
            href="https://doi.org/10.1038/s41598-020-67666-3">Finding
            hotspots: Development of an adaptive spatial sampling
            approach</a>. <em>Scientific Reports</em>, 10, 10939.
            </div>
            <div id="ref-Angilletta2004TemGro" class="csl-entry"
            role="doc-biblioentry">
            Angilletta, M.J., Jr., Steury, T.D. &amp; Sears, M.W.
            (2004). <a
            href="https://doi.org/10.1093/icb/44.6.498">Temperature,
            Growth Rate, and Body Size in Ectotherms: Fitting Pieces of
            a Life-History Puzzle1</a>. <em>Integrative and Comparative
            Biology</em>, 44, 498–509.
            </div>
            <div id="ref-Araujo2011Usinga" class="csl-entry"
            role="doc-biblioentry">
            Araujo, M.B., Rozenfeld, A., Rahbek, C. &amp; Marquet, P.A.
            (2011). <a
            href="https://doi.org/10.1111/j.1600-0587.2011.06919.x">Using
            species co-occurrence networks to assess the impacts of
            climate change</a>. <em>Ecography</em>, 34, 897–908.
            </div>
            <div id="ref-Bartomeus2016ComFra" class="csl-entry"
            role="doc-biblioentry">
            Bartomeus, I., Gravel, D., Tylianakis, J.M., Aizen, M.A.,
            Dickie, I.A. &amp; Bernard-Verdier, M. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12666">A common
            framework for identifying linkage rules across different
            types of interactions</a>. <em>Functional Ecology</em>, 30,
            1894–1903.
            </div>
            <div id="ref-Bascompte2003NesAss" class="csl-entry"
            role="doc-biblioentry">
            Bascompte, J., Jordano, P., Melián, C.J. &amp; Olesen, J.M.
            (2003). <a
            href="https://doi.org/10.1073/pnas.1633576100">The nested
            assembly of plantanimal mutualistic networks</a>.
            <em>Proceedings of the National Academy of Sciences</em>,
            100, 9383–9387.
            </div>
            <div id="ref-Berlow2004Interaction" class="csl-entry"
            role="doc-biblioentry">
            Berlow, E.L., Neutel, A.-M., Cohen, J.E., De Ruiter, P.C.,
            Ebenman, B., Emmerson, M., <em>et al.</em> (2004). <a
            href="https://doi.org/10.1111/j.0021-8790.2004.00833.x">Interaction
            strengths in food webs: Issues and opportunities</a>.
            <em>Journal of Animal Ecology</em>, 73, 585–598.
            </div>
            <div id="ref-Blanchet2020Cooccurrencea" class="csl-entry"
            role="doc-biblioentry">
            Blanchet, F.G., Cazelles, K. &amp; Gravel, D. (2020). <a
            href="https://doi.org/10.1111/ele.13525">Co-occurrence is
            not evidence of ecological interactions</a>. <em>Ecology
            Letters</em>, 23, 1050–1063.
            </div>
            <div id="ref-Bohan2017Nextgenerationa" class="csl-entry"
            role="doc-biblioentry">
            Bohan, D.A., Vacher, C., Tamaddoni-Nezhad, A., Raybould, A.,
            Dumbrell, A.J. &amp; Woodward, G. (2017). <a
            href="https://doi.org/10.1016/j.tree.2017.03.001">Next-Generation
            Global Biomonitoring: Large-scale, Automated Reconstruction
            of Ecological Networks</a>. <em>Trends in Ecology &amp;
            Evolution</em>, 32, 477–487.
            </div>
            <div id="ref-Borrett2019Walk" class="csl-entry"
            role="doc-biblioentry">
            Borrett, S.R. &amp; Scharler, U.M. (2019). <a
            href="https://doi.org/10.1016/j.ecolind.2019.105451">Walk
            partitions of flow in Ecological Network Analysis: Review
            and synthesis of methods and indicators</a>. <em>Ecological
            Indicators</em>, 106, 105451.
            </div>
            <div id="ref-Canard2012Emergencea" class="csl-entry"
            role="doc-biblioentry">
            Canard, E., Mouquet, N., Marescot, L., Gaston, K.J., Gravel,
            D. &amp; Mouillot, D. (2012). <a
            href="https://doi.org/10.1371/journal.pone.0038295">Emergence
            of Structural Patterns in Neutral Trophic Networks</a>.
            <em>PLOS ONE</em>, 7, e38295.
            </div>
            <div id="ref-Catchen2023Missinga" class="csl-entry"
            role="doc-biblioentry">
            Catchen, M.D., Poisot, T., Pollock, L.J. &amp; Gonzalez, A.
            (2023). The missing link: Discerning true from false
            negatives when sampling species interaction networks.
            </div>
            <div id="ref-Cazelles2016Theorya" class="csl-entry"
            role="doc-biblioentry">
            Cazelles, K., Araujo, M.B., Mouquet, N. &amp; Gravel, D.
            (2016). <a
            href="https://doi.org/10.1007/s12080-015-0281-9">A theory
            for species co-occurrence in interaction networks</a>.
            <em>Theoretical Ecology</em>, 9, 39–48.
            </div>
            <div id="ref-Choh2012PreRol" class="csl-entry"
            role="doc-biblioentry">
            Choh, Y., Ignacio, M., Sabelis, M.W. &amp; Janssen, A.
            (2012). <a
            href="https://doi.org/10.1038/srep00728">Predator-prey role
            reversals, juvenile experience and adult antipredator
            behaviour</a>. <em>Scientific Reports</em>, 2, 728.
            </div>
            <div id="ref-Cirtwill2019QuaFra" class="csl-entry"
            role="doc-biblioentry">
            Cirtwill, A.R., Eklöf, A., Roslin, T., Wootton, K. &amp;
            Gravel, D. (2019). <a
            href="https://doi.org/10.1111/2041-210X.13180">A
            quantitative framework for investigating the reliability of
            empirical network construction</a>. <em>Methods in Ecology
            and Evolution</em>, 10, 902–911.
            </div>
            <div id="ref-Dunne2006Network" class="csl-entry"
            role="doc-biblioentry">
            Dunne, J.A. (2006). The Network Structure of Food Webs. In:
            <em>Ecological networks: Linking structure and dynamics</em>
            (eds. Dunne, J.A. &amp; Pascual, M.). Oxford University
            Press, pp. 27–86.
            </div>
            <div id="ref-Elmasri2020HieBay" class="csl-entry"
            role="doc-biblioentry">
            Elmasri, M., Farrell, M.J., Davies, T.J. &amp; Stephens,
            D.A. (2020). <a href="https://doi.org/10.1214/19-AOAS1296">A
            hierarchical Bayesian model for predicting ecological
            interactions using scaled evolutionary relationships</a>.
            <em>The Annals of Applied Statistics</em>, 14, 221–240.
            </div>
            <div id="ref-Elton2001Animal" class="csl-entry"
            role="doc-biblioentry">
            Elton, C.S. (2001). <em>Animal Ecology</em>. University of
            Chicago Press, Chicago, IL.
            </div>
            <div id="ref-Emmerson2004Predatora" class="csl-entry"
            role="doc-biblioentry">
            Emmerson, M.C. &amp; Raffaelli, D. (2004). <a
            href="https://doi.org/10.1111/j.0021-8790.2004.00818.x">Predatorprey
            body size, interaction strength and the stability of a real
            food web</a>. <em>Journal of Animal Ecology</em>, 73,
            399–409.
            </div>
            <div id="ref-Fortuna2006HabLos" class="csl-entry"
            role="doc-biblioentry">
            Fortuna, M.A. &amp; Bascompte, J. (2006). <a
            href="https://doi.org/10.1111/j.1461-0248.2005.00868.x">Habitat
            loss and the structure of plantanimal mutualistic
            networks</a>. <em>Ecology Letters</em>, 9, 281–286.
            </div>
            <div id="ref-Galiana2018Spatiala" class="csl-entry"
            role="doc-biblioentry">
            Galiana, N., Lurgi, M., Claramunt-López, B., Fortin, M.-J.,
            Leroux, S., Cazelles, K., <em>et al.</em> (2018). <a
            href="https://doi.org/10.1038/s41559-018-0517-3">The spatial
            scaling of species interaction networks</a>. <em>Nature
            Ecology &amp; Evolution</em>, 2, 782–790.
            </div>
            <div id="ref-Gilman2010FraCom" class="csl-entry"
            role="doc-biblioentry">
            Gilman, S.E., Urban, M.C., Tewksbury, J., Gilchrist, G.W.
            &amp; Holt, R.D. (2010). <a
            href="https://doi.org/10.1016/j.tree.2010.03.002">A
            framework for community interactions under climate
            change</a>. <em>Trends in Ecology &amp; Evolution</em>, 25,
            325–331.
            </div>
            <div id="ref-Godsoe2022Species" class="csl-entry"
            role="doc-biblioentry">
            Godsoe, W., Murray, R. &amp; Iritani, R. (2022). <a
            href="https://doi.org/10.1111/oik.09282">Species
            interactions and diversity: A unified framework using Hill
            numbers</a>. <em>Oikos</em>, n/a, e09282.
            </div>
            <div id="ref-Golubski2011ModMod" class="csl-entry"
            role="doc-biblioentry">
            Golubski, A.J. &amp; Abrams, P.A. (2011). <a
            href="https://doi.org/10.1111/j.1365-2656.2011.01852.x">Modifying
            modifiers: What happens when interspecific interactions
            interact?</a> <em>Journal of Animal Ecology</em>, 80,
            1097–1108.
            </div>
            <div id="ref-Gomez2010EcoInt" class="csl-entry"
            role="doc-biblioentry">
            Gomez, J.M., Verdu, M. &amp; Perfectti, F. (2010). <a
            href="https://doi.org/10.1038/nature09113">Ecological
            interactions are evolutionarily conserved across the entire
            tree of life</a>. <em>Nature</em>, 465, 918–U6.
            </div>
            <div id="ref-Gonzalez-Varo2016Labilea" class="csl-entry"
            role="doc-biblioentry">
            Gonzalez-Varo, J.P. &amp; Traveset, A. (2016). <a
            href="https://doi.org/10.1016/j.tree.2016.06.009">The Labile
            Limits of Forbidden Interactions</a>. <em>Trends in Ecology
            &amp; Evolution</em>, 31, 700–710.
            </div>
            <div id="ref-Gravel2019BriElt" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Baiser, B., Dunne, J.A., Kopelke, J.-P.,
            Martinez, N.D., Nyman, T., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/ecog.04006">Bringing Elton and
            Grinnell together: A quantitative framework to represent the
            biogeography of ecological interaction networks</a>.
            <em>Ecography</em>, 42, 401–415.
            </div>
            <div id="ref-Gravel2013InfFooa" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Poisot, T., Albouy, C., Velez, L. &amp;
            Mouillot, D. (2013). <a
            href="https://doi.org/10.1111/2041-210X.12103">Inferring
            food web structure from predatorprey body size
            relationships</a>. <em>Methods in Ecology and
            Evolution</em>, 4, 1083–1090.
            </div>
            <div id="ref-Guimaraes2020Structurea" class="csl-entry"
            role="doc-biblioentry">
            Guimarães, P.R. (2020). <a
            href="https://doi.org/10.1146/annurev-ecolsys-012220-120819">The
            Structure of Ecological Networks Across Levels of
            Organization</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 51, 433–460.
            </div>
            <div id="ref-Guimera2009MisSpu" class="csl-entry"
            role="doc-biblioentry">
            Guimerà, R. &amp; Sales-Pardo, M. (2009). <a
            href="https://doi.org/10.1073/pnas.0908366106">Missing and
            spurious interactions and the reconstruction of complex
            networks</a>. <em>Proceedings of the National Academy of
            Sciences</em>, 106, 22073–22078.
            </div>
            <div id="ref-Hemprich-Bennett2021AssImp" class="csl-entry"
            role="doc-biblioentry">
            Hemprich-Bennett, D.R., Oliveira, H.F.M., Le Comber, S.C.,
            Rossiter, S.J. &amp; Clare, E.L. (2021). <a
            href="https://doi.org/10.1002/ecy.3256">Assessing the impact
            of taxon resolution on network structure</a>.
            <em>Ecology</em>, 102, e03256.
            </div>
            <div id="ref-Hortal2015SevSho" class="csl-entry"
            role="doc-biblioentry">
            Hortal, J., de Bello, F., Diniz-Filho, J.A.F., Lewinsohn,
            T.M., Lobo, J.M. &amp; Ladle, R.J. (2015). <a
            href="https://doi.org/10.1146/annurev-ecolsys-112414-054400">Seven
            Shortfalls that Beset Large-Scale Knowledge of
            Biodiversity</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 46, 523–549.
            </div>
            <div id="ref-Jordano2016Samplingb" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12763">Sampling
            networks of ecological interactions</a>. <em>Functional
            Ecology</em>, 30, 1883–1893.
            </div>
            <div id="ref-Jordano2003Invarianta" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P., Bascompte, J. &amp; Olesen, J.M. (2003). <a
            href="https://doi.org/10.1046/j.1461-0248.2003.00403.x">Invariant
            properties in coevolutionary networks of plantanimal
            interactions</a>. <em>Ecology Letters</em>, 6, 69–81.
            </div>
            <div id="ref-Kefi2015NetStr" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Berlow, E.L., Wieters, E.A., Joppa, L.N., Wood,
            S.A., Brose, U., <em>et al.</em> (2015). <a
            href="https://doi.org/10.1890/13-1424.1">Network structure
            beyond food webs: Mapping non-trophic and trophic
            interactions on Chilean rocky shores</a>. <em>Ecology</em>,
            96, 291–303.
            </div>
            <div id="ref-Kefi2012MorMea" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Berlow, E.L., Wieters, E.A., Navarrete, S.A.,
            Petchey, O.L., Wood, S.A., <em>et al.</em> (2012). <a
            href="https://doi.org/10.1111/j.1461-0248.2011.01732.x">More
            than a meal… integrating non-feeding interactions into food
            webs</a>. <em>Ecology Letters</em>, 15, 291–300.
            </div>
            <div id="ref-Kefi2016HowStr" class="csl-entry"
            role="doc-biblioentry">
            Kéfi, S., Miele, V., Wieters, E.A., Navarrete, S.A. &amp;
            Berlow, E.L. (2016). <a
            href="https://doi.org/10.1371/journal.pbio.1002527">How
            Structured Is the Entangled Bank? The Surprisingly Simple
            Organization of Multiplex Ecological Networks Leads to
            Increased Persistence and Resilience</a>. <em>PLOS
            Biology</em>, 14, e1002527.
            </div>
            <div id="ref-Klecka2014EffHab" class="csl-entry"
            role="doc-biblioentry">
            Klecka, J. &amp; Boukal, D.S. (2014). <a
            href="https://doi.org/10.1007/s00442-014-3007-6">The effect
            of habitat structure on prey mortality depends on predator
            and prey microhabitat use</a>. <em>Oecologia</em>, 176,
            183–191.
            </div>
            <div id="ref-Kopelke2017FooStra" class="csl-entry"
            role="doc-biblioentry">
            Kopelke, J.-P., Nyman, T., Cazelles, K., Gravel, D.,
            Vissault, S. &amp; Roslin, T. (2017). <a
            href="https://doi.org/10.1002/ecy.1832">Food-web structure
            of willow-galling sawflies and their natural enemies across
            Europe</a>. <em>Ecology</em>, 98, 1730–1730.
            </div>
            <div id="ref-McCann2007ProBio" class="csl-entry"
            role="doc-biblioentry">
            McCann, K. (2007). Protecting biostructure. <em>Nature</em>,
            446, 29–29.
            </div>
            <div id="ref-McCann2011FooWeb" class="csl-entry"
            role="doc-biblioentry">
            McCann, K.S. (2011). Food webs (MPB-50). In: <em>Food Webs
            (MPB-50)</em>. Princeton University Press.
            </div>
            <div id="ref-McLeod2021Sampling" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A., Leroux, S.J., Gravel, D., Chu, C., Cirtwill,
            A.R., Fortin, M.-J., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/oik.08650">Sampling and
            asymptotic network properties of spatial multi-trophic
            networks</a>. <em>Oikos</em>, n/a.
            </div>
            <div id="ref-McLeod2020EffSpe" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A.M., Leroux, S.J. &amp; Chu, C. (2020). <a
            href="https://doi.org/10.1002/ecs2.3018">Effects of species
            traits, motif profiles, and environment on spatial variation
            in multi-trophic antagonistic networks</a>.
            <em>Ecosphere</em>, 11, e03018.
            </div>
            <div id="ref-Ohlmann2019Diversity" class="csl-entry"
            role="doc-biblioentry">
            Ohlmann, M., Miele, V., Dray, S., Chalmandrier, L.,
            O’Connor, L. &amp; Thuiller, W. (2019). <a
            href="https://doi.org/10.1111/ele.13221">Diversity indices
            for ecological networks: A unifying framework using Hill
            numbers</a>. <em>Ecology Letters</em>, 22, 737–747.
            </div>
            <div id="ref-Olesen2010Missing" class="csl-entry"
            role="doc-biblioentry">
            Olesen, J.M., Bascompte, J., Dupont, Y.L., Elberling, H.,
            Rasmussen, C. &amp; Jordano, P. (2010). <a
            href="https://doi.org/10.1098/rspb.2010.1371">Missing and
            forbidden links in mutualistic networks</a>. <em>Proceedings
            of the Royal Society B: Biological Sciences</em>, 278,
            725–732.
            </div>
            <div id="ref-Pascual2006Ecologicala" class="csl-entry"
            role="doc-biblioentry">
            Pascual, M., Dunne, J.A. &amp; Dunne, J.A. (2006).
            <em>Ecological Networks: Linking Structure to Dynamics in
            Food Webs</em>. Oxford University Press, USA.
            </div>
            <div id="ref-Petchey2008SizFor" class="csl-entry"
            role="doc-biblioentry">
            Petchey, O.L., Beckerman, A.P., Riede, J.O. &amp; Warren,
            P.H. (2008). <a
            href="https://doi.org/10.1073/pnas.0710672105">Size,
            foraging, and food web structure</a>. <em>Proceedings of the
            National Academy of Sciences</em>, 105, 4191–4196.
            </div>
            <div id="ref-Pilosof2017MulNat" class="csl-entry"
            role="doc-biblioentry">
            Pilosof, S., Porter, M.A., Pascual, M. &amp; Kéfi, S.
            (2017). <a
            href="https://doi.org/10.1038/s41559-017-0101">The
            multilayer nature of ecological networks</a>. <em>Nature
            Ecology &amp; Evolution</em>, 1, 1–9.
            </div>
            <div id="ref-Poisot2012Dissimilaritya" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Canard, E., Mouillot, D., Mouquet, N. &amp;
            Gravel, D. (2012). <a
            href="https://doi.org/10.1111/ele.12002">The dissimilarity
            of species interaction networks</a>. <em>Ecology
            Letters</em>, 15, 1353–1361.
            </div>
            <div id="ref-Poisot2016Structure" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D.,
            Fortin, M.-J. &amp; Stouffer, D.B. (2016). <a
            href="https://doi.org/10.1111/2041-210X.12468">The structure
            of probabilistic networks</a>. <em>Methods in Ecology and
            Evolution</em>, 7, 303–312.
            </div>
            <div id="ref-Poisot2015Speciesa" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Stouffer, D.B. &amp; Gravel, D. (2015). <a
            href="https://doi.org/10.1111/oik.01719">Beyond species: Why
            ecological interaction networks vary through space and
            time</a>. <em>Oikos</em>, 124, 243–251.
            </div>
            <div id="ref-Pollock2014UndCoo" class="csl-entry"
            role="doc-biblioentry">
            Pollock, L.J., Tingley, R., Morris, W.K., Golding, N.,
            O’Hara, R.B., Parris, K.M., <em>et al.</em> (2014). <a
            href="https://doi.org/10.1111/2041-210X.12180">Understanding
            co-occurrence bymodelling species simultaneously with a
            Joint Species DistributionModel (JSDM)</a>. <em>Methods in
            Ecology and Evolution</em>, 5, 397–406.
            </div>
            <div id="ref-Pomeranz2019InfPre" class="csl-entry"
            role="doc-biblioentry">
            Pomeranz, J.P.F., Thompson, R.M., Poisot, T. &amp; Harding,
            J.S. (2019). <a
            href="https://doi.org/10.1111/2041-210X.13125">Inferring
            predatorprey interactions in food webs</a>. <em>Methods in
            Ecology and Evolution</em>, 10, 356–367.
            </div>
            <div id="ref-Pringle2020Resolving" class="csl-entry"
            role="doc-biblioentry">
            Pringle, R.M. &amp; Hutchinson, M.C. (2020). <a
            href="https://doi.org/10.1146/annurev-ecolsys-110218-024908">Resolving
            Food-Web Structure</a>. <em>Annual Review of Ecology,
            Evolution, and Systematics</em>, 51, 55–80.
            </div>
            <div id="ref-Proulx2005Networka" class="csl-entry"
            role="doc-biblioentry">
            Proulx, S.R., Promislow, D.E.L. &amp; Phillips, P.C. (2005).
            <a href="https://doi.org/10.1016/j.tree.2005.04.004">Network
            thinking in ecology and evolution</a>. <em>Trends in Ecology
            &amp; Evolution</em>, SPECIAL ISSUE: BUMPER BOOK REVIEW, 20,
            345–353.
            </div>
            <div id="ref-Pulliam1974Theory" class="csl-entry"
            role="doc-biblioentry">
            Pulliam, H.R. (1974). <a
            href="https://doi.org/10.1086/282885">On the Theory of
            Optimal Diets</a>. <em>The American Naturalist</em>, 108,
            59–74.
            </div>
            <div id="ref-Rooney2012IntFoo" class="csl-entry"
            role="doc-biblioentry">
            Rooney, N. &amp; McCann, K.S. (2012). <a
            href="https://doi.org/10.1016/j.tree.2011.09.001">Integrating
            food web diversity, structure and stability</a>. <em>Trends
            in Ecology &amp; Evolution</em>, 27, 40–46.
            </div>
            <div id="ref-Salim2022Data" class="csl-entry"
            role="doc-biblioentry">
            Salim, J.A., Saraiva, A.M., Zermoglio, P.F., Agostini, K.,
            Wolowski, M., Drucker, D.P., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1093/gigascience/giac043">Data
            standardization of plantpollinator interactions</a>.
            <em>GigaScience</em>, 11, giac043.
            </div>
            <div id="ref-Saravia2022Ecological" class="csl-entry"
            role="doc-biblioentry">
            Saravia, L.A., Marina, T.I., Kristensen, N.P., De Troch, M.
            &amp; Momo, F.R. (2022). <a
            href="https://doi.org/10.1111/1365-2656.13652">Ecological
            network assembly: How the regional metaweb influences local
            food webs</a>. <em>Journal of Animal Ecology</em>, 91,
            630–642.
            </div>
            <div id="ref-Schmeller2015GloTer" class="csl-entry"
            role="doc-biblioentry">
            Schmeller, D.S., Julliard, R., Bellingham, P.J., Böhm, M.,
            Brummitt, N., Chiarucci, A., <em>et al.</em> (2015). <a
            href="https://doi.org/10.1016/j.jnc.2015.03.003">Towards a
            global terrestrial species monitoring program</a>.
            <em>Journal for Nature Conservation</em>, 25, 51–57.
            </div>
            <div id="ref-Singer2012GeoMos" class="csl-entry"
            role="doc-biblioentry">
            Singer, M.C. &amp; McBride, C.S. (2012). <a
            href="https://doi.org/10.1890/11-2078.1">Geographic mosaics
            of species’ association: A definition and an example driven
            by plantinsect phenological synchrony</a>. <em>Ecology</em>,
            93, 2658–2673.
            </div>
            <div id="ref-Stouffer2011RolBod" class="csl-entry"
            role="doc-biblioentry">
            Stouffer, D.B., Rezende, E.L. &amp; Amaral, L.A.N. (2011).
            <a
            href="https://doi.org/10.1111/j.1365-2656.2011.01812.x">The
            role of body mass in diet contiguity and food-web
            structure</a>. <em>Journal of Animal Ecology</em>, 80,
            632–639.
            </div>
            <div id="ref-Strydom2022Food" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron,
            D., Farrell, M.J., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1111/2041-210X.13835">Food web
            reconstruction through phylogenetic transfer of low-rank
            network representation</a>. <em>Methods in Ecology and
            Evolution</em>, 13.
            </div>
            <div id="ref-Strydom2021Roadmapa" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Catchen, M.D., Banville, F., Caron, D.,
            Dansereau, G., Desjardins-Proulx, P., <em>et al.</em>
            (2021). <a href="https://doi.org/10.1098/rstb.2021.0063">A
            roadmap towards predicting species interaction networks
            (across space and time)</a>. <em>Philosophical Transactions
            of the Royal Society B-Biological Sciences</em>, 376,
            20210063.
            </div>
            <div id="ref-Tylianakis2007HabMod" class="csl-entry"
            role="doc-biblioentry">
            Tylianakis, J.M., Tscharntke, T. &amp; Lewis, O.T. (2007).
            <a href="https://doi.org/10.1038/nature05429">Habitat
            modification alters the structure of tropical hostparasitoid
            food webs</a>. <em>Nature</em>, 445, 202–205.
            </div>
            <div id="ref-VazquezSS2022Ecological" class="csl-entry"
            role="doc-biblioentry">
            Vázquez, D.P., Peralta, G., Cagnolo, L., Santos, M. &amp;
            Igual, E. autores contribuyeron por. (2022). <a
            href="https://doi.org/10.25260/EA.22.32.2.1.1846">Ecological
            interaction networks. What we know, what we don’t, and why
            it matters</a>. <em>Ecología Austral</em>, 32, 670–697.
            </div>
            <div id="ref-Wood2015Effects" class="csl-entry"
            role="doc-biblioentry">
            Wood, S.A., Russell, R., Hanson, D., Williams, R.J. &amp;
            Dunne, J.A. (2015). <a
            href="https://doi.org/10.1002/ece3.1640">Effects of spatial
            scale of sampling on food web structure</a>. <em>Ecology and
            Evolution</em>, 5, 3769–3782.
            </div>
            <div id="ref-Woodward2010ChaEco" class="csl-entry"
            role="doc-biblioentry">
            Woodward, G., Benstead, J.P., Beveridge, O.S., Blanchard,
            J., Brey, T., Brown, L.E., <em>et al.</em> (2010). <a
            href="https://doi.org/10.1016/B978-0-12-381363-3.00002-2">Chapter
            2 - Ecological Networks in a Changing Climate</a>. In:
            <em>Advances in Ecological Research</em>, Ecological
            Networks (ed. Woodward, G.). Academic Press, pp. 71–138.
            </div>
            <div id="ref-Woodward2012CliCha" class="csl-entry"
            role="doc-biblioentry">
            Woodward, G., Brown, L.E., Edwards, F.K., Hudson, L.N.,
            Milner, A.M., Reuman, D.C., <em>et al.</em> (2012). <a
            href="https://doi.org/10.1098/rstb.2012.0245">Climate change
            impacts in multispecies systems: Drought alters food web
            size structure in a field experiment</a>. <em>Philosophical
            Transactions of the Royal Society B: Biological
            Sciences</em>, 367, 2990–2997.
            </div>
            </div>
        </article>
    </main>

    <footer>
                <div class="container">
            <h2>Supplementary Material</h2>
            <span class="appendix--list">
                <a href="appendix/Banville_et_al_2023_ecological_interpretation_s1.pdf" target="_blank">
                    <b>1:</b>&nbsp;Supplementary Material
                </a><br />
            </span>
        </div>
                <div class="container">
            <h2>Affiliations</h2>
            <sup>1</sup>&nbsp;Université de
            Montréal; <sup>2</sup>&nbsp;Université de
            Sherbrooke; <sup>3</sup>&nbsp;Quebec Centre for Biodiversity
            Science
            <br />
            
        </div>
    </footer>

</body>

</html>