<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>The biological interpretation of probabilistic food webs</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
    integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);
            
            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article > p {
            line-height: 135%;
        }
        article > h1 {
            padding-top: 3em;
        }
        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }
        a:hover {
            font-weight: 500;
        }
        .citation {
            color: var(--citation);
        }
        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }
        header > .container {
            margin-top: 4em;
        }
        code, pre {
            font-family: "Roboto Mono", monospace;
        }
        pre {
            white-space: pre-wrap;
        }
        h1, h2, h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }
        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }
        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }
        .header--authors > span {
            color: var(--light);
            font-weight: 400;
        }
        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }
        footer {
            background-color: var(--grey);
            font-weight: 300;
        }
        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }
        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }
        figure > img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }
        figure > figcaption {
            padding: var(--inner-space);
        }
        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }
        table > caption {
            padding: var(--inner-space);
        }
        figcaption, caption {
            color: var(--light);
            font-weight: 300;
        }
        thead {
            color: var(--structure);
        }
        td, th {
            padding: 3px;
        }
        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }
        tbody > tr:nth-child(even) {background: var(--grey)};
        tbody {
            border-bottom: 1px solid var(--structure);
        }
        #refs {
            font-size: 90%;
            column-count: 2;
        }
        #refs > .csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }
        .hanging-indent > .csl-entry {
            padding-left: 15px ;
            text-indent: -15px ;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.css"
    integrity="sha384-D+9gmBxUQogRLqvARvNLmA9hS2x//eK1FhVb9PiU86gmcrBrJAQT8okdJ4LMp2uv" crossorigin="anonymous" />
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.js"
    integrity="sha384-483A6DwYfKeDa0Q52fJmxFXkcPCFfnXMoXblOkJ4JcA8zATN6Tm78UNL72AKk+0O"
    crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/contrib/auto-render.min.js"
    integrity="sha384-yACMu8JWxKzSp/C1YV86pzGiQ/l1YUfE8oPuahJQxzehAjEt2GiQuy/BIvl9KyeF"
    crossorigin="anonymous"></script>
    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                { left: "$$", right: "$$", display: true },
                { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">The biological interpretation of probabilistic food webs</div>
            <div class="header--authors">
                <span class="header--author">
                <a href="https://orcid.org/0000-0001-9051-0597" target="_blank">
                    Francis Banville
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">2</sup>
                <sup class="affilcode">3</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0001-6067-1349" target="_blank">
                    Tanya Strydom
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">3</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">3</sup>
                </span>
            </div>
        </div>
    </header>
    
    <main class="container">
        <section class="abstract">
            

            <dl>
                    <dt>1</dt>
                <dd>Community ecologists are increasingly thinking probabilistically when it comes to food webs and other ecological networks. Assuredly, the benefits of representing ecological interactions as probabilistic events (e.g., how likely are species to interact?) instead of binary objects (e.g., do species interact?) are numerous, from a better assessment of the spatiotemporal variation of trophic interactions to an increase capacity to reconstruct networks from sparse data. However, probabilities need to be used with caution when working with species interactions. Indeed, depending on the system at hand and the method used to build probabilistic networks, probabilities can have different interpretations that imply different ways to manipulate them. At the core of these differences lie the distinction between assessing the likelihood that two groups of individuals <em>can</em> interact and the likelihood that they <em>will</em> interact. This impacts the spatial, temporal, and taxonomic scaling of interaction probabilities, thus enlightening the need to properly define them in their ecological context. Published data on probabilistic species interactions are poorly documented in the literature, which impedes our ability to use them appropriately. With these challenges in mind, we propose a general approach to thinking about probabilities in regards to ecological interactions, with a strong focus on food webs, and call for better definitions and conceptualizations of probabilistic ecological networks, both at the local and regional scales.</dd>
                </dl>

            <div class="artifacts">
                <a href="draft.pdf" title="Draft PDF" target="_blank">draft</a>
                <a href="preprint.pdf" title="Preprint PDF" target="_blank">preprint</a>
            </div>
        </section>
        
        <article>
            <h1 id="introduction">Introduction</h1>
            <p>Cataloging species interactions across space is a gargantuan task. At the core of this challenge lies the spatiotemporal variability of ecological networks <span class="citation" data-cites="Poisot2012DisSpe Poisot2015SpeWhy">(Poisot <em>et al.</em> 2012, 2015)</span>, which makes documenting the location and timing of interactions difficult. Indeed, it is not sufficient to know if two species have the biological capacity to interact to infer the realization of their interaction at a specific time and space [TK]. Taking food webs as an example, a predator and its potential prey must first co-occur on the same territory in order for a trophic interaction to take place <span class="citation" data-cites="Blanchet2020CooNot">(Blanchet <em>et al.</em> 2020)</span>. They must then encounter, which is conditional on their relative abundances in the ecosystem and the matching of their phenology <span class="citation" data-cites="Poisot2015SpeWhy">(Poisot <em>et al.</em> 2015)</span>. Finally, the interaction occurs only if the predators have a desire to consume their prey and are able to capture and ingest them <span class="citation" data-cites="Pulliam1974TheOpt">(Pulliam 1974)</span>. Environmental (e.g. temperature and presence of shelters) and biological (e.g. physiological state of both species and availability of other prey species) factors contribute to this variability by impacting species co-occurrence [TK] and the realization of their interactions [TK]. In this context, it is unsurprising that computational methods are being developed in ecology to help alleviate the colossal sampling efforts required to document species interactions across time and space <span class="citation" data-cites="Strydom2021RoaPre">(Strydom <em>et al.</em> 2021)</span>. Having a better portrait of species interactions and the emerging structure of food webs is important since it lays the groundwork for understanding the functioning [TK], dynamics [TK], and resilience [TK] of ecosystems worldwide.</p>
            <p>The recognition of the variability of species interactions and the emergence of numerical methods have led ecologists to rethink their representation of ecological networks, slowly moving from a binary to a probabilistic view of species interactions <span class="citation" data-cites="Poisot2016StrProa">(<strong>Poisot2016StrProa?</strong>)</span>. This has several benefits. For example, probabilities represent the limit of our knowledge about species interactions and can indicate the expected frequency of two species interacting with each other [TK]. They are also very helpful in predictive models when modeling the spatial distribution of species <span class="citation" data-cites="Cazelles2016TheSpe">(Cazelles <em>et al.</em> 2016)</span> and the temporal variability of ecological networks [TK], generating new ecological data <span class="citation" data-cites="Strydom2022Food">(e.g., Strydom <em>et al.</em> 2022)</span>, and identifying priority sampling locations <span class="citation" data-cites="Andrade-Pacheco2020FinHot">(see Andrade-Pacheco <em>et al.</em> 2020 for an ecological example of a sampling optimization problem)</span>. Moreover, the high rate of false negatives in ecological network data, resulting from the difficulty of witnessing interactions between rare species, makes it hard to interpret non-observations of species interactions ecologically <span class="citation" data-cites="Catchen2023MisLin">(Catchen <em>et al.</em> 2023)</span>. Using probabilities instead of yes-no interactions accounts for these observation errors; in that case, only forbidden interactions <span class="citation" data-cites="Jordano2003InvPro Olesen2010MisFor">(Jordano <em>et al.</em> 2003; Olesen <em>et al.</em> 2010)</span> would have a probability value of zero <span class="citation" data-cites="Gonzalez-Varo2016LabLim">(but see Gonzalez-Varo &amp; Traveset 2016)</span>.</p>
            <p>However, representing species interactions probabilistically can also be challenging. Beyond methodological difficulties in estimating these numbers, there are important conceptual challenges in defining what we mean by “probability of interactions.” To the best of our knowledge, because the building blocks of this mathematical representation of food webs are still being laid, there is no clear definition found in the literature. This is worrisome, since working with probabilistic species interactions without clear guidelines could be misleading as much for field ecologists as for computational ecologists who use and generate these data. In this contribution, we outline different ways to define and interpret interactions probabilities in network ecology and propose an approach to thinking about them. These definitions mostly depend on the study system (e.g. local network or metaweb) and on the method used to generate them. We show that different definitions can have different ecological implications, especially regarding spatial, temporal, and taxonomic scaling. Although we will focus on food webs, our observations and advice can be applied to all types of ecological networks, from plant-pollinator to host-parasite networks. Specifically, we argue that probabilities should be better documented, defined mathematically, and used with caution when describing species interactions.</p>
            <h1 id="definitions-and-interpretations">Definitions and interpretations</h1>
            <h2 id="overview-of-interaction-probabilities">Overview of interaction probabilities</h2>
            <p>How are interaction probabilities defined in the literature? It might not be as intuitive as one would think.</p>
            <ul>
            <li>There is a big difference in how we interpret the probability that two species <em>can</em> interact (metaweb) and the probability that they <em>will</em> interact (realized networks).</li>
            <li>Interaction probabilities can be used to describe Boolean interactions (e.g., the probability that two species interact) and weighted interactions (e.g., the probability distribution of the amount of energy flow between two species).</li>
            <li>In many studies, it is not obvious if authors use interaction scores or probabilities (in the latter case, it is rarely specified what these probabilities truly represent).</li>
            </ul>
            <h2 id="probabilistic-metawebs">Probabilistic metawebs</h2>
            <p>What does a probability in the context of a metaweb mean?</p>
            <ul>
            <li>It means the probability that two taxa can interact, regardless of biological plasticity, environmental variability, or co-occurrence.</li>
            <li>One observation is enough to set this probability to one.</li>
            <li>Can we turn this into a local network realisation that is also probabilistic and intuitive?</li>
            </ul>
            <p><strong>Papers:</strong> <span class="citation" data-cites="Strydom2022FooWeb">(<strong>Strydom2022FooWeb?</strong>)</span></p>
            <h2 id="probabilistic-local-networks">Probabilistic local networks</h2>
            <p>What does a probability in the context of a local network mean? A cautionary tale of how we define probabilities.</p>
            <ul>
            <li>It means the probability that two taxa will interact at a given location.</li>
            <li>What do we mean by saying that two taxa will interact? We usually mean that at least one individual from one group will interact with (e.g., predate) at least one other individual from the other group.</li>
            <li>The probability is conditional on the environmental and local abundance contexts.</li>
            <li>We should expect a certain number of interactions to be realized depending on the probability value. This number depends on the number of trials, which also depends on the ecological context (e.g., environmental conditions, scale) in which probabilities were estimated. This is in contrast with probabilities in metawebs.</li>
            </ul>
            <h1 id="scaling">Scaling</h1>
            <h2 id="spatial-and-temporal-scales">Spatial and temporal scales</h2>
            <p>How do interaction probabilities scale spatially and temporally?</p>
            <ul>
            <li>Why do probabilistic local food webs scale with area and time but not probabilistic metawebs?</li>
            <li>In metawebs, interaction probabilities do not scale with space and time because they depend solely on the biological capacity of two species to interact.</li>
            <li>In local food webs, interaction probabilities scale with space and time because there are more opportunities of interactions (e.g., more environmental conditions) in a larger area and longer time period.</li>
            <li>What are some network area relationships in probabilistic local food webs?</li>
            <li>We know that local networks can inform regional networks. However, can regional networks inform local networks?</li>
            </ul>
            <p><strong>Figure:</strong> Empirical example of the association between the number of interactions in realized local food webs and the number of interactions in the corresponding species subnetworks of regional networks. We should expect the interaction to be linear below the 1:1 line.</p>
            <p><strong>Papers:</strong> there might be something in these <span class="citation" data-cites="McLeod2020EffSpe">McLeod <em>et al.</em> (2020)</span>; <span class="citation" data-cites="McLeod2021SamAsy">(<strong>McLeod2021SamAsy?</strong>)</span>; <span class="citation" data-cites="Wood2015EffSpa">(<strong>Wood2015EffSpa?</strong>)</span></p>
            <h2 id="taxonomic-scale">Taxonomic scale</h2>
            <p>How do interaction probabilities scale taxonomically?</p>
            <ul>
            <li>There are different biological interpretations of probabilities in food webs at the individual level and at higher taxonomic levels.</li>
            <li>How does the scaling up of the nodes from an individual to population to any higher taxonomic group change our interpretation of interaction probabilities? How does the aggregation change our interpretation?</li>
            <li>How is it similar and different to spatial and temporal scaling? Basically, all kinds of scaling are just different ways to aggregate individuals or nodes.</li>
            </ul>
            <p><strong>Figure:</strong> Conceptual figure of how a scale up of the nodes from an individual to a population to any higher taxonomic group change our interpretation of the probability of interaction.</p>
            <h1 id="concluding-remarks">Concluding remarks</h1>
            <p>Here we present some advice moving forward.</p>
            <ul>
            <li>What can we learn from other systems/fields (e.g., social networks, probabilistic graph theory)?</li>
            <li>What even are probabilities? What is the probability that we will ever know the answer to that?</li>
            <li>Be careful of how we define probabilities. Be sure to be explicit about these things. Be sure to specify the type of interaction, the spatial, temporal, and taxonomic scale when presenting new data on interaction probabilities. We need better metadata documentation.</li>
            <li>Be careful to use and manipulate interaction probabilities properly depending on how they were defined and obtained. Different interpretations imply different scaling, and thus different ways to manipulate these numbers.</li>
            <li>Maybe mention thinking about a workflow to predict probabilistic local food webs from probabilistic metawebs.</li>
            </ul>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
            <div id="ref-Andrade-Pacheco2020FinHot" class="csl-entry" role="doc-biblioentry">
            Andrade-Pacheco, R., Rerolle, F., Lemoine, J., Hernandez, L., Meïté, A., Juziwelo, L., <em>et al.</em> (2020). Finding hotspots: Development of an adaptive spatial sampling approach. <em>Scientific Reports</em>, 10, 10939.
            </div>
            <div id="ref-Blanchet2020CooNot" class="csl-entry" role="doc-biblioentry">
            Blanchet, F.G., Cazelles, K. &amp; Gravel, D. (2020). Co-occurrence is not evidence of ecological interactions. <em>Ecology Letters</em>, 23, 1050–1063.
            </div>
            <div id="ref-Catchen2023MisLin" class="csl-entry" role="doc-biblioentry">
            Catchen, M.D., Poisot, T., Pollock, L.J. &amp; Gonzalez, A. (2023). The missing link: Discerning true from false negatives when sampling species interaction networks.
            </div>
            <div id="ref-Cazelles2016TheSpe" class="csl-entry" role="doc-biblioentry">
            Cazelles, K., Araujo, M.B., Mouquet, N. &amp; Gravel, D. (2016). A theory for species co-occurrence in interaction networks. <em>Theoretical Ecology</em>, 9, 39–48.
            </div>
            <div id="ref-Gonzalez-Varo2016LabLim" class="csl-entry" role="doc-biblioentry">
            Gonzalez-Varo, J.P. &amp; Traveset, A. (2016). The Labile Limits of Forbidden Interactions. <em>Trends in Ecology &amp; Evolution</em>, 31, 700–710.
            </div>
            <div id="ref-Jordano2003InvPro" class="csl-entry" role="doc-biblioentry">
            Jordano, P., Bascompte, J. &amp; Olesen, J.M. (2003). Invariant properties in coevolutionary networks of plantanimal interactions. <em>Ecology Letters</em>, 6, 69–81.
            </div>
            <div id="ref-McLeod2020EffSpe" class="csl-entry" role="doc-biblioentry">
            McLeod, A.M., Leroux, S.J. &amp; Chu, C. (2020). Effects of species traits, motif profiles, and environment on spatial variation in multi-trophic antagonistic networks. <em>Ecosphere</em>, 11, e03018.
            </div>
            <div id="ref-Olesen2010MisFor" class="csl-entry" role="doc-biblioentry">
            Olesen, J.M., Bascompte, J., Dupont, Y.L., Elberling, H., Rasmussen, C. &amp; Jordano, P. (2010). Missing and forbidden links in mutualistic networks. <em>Proceedings of the Royal Society B: Biological Sciences</em>, 278, 725–732.
            </div>
            <div id="ref-Poisot2012DisSpe" class="csl-entry" role="doc-biblioentry">
            Poisot, T., Canard, E., Mouillot, D., Mouquet, N. &amp; Gravel, D. (2012). The dissimilarity of species interaction networks. <em>Ecology Letters</em>, 15, 1353–1361.
            </div>
            <div id="ref-Poisot2015SpeWhy" class="csl-entry" role="doc-biblioentry">
            Poisot, T., Stouffer, D.B. &amp; Gravel, D. (2015). Beyond species: Why ecological interaction networks vary through space and time. <em>Oikos</em>, 124, 243–251.
            </div>
            <div id="ref-Pulliam1974TheOpt" class="csl-entry" role="doc-biblioentry">
            Pulliam, H.R. (1974). On the Theory of Optimal Diets. <em>The American Naturalist</em>, 108, 59–74.
            </div>
            <div id="ref-Strydom2022Food" class="csl-entry" role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron, D., Farrell, M.J., <em>et al.</em> (2022). Food web reconstruction through phylogenetic transfer of low-rank network representation. <em>Methods in Ecology and Evolution</em>, 13.
            </div>
            <div id="ref-Strydom2021RoaPre" class="csl-entry" role="doc-biblioentry">
            Strydom, T., Catchen, M.D., Banville, F., Caron, D., Dansereau, G., Desjardins-Proulx, P., <em>et al.</em> (2021). A roadmap towards predicting species interaction networks (across space and time). <em>Philosophical Transactions of the Royal Society B-Biological Sciences</em>, 376, 20210063.
            </div>
            </div>
        </article>
    </main>

    <footer>
        <div class="container">
            <h2>Affiliations</h2> 
            <sup>1</sup>&nbsp;Université de Montréal; <sup>2</sup>&nbsp;Université de Sherbrooke; <sup>3</sup>&nbsp;Quebec Centre for Biodiversity Science
            <br />
            <sup>‡</sup>&nbsp;These authors contributed equally to the work
        </div>
    </footer>
    
</body>
</html>