<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>The biological interpretation of probabilistic food webs</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
    integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);
            
            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article > p {
            line-height: 135%;
        }
        article > h1 {
            padding-top: 3em;
        }
        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }
        a:hover {
            font-weight: 500;
        }
        .citation {
            color: var(--citation);
        }
        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }
        header > .container {
            margin-top: 4em;
        }
        code, pre {
            font-family: "Roboto Mono", monospace;
        }
        pre {
            white-space: pre-wrap;
        }
        h1, h2, h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }
        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }
        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }
        .header--authors > span {
            color: var(--light);
            font-weight: 400;
        }
        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }
        footer {
            background-color: var(--grey);
            font-weight: 300;
        }
        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }
        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }
        figure > img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }
        figure > figcaption {
            padding: var(--inner-space);
        }
        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }
        table > caption {
            padding: var(--inner-space);
        }
        figcaption, caption {
            color: var(--light);
            font-weight: 300;
        }
        thead {
            color: var(--structure);
        }
        td, th {
            padding: 3px;
        }
        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }
        tbody > tr:nth-child(even) {background: var(--grey)};
        tbody {
            border-bottom: 1px solid var(--structure);
        }
        #refs {
            font-size: 90%;
            column-count: 2;
        }
        #refs > .csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }
        .hanging-indent > .csl-entry {
            padding-left: 15px ;
            text-indent: -15px ;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.css"
    integrity="sha384-D+9gmBxUQogRLqvARvNLmA9hS2x//eK1FhVb9PiU86gmcrBrJAQT8okdJ4LMp2uv" crossorigin="anonymous" />
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/katex.min.js"
    integrity="sha384-483A6DwYfKeDa0Q52fJmxFXkcPCFfnXMoXblOkJ4JcA8zATN6Tm78UNL72AKk+0O"
    crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.0-rc.1/dist/contrib/auto-render.min.js"
    integrity="sha384-yACMu8JWxKzSp/C1YV86pzGiQ/l1YUfE8oPuahJQxzehAjEt2GiQuy/BIvl9KyeF"
    crossorigin="anonymous"></script>
    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                { left: "$$", right: "$$", display: true },
                { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">The biological interpretation of probabilistic food webs</div>
            <div class="header--authors">
                <span class="header--author">
                <a href="https://orcid.org/0000-0001-9051-0597" target="_blank">
                    Francis Banville
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">2</sup>
                <sup class="affilcode">3</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0001-6067-1349" target="_blank">
                    Tanya Strydom
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">3</sup>
                <sup class="affilcode">‡</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affilcode">1</sup>
                <sup class="affilcode">3</sup>
                </span>
            </div>
        </div>
    </header>
    
    <main class="container">
        <section class="abstract">
            Community ecologists are increasingly thinking probabilistically when it comes to food webs and other ecological networks. Assuredly, the benefits of representing ecological interactions as probabilistic events (e.g., how likely are species to interact?) instead of binary objects (e.g., do species interact?) are numerous, from a better assessment of the spatiotemporal variation of trophic interactions to an increase capacity to reconstruct networks from sparse data. However, probabilities need to be used with caution when working with species interactions. Indeed, depending on the system at hand and the method used to build probabilistic networks, probabilities can have different interpretations that imply different ways to manipulate them. At the core of these differences lie the distinction between assessing the likelihood that two groups of individuals <em>can</em> interact and the likelihood that they <em>will</em> interact. This impacts the spatial, temporal, and taxonomic scaling of interaction probabilities, thus enlightening the need to properly define them in their ecological context. Published data on probabilistic species interactions are poorly documented in the literature, which impedes our ability to use them appropriately. With these challenges in mind, we propose a general approach to thinking about probabilities in regards to ecological interactions, with a strong focus on food webs, and call for better definitions and conceptualizations of probabilistic ecological networks, both at the local and regional scales.


            <div class="artifacts">
                <a href="draft.pdf" title="Draft PDF" target="_blank">draft</a>
                <a href="preprint.pdf" title="Preprint PDF" target="_blank">preprint</a>
            </div>
        </section>
        
        <article>
            <h1 id="introduction">Introduction</h1>
            <p>Cataloging species interactions across space is a gargantuan task. At the core of this challenge lies the spatiotemporal variability of ecological networks <span class="citation" data-cites="Poisot2012DisSpe Poisot2015SpeWhy">(<strong>Poisot2012DisSpe?</strong>; <strong>Poisot2015SpeWhy?</strong>)</span>, which makes documenting the location and timing of interactions difficult. Indeed, it is not sufficient to know that two species have the biological capacity to interact to infer the realization of their interaction at a specific time and space <span class="citation" data-cites="Dunne2006Network">(Dunne 2006)</span>. Taking food webs as an example, a predator species and its potential prey must first co-occur on the same territory in order for a trophic interaction to take place <span class="citation" data-cites="Blanchet2020CooNot">(<strong>Blanchet2020CooNot?</strong>)</span>. They must then encounter, which is conditional on their relative abundances in the ecosystem and the matching of their phenology <span class="citation" data-cites="Poisot2015SpeWhy">(<strong>Poisot2015SpeWhy?</strong>)</span>. Finally, the interaction occurs only if the predators have a desire to consume their prey and are able to capture and ingest them <span class="citation" data-cites="Pulliam1974TheOpt">(<strong>Pulliam1974TheOpt?</strong>)</span>. Environmental (e.g. temperature and presence of shelters) and biological (e.g. physiological state of both species and availability of other prey species) factors contribute to this variability by impacting species co-occurrence <span class="citation" data-cites="Araujo2011UsiSpe">(Araujo <em>et al.</em> 2011)</span> and the realization of their interactions <span class="citation" data-cites="Poisot2015SpeWhy">(<strong>Poisot2015SpeWhy?</strong>)</span>. In this context, it is unsurprising that computational methods are being developed in ecology to help alleviate the colossal sampling efforts required to document species interactions across time and space <span class="citation" data-cites="Strydom2021RoaPre">(<strong>Strydom2021RoaPre?</strong>)</span>. Having a better portrait of species interactions and the emerging structure of their food webs is important since it lays the groundwork for understanding the functioning, dynamics, and resilience of ecosystems worldwide <span class="citation" data-cites="Proulx2005NetThi Pascual2006EcoNet Delmas2019AnaEco">(e.g., Proulx <em>et al.</em> 2005; Pascual <em>et al.</em> 2006; Delmas <em>et al.</em> 2019)</span>.</p>
            <p>The recognition of the intrinsic variability of species interactions and the emergence of numerical methods have led ecologists to rethink their representation of ecological networks, slowly moving from a binary to a probabilistic view of species interactions <span class="citation" data-cites="Poisot2016Structure">(Poisot <em>et al.</em> 2016)</span>. This has several benefits. For example, probabilities represent the limit of our knowledge about species interactions and can inform us about the expected number of interactions and emerging network properties despite this limited knowledge <span class="citation" data-cites="Poisot2016Structure">(Poisot <em>et al.</em> 2016)</span>. They are also very helpful in predicting the spatial distribution of species within networks <span class="citation" data-cites="Cazelles2016TheSpe">(<strong>Cazelles2016TheSpe?</strong>)</span> and the temporal variability of interactions <span class="citation" data-cites="Poisot2015Speciesa">(Poisot <em>et al.</em> 2015)</span>, generating new ecological data <span class="citation" data-cites="Strydom2022Food">(e.g., Strydom <em>et al.</em> 2022)</span>, and identifying priority sampling locations of species interactions <span class="citation" data-cites="Andrade-Pacheco2020FinHot">(see <strong>Andrade-Pacheco2020FinHot?</strong> for an ecological example of a sampling optimization problem)</span>. Moreover, the high rate of false negatives in ecological network data, resulting from the difficulty of witnessing interactions between rare species, makes it hard to interpret non-observations of species interactions ecologically <span class="citation" data-cites="Catchen2023MisLin">(<strong>Catchen2023MisLin?</strong>)</span>. Using probabilities instead of yes-no interactions accounts for these observation errors; in that case, only forbidden interactions <span class="citation" data-cites="Jordano2003InvPro Olesen2010MisFor">(<strong>Jordano2003InvPro?</strong>; <strong>Olesen2010MisFor?</strong>)</span> would have a probability value of zero <span class="citation" data-cites="Gonzalez-Varo2016LabLim">(but see <strong>Gonzalez-Varo2016LabLim?</strong>)</span>. Many measures have been developed to describe the structure <span class="citation" data-cites="Poisot2016Structure">(Poisot <em>et al.</em> 2016)</span> and diversity <span class="citation" data-cites="Ohlmann2019DivInd GodsoeSpeInt">(Ohlmann <em>et al.</em> 2019; Godsoe <em>et al.</em> n.d.)</span> of probabilistic interactions, which shows the potential of this framework in the study of a variety of ecological phenomena.</p>
            <p>However, representing species interactions probabilistically can also be challenging. Beyond methodological difficulties in estimating these numbers, there are important conceptual challenges in defining what we mean by “probability of interactions.” To the best of our knowledge, because the building blocks of this mathematical representation of food webs are still being laid, there is no clear definition found in the literature. This is worrisome, since working with probabilistic species interactions without clear guidelines could be misleading as much for field ecologists as for computational ecologists who use and generate these data. In this contribution, we outline different ways to define and interpret interactions probabilities in network ecology and propose an approach to thinking about them. These definitions mostly depend on the study system (e.g. local network or metaweb) and on the method used to generate them. We show that different definitions can have different ecological implications, especially regarding spatial, temporal, and taxonomic scaling. Although we will focus on food webs, our observations and advice can be applied to all types of ecological networks, from plant-pollinator to host-parasite networks. Specifically, we argue that probabilities should be better documented, defined mathematically, and used with caution when describing species interactions.</p>
            <h1 id="definitions-and-interpretations">Definitions and interpretations</h1>
            <p>The basic unit of food webs and other ecological networks are individuals that interact with each others <span class="citation" data-cites="Elton2001AniEco">(e.g., by predation; Elton 2001)</span>. The aggregation of these individuals into more or less homogeneous groups (e.g., populations, species, trophic species) allows us to represent networks at different scales, which impacts the properties and behaviour of these systems <span class="citation" data-cites="Guimaraes2020StrEco">(Guimarães 2020)</span>. A network’s nodes can thus designate distinct levels of organization, whereas the edges linking these nodes can describe a variety of interaction measures. When using a Boolean (yes-no) representation of biotic interactions, the observation that one individual from group (or node) <span class="math inline"><em>i</em></span> interacts with another individual from group <span class="math inline"><em>j</em></span> is enough to set the interaction <span class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub></span> to 1. This simplified representation of food webs is a highly valuable source of ecological information <span class="citation" data-cites="Pascual2006EcoNet">(Pascual <em>et al.</em> 2006)</span> even though it overlooks important factors regarding interaction strengths. These, in turn, can be represented using weighted interactions, which better describe the energy flows, demographic impacts or frequencies of interactions between nodes <span class="citation" data-cites="Berlow2004IntStr Borrett2019WalPar">(Berlow <em>et al.</em> 2004; Borrett &amp; Scharler 2019)</span>, with <span class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> ∈ ℕ</span> or <span class="math inline">ℝ</span> depending on the measure. For example, they can be used to estimate the average number of prey individuals consumed by the predators in a given time period. This extra amount of ecological information typically comes at a cost of greater sampling effort or data requirement in predictive models <span class="citation" data-cites="Strydom2021Roadmapa">(Strydom <em>et al.</em> 2021)</span>, which can lead to high uncertainties when building these types of networks.</p>
            <p>The uncertainty and spatiotemporal variability of both types of biotic interactions (Boolean and weighted) can be represented probabilistically. On one hand, Boolean interactions follow a Bernoulli distribution <span class="math inline">$A_{i,j} \sim {\rm Bernoulli}(p)$</span>, with <span class="math inline"><em>p</em></span> being the probability of interactions. The only two possible outcomes are the presence (<span class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> = 1</span>) or absence (<span class="math inline"><em>A</em><sub><em>i</em>, <em>j</em></sub> = 0</span>) of an interaction between the two nodes. Weighted interactions, on the other hand, can follow various probability distributions depending on the measure used. In this case, the event’s outcome is the value of interaction strength. For instance, weights can follow a Poisson distribution <span class="math inline">$A_{i,j} \sim {\rm Poisson}(\lambda)$</span> when predicting frequencies of interactions between pairs of nodes, with <span class="math inline"><em>λ</em></span> being the expected rate of interaction. The definition and interpretation of parameters like <span class="math inline"><em>p</em></span> and <span class="math inline"><em>λ</em></span> are inextricably linked to environmental and biological factors such as species relative abundance, traits, area, and time, depending on the type of interaction. Because Boolean species interactions are much more documented in the literature, our primary focus in this contribution will be on addressing the challenges in defining and interpretating <span class="math inline"><em>p</em></span> for pairwise species interactions.</p>
            <p>The first aspect to take into consideration when estimating or using probabilities of interactions is knowing if they describe the likelihood of potential or realized interactions. A potential interaction is defined as the biological capacity of two species to interact (i.e., the probability that they <em>can</em> interact) whereas a realized interaction refers to the materialization or observation of this interaction in a delineated space and time period (i.e., the probability that they <em>will</em> interact). Here, we will use the terms <em>metaweb</em> to designate networks of potential interactions and <em>local networks</em> for those of realized interactions. Frequent confusion arises among ecologists over the use of these two terms, especially in a probabilistic context. Indeed, in many studies of probabilistic ecological networks, it remains unclear when authors describe potential or realized interactions, or when so-called probabilities are actually <em>interaction scores</em>. A better understanding of these differences would alleviate interpretation errors and help ecologists use these numbers more appropriately.</p>
            <h2 id="probabilistic-metawebs">Probabilistic metawebs</h2>
            <p>What does a probability in the context of a metaweb mean?</p>
            <p><span class="math display"><em>P</em>(<em>i</em> → <em>j</em>)</span></p>
            <ul>
            <li>It means the probability that two taxa can interact, regardless of biological plasticity, environmental variability, or co-occurrence.</li>
            <li>One observation is enough to set this probability to one.</li>
            <li>Can we turn this into a local network realisation that is also probabilistic and intuitive?</li>
            <li>Low probability with rare interactions?</li>
            </ul>
            <p><strong>Papers:</strong> <span class="citation" data-cites="Strydom2022FooWeb">(<strong>Strydom2022FooWeb?</strong>)</span></p>
            <h2 id="probabilistic-local-networks">Probabilistic local networks</h2>
            <p><span class="math display"><em>P</em>(<em>i</em> → <em>j</em>|<em>C</em>, <em>A</em>, <em>N</em>, <em>E</em>, <em>t</em>)</span></p>
            <p>i = predator j = prey C = co-occurrence (and relative abundance?) A = area N = relative abundance (or network?) E = environment (including network?) t = time</p>
            <p>What does a probability in the context of a local network mean? A cautionary tale of how we define probabilities.</p>
            <ul>
            <li>It means the probability that two taxa will interact at a given location.</li>
            <li>What do we mean by saying that two taxa will interact? We usually mean that at least one individual from one group will interact with (e.g., predate) at least one other individual from the other group.</li>
            <li>The probability is conditional on the environmental and local abundance contexts.</li>
            <li>We should expect a certain number of interactions to be realized depending on the probability value. This number depends on the number of trials, which also depends on the ecological context (e.g., environmental conditions, scale) in which probabilities were estimated. This is in contrast with probabilities in metawebs.</li>
            </ul>
            <h1 id="scaling">Scaling</h1>
            <h2 id="spatial-and-temporal-scales">Spatial and temporal scales</h2>
            <p>How do interaction probabilities scale spatially and temporally?</p>
            <ul>
            <li>Why do probabilistic local food webs scale with area and time but not probabilistic metawebs?</li>
            <li>In metawebs, interaction probabilities do not scale with space and time because they depend solely on the biological capacity of two species to interact.</li>
            <li>In local food webs, interaction probabilities scale with space and time because there are more opportunities of interactions (e.g., more environmental conditions) in a larger area and longer time period.</li>
            <li>What are some network area relationships in probabilistic local food webs?</li>
            <li>We know that local networks can inform regional networks. However, can regional networks inform local networks?</li>
            </ul>
            <p><strong>Figure:</strong> Empirical example of the association between the number of interactions in realized local food webs and the number of interactions in the corresponding species subnetworks of regional networks. We should expect the interaction to be linear below the 1:1 line.</p>
            <p><strong>Papers:</strong> there might be something in these <span class="citation" data-cites="McLeod2020EffSpe">McLeod <em>et al.</em> (2020)</span>; <span class="citation" data-cites="McLeod2021SamAsy">(<strong>McLeod2021SamAsy?</strong>)</span>; <span class="citation" data-cites="Wood2015EffSpa">(<strong>Wood2015EffSpa?</strong>)</span></p>
            <h2 id="taxonomic-scale">Taxonomic scale</h2>
            <p>How do interaction probabilities scale taxonomically?</p>
            <ul>
            <li>There are different biological interpretations of probabilities in food webs at the individual level and at higher taxonomic levels.</li>
            <li>How does the scaling up of the nodes from an individual to population to any higher taxonomic group change our interpretation of interaction probabilities? How does the aggregation change our interpretation?</li>
            <li>How is it similar and different to spatial and temporal scaling? Basically, all kinds of scaling are just different ways to aggregate individuals or nodes.</li>
            </ul>
            <p><strong>Figure:</strong> Conceptual figure of how a scale up of the nodes from an individual to a population to any higher taxonomic group change our interpretation of the probability of interaction.</p>
            <h1 id="concluding-remarks">Concluding remarks</h1>
            <p>Here we present some advice moving forward.</p>
            <ul>
            <li>What can we learn from other systems/fields (e.g., social networks, probabilistic graph theory)?</li>
            <li>What even are probabilities? What is the probability that we will ever know the answer to that?</li>
            <li>Be careful of how we define probabilities. Be sure to be explicit about these things. Be sure to specify the type of interaction, the spatial, temporal, and taxonomic scale when presenting new data on interaction probabilities. We need better metadata documentation.</li>
            <li>Be careful to use and manipulate interaction probabilities properly depending on how they were defined and obtained. Different interpretations imply different scaling, and thus different ways to manipulate these numbers.</li>
            <li>Maybe mention thinking about a workflow to predict probabilistic local food webs from probabilistic metawebs.</li>
            </ul>
            <h1 id="acknowledgement">Acknowledgement</h1>
            <p>We acknowledge that this study was conducted on land within the traditional unceded territory of the Saint Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat, and Omàmiwininiwak nations. This work was supported by the Institute for Data Valorisation (IVADO) and the Natural Sciences and Engineering Research Council of Canada (NSERC) Collaborative Research and Training Experience (CREATE) program, through the Computational Biodiversity Science and Services (BIOS²) program. A special thank to all members of the Black Holes and Revelations working group (organized by BIOS²) for their insightful discussions and valuable feedbacks on this manuscript.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
            <div id="ref-Araujo2011UsiSpe" class="csl-entry" role="doc-biblioentry">
            Araujo, M.B., Rozenfeld, A., Rahbek, C. &amp; Marquet, P.A. (2011). Using species co-occurrence networks to assess the impacts of climate change. <em>Ecography</em>, 34, 897–908.
            </div>
            <div id="ref-Berlow2004IntStr" class="csl-entry" role="doc-biblioentry">
            Berlow, E.L., Neutel, A.-M., Cohen, J.E., De Ruiter, P.C., Ebenman, B., Emmerson, M., <em>et al.</em> (2004). Interaction strengths in food webs: Issues and opportunities. <em>Journal of Animal Ecology</em>, 73, 585–598.
            </div>
            <div id="ref-Borrett2019WalPar" class="csl-entry" role="doc-biblioentry">
            Borrett, S.R. &amp; Scharler, U.M. (2019). Walk partitions of flow in Ecological Network Analysis: Review and synthesis of methods and indicators. <em>Ecological Indicators</em>, 106, 105451.
            </div>
            <div id="ref-Delmas2019AnaEco" class="csl-entry" role="doc-biblioentry">
            Delmas, E., Besson, M., Brice, M.-H., Burkle, L.A., Dalla Riva, G.V., Fortin, M.-J., <em>et al.</em> (2019). Analysing ecological networks of species interactions. <em>Biological Reviews</em>, 94, 16–36.
            </div>
            <div id="ref-Dunne2006Network" class="csl-entry" role="doc-biblioentry">
            Dunne, J.A. (2006). The Network Structure of Food Webs. In: <em>Ecological networks: Linking structure and dynamics</em> (eds. Dunne, J.A. &amp; Pascual, M.). Oxford University Press, pp. 27–86.
            </div>
            <div id="ref-Elton2001AniEco" class="csl-entry" role="doc-biblioentry">
            Elton, C.S. (2001). <em>Animal Ecology</em>. University of Chicago Press, Chicago, IL.
            </div>
            <div id="ref-GodsoeSpeInt" class="csl-entry" role="doc-biblioentry">
            Godsoe, W., Murray, R. &amp; Iritani, R. (n.d.). Species interactions and diversity: A unified framework using Hill numbers. <em>Oikos</em>, n/a, e09282.
            </div>
            <div id="ref-Guimaraes2020StrEco" class="csl-entry" role="doc-biblioentry">
            Guimarães, P.R. (2020). The Structure of Ecological Networks Across Levels of Organization. <em>Annual Review of Ecology, Evolution, and Systematics</em>, 51, 433–460.
            </div>
            <div id="ref-McLeod2020EffSpe" class="csl-entry" role="doc-biblioentry">
            McLeod, A.M., Leroux, S.J. &amp; Chu, C. (2020). Effects of species traits, motif profiles, and environment on spatial variation in multi-trophic antagonistic networks. <em>Ecosphere</em>, 11, e03018.
            </div>
            <div id="ref-Ohlmann2019DivInd" class="csl-entry" role="doc-biblioentry">
            Ohlmann, M., Miele, V., Dray, S., Chalmandrier, L., O’Connor, L. &amp; Thuiller, W. (2019). Diversity indices for ecological networks: A unifying framework using Hill numbers. <em>Ecology Letters</em>, 22, 737–747.
            </div>
            <div id="ref-Pascual2006EcoNet" class="csl-entry" role="doc-biblioentry">
            Pascual, M., Dunne, J.A. &amp; Dunne, J.A. (2006). <em>Ecological Networks: Linking Structure to Dynamics in Food Webs</em>. Oxford University Press, USA.
            </div>
            <div id="ref-Poisot2016Structure" class="csl-entry" role="doc-biblioentry">
            Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D., Fortin, M.-J. &amp; Stouffer, D.B. (2016). The structure of probabilistic networks. <em>Methods in Ecology and Evolution</em>, 7, 303–312.
            </div>
            <div id="ref-Poisot2015Speciesa" class="csl-entry" role="doc-biblioentry">
            Poisot, T., Stouffer, D.B. &amp; Gravel, D. (2015). Beyond species: Why ecological interaction networks vary through space and time. <em>Oikos</em>, 124, 243–251.
            </div>
            <div id="ref-Proulx2005NetThi" class="csl-entry" role="doc-biblioentry">
            Proulx, S.R., Promislow, D.E.L. &amp; Phillips, P.C. (2005). Network thinking in ecology and evolution. <em>Trends in Ecology &amp; Evolution</em>, SPECIAL issue: BUMPER book REVIEW, 20, 345–353.
            </div>
            <div id="ref-Strydom2022Food" class="csl-entry" role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron, D., Farrell, M.J., <em>et al.</em> (2022). Food web reconstruction through phylogenetic transfer of low-rank network representation. <em>Methods in Ecology and Evolution</em>, 13.
            </div>
            <div id="ref-Strydom2021Roadmapa" class="csl-entry" role="doc-biblioentry">
            Strydom, T., Catchen, M.D., Banville, F., Caron, D., Dansereau, G., Desjardins-Proulx, P., <em>et al.</em> (2021). A roadmap towards predicting species interaction networks (across space and time). <em>Philosophical Transactions of the Royal Society B-Biological Sciences</em>, 376, 20210063.
            </div>
            </div>
        </article>
    </main>

    <footer>
        <div class="container">
            <h2>Affiliations</h2> 
            <sup>1</sup>&nbsp;Université de Montréal; <sup>2</sup>&nbsp;Université de Sherbrooke; <sup>3</sup>&nbsp;Quebec Centre for Biodiversity Science
            <br />
            <sup>‡</sup>&nbsp;These authors contributed equally to the work
        </div>
    </footer>
    
</body>
</html>