%!TEX TS-program = xelatex
\documentclass[10pt,oneside]{article}
\usepackage[fontsize=9pt]{scrextend}

\usepackage[english]{babel}

\usepackage{amsmath,amssymb,amsfonts}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{stix2}
\usepackage[scaled]{helvet}
\usepackage[scaled]{inconsolata}

\usepackage{lastpage}

\usepackage{setspace}

\usepackage{ccicons}

\usepackage[hang,flushmargin]{footmisc}

\usepackage{geometry}

\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}

\usepackage{fancyhdr}
\renewcommand{\headrulewidth}{0pt}\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\makeatletter
\newcounter{tableno}
\newenvironment{tablenos:no-prefix-table-caption}{
  \caption@ifcompatibility{}{
    \let\oldthetable\thetable
    \let\oldtheHtable\theHtable
    \renewcommand{\thetable}{tableno:\thetableno}
    \renewcommand{\theHtable}{tableno:\thetableno}
    \stepcounter{tableno}
    \captionsetup{labelformat=empty}
  }
}{
  \caption@ifcompatibility{}{
    \captionsetup{labelformat=default}
    \let\thetable\oldthetable
    \let\theHtable\oldtheHtable
    \addtocounter{table}{-1}
  }
}
\makeatother

\usepackage{array}
\newcommand{\PreserveBackslash}[1]{\let\temp=\\#1\let\\=\temp}
\let\PBS=\PreserveBackslash

\usepackage[breaklinks=true]{hyperref}
\hypersetup{colorlinks,%
citecolor=blue,%
filecolor=blue,%
linkcolor=blue,%
urlcolor=blue}
\usepackage{url}

\usepackage{caption}
\setcounter{secnumdepth}{0}
\usepackage{cleveref}

\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth
\else\Gin@nat@width\fi}
\makeatother
\let\Oldincludegraphics\includegraphics
\renewcommand{\includegraphics}[1]{\Oldincludegraphics[width=\maxwidth]{#1}}

\usepackage{longtable}
\usepackage{booktabs}

\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}

\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for \widthof, \maxof
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\maxof{\widthof{#1}}{\csllabelwidth}}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}\usepackage[table,dvipsnames]{xcolor}

\geometry{includemp,
            letterpaper,
            top=2.4cm,
            bottom=2.4cm,
            left=1.0cm,
            right=1.0cm,
            marginparwidth=5cm,
            marginparsep=1.0cm}

\usepackage[singlelinecheck=off]{caption}

\captionsetup{
  font={small},
  labelfont={bf},
  format=plain,
  labelsep=quad
}

\usepackage{floatrow}

\floatsetup[figure]{margins=hangright,
              facing=no,
              capposition=beside,
              capbesideposition={center,outside},
              floatwidth=\textwidth}

% \floatsetup[table]{margins=hangright,
%              facing=no,
%              capposition=beside,
%              capbesideposition={center,outside},
%              floatwidth=\textwidth}

\pagestyle{plain}

\setcounter{secnumdepth}{5}

\usepackage{titlesec}

\titleformat{\section}[block]
{\normalfont\large\sffamily}
{\thesection}{.5em}{\titlerule\\[.8ex]\bfseries}

\titleformat{\subsection}[runin]
{\normalfont\fontseries{b}\selectfont\filright\sffamily}
{\thesubsection.}{.5em}{}

\titleformat{\subsubsection}[runin]
{\normalfont\itshape\rmfamily\bfseries}{\thesubsubsection}{1em}{}

\fancypagestyle{firstpage}
{
   \fancyhf{}
   \renewcommand{\headrulewidth}{0pt}
   \fancyfoot[R]{\footnotesize\ccby}
   \fancyfoot[L]{\footnotesize\sffamily\today}
}

\fancypagestyle{normal}
{
  \fancyhf{}
  \fancyfoot[R]{\footnotesize\sffamily\thepage\ of \pageref*{LastPage}}
}

\usepackage{tikz}
\begin{document}
\tikz [remember picture, overlay] %
\node [shift={(-0.6in,1.1cm)},scale=0.2,opacity=0.4] at (current page.south east)[anchor=south east]{\includegraphics{logo}};%
\pagestyle{normal}
\thispagestyle{firstpage}

\newcommand{\colorRule}[3][black]{\textcolor[HTML]{#1}{\rule{#2}{#3}}}

\noindent {\LARGE \textbf{\textsf{The biological interpretation of
probabilistic food webs}}}

\medskip
\begin{flushleft}
{\small
%
\href{https://orcid.org/0000-0001-9051-0597}{Francis\,Banville}%
%
\,\textsuperscript{1,2,3,‡}, %
\href{https://orcid.org/0000-0001-6067-1349}{Tanya\,Strydom}%
%
\,\textsuperscript{1,3,‡}, %
\href{https://orcid.org/0000-0002-0735-5184}{Timothée\,Poisot}%
%
\,\textsuperscript{1,3}
\vskip 1em
\textsuperscript{1}\,Université de
Montréal; \textsuperscript{2}\,Université de
Sherbrooke; \textsuperscript{3}\,Quebec Centre for Biodiversity
Science\\
\textsuperscript{‡}\,These authors contributed equally to the work\\
\vskip 1em
\textbf{Correspondance to:}\\
Francis Banville --- \texttt{francis.banville@umontreal.ca}\\
}
\end{flushleft}

\vskip 2em
\makebox[0pt][l]{\colorRule[CCCCCC]{2.0\textwidth}{0.5pt}}
\vskip 2em
\noindent

\marginpar{\vskip 1em\flushright
{\small{\bfseries Keywords}:\par
ecological networks\\food webs\\metaweb\\probabilistic networks\\spatial
scale\\species interactions\\temporal scale\\}
}


Community ecologists are increasingly thinking probabilistically when it
comes to food webs and other ecological networks. Assuredly, the
benefits of representing ecological interactions as probabilistic events
(e.g., how likely are species to interact?) instead of binary objects
(e.g., do species interact?) are numerous, from a better assessment of
the spatiotemporal variation of trophic interactions to an increase
capacity to reconstruct networks from sparse data. However,
probabilities need to be used with caution when working with species
interactions. Indeed, depending on the system at hand and the method
used to build probabilistic networks, probabilities can have different
interpretations that imply different ways to manipulate them. At the
core of these differences lie the distinction between assessing the
likelihood that two groups of individuals \emph{can} interact and the
likelihood that they \emph{will} interact. This impacts the spatial,
temporal, and taxonomic scaling of interaction probabilities, thus
enlightening the need to properly define them in their ecological
context. Published data on probabilistic species interactions are poorly
documented in the literature, which impedes our ability to use them
appropriately. With these challenges in mind, we propose a general
approach to thinking about probabilities in regards to ecological
interactions, with a strong focus on food webs, and call for better
definitions and conceptualizations of probabilistic ecological networks,
both at the local and regional scales.




\vskip 2em
\makebox[0pt][l]{\colorRule[CCCCCC]{2.0\textwidth}{0.5pt}}
\vskip 2em

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

Cataloging species interactions across space is a gargantuan task. At
the core of this challenge lies the spatiotemporal variability of
ecological networks (Poisot \emph{et al.} 2012, 2015), which makes
documenting the location and timing of interactions difficult. Indeed,
it is not sufficient to know that two species have the biological
capacity to interact to infer the realization of their interaction at a
specific time and space (Dunne 2006). Taking food webs as an example, a
predator species and its potential prey must first co-occur in order for
a trophic interaction to take place (Blanchet \emph{et al.} 2020). They
must then encounter, which is conditional on their relative abundances
in the ecosystem and the matching of their phenology (Poisot \emph{et
al.} 2015). Finally, the interaction occurs only if the predators have a
desire to consume their prey and are able to capture and ingest them
(Pulliam 1974). Environmental (e.g.~temperature and presence of
shelters) and biological (e.g.~physiological state of both species and
availability of other prey species) factors contribute to this
variability by impacting species co-occurrence (Araujo \emph{et al.}
2011) and the realization of their interactions (Poisot \emph{et al.}
2015). In this context, the development of computational methods in
ecology can help alleviate the colossal sampling efforts required to
document species interactions across time and space (Strydom \emph{et
al.} 2021). Having a better portrait of species interactions and the
emerging structure of their food webs is important since it lays the
groundwork for understanding the functioning, dynamics, and resilience
of ecosystems worldwide (e.g., Proulx \emph{et al.} 2005; Pascual
\emph{et al.} 2006; Delmas \emph{et al.} 2019).

The recognition of the intrinsic variability of species interactions and
the emergence of numerical methods have led ecologists to rethink their
representation of ecological networks, slowly moving from a binary to a
probabilistic view of species interactions (Poisot \emph{et al.} 2016).
This has several benefits. For example, probabilities represent the
limit of our knowledge about species interactions and can inform us
about the expected number of interactions and emerging network
properties despite this limited knowledge (Poisot \emph{et al.} 2016).
They are also very helpful in predicting the spatial distribution of
species within networks (Cazelles \emph{et al.} 2016) and the temporal
variability of interactions (Poisot \emph{et al.} 2015), generating new
ecological data (e.g., Strydom \emph{et al.} 2022), and identifying
priority sampling locations of species interactions (see Andrade-Pacheco
\emph{et al.} 2020 for an ecological example of a sampling optimization
problem). Moreover, the high rate of false negatives in ecological
network data, resulting from the difficulty of witnessing interactions
between rare species, makes it hard to interpret non-observations of
species interactions ecologically (Catchen \emph{et al.} 2023). Using
probabilities instead of yes-no interactions accounts for these
observation errors; in that case, only forbidden interactions (Jordano
\emph{et al.} 2003; Olesen \emph{et al.} 2010) would have a probability
value of zero (but see Gonzalez-Varo \& Traveset 2016). Many measures
have been developed to describe the structure (Poisot \emph{et al.}
2016) and diversity (Ohlmann \emph{et al.} 2019; Godsoe \emph{et al.}
2022) of probabilistic interactions, which shows the potential of this
framework in the study of a variety of ecological phenomena.

However, representing species interactions probabilistically can also be
challenging. Beyond methodological difficulties in estimating these
numbers, there are important conceptual challenges in defining what we
mean by ``probability of interactions.'' To the best of our knowledge,
because the building blocks of this mathematical representation of food
webs are still being laid, there is no clear definition found in the
literature or data standard when it comes to publishing data on
probabilistic interactions (see Salim \emph{et al.} 2022 for a
discussion on data standardization for mutualistic networks). This is
worrisome, since working with probabilistic species interactions without
clear guidelines could be misleading as much for field ecologists as for
computational ecologists who use and generate these data. In this
contribution, we outline different ways to define and interpret
interactions probabilities in network ecology and propose an approach to
thinking about them. These definitions mostly depend on the study system
(e.g.~local network or metaweb) and on the method used to generate them.
We show that different definitions can have different ecological
implications, especially regarding spatial, temporal, and taxonomic
scaling. Although we will focus on food webs, our observations and
advice can be applied to all types of ecological networks, from
plant-pollinator to host-parasite networks. Indeed, all ecological
networks, whether they are unipartite or bipartite, share fundamental
commonalities in their biological conceptualization and mathematical
representation that support these comparisons (i.e., they all describe
groups of individuals interacting with each other). Regardless of the
study system, we argue that probabilities should be better documented,
defined mathematically, and used with caution when describing species
interactions.

\hypertarget{definitions-and-interpretations}{%
\section{Definitions and
interpretations}\label{definitions-and-interpretations}}

\hypertarget{food-web-representations}{%
\subsection{Food-web representations}\label{food-web-representations}}

The basic unit of food webs and other ecological networks are
individuals that interact with each others (e.g., by predation; Elton
2001), forming individual-based networks. The aggregation of these
individuals into more or less homogeneous groups (e.g., populations,
species, trophic species, families) allows us to represent networks at
broader scales, which impacts the properties and behaviour of these
systems (Guimarães 2020). A network's nodes can thus designate distinct
levels of organization, whereas the edges linking these nodes can
describe a variety of interaction measures. When using a Boolean
(yes-no) representation of biotic interactions, the observation that one
individual from group (or node) \(i\) interacts with another individual
from group \(j\) is enough to set the interaction \(A_{i,j}\) to 1. This
simplified representation of food webs is a highly valuable source of
ecological information (Pascual \emph{et al.} 2006) even though it
overlooks important factors regarding interaction strengths. These, in
turn, can be represented using weighted interactions, which better
describe the energy flows, demographic impacts or frequencies of
interactions between nodes (Berlow \emph{et al.} 2004; Borrett \&
Scharler 2019), with \(A_{i,j} \in \mathbb{N}\) or \(\mathbb{R}\)
depending on the measure. For example, they can be used to estimate the
average number of prey individuals consumed by the predators in a given
time period (e.g., the average number of fish in the stomach of a
piscivorous species). Interaction strengths can also be used as good
estimators of the parameters describing species interactions in a
Lotka-Volterra model (e.g., Emmerson \& Raffaelli 2004). This extra
amount of ecological information typically comes at a cost of greater
sampling effort or data requirement in predictive models (Strydom
\emph{et al.} 2021), which can lead to high uncertainties when building
these types of networks. Therefore, important methodological and
conceptual decisions must be made when sampling and building food webs.

The uncertainty and spatiotemporal variability of both types of trophic
interactions (Boolean and weighted) can be represented
probabilistically. On one hand, Boolean interactions follow a Bernoulli
distribution \(A_{i,j} \sim {\rm Bernoulli}(p)\), with \(p\) being the
probability of interactions. The only two possible outcomes are the
presence (\(A_{i,j} = 1\)) or absence (\(A_{i,j} = 0\)) of an
interaction between the two nodes. Weighted interactions, on the other
hand, can follow various probability distributions depending on the
measure used. In this case, the event's outcome is the value of
interaction strength. For instance, weights can follow a Poisson
distribution \(A_{i,j} \sim {\rm Poisson}(\lambda)\) when predicting
frequencies of interactions between pairs of nodes, with \(\lambda\)
being the expected rate of interaction. Note that weighted interactions
can be converted to probabilistic interactions by normalizing. The
definition and interpretation of parameters like \(p\) and \(\lambda\)
are inextricably linked to environmental and biological factors such as
species relative abundance, traits, area, and time, depending on the
type of interaction. Because Boolean species interactions are much more
documented in the literature, our primary focus in this contribution
will be on addressing the challenges in defining and interpretating
\(p\) for pairwise species interactions.

The first aspect to take into consideration when estimating or
interpreting probabilities of interactions is knowing if they describe
the likelihood of potential or realized interactions. A potential
interaction is defined as the biological capacity of two species to
interact (i.e., the probability that they \emph{can} interact) whereas a
realized interaction refers to the materialization or observation of
this interaction in a delineated space and time period (i.e., the
probability that they interact). Here, we will use the terms
\emph{metaweb} to designate networks of potential interactions and
\emph{local networks} for those of realized interactions. Frequent
confusion arises among ecologists over the use of these two terms,
especially in a probabilistic context. Indeed, in many studies of
probabilistic ecological networks, it remains unclear when authors
describe potential or realized interactions, or when so-called
probabilities are actually \emph{interaction scores}. Likewise,
probabilistic potential interactions are often used as realized
interactions (and conversely), even when the type of interaction is
clearly indicated. We believe that a better understanding of these
differences and concepts would alleviate interpretation errors and help
ecologists use these numbers more appropriately.

\hypertarget{probabilistic-metawebs}{%
\subsection{Probabilistic metawebs}\label{probabilistic-metawebs}}

Metawebs are networks of potential interactions, representing the
probability that two taxa can interact regardless of biological
plasticity, environmental variability or co-occurrence. Instead of
describing stochastic biological processes occurring in nature,
probabilistic potential interactions can be thought of as a measure of
imperfect knowledge about the capacity of two taxa to interact. They are
the network analogue to the species pool, where local networks originate
from a subset of both species (nodes) and interactions (edges). For this
reason, they have been initially designed for broad spatial, temporal,
and taxonomic scales (e.g, species food webs at the continental scale).
However, in the next section, we argue that this concept can also be
used at smaller scales, with similar ecological meaning.

We can express the probability that two taxa \(i\) and \(j\) can
interact in a metaweb \(M\) as

\begin{equation}\protect\hypertarget{eq:metaweb}{}{P_{M}(i \rightarrow j),}\label{eq:metaweb}\end{equation}

which is context independent. In other words, the probability that two
species can interact is not contingent on location, time, and
environmental factors. Nevertheless, one aspect of a metaweb that could
be conditional on these factors is the list of species (or taxa) it is
built from when assembled for a specific region.

Starting from a selected set of species, metawebs can be built using
different data sources, including literature review, field work, and
predictive models (e.g., the metaweb of Canadian mammals inferred by
Strydom \emph{et al.} 2022). Every pair of species that has been
observed to interact at least once can be given a probability of
interaction of \(1\); we know that they \emph{can} interact. This means
that rare interactions can technically be given high probabilities in
the metaweb. Unobserved interactions, on the other hand, are given lower
probabilities, going as low as \(0\) for forbidden links. Two important
nuances must however be made here. Because of observation errors due to
taxonomic misidentifications and ecological misinterpretations (e.g.,
due to cryptic species and interactions; Pringle \& Hutchinson 2020),
many observations of interactions are actually false positives.
Similarly, forbidden interactions can be false negatives if e.g.~they
have been assessed for specific phenotypes, locations or time.
Implementing a Bayesian framework, which updates prior probabilities of
interactions with empirical data, could lessen these errors.

\hypertarget{probabilistic-local-networks}{%
\subsection{Probabilistic local
networks}\label{probabilistic-local-networks}}

As opposed to metawebs, probabilistic local food webs represent the
likelihood that two species will interact at a specific location and
within a given time period; in other words, they are context dependant.
They could also represent the likelihood of observing these interactions
within a given area and time. To be specific, space is defined here as
the geographic coordinates \((x, y)\) of the spatial boundaries
delineating the system, whereas time is the time interval \(t\) during
which interactions were sampled or for which they were predicted. We
want to point out that they are not single values, but rather continued
dimensions that could be outlined differently depending on the study
system. Regardless of how they were defined, they always delineate a
specific area \(A\) and duration \(t\). These could refer to the sampled
area and duration or to the targeted location and time period.

Many factors could be taken into consideration when estimating the
probability that a predator species \(i\) interacts with a given prey
species \(j\) locally. One of the most important is species
co-occurrence \(C\), which is a Boolean describing if both species can
be found at location and time (\(x\), \(y\), \(t\)). Surely, the
probability that the interaction is realized must be \(0\) when species
do not co-occur (\(C = 0\)). Interaction probabilities can also be
conditional on other biological and environmental variables, such as
temperature, precipitation, presence of shelters, phenotypic plasticity,
phenology, and presence of other interacting species in the network.
These conditions can affect species traits, which greatly impact the
capacity of species to interact (Poisot \emph{et al.} 2015). Similarly,
species relative abundance is another important predictor of the
probability of interaction, because it impacts the probability that
species will randomly encounter (Canard \emph{et al.} 2012; Canard
\emph{et al.} 2014; Poisot \emph{et al.} 2015). Here, we will use the
variable \(\Omega\) as a substitute for the biological and ecological
context in which interaction probabilities were estimated, including the
presence of higher-order interactions. This gives us the following
equation for the probability of realized interaction between species (or
taxa) \(i\) and \(j\) in a local network \(N\):

\begin{equation}\protect\hypertarget{eq:local}{}{P_{N}(i \rightarrow j | A, t, C, \Omega),}\label{eq:local}\end{equation}

which can be read as the probability of local interaction between the
two species in an area \(A\) and time interval \(t\), given their
co-occurrence \(C\) and specific environmental and biological conditions
\(\Omega\). These conditions do not systematically need to be specified
for all studies. However, when they are, they should be made explicit in
the metadada.

Multiple difficulties of interpretation arise when the conditions are
not clearly specified, which we found is often the case in the
literature. For example, if \(P_{N}(i \rightarrow j | C = 1)\)
represents the probability that two co-occurring species interact (i.e.,
the edge's probability value), \(P_{N}(i \rightarrow j)\) denotes
instead the probability of interaction without knowing if they co-occur
(i.e., the product of the nodes and edge's probability values). For
practical reasons, probabilistic ecological networks are generally
represented as matrices of probabilities (i.e., matrices of edges
without node values), whose elements are thus hard to interpret without
clear indications about \(C\). Overall, when probabilities of
interactions are estimated using specific values of \(A\), \(t\), \(C\),
and \(\Omega\), ecologists should make them explicit in their metadata,
preferably using mathematical equations to avoid any ambiguity. Below,
we will see examples of why this matters when it comes to spatial,
temporal, and taxonomic scaling of biotic interactions.

{[}Table 1 about here{]}. Articles using probabilistic interactions and
the definitions and variables they considered.

\hypertarget{applications-of-probabilistic-interactions-data}{%
\section{Applications of probabilistic interactions
data}\label{applications-of-probabilistic-interactions-data}}

\hypertarget{inferring-probabilistic-local-food-webs-from-metawebs}{%
\subsection{Inferring probabilistic local food webs from
metawebs}\label{inferring-probabilistic-local-food-webs-from-metawebs}}

Even though the spatiotemporal variability of interactions is not
considered in metawebs, they can still be useful to reconstruct local
networks of realized interactions. Indeed, local networks are formed
from subsets of their metaweb (called subnetworks), which are obtained
by selecting a subset of both species and interactions (Dunne 2006).
Because a community's composition is arguably easier to sample (or
predict) than its interactions, the biggest challenge is to sample links
from the metaweb. This becomes a conceptual issue when we consider how
potential and realized interactions differ. Despite these concerns,
metawebs remain an important source of ecological information that can
be leveraged for realistically predicting spatially explicit networks.
First, metawebs set the upper limit of species interactions (McLeod
\emph{et al.} 2021), i.e. the probability that two species interact at a
specific location is always lower or equal to the probability of their
potential interaction:

\begin{equation}\protect\hypertarget{eq:switch}{}{P_{N}(i \rightarrow j | A, t, C, \Omega) \le
P_M(i \rightarrow j).}\label{eq:switch}\end{equation}

Therefore, inferring local networks from their metaweb keeping the same
values of interaction probability would generate systematic biases in
the prediction. In that case, these networks would instead be called
\emph{spatially explicit} or \emph{local} metawebs (i.e., smaller-scale
networks of potential interactions). Second, the structure of local
networks is constrained by the one of their metaweb (Saravia \emph{et
al.} 2022). This suggests that a metaweb not only constrains the
pairwise interactions of its corresponding local networks, but also
their emerging properties. Inferring the structure of local networks
from the metaweb could thus help estimate more realistically the
likelihood that potential interactions are realized and observed locally
(Strydom \emph{et al.} 2021).

{[}Figure 1 about here{]}. Empirical example of the association between
the number of interactions in realized local food webs and the number of
interactions in the corresponding subnetworks of their regional metaweb.
We should expect the association to be linear below the 1:1 line,
illustrating eq.~\ref{eq:switch}.

\hypertarget{sampling-random-draws-from-probabilistic-food-webs}{%
\subsection{Sampling random draws from probabilistic food
webs}\label{sampling-random-draws-from-probabilistic-food-webs}}

Another conceptual challenge encountered when using probabilistic food
webs is the prediction of Boolean networks across space. Lets take
\(n \times n\) grid cells each representing a probabilistic food web. If
they contain potential interactions, a single random trial must be
conducted for each pairwise interaction across the region (i.e., we
should have only one random realization of the regional metaweb). On the
contrary, if they represent probabilities of realized interactions, each
food web must be independently sampled (i.e., \(n^2\) independent random
draws). This has direct implications on the spatial scaling of
interactions. For example, let \(N_2\) be another network of area
\(A_2 < A_0\) within \(A_0\) and disjoint from \(N_1\), such as \(N_1\)
and \(N_2\) form \(N_0\) (think of two contiguous cells that together
delineate \(N_0\)). All other things being equal, we should expect the
probability that \(i\) and \(j\) interacts in \(A_0\) to be
\(P_{N_0}(i \rightarrow j) = 1 - (1 - P_{N_1}(i \rightarrow j)) \times (1 - P_{N_2}(i \rightarrow j))\)
if \(N_1\) and \(N_2\) are independently sampled. This also implies that
we should expect interactions to be realized in a certain number of
local networks depending on the probability value, which is not the case
with metawebs. Note that spatial auto-correlation and the concept of
meta-network (i.e., networks of networks) could invalidate the
statistical assumption of independence. Nevertheless, the fundamental
difference in sampling metawebs and local networks stands even when
considering these factors. This difference in sampling further sheds
light on the importance of clearly defining interaction probabilities.
What we consider as a \emph{Bernoulli trial}, when randomly drawing
deterministic networks from probabilistic food webs, depends on our
biological interpretation of these probabilities.

\hypertarget{describing-the-spatial-and-temporal-scaling-of-probabilistic-interactions}{%
\subsection{Describing the spatial and temporal scaling of probabilistic
interactions}\label{describing-the-spatial-and-temporal-scaling-of-probabilistic-interactions}}

Metawebs and local networks intrinsically differ in their relation to
scale. On one hand, as mentioned above, probabilistic metawebs are
context independent, i.e., probabilistic pairwise interactions do not
scale with space and time because they depend solely on the biological
capacity of the two taxa to interact. This implies that the estimated
likelihood that two species can potentially interact should be the same
among all metawebs in which they are present. In practice, this is
rarely the case because ecologists use different methods and data to
estimate these probabilities of interactions (e.g., different sampling
area and time period). However, in the case where local metawebs
\(M_{x,y}\) are subsampled from their regional counterpart \(M_0\), we
should expect edge values to be identical among all networks, regardless
of their spatial scale,
i.e.~\(P_{M_{x,y}}(i \rightarrow j) = P_{M_0}(i \rightarrow j)\). On the
other hand, local probabilistic networks are indissociable from their
spatial and temporal contexts because there are more opportunities of
interactions (e.g., more individuals, more trait variations, more chance
of encounter) in a larger area and longer time period (McLeod \emph{et
al.} 2020). Let \(N_0\) be a local probabilistic food web delineated in
an area \(A_0\) and \(N_1\) a network of area \(A_1 < A_0\) within
\(A_0\). We should expect the probability that \(i\) and \(j\) interacts
to scale spatially,
i.e.~\(P_{N_1}(i \rightarrow j | A < A_0) \le P_{N_0}(i \rightarrow j | A = A_0)\).
Similarly, the temporal scaling of probabilistic local food webs could
be manifested through the effect of sampling effort on the observation
of interactions (Jordano 2016; McLeod \emph{et al.} 2021) or of time
itself on their realization (Poisot \emph{et al.} 2012). There are many
network-area relationships (e.g., Wood \emph{et al.} 2015; Galiana
\emph{et al.} 2018) and interaction accumulation curves (e.g, Jordano
2016) explored in the literature. These could inspire the development
and testing of different equations describing the spatiotemporal scaling
of probabilistic pairwise interactions in local networks, which are over
the scope of this manuscript.

{[}Figure 2 about here{]}. Conceptual figure showing (1) the
spatiotemporal scaling of probabilistic metawebs and local food webs and
(2) the spatial sampling of metawebs and local food webs into Boolean
networks.

\hypertarget{making-probabilistic-interactions-spatiotemporally-explicit}{%
\subsection{Making probabilistic interactions spatiotemporally
explicit}\label{making-probabilistic-interactions-spatiotemporally-explicit}}

The variability of species interactions spurred the development of
methods aiming at predicting ecological networks at fine spatial and
temporal scales. For example, Bohan \emph{et al.} (2017) proposed a
framework to reconstruct networks in real time using continuous
biomonitoring. Here, we will build on these studies by proposing a
simple model to make probabilistic local networks spatiotemporally
explicit. These types of models could prove useful when inferring food
webs across time and space from sparse data. However, they are not
suitable for metawebs, which are static objects.

One way that probabilistic food webs can be made spatiotemporally
explicit is by modelling interactions between co-occurring species as a
Poisson process with rate \(\lambda\). Specifically, if the total
observation time for a location is \(t_0\), the probability that two
co-occurring species \(i\) and \(j\) will interact during this time
period is
\(P_N(i \rightarrow j | C_{i,j} = 1, t = t_0) = 1-e^{-\lambda t_0}\),
which approaches \(1\) when \(t \to \infty\). The value of the parameter
\(\lambda\) could be estimated using prior data on interaction
strengths, if available. Additionally, we can estimate the probability
of co-occurrence at location \((x,y)\) with
\(P_{x,y}(C_{i,j} = 1) = P_{x,y}(i) P_{x,y}(j)\gamma\), where
\(P_{x,y}(i)\) and \(P_{x,y}(j)\) are respectively the probabilities of
occurrence of species \(i\) and \(j\) and \(\gamma\) is the strength of
association between occurrence and co-occurrence, as defined in Cazelles
\emph{et al.} (2016). Note that in empirical networks, \(\gamma\) is
typically \(> 1\) (Catchen \emph{et al.} 2023). The observation of this
interaction would thus follow a Bernoulli distribution with parameter
\(p = p_A(x,y)p_B(x,y)\gamma(1-e^{-\lambda t_0})\). This simple model
could be customized in many ways, e.g.~by linking \(\lambda\) with given
environmental variables or by adding in observation errors (i.e.,
probability of false negatives and false positives; Catchen \emph{et
al.} (2023)).

\hypertarget{exploring-different-taxonomic-resolutions}{%
\subsection{Exploring different taxonomic
resolutions}\label{exploring-different-taxonomic-resolutions}}

The properties of ecological networks depend on their level of
organization (Guimarães 2020). Indeed, at different taxonomic scales,
different behaviours and dynamics can be observed and distinct
ecological questions can be answered (e.g., exploring evolutionary
dynamics at broad taxonomic scales). Because of these reasons, it could
be important to analyse the same network at different taxonomic scales.
However, we want to emphasize here that many networks do not have an
homogenous level of organisation (Vázquez \emph{et al.} 2022). Indeed,
different nodes within the same network can be represented at different
taxonomic scales (e.g., a network composed of species and trophic
species). This becomes important when we consider that the biological
interpretation of interaction probabilities depends on the nodes'
resolution. For example, in individual-based networks, the probability
that two individuals interact could represent the degree of belief that
one will actually consume the other. In species-based networks, the
probability that two species interact could rather represent the degree
of belief that \emph{at least} one individual from the predator species
will eat \emph{at least} another individual from the prey species. This
distinction in interpretation impacts the way probability values change
with taxonomic scale.

There are a lot of similarities between taxonomic and spatiotemporal
scaling of probabilistic interactions. Fundamentally, these types of
scaling are just different ways to aggregate individuals into broader
nodes, either spatially, temporally, or taxonomically. However, there
are also important differences between them. First, in metawebs, if we
know that two species have the capacity to interact, we can infer that
their respective genus should also be able to interact (i.e., there
should be at least two individuals within these genus that can
interact). On the contrary, knowing that two genus can interact does not
mean that all pairwise combinations of species within these genus can
also interact among themselves. This observation also applies to local
networks. When it comes to probabilistic networks, interaction
probabilities at broader taxonomic scales can be directly obtained from
probabilities at finer scales when aggregating nodes. For example, if we
have in a network \(n_A\) species from genus \(A\) and \(n_B\) species
from genus \(B\), we can calculate the probability that the two genus
interact as
\(P_N(A \rightarrow B) = 1 - \prod_{i = 1}^{n_1}\prod_{j = 1}^{n_2}(1 - P_N(A_i \rightarrow B_j))\),
where \(A_i\) and \(B_j\) are the species of the corresponding genus.
However, more sophisticated models need to be used when building
probabilistic networks at smaller taxonomic resolutions (e.g., when
building a species-level network from a genus-level network). One could,
for example, estimate the probabilities of all pairwise species
interactions by using a Beta distribution parametrised by the
broader-scale network.

{[}Figure 3 about here{]}. Conceptual figure of how a scale up of the
nodes from an individual to a population to any higher taxonomic group
change our interpretation of the probability of interaction.

\hypertarget{conclusion}{%
\section{Conclusion}\label{conclusion}}

The emergence of probabilistic thinking in network ecology has paved the
way to a better assessment of the spatiotemporal variability and
uncertainty of biotic interactions. However, measuring probabilities
empirically can be strenuous given the difficulties of deciphering
species and interactions (Pringle \& Hutchinson 2020). In this context,
the development of computational methods makes it possible to estimate
interaction probabilities at large scales, which in turn can pinpoint
where we should go to optimise our sampling effort for better resolving
local food webs.

In this contribution, we showed that network metadata are perhaps as
important as interaction data themselves when it comes to interpreting
probabilistic food webs in ecological terms. First, the type of
probabilistic interaction (potential or realized) impacts the importance
of scale, with interactions in metawebs being scale independent, both
spatially and temporally. Second, the conditions in which local networks
were estimated (e.g., area, time, biological and environmental factors)
and the attributes of the interacting species that were considered
(e.g., species co-occurrence) are essential contextual factors that
impact the mathematical representation of probabilities and their
resulting behaviour. Third, the biological interpretation of
probabilities changes with the level of organization of the network due
to the aggregation of individuals into broader groups. All these
information should be available as clear metadata so that ecologists can
use probabilistic network data appropriately.

Moreover, many statistical models in ecology that yield accurate
predictions of biotic interactions are black boxes difficult to
interpret. Ecologists should be careful before using the output of these
models as probabilistic objects, since there is often a thin line
between a real probability and a non-probabilistic predictive number (or
score). Probabilities are numbers between \(0\) and \(1\) that sum to
\(1\) and either represent the expected frequency of a phenomenon or the
degree of belief that it will be realized. Non-probabilistic scores,
which are more akin to interaction weights, have different mathematical
properties, which impacts how we should handle these numbers in a
spatially or temporally explicit context. Therefore, researchers should
use their expertise to assess if their interaction data are actually
probabilities or scores. This should also be added to the metadata
before sharing them, as well as the methods used to build the networks.

Better metadata documentation would allow researchers to use and
manipulate probabilistic ecological interactions according to how they
were actually defined and obtained. This would support better scientific
practices, in particular when these data are used for ecological
prediction and forecasting. For instance, we showed that building a
rigorous workflow to predict local networks from a probabilistic metaweb
requires a good understanding of the data at hand. Similarly, explicitly
stating the context in which probabilistic data were estimated would
help using forecasting food-web models more rigorously under specific
climate change and habitat use scenarios. Regardless of the method and
application, fostering a better foundation for probabilistic reasonings
in network ecology, from the very nature of probabilities and biotic
interactions, is essential.

\hypertarget{acknowledgement}{%
\section{Acknowledgement}\label{acknowledgement}}

We acknowledge that this study was conducted on land within the
traditional unceded territory of the Saint Lawrence Iroquoian,
Anishinabewaki, Mohawk, Huron-Wendat, and Omàmiwininiwak nations. This
work was supported by the Institute for Data Valorisation (IVADO) and
the Natural Sciences and Engineering Research Council of Canada (NSERC)
Collaborative Research and Training Experience (CREATE) program, through
the Computational Biodiversity Science and Services (BIOS²) program. A
special thank to all members of the Black Holes and Revelations working
group (organized by BIOS²) for their insightful discussions and valuable
feedbacks on this manuscript.

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\hypertarget{ref-Andrade-Pacheco2020Finding}{}%
Andrade-Pacheco, R., Rerolle, F., Lemoine, J., Hernandez, L., Meïté, A.,
Juziwelo, L., \emph{et al.} (2020). Finding hotspots: Development of an
adaptive spatial sampling approach. \emph{Scientific Reports}, 10,
10939.

\leavevmode\hypertarget{ref-Araujo2011Usinga}{}%
Araujo, M.B., Rozenfeld, A., Rahbek, C. \& Marquet, P.A. (2011). Using
species co-occurrence networks to assess the impacts of climate change.
\emph{Ecography}, 34, 897--908.

\leavevmode\hypertarget{ref-Berlow2004Interaction}{}%
Berlow, E.L., Neutel, A.-M., Cohen, J.E., De Ruiter, P.C., Ebenman, B.,
Emmerson, M., \emph{et al.} (2004). Interaction strengths in food webs:
Issues and opportunities. \emph{Journal of Animal Ecology}, 73,
585--598.

\leavevmode\hypertarget{ref-Blanchet2020Cooccurrencea}{}%
Blanchet, F.G., Cazelles, K. \& Gravel, D. (2020). Co-occurrence is not
evidence of ecological interactions. \emph{Ecology Letters}, 23,
1050--1063.

\leavevmode\hypertarget{ref-Bohan2017NexGlo}{}%
Bohan, D.A., Vacher, C., Tamaddoni-Nezhad, A., Raybould, A., Dumbrell,
A.J. \& Woodward, G. (2017). Next-Generation Global Biomonitoring:
Large-scale, Automated Reconstruction of Ecological Networks.
\emph{Trends in Ecology \& Evolution}, 32, 477--487.

\leavevmode\hypertarget{ref-Borrett2019Walk}{}%
Borrett, S.R. \& Scharler, U.M. (2019). Walk partitions of flow in
Ecological Network Analysis: Review and synthesis of methods and
indicators. \emph{Ecological Indicators}, 106, 105451.

\leavevmode\hypertarget{ref-Canard2014Empiricala}{}%
Canard, E.F., Mouquet, N., Mouillot, D., Stanko, M., Miklisova, D. \&
Gravel, D. (2014). Empirical Evaluation of Neutral Interactions in
Host-Parasite Networks. \emph{The American Naturalist}, 183, 468--479.

\leavevmode\hypertarget{ref-Canard2012Emergencea}{}%
Canard, E., Mouquet, N., Marescot, L., Gaston, K.J., Gravel, D. \&
Mouillot, D. (2012). Emergence of Structural Patterns in Neutral Trophic
Networks. \emph{PLOS ONE}, 7, e38295.

\leavevmode\hypertarget{ref-Catchen2023Missinga}{}%
Catchen, M.D., Poisot, T., Pollock, L.J. \& Gonzalez, A. (2023). The
missing link: Discerning true from false negatives when sampling species
interaction networks.

\leavevmode\hypertarget{ref-Cazelles2016Theorya}{}%
Cazelles, K., Araujo, M.B., Mouquet, N. \& Gravel, D. (2016). A theory
for species co-occurrence in interaction networks. \emph{Theoretical
Ecology}, 9, 39--48.

\leavevmode\hypertarget{ref-Delmas2019Analysing}{}%
Delmas, E., Besson, M., Brice, M.-H., Burkle, L.A., Dalla Riva, G.V.,
Fortin, M.-J., \emph{et al.} (2019). Analysing ecological networks of
species interactions. \emph{Biological Reviews}, 94, 16--36.

\leavevmode\hypertarget{ref-Dunne2006Network}{}%
Dunne, J.A. (2006). The Network Structure of Food Webs. In:
\emph{Ecological networks: Linking structure and dynamics} (eds. Dunne,
J.A. \& Pascual, M.). Oxford University Press, pp. 27--86.

\leavevmode\hypertarget{ref-Elton2001Animal}{}%
Elton, C.S. (2001). \emph{Animal Ecology}. University of Chicago Press,
Chicago, IL.

\leavevmode\hypertarget{ref-Emmerson2004PrePre}{}%
Emmerson, M.C. \& Raffaelli, D. (2004). Predatorprey body size,
interaction strength and the stability of a real food web. \emph{Journal
of Animal Ecology}, 73, 399--409.

\leavevmode\hypertarget{ref-Galiana2018SpaSca}{}%
Galiana, N., Lurgi, M., Claramunt-López, B., Fortin, M.-J., Leroux, S.,
Cazelles, K., \emph{et al.} (2018). The spatial scaling of species
interaction networks. \emph{Nature Ecology \& Evolution}, 2, 782--790.

\leavevmode\hypertarget{ref-Godsoe2022Species}{}%
Godsoe, W., Murray, R. \& Iritani, R. (2022). Species interactions and
diversity: A unified framework using Hill numbers. \emph{Oikos}, n/a,
e09282.

\leavevmode\hypertarget{ref-Gonzalez-Varo2016Labilea}{}%
Gonzalez-Varo, J.P. \& Traveset, A. (2016). The Labile Limits of
Forbidden Interactions. \emph{Trends in Ecology \& Evolution}, 31,
700--710.

\leavevmode\hypertarget{ref-Guimaraes2020Structurea}{}%
Guimarães, P.R. (2020). The Structure of Ecological Networks Across
Levels of Organization. \emph{Annual Review of Ecology, Evolution, and
Systematics}, 51, 433--460.

\leavevmode\hypertarget{ref-Jordano2016SamNet}{}%
Jordano, P. (2016). Sampling networks of ecological interactions.
\emph{Functional Ecology}, 30, 1883--1893.

\leavevmode\hypertarget{ref-Jordano2003Invarianta}{}%
Jordano, P., Bascompte, J. \& Olesen, J.M. (2003). Invariant properties
in coevolutionary networks of plantanimal interactions. \emph{Ecology
Letters}, 6, 69--81.

\leavevmode\hypertarget{ref-McLeod2021Sampling}{}%
McLeod, A., Leroux, S.J., Gravel, D., Chu, C., Cirtwill, A.R., Fortin,
M.-J., \emph{et al.} (2021). Sampling and asymptotic network properties
of spatial multi-trophic networks. \emph{Oikos}, n/a.

\leavevmode\hypertarget{ref-McLeod2020EffSpe}{}%
McLeod, A.M., Leroux, S.J. \& Chu, C. (2020). Effects of species traits,
motif profiles, and environment on spatial variation in multi-trophic
antagonistic networks. \emph{Ecosphere}, 11, e03018.

\leavevmode\hypertarget{ref-Ohlmann2019Diversity}{}%
Ohlmann, M., Miele, V., Dray, S., Chalmandrier, L., O'Connor, L. \&
Thuiller, W. (2019). Diversity indices for ecological networks: A
unifying framework using Hill numbers. \emph{Ecology Letters}, 22,
737--747.

\leavevmode\hypertarget{ref-Olesen2010Missing}{}%
Olesen, J.M., Bascompte, J., Dupont, Y.L., Elberling, H., Rasmussen, C.
\& Jordano, P. (2010). Missing and forbidden links in mutualistic
networks. \emph{Proceedings of the Royal Society B: Biological
Sciences}, 278, 725--732.

\leavevmode\hypertarget{ref-Pascual2006Ecologicala}{}%
Pascual, M., Dunne, J.A. \& Dunne, J.A. (2006). \emph{Ecological
Networks: Linking Structure to Dynamics in Food Webs}. Oxford University
Press, USA.

\leavevmode\hypertarget{ref-Poisot2012Dissimilaritya}{}%
Poisot, T., Canard, E., Mouillot, D., Mouquet, N. \& Gravel, D. (2012).
The dissimilarity of species interaction networks. \emph{Ecology
Letters}, 15, 1353--1361.

\leavevmode\hypertarget{ref-Poisot2016Structure}{}%
Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D., Fortin, M.-J. \&
Stouffer, D.B. (2016). The structure of probabilistic networks.
\emph{Methods in Ecology and Evolution}, 7, 303--312.

\leavevmode\hypertarget{ref-Poisot2015Speciesa}{}%
Poisot, T., Stouffer, D.B. \& Gravel, D. (2015). Beyond species: Why
ecological interaction networks vary through space and time.
\emph{Oikos}, 124, 243--251.

\leavevmode\hypertarget{ref-Pringle2020Resolving}{}%
Pringle, R.M. \& Hutchinson, M.C. (2020). Resolving Food-Web Structure.
\emph{Annual Review of Ecology, Evolution, and Systematics}, 51, 55--80.

\leavevmode\hypertarget{ref-Proulx2005Networka}{}%
Proulx, S.R., Promislow, D.E.L. \& Phillips, P.C. (2005). Network
thinking in ecology and evolution. \emph{Trends in Ecology \&
Evolution}, SPECIAL issue: BUMPER book REVIEW, 20, 345--353.

\leavevmode\hypertarget{ref-Pulliam1974Theory}{}%
Pulliam, H.R. (1974). On the Theory of Optimal Diets. \emph{The American
Naturalist}, 108, 59--74.

\leavevmode\hypertarget{ref-Salim2022DatSta}{}%
Salim, J.A., Saraiva, A.M., Zermoglio, P.F., Agostini, K., Wolowski, M.,
Drucker, D.P., \emph{et al.} (2022). Data standardization of
plantpollinator interactions. \emph{GigaScience}, 11, giac043.

\leavevmode\hypertarget{ref-Saravia2022Ecological}{}%
Saravia, L.A., Marina, T.I., Kristensen, N.P., De Troch, M. \& Momo,
F.R. (2022). Ecological network assembly: How the regional metaweb
influences local food webs. \emph{Journal of Animal Ecology}, 91,
630--642.

\leavevmode\hypertarget{ref-Strydom2022Food}{}%
Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron, D., Farrell,
M.J., \emph{et al.} (2022). Food web reconstruction through phylogenetic
transfer of low-rank network representation. \emph{Methods in Ecology
and Evolution}, 13.

\leavevmode\hypertarget{ref-Strydom2021Roadmapa}{}%
Strydom, T., Catchen, M.D., Banville, F., Caron, D., Dansereau, G.,
Desjardins-Proulx, P., \emph{et al.} (2021). A roadmap towards
predicting species interaction networks (across space and time).
\emph{Philosophical Transactions of the Royal Society B-Biological
Sciences}, 376, 20210063.

\leavevmode\hypertarget{ref-VazquezSS2022EcoInt}{}%
Vázquez, D.P., Peralta, G., Cagnolo, L., Santos, M. \& Igual, E. autores
contribuyeron por. (2022). Ecological interaction networks. What we
know, what we don't, and why it matters. \emph{Ecología Austral}, 32,
670--697.

\leavevmode\hypertarget{ref-Wood2015Effects}{}%
Wood, S.A., Russell, R., Hanson, D., Williams, R.J. \& Dunne, J.A.
(2015). Effects of spatial scale of sampling on food web structure.
\emph{Ecology and Evolution}, 5, 3769--3782.

\end{CSLReferences}

\end{document}
